-- MySQL dump 10.13  Distrib 5.5.28, for Linux (x86_64)
--
-- Host: localhost    Database: dokhlab
-- ------------------------------------------------------
-- Server version	5.5.28

/*!40101 SET @OLD_CHARACTER_SET_CLIENT=@@CHARACTER_SET_CLIENT */;
/*!40101 SET @OLD_CHARACTER_SET_RESULTS=@@CHARACTER_SET_RESULTS */;
/*!40101 SET @OLD_COLLATION_CONNECTION=@@COLLATION_CONNECTION */;
/*!40101 SET NAMES utf8 */;
/*!40103 SET @OLD_TIME_ZONE=@@TIME_ZONE */;
/*!40103 SET TIME_ZONE='+00:00' */;
/*!40014 SET @OLD_UNIQUE_CHECKS=@@UNIQUE_CHECKS, UNIQUE_CHECKS=0 */;
/*!40014 SET @OLD_FOREIGN_KEY_CHECKS=@@FOREIGN_KEY_CHECKS, FOREIGN_KEY_CHECKS=0 */;
/*!40101 SET @OLD_SQL_MODE=@@SQL_MODE, SQL_MODE='NO_AUTO_VALUE_ON_ZERO' */;
/*!40111 SET @OLD_SQL_NOTES=@@SQL_NOTES, SQL_NOTES=0 */;

--
-- Table structure for table `GroupMeeting`
--

DROP TABLE IF EXISTS `GroupMeeting`;
/*!40101 SET @saved_cs_client     = @@character_set_client */;
/*!40101 SET character_set_client = utf8 */;
CREATE TABLE `GroupMeeting` (
  `ID` int(11) NOT NULL AUTO_INCREMENT,
  `Date` varchar(10) DEFAULT NULL,
  `Name` varchar(20) DEFAULT NULL,
  `Content` varchar(100) DEFAULT NULL,
  PRIMARY KEY (`ID`)
) ENGINE=MyISAM AUTO_INCREMENT=962 DEFAULT CHARSET=latin1;
/*!40101 SET character_set_client = @saved_cs_client */;

--
-- Dumping data for table `GroupMeeting`
--

LOCK TABLES `GroupMeeting` WRITE;
/*!40000 ALTER TABLE `GroupMeeting` DISABLE KEYS */;
INSERT INTO `GroupMeeting` VALUES (67,'11/07','Tamas Hegedus',''),(66,'10/31','Adrian Serohijos',''),(15,'06/27','Kyle Wilcox',''),(16,'07/11','Sagar Khare',''),(17,'07/18','Shantanu Sharma',''),(18,'07/25','Gong Peng',''),(19,'08/01','Joshua J. LaRocque',''),(20,'08/08','Huifen Nie',''),(21,'08/15','Micheal Chen',''),(22,'08/22','Feng Ding',''),(65,'10/24','Shantanu Sharma',''),(64,'10/17','Sagar Khare',''),(63,'10/10','Feng Ding',''),(62,'10/03','Kyle Wilcox',''),(61,'09/26','',''),(60,'09/19','Peng Gong',''),(59,'09/12','',''),(58,'09/05','',''),(57,'08/29','Feng Ding',''),(68,'11/14','Feng Ding',''),(69,'11/21','Yiwen Chen',''),(70,'11/28','Huifen Nie',''),(71,'12/05','Kyle Wilcox',''),(72,'12/12','Adrian Serohijos',''),(73,'12/19','Shantanu Sharma',''),(74,'01/09','Feng Ding',''),(75,'01/23','Yiwen Chen',''),(76,'01/30','Huifen Nie',''),(77,'02/06','Srinivas Ramachandra',''),(78,'02/13','Charles Davis',''),(79,'02/20','Adrian Serohijos',''),(80,'02/27','Peng Gong',''),(81,'02/02','Sangwook Wu',''),(82,'02/09','Pavel Zhuravlev',''),(83,'02/16','Lanyuan Lu',''),(84,'02/23','Feng Ding',''),(85,'02/06','Srinivas Ramachandra',''),(86,'02/13','Charles Davis',''),(87,'02/20','Adrian Serohijos',''),(88,'02/27','Peng Gong',''),(89,'01/30','Huifen Nie',''),(90,'02/06','Srinivas Ramachandra',''),(91,'02/13','Charles Davis',''),(92,'02/20','Adrian Serohijos',''),(93,'02/27','Peng Gong',''),(94,'03/06','Shantanu Sharma',''),(95,'03/13','Kyle Wilcox',''),(96,'03/20','Feng Ding',''),(97,'03/27','',''),(98,'04/03','Yiwen Chen',''),(102,'05/01','',''),(99,'04/10','Huifen Nie',''),(100,'04/17','',''),(101,'04/24','Adrian Serohijos',''),(110,'06/26','Adrian Serohijos',''),(103,'05/08','Kyle Wilcox',''),(104,'05/15','Feng Ding',''),(105,'05/22','',''),(106,'05/29','',''),(107,'06/05','Yiwen Chen',''),(108,'06/12','Huifen Nie',''),(109,'06/19','Kyle Wilcox',''),(111,'07/03','Shantanu Sharma',''),(112,'07/10','Feng Ding',''),(113,'07/17','',''),(114,'07/24','Yiwen Chen',''),(115,'07/31','Huifen Nie',''),(116,'08/07','',''),(117,'08/14','Kyle Wilcox',''),(118,'08/21','Adrian Serohijos',''),(119,'08/28','Shantanu Sharma',''),(120,'09/04','',''),(121,'09/11','Feng Ding',''),(122,'09/18','Yiwen Chen',''),(123,'09/25','Huifen Nie',''),(124,'10/02','Kyle Wilcox',''),(125,'10/09','Adrian Serohijos',''),(126,'10/16','Shantanu Sharma',''),(127,'10/23','Shuanye Yin',''),(128,'10/31','Adrian Serohijos',''),(129,'11/7','Feng Ding',''),(130,'11/14','Feng Ding',''),(131,'11/21','Yiwen Chen',''),(132,'11/28','Huifen Nie',''),(133,'10/17','Shantanu Sharma',''),(134,'11/24','Shuangye Yin',''),(135,'10/17','Shantanu Sharma',''),(136,'10/24','Shuanye Yin',''),(137,'10/31','Adrian Serohijos',''),(138,'11/07','Tamas Hegedus',''),(139,'11/14','Feng Ding',''),(140,'11/21','Yiwen Chen',''),(141,'11/28','Huifen Nie',''),(142,'12/05','Kyle Wilcox',''),(143,'12/12','Adrian Serohijos',''),(144,'12/19','Shantanu Sharma',''),(145,'12/26','',''),(146,'01/02','Shuangye Yin',''),(147,'01/09','Feng Ding',''),(148,'12/21','Kyle Wilcox',''),(149,'12/28','Santa',''),(150,'01/04','Feng Ding',''),(151,'01/11','Huifen Nie',''),(152,'12/05','Kyle Wilcox',''),(153,'12/12','Adrian Serohijos',''),(154,'12/19','Shantanu Sharma',''),(155,'12/26','',''),(156,'01/02','Shuangye Yin',''),(157,'01/09','Feng Ding',''),(158,'01/16','Tamas Hegedus',''),(159,'01/23','Yiwen Chen',''),(160,'01/30','Huifen Nie',''),(161,'02/06','Srinivas Ramachandra',''),(162,'02/13','Kyle Wilcox',''),(163,'02/20','Adrian Serohijos',''),(164,'02/27','Shantanu Sharma',''),(165,'03/06','Shuangye Yin',''),(166,'03/13','Feng Ding',''),(167,'03/20','Tamas Hegdus',''),(168,'03/27','Yiwen Chen',''),(169,'01/29','Yiwen Chen',''),(170,'02/05','',''),(171,'02/12','Huifen Nie',''),(172,'02/19','Barry Kesner',''),(173,'02/26','Yiwen Chen',''),(174,'03/05','Srinivas Ramachandra',''),(175,'03/12','Kyle Wilcox',''),(176,'03/19','Adrian Serohijos',''),(177,'03/26','Shantanu Sharma',''),(178,'04/02','Shuangye Yin',''),(179,'04/02','Shuangye Yin',''),(180,'04/09','Feng Ding',''),(181,'04/16','Tamas Hegdus',''),(182,'04/23','Yiwen Chen',''),(183,'04/30','Barry Kesner',''),(184,'05/07','Srinivas Ramachandra',''),(185,'05/14','Kyle Wilcox',''),(186,'05/21','Adrian Serohijos',''),(187,'5/28','Shantanu Sharma',''),(188,'6/4','Shuangye','GM'),(189,'6/11','Feng Ding',''),(190,'6/18','Barry Kesner',''),(191,'6/25','Srinivas Ramachandra',''),(192,'7/2','Kyle Wilcox',''),(193,'7/9','Kyle Wilcox',''),(194,'7/16','Tamas Hegdus',''),(195,'7/23','Shantanu Sharma',''),(196,'7/30','Adrian Serohijos',''),(202,'8/13','Feng Ding',''),(201,'8/6','Shuangye Yin',''),(203,'8/20','',''),(204,'8/27','Barry Kesner',''),(205,'9/3','',''),(206,'9/10','Douglas Tsao',''),(207,'9/17','Kyle Wilcox',''),(208,'9/24','Tamas Hegdus',''),(209,'10/1','Shantanu Sharma',''),(210,'10/8','Adrian Serohijos',''),(211,'10/15','Shuangye Yin',''),(212,'10/22','Ben Stranges',''),(213,'10/29','Pradeep Kota',''),(214,'11/5','Pradeep Kota/Douglas',''),(215,'11/12','Douglas Tsao',''),(216,'11/19','Srinivas Ramachandra',''),(217,'11/26','Feng Ding',''),(218,'12/3','Kyle Wilcox',''),(219,'12/10','Barry Kesner',''),(220,'12/17','Shantanu Sharma',''),(221,'1/7','',''),(222,'1/14','Shuangye Yin',''),(223,'1/28','Tamas Hegdus',''),(224,'2/4','Adrian Serohijos',''),(225,'2/11','Douglas Tsao',''),(226,'2/18','Srinivas Ramachandra',''),(227,'2/25','Feng Ding',''),(228,'3/3','Kyle Wilcox',''),(229,'3/10','Barry Kesner',''),(230,'3/17','Shantanu Sharma',''),(231,'3/24','Shantanu Sharma',''),(232,'3/31','Adrian Serohijos',''),(233,'4/7','Shuangye Yin',''),(234,'4/14','Adrian Serohijos',''),(235,'4/21','Tamas Hegedus',''),(236,'4/28','Barry Kesner',''),(237,'5/5','Srinivas Ramachandra',''),(238,'5/12','Feng Ding ',''),(239,'5/19','Douglas Tsao',''),(330,'1/12','Rachel Redler',''),(241,'6/2','Srinivas Ramachandra',''),(242,'6/9','Kyle Wilcox',''),(243,'6/16',' Shantanu Sharma',''),(244,'6/23','Adrian Serohijos',''),(245,'6/30','Shuangye Yin',''),(246,'7/7','Douglas Tsao',''),(247,'7/14','Pradeep Kota',''),(248,'7/21','Brittany Fotsch',''),(249,'7/28','Barry Kesner',''),(250,'8/4','Feng Ding',''),(251,'8/11','Douglas Tsao',''),(252,'8/18','Srinivas Ramachandra',''),(253,'8/25','Kyle Wilcox',''),(254,'9/1','',''),(255,'9/8','Kyle Wilcox',''),(256,'9/15','Elizabeth Proctor',''),(257,'9/22','Rachel Redler',''),(258,'9/29','Pradeep Kota',''),(259,'10/6','Tamas Hegdus',''),(260,'10/13','Barry Kesner',''),(261,'10/20','Feng Ding',''),(262,'10/27','Douglas Tsao',''),(263,'11/2','Srinivas Ramachandra',''),(264,'11/10','Kyle Wilcox',''),(265,'11/17','Shantanu Sharma',''),(266,'11/24','Adrian Serohijos',''),(267,'12/1','Pradeep Kota',''),(268,'12/8','Barry Kesner',''),(269,'12/15',' Shuangye Yin',''),(270,'12/22','Feng Ding',''),(271,'1/5/2009','',''),(272,'1/12/2009','Douglas Tsao',''),(273,'1/19/2009','Srinivas Ramachandra',''),(274,'1/26','Tamas Hegdus',''),(275,'2/2','Kyle Wilcox',''),(276,'2/9','Pradeep Kota',''),(277,'2/16','Douglas Tsao',''),(278,'2/23','Adrian Serohijos',''),(279,'2/23','Adrian Serohijos',''),(280,'3/2','Barry Kesner',''),(281,'3/9','Shuangye Yin',''),(282,'3/16','',''),(283,'3/23','Feng Ding',''),(284,'3/20','Tamas Hegdus ',''),(285,'4/6','Srinivas Ramachandra',''),(286,'4/13','Kyle Wilcox',''),(287,'4/20','Pradeep Kota',''),(288,'4/27','Barry Kesner',''),(289,'5/4','Douglas Tsao',''),(290,'5/11','Shuangye Yin',''),(291,'5/18','Feng Ding',''),(292,'5/25','',''),(294,'6/1','Srinivas Ramachandra',''),(295,'6/8','Douglas Tsao',''),(296,'6/15','Feng Ding',''),(297,'6/23','Barry Kesner',''),(293,'5/26','Elizabeth Proctor',''),(299,'6/29','Shuangye Yin',''),(300,'7/6','Douglas Tsao',''),(301,'7/13','Pradeep Kota',''),(302,'7/20','Tamas Hegdus',''),(303,'7/27','Feng Ding',''),(304,'8/3','Srinivas Ramachandra',''),(305,'8/10','Kyle Wilcox',''),(306,'8/17','Douglas Tsao',''),(307,'8/24','Pradeep Kota',''),(308,'9/1',' Brittany Fotsch',''),(309,'9/8','Kyle Wilcox',''),(310,'9/15','Elizabeth Proctor',''),(311,'9/22','Rachel Redler',''),(313,'9/29','Feng Ding',''),(314,'10/6','Srinivas Ramachandra',''),(315,'10/13','Shuangye Yin',''),(316,'10/20','Douglas Tsao',NULL),(317,'10/27','Pradeep Kota',NULL),(318,'11/3','Elizabeth Proctor',NULL),(319,'11/10','Rachel Redler',NULL),(320,'11/17','Feng Ding',''),(321,'11/24','Srinivas Ramachandra',''),(322,'12/1','Shuangye Yin',''),(323,'12/8','Douglas Tsao',''),(325,'12/15','Vinal Lakhani',NULL),(329,'1/5','Elizabeth Proctor',''),(328,'12/22','Pradeep Kota',''),(331,'1/19','Feng Ding',''),(332,'1/26','Srinivas Ramachandra',''),(333,'2/2','Shuangye Yin',''),(335,'2/9',' Douglas Tsao',''),(336,'2/16','Vinal Lakhani',''),(337,'2/23','Elizabeth Proctor',''),(338,'3/2','Meeting with Dave',''),(339,'3/9','Lanette Fee',''),(340,'3/16','Pradeep Kota',''),(341,'3/23','Rachel Redler',''),(342,'3/30','Feng Ding',''),(343,'4/6','Srinivas Ramachandra',''),(344,'4/13','Shuangye Yin',''),(345,'4/20','Douglas Tsao',''),(346,'4/27','Elizabeth Proctor',''),(347,'5/4','Lanette Fee',''),(348,'5/11','Pradeep Kota',''),(349,'5/18','Rachel Redler',''),(350,'5/25','David Shirvanyants',''),(351,'6/1','Jeremy Hanh',''),(352,'6/8','Feng Ding',''),(353,'6/15','Douglas Tsao',''),(354,'6/22','Srinivas Ramachandra',''),(355,'6/29','Shuangye Yin',''),(356,'7/6','Elizabeth Proctor',''),(357,'7/13','Pradeep Kota',''),(358,'7/20','Lanette Fee',''),(359,'7/27','Rachel Redler',''),(360,'8/3','David Shirvanyants',''),(361,'8/10','Srinivas Ramachandra',''),(362,'8/17','Feng Ding',''),(364,'8/24','Douglas Tsao',NULL),(365,'8/31','Jeremy Hahn',NULL),(366,'9/7','Elizabeth Proctor',''),(367,'9/14','Shuangye Yin',''),(368,'9/21','Lanette Fee',''),(369,'9/28','Pradeep Kota',''),(370,'10/5','Rachel Redler',''),(371,'10/12','Gergely Gyimesi ',''),(372,'10/19','David Shirvanyants',''),(373,'10/26','',''),(374,'11/2','',''),(375,'11/9','Elizabeth Proctor',''),(376,'11/16','Srinivas Ramachandra',''),(377,'11/23','Douglas Tsao',''),(378,'12/14','Shuangye Yin',''),(379,'12/21','Lanette Fee',''),(380,'12/28','Elizabeth Proctor',''),(381,'1/4','',''),(382,'1/11','Feng Ding',''),(383,'1/18','Rachel Redler',''),(384,'1/25','Pradeep Kota',''),(385,'2/1','David Shirvanyants',''),(386,'2/8','Elizabeth Proctor',''),(387,'2/15','Srinivas Ramachandra',''),(388,'2/22','Pradeep Kota',''),(389,'3/1','Douglas Tsao',''),(390,'3/8','',''),(391,'3/15','Lanette Fee',''),(392,'3/22','Elizabeth Proctor',''),(393,'3/29','',''),(394,'4/5','Feng Ding',''),(395,'4/12','Rachel Redler',''),(396,'4/19','Pradeep Kota',''),(397,'4/26','',''),(398,'5/3','David Shirvanyants',''),(399,'5/10','Elizabeth Proctor',''),(400,'5/17','Srinivas Ramachandra',''),(401,'5/24','',''),(402,'5/31','Lanette Fee',''),(403,'6/7','Douglas Tsao',''),(404,'6/14','Elizabeth Proctor',''),(405,'6/21','Elizabeth Proctor',''),(406,'6/28','Feng Ding',''),(407,'7/5','Rachel Redler',''),(408,'7/12','David Shirvanyants',''),(409,'7/19','Onur Dagliyan',''),(410,'7/26','Srinivas Ramachandra',''),(411,'8/2','Mahmoud',''),(412,'8/9','Elizabeth Proctor',''),(417,'9/13','Rachel Redler',''),(413,'8/16','Lanette Fee',''),(414,'8/23','',''),(415,'8/30','Pradeep Kota',''),(416,'9/6','Feng Ding',''),(418,'9/20','David Shirvanyants',''),(419,'9/27','Elizabeth Proctor',''),(420,'10/4','',''),(421,'10/11','Onur Dagliyan',''),(422,'10/18','Arpit Tandon',''),(423,'10/25','Srinivas Ramachandra',''),(424,'11/1','Feng Ding',''),(425,'11/8','Feng Ding',''),(426,'11/15','Lanette Fee',''),(427,'11/22','Elizabeth Proctor',''),(428,'11/29','Pradeep Kota',''),(429,'12/6','Rachel Redler',''),(430,'12/13','Travis Hoppe',''),(431,'12/20','David Shirvanyants',''),(432,'12/27','Onur Dagliyan',''),(433,'1/3','Srinivas Ramachandra',''),(434,'1/10','Feng Ding',''),(435,'1/17','Lanette Fee',''),(436,'1/24','Elizabeth Proctor',''),(437,'1/31','Pradeep Kota',''),(438,'2/7','Rachel Redler',''),(439,'2/14','David Shirvanyants',''),(440,'2/21','Onur Dagliyan',''),(441,'2/28','Srinivas Ramachandra',''),(442,'3/6','Feng Ding',''),(443,'3/13','Lanette Fee',''),(444,'3/20','Elizabeth Proctor',''),(445,'3/27','Pradeep Kota',''),(446,'4/3','',''),(447,'4/10','Rachel Redler',''),(448,'4/17','David Shirvanyants',''),(449,'4/24','Patrick McCarter',''),(450,'5/1','Onur Dagliyan',NULL),(451,'5/8','Srinivas Ramachandra',NULL),(452,'5/15','Lanette Fee',NULL),(453,'5/22','Elizabeth Proctor',NULL),(454,'5/29','Pradeep Kota',NULL),(455,'6/5','Rachel Redler',''),(456,'6/12','Arpit Tandon',''),(457,'6/19','David Shirvanyants',''),(458,'6/26','Onur Dagliyan',''),(459,'7/3','Pradeep Kota',''),(460,'7/10','Pradeep Kota',''),(461,'7/17','Elizabeth Proctor',''),(462,'7/24','Lanette Fee',''),(464,'8/7','David Shirvanyants',''),(463,'7/31','Rachel Redler',''),(465,'8/14','Elizabeth Proctor',''),(466,'8/21','Onur Dagliyan',''),(467,'8/28','Gurpreet Kaur',''),(468,'9/4','Arpit Tandon',''),(469,'9/11','Pradeep Kota',''),(470,'9/18','Rachel Redler',''),(471,'9/25','Lanette Fee',''),(472,'10/2','David Shirvanyants',NULL),(473,'10/9','Gurpreet Kaur',NULL),(474,'10/16','Jhuma Das',NULL),(475,'9/18','Jimmy Fay',NULL),(476,'9/25','Rachel Redler',NULL),(477,'10/2','Lanette Fee',NULL),(478,'10/9','David Shirvanyants',NULL),(479,'10/16','Jhuma Das',NULL),(480,'10/23','Gurpreet Kaur',NULL),(481,'10/30','Elizabeth Proctor',NULL),(482,'11/6','Rachel Redler',NULL),(483,'11/27','Kevin Houlihan',NULL),(484,'11/27','Rachel Cohen',NULL),(485,'11/27','Rachel Cohen',NULL),(486,'11/27','Kevin Houlihan',NULL),(487,'11/27','Rachel Cohen',NULL),(488,'11/27','Kevin Houlihan',NULL),(489,'11/27','Kevin HouliHan Rache',NULL),(490,'11/27','Kevin HouliHan Rache',NULL),(491,'11/27','Kevin HouliHan,Rache',NULL),(492,'11/27','Kevin HouliHan Rache',NULL),(493,'11/27','Kevin HouliHan','Rachel Cohen'),(494,'11/27','Kevin,  Rachel',NULL),(495,'10/23','Gurpreet Kaur',NULL),(496,'10/30','Elizabeth Proctor',NULL),(497,'11/6','Rachel Redler',NULL),(498,'11/13','Reed, Mayukh',NULL),(499,'11/20','Chanin, Mahmoud',NULL),(500,'11/6','Jimmy Fay',NULL),(501,'11/6','Rachel Redler',''),(502,'12/4','Jimmy Fay',''),(503,'12/11','Lanette Fee',''),(504,'12/18','David Shirvanyants',''),(505,'12/11','Arpit Tandon',''),(506,'12/18','Lanette Fee',''),(507,'11/27','Kevin, Rachel',''),(508,'12/11','Lanette Fee',''),(509,'12/18','Arpit Tandon',''),(510,'11/13','Reed, Mayukh',''),(511,'11/13','Chanin, Mahmoud',''),(512,'11/13','Reed, Mayukh',''),(513,'11/18','Chanin, Mahmoud',''),(514,'11/20','Chanin, Mahmoud',''),(515,'12/4','Jimmy Fay',''),(516,'11/6','Rachel Redler',''),(517,'12/18','Onur Dagliyan',''),(518,'01/08','Arpit Tandon',''),(519,'12/11','Lanette Fee',''),(520,'1/8','Arpit Tandon',''),(521,'01/8','Arpit Tandon',''),(522,'1/8/2013','Arpit Tandon',''),(523,'1/8/13','Arpit Tandon',''),(524,'1/8','Arpit Tandon',''),(525,'1/8','David Shirvanyants',''),(526,'1/15','Arpit Tandon',''),(527,'1/22','Jhuma Das',''),(528,'1/31','Elizabeth Proctor',''),(529,'1/29','Elizabeth Proctor',''),(530,'2/5','Jimmy Fay',''),(531,'2/12','Rachel Redler',''),(532,'2/5','Gurpreet Kaur',''),(533,'2/12','Jimmy Fay',''),(534,'2/19','Rachel Redler',''),(535,'2/26','Lanette Fee',''),(536,'2/5','Jimmy Fay',''),(537,'2/12','Gurpreet Kaur',''),(538,'3/5','Onur Dagliyan',''),(539,'3/12','David Shirvanyants',''),(540,'3/19','Arpit Tandon',''),(541,'3/26','Jhuma Das',''),(542,'3/5','Nikolay Dokholyan',''),(543,'3/12','Onur Dagliyan',''),(544,'3/19','David Shirvanyants',''),(545,'3/26','Arpit Tandon',''),(546,'4/2','Jhuma Das',''),(547,'4/9','Elizabeth Proctor',''),(548,'4/16','Marino Convertino',''),(549,'4/23','Jimmy Fay',''),(550,'4/30','Gurpreet Kaur',''),(551,'3/12','Arpit Tandon',''),(552,'3/19','Arpit Tandon',''),(553,'3/19','David Shirvanyants',''),(554,'3/26','Arpit Tandon',''),(555,'3/12','Onur Dagliyan',''),(556,'4/2','Jhuma Das',''),(557,'5/7','Raul Mendez',''),(558,'5/14','Rachel Redler',''),(559,'4/9','Elizabeth Proctor',''),(560,'4/16','Marino Convertino',''),(561,'4/23','Jimmy Fay',''),(562,'4/30','Gurpreet Kaur',''),(563,'5/7','Ardeshir Goliaei, Ko',''),(564,'5/7','Ardeshir Kossi',''),(565,'5/7','Ardeshir,Kossi',''),(566,'5/14','Raul Mendez',''),(567,'5/21','Rachel Redler',''),(568,'5/28','Lanette Fee',''),(569,'6/4','Andrey Krokhotin',''),(570,'6/11','Onur Dagliyan',''),(571,'6/18','David Shirvanyants',''),(572,'6/25','Jhuma Das',''),(573,'6/11','Lanette Fee',''),(574,'6/18','Andrey Krokhotin',''),(575,'6/25','Onur Dagliyan',''),(576,'7/2','David Shirvanyants',''),(577,'7/9','Jhuma Das',''),(578,'6/4','Elizabeth Proctor',''),(579,'7/16','Mahmoud Shobair',''),(580,'7/23','Elizabeth Proctor',NULL),(581,'7/30','Marino Convertino',NULL),(582,'8/6','James Fay',NULL),(583,'8/6','Reed Jacob',NULL),(584,'8/13','James Fay',NULL),(585,'8/20','Arpit Tandon',NULL),(586,'8/27','Raul Mendez',NULL),(587,'9/3','Rachel Redler',NULL),(588,'9/10','Andrey Krokhotin',NULL),(589,'9/17','Soonmin Jang',NULL),(590,'9/17','Onur Dagliyan',NULL),(591,'9/17','Soonmin Jang',NULL),(592,'9/24','Onur Dagliyan',NULL),(593,'10/1','David Shirvanyants',NULL),(594,'10/8','Jhuma Das',NULL),(595,'10/15','Mahmoud Shobair',NULL),(596,'10/15','Elizabeth Proctor',NULL),(597,'10/22','James Fay',NULL),(598,'10/29','Mahmoud Shobair',NULL),(599,'11/5','Marino Convertino',NULL),(600,'11/5','Arpit Tandon',NULL),(601,'11/12','Reed Jacob',NULL),(602,'10/1','Jhuma Das',NULL),(603,'10/8','David Shirvanyants',NULL),(604,'11/19','Raul Mendez',NULL),(605,'11/26','Marino Convertino',NULL),(606,'10/15','Mahmoud Shobair',NULL),(607,'10/29','Elizabeth Proctor',NULL),(608,'10/22','James Fay',NULL),(609,'11/5','Mahmoud Shobair',NULL),(610,'11/12','Arpit Tandon',NULL),(611,'11/19','Mahmoud Shobair',NULL),(612,'11/26','Reed Jacob',NULL),(613,'12/3','Benfeard Williams',NULL),(614,'12/10','Mahmoud Shobair',NULL),(615,'12/17','Raul Mendez',NULL),(616,'12/17','Reed Jacob',NULL),(617,'11/26','Mahmoud Shobair',NULL),(618,'11/19','No Lab Meeting',NULL),(619,'12/17','Benfeard Williams',NULL),(620,'12/3','Marino Convertino',NULL),(621,'12/24','No Lab meeting',NULL),(622,'12/31','No Lab meeting',NULL),(623,'1/7','Reed Jacob',NULL),(624,'1/7','Mahmoud Shobair',NULL),(625,'1/7','Reed Jacob',NULL),(626,'1/14','Mahmoud Shobair',NULL),(627,'1/21','Joe Kousouros',NULL),(628,'1/28','Raul Mendez',NULL),(629,'2/4','Andrey Krokhotin',NULL),(630,'2/11','Onur Dagliyan',NULL),(631,'2/18','Jimmy Fay',NULL),(632,'2/25','Arpit Tandon',NULL),(633,'3/4','Dominique S., Matt S',NULL),(634,'3/4','Dominique Soroka & M',NULL),(635,'3/4','Dominique S. & Matt ',NULL),(636,'3/4','Dominique Soroka Mat',NULL),(637,'3/4','Dominique S. Matt S.',NULL),(638,'3/4','Dominique S. and Mat',NULL),(639,'3/4','Dominique S.; Matt S',NULL),(640,'3/11','Cheng Zhu',NULL),(641,'3/18','Marino Convertino',NULL),(642,'3/25','Jhuma Das',NULL),(643,'3/11','Dominique Soroka',NULL),(644,'3/25','Cheng Zhu',NULL),(645,'3/4','Cancelled-weather',NULL),(646,'4/1','Jhuma Das',NULL),(647,'4/8','Yazhong Tao',NULL),(648,'4/15','Reed Jacob',NULL),(649,'4/22','Mahmoud Shobair',NULL),(650,'4/29','Joe Kousouros',NULL),(651,'5/6','Raul Mendez',NULL),(652,'5/13','Andrey Krokhotin',NULL),(653,'5/20','Onur Dagliyan',NULL),(654,'5/27','Jimmy Fay',NULL),(655,'6/3','Regina Politi',NULL),(656,'6/10','Dominique Soroka',NULL),(657,'6/17','Cheng Zhu',NULL),(658,'6/24','Arpit Tandon',NULL),(659,'5/27','Mahmoud Shobair',NULL),(660,'6/3','Jimmy Fay',NULL),(661,'6/10','Mahmoud Shobair',NULL),(662,'6/17','Regina Politi',NULL),(663,'6/24','Cheng Zhu',NULL),(664,'6/24','Dominique Soroka',NULL),(665,'6/24','Cheng Zhu',NULL),(666,'6/17','Dominique Soroka',NULL),(667,'7/1','Arpit Tandon',NULL),(668,'7/8','Jhuma Das',NULL),(669,'7/15','Regina Politi',NULL),(670,'7/22','Marino Convertino',NULL),(671,'7/29','Joe Kousouros',NULL),(672,'7/1','Marino Convertino',NULL),(673,'7/8','Benfeard Williams',NULL),(674,'7/15','Jhuma Das',NULL),(675,'7/22','Regina Politi',NULL),(676,'7/31','Dominique Soroka',NULL),(677,'7/15','Raul Mendez',NULL),(678,'7/22','Jhuma Das',NULL),(679,'8/5','Yazhong Tao',NULL),(680,'8/12','Reed Jacob',NULL),(681,'8/19','Mahmoud Shobair',NULL),(682,'8/26','Andrey Krokhotin',NULL),(683,'8/19','Reed Jacob',NULL),(684,'8/12','Mahmoud Shobair',NULL),(685,'9/2','Onur Dagliyan',NULL),(686,'9/9','Jimmy Fay',NULL),(687,'9/16','Dominique Soroka',NULL),(688,'9/23','Cheng Zhu',NULL),(689,'9/30','Jhuma Das',NULL),(690,'9/23','Jhuma Das',NULL),(691,'9/30','Cheng Zhu',NULL),(692,'9/23','Jimmy Fay',NULL),(693,'9/9','Jimmy and Reed',NULL),(694,'10/7','Jhuma Das',NULL),(695,'10/14','Marino Convertino',NULL),(696,'10/21','Benfeard Williams',NULL),(697,'10/28','Reed Jacob',NULL),(698,'9/16','Dominique Soroka',NULL),(699,'10/28','Aspen Gutgsell',NULL),(700,'11/4','Mahmoud Shobair',NULL),(701,'11/11','Reed Jacob',NULL),(702,'11/18','Yazhong Tao',NULL),(703,'11/25','Raul Mendez',NULL),(704,'12/2','Andrey Krokhotin',NULL),(705,'12/9','Onur Dagliyan',NULL),(706,'12/16','Jimmy Fay',NULL),(707,'12/23','Cheng Zhu',NULL),(708,'12/30','Jhuma Das',NULL),(709,'12/9','Jimmy Fay',NULL),(710,'12/16','Jhuma Das',NULL),(711,'12/23','Marino Convertino',NULL),(712,'12/30','Dominique Soroka',NULL),(713,'11/25','Dept. Retreat',NULL),(714,'12/2','Raul Mendez',NULL),(715,'12/9','Andrey Krokhotin',NULL),(716,'12/16','Jimmy Fay',NULL),(717,'12/23','Jhuma Das',NULL),(718,'12/30','Marino Convertino',NULL),(719,'1/6','Dominique Soroka',NULL),(720,'1/13','Onur Dagliyan',NULL),(721,'1/20','Cheng Zhu',NULL),(722,'1/27','David Mowrey',NULL),(723,'1/6','Marino Convertino',NULL),(724,'2/3','Frank Teets',NULL),(725,'2/10','Benfeard Williams',NULL),(726,'2/17','Mahmoud Shobair',NULL),(727,'2/17','Reed Jacob',NULL),(728,'2/24','Mahmoud Shobair',NULL),(729,'1/27','Ebrahim Kolahdouz',NULL),(730,'2/10','David Mowrey',NULL),(731,'2/17','Benfeard Williams',NULL),(732,'2/24','Reed Jacob',NULL),(733,'1/20','Cheng Zhu',NULL),(734,'1/13','Onur Dagliyan',NULL),(735,'2/3','Frank Teets',NULL),(736,'2/17','GM cancelled',NULL),(737,'2/24','Benfeard Williams',NULL),(738,'3/3','Reed Jacob',NULL),(739,'3/10','Mahmoud Shobair',NULL),(740,'3/17','Yazhong Tao',NULL),(741,'3/24','Andrey Krokhotin',NULL),(742,'3/31','Jimmy Fay',NULL),(743,'3/17','Mahmoud Shobair',NULL),(744,'3/24','Yazhong Tao',NULL),(745,'3/31','Andrey Krokhotin',NULL),(746,'4/7','Alex Carlson',NULL),(747,'4/14','Jimmy Fay',NULL),(748,'4/21','Jhuma Das',NULL),(749,'4/28','Konstantin Popov',NULL),(750,'4/21','Konstantin Popov',NULL),(751,'4/28','Jhuma Das',NULL),(752,'5/5','Marino Convertino',NULL),(753,'5/5','Joe Kousouros',NULL),(754,'5/12','Marino Convertino',NULL),(755,'5/19','Onur Dagliyan',NULL),(756,'5/26','Cheng Zhu',NULL),(757,'6/2','David Mowrey',NULL),(758,'6/9','Yazhong Tao',NULL),(759,'6/16','Mahmoud Shobair',NULL),(760,'6/23','Andrey Krokhotin',NULL),(761,'6/30','Benfeard Williams',NULL),(762,'6/2','Cheng Zhu',NULL),(763,'5/26','David Mowrey',NULL),(764,'6/2','Mahmoud Shobair',NULL),(765,'6/16','Cheng Zhu',NULL),(766,'7/7','Jimmy Fay',NULL),(767,'7/14','Konstantin Popov',NULL),(768,'7/21','Jhuma Das',NULL),(769,'7/28','Reed Jacob',NULL),(770,'8/4','Hanna Trzeciakiewicz',NULL),(771,'8/11','Marino Convertino',NULL),(772,'8/18','Aspen Gutgsell',NULL),(773,'8/25','Onur Dagliyan',NULL),(774,'8/4','Cheng Zhu',NULL),(775,'9/1','David Mowrey',NULL),(776,'9/8','Cheng Zhu',NULL),(777,'9/15','Yazhong Tao',NULL),(778,'9/22','Mahmoud Shobair',NULL),(779,'9/29','Andrey Krokhotin',NULL),(780,'9/8','Yazhong Tao',NULL),(781,'9/15','Mahmoud Shobair',NULL),(782,'9/22','Andrey Krokhotin',NULL),(783,'9/29','Benfeard Williams',NULL),(784,'8/18','Aspen Gutgsell',NULL),(785,'8/25','Onur Dagliyan',NULL),(786,'9/1','David Mowrey',NULL),(787,'10/6','Konstantin Popov',NULL),(788,'10/13','Reed Jacob',NULL),(789,'10/20','Jhuma Das',NULL),(790,'10/27','Cheng Zhu',NULL),(791,'10/20','Cheng Zhu',NULL),(792,'10/27','Jhuma Das',NULL),(793,'11/3','Stephan Kudlacek, Ja',NULL),(794,'11/3','Stephan K., Jack M.',NULL),(795,'11/10','Marino Convertino',NULL),(796,'11/17','Onur Dagliyan',NULL),(797,'11/24','Aspen Gutgsell',NULL),(798,'12/1','David Mowrey',NULL),(799,'12/8','Mahmoud Shobair',NULL),(800,'12/8','Yazhong Tao',NULL),(801,'12/15','Mahmoud Shobair',NULL),(802,'12/22','Andrey Krokhotin',NULL),(803,'12/29','Benfeard Williams',NULL),(804,'11/24','Elizabeth Proctor',NULL),(805,'12/1','Aspen Gutgsell',NULL),(806,'12/8','David Mowrey',NULL),(807,'12/15','Yazhong Tao',NULL),(808,'12/22','Mahmoud Shobair',NULL),(809,'12/29','Andrey Krokhotin',NULL),(810,'1/5','Andrey Krokhotin',NULL),(811,'1/12','Benfeard Williams',NULL),(812,'1/19','Konstantin Popov',NULL),(813,'1/26','Jhuma Das',NULL),(814,'2/2','Jessica Hobson',NULL),(815,'2/9','Marino Convertino',NULL),(816,'2/9','Onur Dagliyan',NULL),(817,'2/16','Marino Convertino',NULL),(818,'2/23','David Mowrey',NULL),(819,'1/26','Practice talks',NULL),(820,'2/9','Jhuma Das',NULL),(821,'2/16','Onur Dagliyan',NULL),(822,'2/23','Reed Jacob',NULL),(823,'3/1','Marino Convertino',NULL),(824,'3/8','David Mowrey',NULL),(825,'3/15','Mahmoud Shobair',NULL),(826,'3/22','Andrey Krokhotin',NULL),(827,'3/29','Benfeard Williams',NULL),(828,'4/5','Konstantin Popov',NULL),(829,'4/12','Jhuma Das',NULL),(830,'4/19','Reed Jacob',NULL),(831,'4/26','Marino Convertino',NULL),(832,'4/19','Jimmy Fay',NULL),(833,'4/26','Reed Jacob',NULL),(834,'5/3','Marino Convertino',NULL),(835,'5/10','Onur Dagliyan',NULL),(836,'5/17','David Mowrey',NULL),(837,'5/3','Marino Convertino',NULL),(838,'5/24','Mahmoud Shobair',NULL),(839,'5/31','Andrey Krokhotin',NULL),(840,'6/7','Benfeard Williams',NULL),(841,'6/14','Konstantin Popov',NULL),(842,'6/21','Jhuma Das',NULL),(843,'6/28','Reed Jacob',NULL),(844,'7/5','Marino Convertino',NULL),(845,'7/12','Onur Dagliyan',NULL),(846,'7/19','David Mowrey',NULL),(847,'7/26','Mahmoud Shobair',NULL),(848,'7/12','Marino Convertino',NULL),(849,'7/5','Onur Dagliyan',NULL),(850,'6/28','Marino Convertino',NULL),(851,'7/12','Reed Jacob',NULL),(852,'7/12','Onur Dagliyan',NULL),(853,'7/5','Reed Jacob',NULL),(854,'8/2','Andrey Krokhotin',NULL),(855,'8/9','Benfeard Williams',NULL),(856,'8/16','Cheng Zhu',NULL),(857,'8/23','Konstantin Popov',NULL),(858,'8/30','Jhuma Das',NULL),(859,'7/26','Mahmoud Shobair',NULL),(860,'7/19','David Mowrey',NULL),(861,'09/06','Reed Jacob',NULL),(862,'09/13','Marino Convertino',NULL),(863,'09/20','David Mowrey',NULL),(864,'09/27','Mahmoud Shobair',NULL),(865,'08/30','Konstantin Popov',NULL),(866,'09/06','Jhuma Das',NULL),(867,'09/13','Reed Jacob',NULL),(868,'09/20','Marino Convertino',NULL),(869,'09/27','David Mowrey',NULL),(870,'09/13','Marino Convertino',NULL),(871,'09/20','Reed Jacob',NULL),(872,'10/4','Mahmoud Shobair',NULL),(873,'10/11','Andrey Krokhotin',NULL),(874,'10/18','Benfeard Williams',NULL),(875,'10/25','Cheng Zhu',NULL),(876,'10/11','Nikolay Dokholyan',NULL),(877,'10/18','Andrey Krokhotin',NULL),(878,'10/25','Benfeard Williams',NULL),(879,'11/1','Kasey Skinner, Alice',NULL),(880,'11/8','Konstantin Popov',NULL),(881,'11/15','Jhuma Das',NULL),(882,'11/22','Reed Jacbo',NULL),(883,'11/22','Reed Jacob',NULL),(884,'11/29','Marino Convertino',NULL),(885,'11/1','Kasey, Alice',NULL),(886,'12/6','David Mowrey',NULL),(887,'12/13','Mahmoud Shobair',NULL),(888,'12/20','Andrey Krokhotin',NULL),(889,'1/3','Andrey Krokhotin',NULL),(890,'1/10','Benfeard Williams',NULL),(891,'1/17','Cheng Zhu',NULL),(892,'1/24','Konstantin Popov',NULL),(893,'1/31','Jhuma Das',NULL),(894,'1/10','UNC closed Snow',NULL),(895,'1/17','Benfeard Williams',NULL),(896,'1/24','Cheng Zhu',NULL),(897,'1/31','Konstantin Popov',NULL),(898,'2/7','Jhuma Das',NULL),(899,'2/7','Reed Jacob',NULL),(900,'2/7','Jhuma Das',NULL),(901,'2/14','Reed Jacob',NULL),(902,'2/21','David Mowrey',NULL),(903,'2/28','Mahmoud Shobair',NULL),(904,'2/28','Reed Jacob',NULL),(905,'2/21','Mahmoud Shobair',NULL),(906,'2/14','David Mowrey',NULL),(907,'1/24','Cheng Zhu',NULL),(908,'1/31','Konstantin Popov',NULL),(909,'2/21','David Mowrey',NULL),(910,'2/14','',NULL),(911,'2/28','Mahmoud Shobair',NULL),(912,'3/7','Reed Jacob',NULL),(913,'3/14','Andrey Krokhotin',NULL),(914,'3/21','Benfeard Williams',NULL),(915,'3/28','Cheng Zhu',NULL),(916,'3/21','Conor Parks',NULL),(917,'4/4','Konstantin Popov',NULL),(918,'4/11','Jhuma Das',NULL),(919,'4/18','Benfeard Williams',NULL),(920,'4/25','Reed Jacob',NULL),(921,'3/28','Enrico Ragni',NULL),(922,'4/4','Cheng Zhu',NULL),(923,'4/11','Konstantin Popov',NULL),(924,'4/25','Jhuma Das',NULL),(925,'5/2','Reed Jacob',NULL),(926,'5/9','Marino Convertino',NULL),(927,'5/16','Andrey Krokhotin',NULL),(928,'5/23','Cheng Zhu',NULL),(929,'5/30','Konstantin Popov',NULL),(930,'6/6','Jhuma Das',NULL),(931,'6/13','Andrey Krokhotin',NULL),(932,'6/20','Cheng Zhu',NULL),(933,'6/27','Konstantin Popov',NULL),(934,'5/30','No GM',NULL),(935,'6/6','Konstantin Popov',NULL),(936,'6/13','Jhuma Das',NULL),(937,'6/20','Andrey Krokhotin',NULL),(938,'6/27','Cheng Zhu',NULL),(939,'6/13','No GM',NULL),(940,'6/20','Jhuma Das',NULL),(941,'6/27','No GM',NULL),(942,'7/18','Ernesto Sevilla',NULL),(943,'8/1','David Bednar',NULL),(944,'8/3','Jhuma Das',NULL),(945,'8/8','Andrey Krokhotin',NULL),(946,'8/15','Cheng Zhu',NULL),(947,'8/22','Konstantin Popov',NULL),(948,'8/29','Jhuma Das',NULL),(949,'8/10','Jhuma Das',NULL),(950,'8/17','Cheng Zhu',NULL),(951,'8/31','Konstantin Popov',NULL),(952,'8/17','Cheng Zhu',NULL),(953,'8/29','David Bednar',NULL),(954,'9/5','Venkat Reddy',NULL),(955,'9/12','Jian Wang',NULL),(956,'9/19','Ashhar Khan',NULL),(957,'9/5','Andrey Krokhotin',NULL),(958,'9/12','Venkat Reddy',NULL),(959,'9/19','Jian Wang',NULL),(960,'9/26','Ashhar Khan',NULL),(961,'8/22','Konstantin Popov',NULL);
/*!40000 ALTER TABLE `GroupMeeting` ENABLE KEYS */;
UNLOCK TABLES;

--
-- Table structure for table `JournalClub`
--

DROP TABLE IF EXISTS `JournalClub`;
/*!40101 SET @saved_cs_client     = @@character_set_client */;
/*!40101 SET character_set_client = utf8 */;
CREATE TABLE `JournalClub` (
  `ID` int(11) NOT NULL AUTO_INCREMENT,
  `Date` varchar(10) DEFAULT NULL,
  `Name` varchar(20) DEFAULT NULL,
  `Content` varchar(100) DEFAULT NULL,
  PRIMARY KEY (`ID`)
) ENGINE=MyISAM AUTO_INCREMENT=926 DEFAULT CHARSET=latin1;
/*!40101 SET character_set_client = @saved_cs_client */;

--
-- Dumping data for table `JournalClub`
--

LOCK TABLES `JournalClub` WRITE;
/*!40000 ALTER TABLE `JournalClub` DISABLE KEYS */;
INSERT INTO `JournalClub` VALUES (1,'07/07','Gong Peng',''),(2,'07/14','Jeremy Deyton',''),(3,'07/21','Joshua J. LaRocque',''),(4,'07/28','Tony Law',''),(5,'08/04','Yiwen Chen',''),(6,'08/11','Huifen Nie',''),(7,'08/18','Feng Ding',''),(8,'08/25','kyle Wilcox',''),(9,'09/01','Sagar Khare',''),(10,'09/08','Shantanu Sharma',''),(11,'09/15','Gong Peng',''),(12,'09/22','Huifen Nie',''),(13,'09/29','Yiwen Chen',''),(14,'10/03','Jeremy Deyton',''),(15,'10/10','Yueheng Lan',''),(16,'10/17','Alexey Savelyev',''),(17,'10/24','Sangwook Wu',''),(18,'10/31','Pavel Zhuravlev',''),(19,'10/06','Yueheng Lan',''),(20,'10/13','Jeremy Deyton',''),(21,'10/20','Alexey Savelyev',''),(22,'10/27','Pavel Zhuravlev',''),(23,'11/03','Sangwook Wu',''),(24,'11/10','Lanyuan Lu',''),(25,'11/17','Feng Ding',''),(26,'11/24','',''),(27,'12/01','Kyle Wilcox',''),(28,'12/08','Huifen Nie',''),(29,'12/15','Yiwen Chen',''),(30,'12/22','Adrian Serohijos',''),(31,'01/05','Shantanu Sharma',''),(32,'01/12','',''),(33,'01/19','Yueheng Lan',''),(35,'02/02','Natsuki Tanaka',''),(34,'01/26','Natsuki Tanaka',''),(36,'02/09','Alexey Savelyev',''),(37,'02/16','Sangwook Wu',''),(38,'02/23','Pavel Zhuravlev',''),(39,'03/02','Lanyuan Lu',''),(40,'03/09','Feng Ding',''),(41,'03/16','Kyle Wilcox',''),(42,'03/23','Peng Gong',''),(43,'03/30','Huifen Nie',''),(44,'04/06','Yiwen Chen',''),(45,'04/13','Adrian Serohijos',''),(46,'04/20','Yueheng Lan',''),(47,'04/27','Alexey Savelyev',''),(48,'05/04','',''),(49,'05/11','Lanyuan Lu',''),(50,'05/18','Feng Ding',''),(51,'05/25','Kyle Wilcox',''),(52,'06/01','Huifen Nie',''),(53,'06/08','',''),(54,'06/15','Yiwen Chen',''),(55,'06/22','Adrian Serohijos',''),(56,'06/29','Shantanu Sharma',''),(57,'07/06','Natsuki Tanaka',''),(58,'07/13','Yueheng Lan',''),(59,'07/20','Alexey Savelyev',''),(60,'07/27','Sangwook Wu',''),(61,'08/03','Pavel Zhuravlev',''),(62,'08/10','Lanyuan Lu',''),(63,'08/17','',''),(64,'08/24','Feng Ding',''),(65,'08/31','Kyle Wilcox',''),(66,'09/07','Huifen Nie',''),(67,'09/14','',''),(68,'09/21','Yiwen Chen',''),(69,'09/28','Adrian Serohijos',''),(70,'10/05','Shantanu Sharma',''),(71,'10/12','Shuangye Yin',''),(72,'10/19','Tamas Hegedus',''),(73,'10/26','Natsuki Tanaka',''),(74,'11/02','',''),(75,'11/09','Yueheng Lan',''),(76,'11/16','Alexey Savelyev',''),(77,'11/23','',''),(78,'11/30','Sangwook Wu',''),(79,'12/07','',''),(80,'12/14','Pavel Zhuravlev',''),(81,'12/21','',''),(82,'12/28','',''),(83,'01/04','Zhancheng zhang',''),(84,'01/11','Feng Ding',''),(85,'01/18','Kyle Wilcox',''),(86,'01/25','Huifen Nie',''),(87,'02/01','Yiwen Chen',''),(88,'02/08','Adrian Serohijos',''),(89,'02/13','Kyle Wilcox',''),(90,'02/20','Adrian Serohijos',''),(91,'02/27','Shantanu Sharma',''),(92,'01/25','Yiwen Chen',''),(93,'02/01','Huifen Nie',''),(94,'02/08','Adrian Serohijos',''),(95,'02/15','Shantanu Sharma',''),(96,'02/22','Shuangye Yin',''),(97,'03/01','Tamas Hegedus',''),(98,'03/08','Barry Kesner',''),(99,'03/15','Natsuki Tanaka',''),(100,'03/22','Yueheng Lan',''),(101,'03/29','Alexey Savelyev',''),(102,'04/05','Chris Materese',''),(103,'04/12','',''),(104,'04/19','Sangwook Wu',''),(105,'04/26','Zhancheng Zhang',''),(106,'05/03','Pavel Zhuravlev',''),(107,'05/10','Charles Davis',''),(108,'05/17','Feng Ding',''),(109,'05/24','Kyle Wilcox',''),(110,'05/31','Adrian Serohijos',''),(111,'06/07','Shantanu Sharma',''),(112,'06/14','Barry Kesner',''),(113,'06/21','Shuangye Yin',''),(114,'6/28','Srinivas Ramachandra',''),(115,'7/5','Natsuki Tanaka',''),(116,'7/12','',''),(117,'7/19','Alexey Savelyev',''),(118,'7/26','Chris Materese',''),(119,'8/2','',''),(120,'8/9','Sangwook Wu',''),(121,'8/16','Pavel Zhuravlev',''),(122,'8/23','Zhancheng Zhang',''),(123,'8/30','Charles Davis',''),(124,'9/6','Feng Ding',''),(125,'9/13','Kyle Wilcox',''),(126,'9/20','',''),(127,'9/27','Adrian Serohijos',''),(128,'10/4','Shantanu Sharma',''),(129,'10/11','Shuangye Yin',''),(130,'10/18','Barry Kesner',''),(131,'10/25','Douglas Tsao',''),(132,'11/1','Srinivas Ramachandra',''),(133,'11/8','',''),(134,'11/15','Natsuki Tanaka',''),(135,'11/22','',''),(136,'11/29','SKIP',''),(137,'12/6','',''),(139,'12/20','Alexey Savelyev',''),(138,'12/13','',''),(140,'1/10','Chris Materese',''),(141,'1/17','',''),(142,'1/24','',''),(143,'1/31','Pavel Zhuravlev',''),(144,'2/7','Zhancheng Zhang',''),(145,'2/14','Feng Ding',''),(146,'2/21','Charles Davis',''),(147,'2/28','Kyle Wilcox',''),(148,'3/6','Adrian Serohijos',''),(149,'3/13','Shantanu Sharma',''),(150,'3/20','Shuangye Yin',''),(151,'3/27','Tamas Hegedus',''),(152,'4/3','Barry Kesner',''),(153,'4/10','Douglas Tsao',''),(154,'4/17','Natsuki Tanaka',''),(155,'4/24','Srinivas Ramachandra',''),(156,'5/1','Alexey Savelyev',''),(157,'5/8','Chris Materese',''),(158,'5/15','Pavel Zhuravlev',''),(159,'5/22','Charles Davis',''),(160,'5/29','Zhancheng Zhang',''),(161,'6/5','Longhua Hu',''),(162,'6/12','Eun & Potoyan',''),(163,'6/19','Ding & Hegedus',''),(164,'6.26','Serohijos & Sharma',''),(165,'7/3','Wilcox & Tsao',''),(166,'7/10','Yin & Kesner',''),(167,'7/17','Ramachandran  & Kota',''),(168,'7/24','Tanaka & Savelyev ',''),(169,'7/31','Materese & Zhuravlev',''),(170,'8/7','Zhang & Davis',''),(171,'8/14','Eun',''),(172,'8/21','Hu & Potoyan',''),(173,'8/28','Ding &  Wilcox ',''),(174,'9/4','Serohijos & Yin',''),(175,'9/11','Hegedus & Tsao',''),(176,'9/18','Sharma & Kesner ',''),(177,'9/25','Ramachandran & Kota',''),(178,'10/2','Tanaka & Savelyev ',''),(179,'10/9','Materese & Zhuravlev',''),(180,'10/16','Zhang & Davis ',''),(181,'10/23','Hu & Eun ',''),(182,'10/30','Potoyan & Ding',''),(183,'11/6','Ding & Wilcox',''),(184,'11/13','Serohijos & Hegdus',''),(185,'11/20','Sharma & Tsao',''),(186,'11/27','',''),(187,'12/4','Kesner & Ramachandra',''),(188,'12/11','Kota & Yin',''),(189,'12/18','',''),(190,'1/8','',''),(191,'1/15','',''),(192,'1/14','Feng Ding',''),(193,'1/21','Rachel Redler',''),(194,'1/28','Shuangye Yin',''),(195,'2/4',' Potoyan & Serohijos',''),(196,'2/11','Wilcox & Ding',''),(197,'2/18','',''),(198,'2/26','Hegedus & Kesner',''),(199,'3/5','Yin & Ramachandran',''),(200,'3/12','Kota & Tanaka',''),(201,'3/19','Savelyev & Materese',''),(202,'3/26','',''),(203,'4/2',' Zhuravlev & Davis',''),(204,'4/9','Hu & Eun',''),(205,'4/16','Potoyan & Wilcox',''),(206,'4/22',' Ding &  Serohijos',''),(207,'4/29','Yin &  Kesner',''),(208,'5/6','Hegedus & Ramachandr',''),(209,'5/13','Kota & Tanaka',''),(210,'5/20','Materese & Zhuravlev',''),(211,'5/27','David Shirvanyants',''),(212,'6/3','Eun &  Davis\r\n',''),(213,'6/10','',''),(214,'6/17','Hegedus & Yin',''),(215,'6/24',' Wilcox & Ding',''),(216,'7/1','Kota & Ramachandra',''),(217,'7/8','Proctor & Kesner',''),(218,'7/15','Tsao & Zhuravlev',''),(219,'7/22','',''),(220,'7/29','Materese & Davis',''),(221,'8/5','Hu & Eun',''),(222,'8/12','Potoyan & Minakova',''),(223,'8/19','Davtyan & Ding',''),(224,'8/26','Wilcox & Ramachandra',''),(225,'9/2','Kota & Redler',''),(226,'9/9','Gergely Gyimesi ',''),(227,'9/16','Douglas Tsao',''),(228,'9/23','Shuangye Yin',''),(230,'9/30','Rachel Redler',''),(231,'10/7','Ramachandran & Davty',''),(232,'10/14','Redler & Proctor',''),(233,'10/21','Kota & Zhuravlev',''),(234,'10/28','Materese & Davis',NULL),(235,'11/5','Shuange Yin','TBD'),(236,'11/12','Srinivas Ramachandra',''),(237,'11/19','Rachel Redler',''),(238,'11/26','',''),(239,'12/3','Elizabeth Proctor',''),(241,'12/10 ','Pradeep Kota',''),(242,'12/17','Douglas Tsao',''),(243,'1/7','Feng Ding',''),(244,'1/14','Feng Ding',''),(245,'1/21','Rachel Redler',''),(246,'1/28','Shuangye Yin',''),(248,'2/11','Srinivas Ramachandra',''),(247,'2/4','Elizabeth Proctor',''),(249,'2/18','Pradeep Kota',''),(250,'2/25','Douglas Tsao',''),(251,'3/4','Elizabeth Proctor',''),(252,'3/11','Feng Ding',''),(253,'3/18','Rachel Redler',''),(254,'3/25','Shuangye Yin',''),(255,'4/1','Elizabeth Proctor',''),(256,'4/8','Srinivas Ramachandra',''),(257,'4/15','Pradeep Kota',''),(258,'4/22','Douglas Tsao',''),(259,'4/29','Elizabeth Proctor',''),(260,'5/6','Shuangye Yin',''),(261,'5/13',' Rachel Redler',''),(262,'5/20','Feng Ding',''),(263,'5/27','David Shirvanyants',''),(266,'6/3','Jeremey Hanhs',''),(267,'6/10','Elizabeth Proctor',''),(268,'6/17','Srinivas Ramachandra',''),(269,'6/24','Pradeep Kota',''),(270,'7/1','Douglas Tsao',''),(271,'7/8','Jeremy Hahn',''),(272,'7/15','Shuangye Yin',''),(273,'7/22','Rachel Redler',''),(274,'7/29','',''),(275,'8/5','David Shirvanyants',''),(276,'8/12','Feng Ding',''),(279,'9/2','Pradeep Kota',''),(277,'8/19','Elizabeth Proctor',''),(278,'8/26','Srinivas Ramachandra',''),(280,'9/9','Gergely Gyimesi ',''),(281,'9/16','Douglas Tsao',''),(282,'9/23','Shuangye Yin',''),(283,'9/30','Rachel Redler',''),(284,'10/7','David Shirvanyants',''),(285,'10/14','Feng Ding',''),(286,'10/21','Elizabeth Proctor',''),(287,'10/28','Srinivas Ramachandra',''),(288,'11/4','Pradeep Kota',''),(289,'11/11','Douglas Tsao',''),(290,'11/18','Rachel Redler',''),(291,'11/25','Shuangye Yin',''),(292,'12/16','David Shirvanyants',''),(293,'12/23','Feng Ding',''),(294,'12/30','',''),(295,'1/6','Elizabeth Proctor',''),(296,'1/13','Srinivas Ramachandra',''),(297,'1/20','Pradeep Kota',''),(298,'1/27','Douglas Tsao',''),(299,'2/3','Rachel Redler',''),(300,'2/10','David Shirvanyants',''),(301,'2/17','Feng Ding',''),(302,'2/24','Elizabeth Proctor',''),(303,'3/3','Srinivas Ramachandra',''),(304,'3/10','Pradeep Kota',''),(305,'3/17','Douglas Tsao',''),(306,'3/24','Rachel Redler',''),(307,'3/31','',''),(308,'4/7','David Shirvanyants',''),(309,'4/14','',''),(310,'4/21','Elizabeth Proctor',''),(311,'4/28','Srinivas Ramachandra',''),(312,'4/21','Feng Ding',''),(313,'4/28','Elizabeth Proctor',''),(314,'5/5','Srinivas Ramachandra',''),(315,'5/12','Pradeep Kota',''),(316,'5/19','Rachel Redler',''),(317,'5/26','Douglas Tsao',''),(318,'6/2','David Shirvanyants',''),(319,'6/9','Feng Ding',''),(320,'6/16','Elizabeth Proctor',''),(321,'6/23','Pradeep Kota',''),(322,'6/30','Srinivas Ramachandra',''),(323,'7/7','Rachel Redler',''),(324,'7/14','Douglas Tsao',''),(325,'7/21','David Shirvanyants',''),(326,'7/28','Feng Ding',''),(327,'8/4','Elizabeth Proctor',''),(328,'8/11','',''),(329,'8/18','Srinivas Ramachandra',''),(330,'8/25','Pradeep Kota',''),(331,'9/1','Rachel Redler',''),(332,'9/8','Onur Dagliyan',''),(333,'9/15','David Shirvanyants',''),(334,'9/22','Feng Ding',''),(335,'9/29','Elizabeth Proctor',''),(336,'10/6','Srinivas Ramachandra',''),(337,'10/13','Pradeep Kota',''),(338,'10/20','Arpit Tandon',''),(339,'10/27','Onur Dagliyan',''),(340,'11/3','David Shirvanyants',''),(341,'11/10','Rachel Redler',''),(342,'11/17','Feng Ding',''),(343,'11/24','',''),(344,'12/1','Elizabeth Proctor',''),(345,'12/8','Srinivas Ramachandra',''),(346,'12/15','Pradeep Kota',''),(347,'12/22','Arpit Tandon',''),(348,'12/29','Onur Dagliyan',''),(349,'1/5','David Shirvanyants',''),(350,'1/12','Srinivas Ramachandra',''),(351,'1/19','Elizabeth Proctor',''),(352,'1/26','Feng Ding',''),(353,'2/2','Pradeep Kota',''),(354,'2/9','Onur Dagliyan',''),(355,'2/16','David Shirvanyants',''),(356,'2/23','',''),(357,'3/1','Rachel Redler',''),(358,'3/8','Feng Ding',''),(359,'3/15','Elizabeth Proctor',''),(360,'3/22','Srinivas Ramachandra',''),(361,'3/29','Pradeep Kota',''),(362,'4/5','Pradeep Kota',''),(363,'4/12','Onur Dagliyan',''),(364,'4/19','David Shirvanyants',''),(365,'4/26','Rachel Redler',''),(366,'5/3','Elizabeth Proctor',NULL),(367,'5/10','Srinivas Ramachandra',NULL),(368,'5/17','Pradeep Kota',NULL),(369,'5/24','Onur Dagliyan',NULL),(370,'5/31','David Shirvanyants',NULL),(371,'6/7','Rachel Redler',''),(372,'6/14','Arpit Tandon',''),(373,'6/21','Elizabeth Proctor',''),(374,'6/28','Srinivas Ramachandra',''),(375,'7/5','Pradeep Kota',''),(376,'7/12','Onur Dagliyan',''),(377,'7/19','David Shirvanyants',''),(378,'7/26','Rachel Redler',''),(379,'8/2','Arpit Tandon',''),(380,'8/9','Elizabeth Proctor',''),(381,'8/16','Pradeep Kota',''),(382,'8/23','Onur Dagliyan',''),(383,'8/30','David Shirvanyants',''),(384,'9/6','Rachel Redler',''),(385,'9/13','Onur Dagliyan',''),(386,'9/20','Arpit Tandon',''),(387,'9/27','Elizabeth Proctor',''),(388,'9/27','Jhuma Das',NULL),(389,'10/4','Elizabeth Proctor',NULL),(390,'10/11','Onur Dagliyan',NULL),(391,'10/18','Rachel Redler',NULL),(392,'10/25','Arpit Tandon',NULL),(393,'11/1','David Shirvanyants',NULL),(394,'11/8','Jhuma Das',NULL),(395,'10/9','Gurpreet Kaur',NULL),(396,'11/15','Elizabeth Proctor',NULL),(397,'11/22','Onur Dagliyan',NULL),(398,'11/29','Rachel Redler',NULL),(399,'11/29','Gurpreet Kaur',NULL),(400,'12/6','Rachel Redler',NULL),(401,'10/25','Rachel Redler',NULL),(402,'11/1','Arpit Tandon',NULL),(403,'11/8','David Shirvanyants',NULL),(404,'11/15','Jhuma Das',NULL),(405,'11/22','Elizabeth Proctor',NULL),(406,'11/29','Onur Dagliyan',NULL),(407,'12/6','Gurpreet Kaur',NULL),(408,'12/13','Rachel Redler',NULL),(409,'12/20','Arpit Tandon',NULL),(410,'11/6','Jimmy Fay',NULL),(411,'12/13','Arpit Tandon',NULL),(412,'12/20','Rachel Redler',NULL),(413,'12/13','Rachel Redler',''),(414,'12/20','Arpit Tandon',''),(415,'11/8','David Shirvanyants',''),(416,'11/15','Jhuma Das',''),(417,'11/22','Elizabeth Proctor',''),(418,'11/29','Onur Dagliyan',''),(419,'12/6','Gurpreet Kaur',''),(420,'11/22','Arpit Tandon',''),(421,'11/29','Elizabeth Proctor',''),(422,'12/6','Onur Dagliyan',''),(423,'12/13','Gurpreet Kaur',''),(424,'12/20','Rachel Redler',''),(425,'11/15','David Shirvanyants',''),(426,'11/22','Jhuma Das',''),(427,'11/29','Arpit Tandon',''),(428,'12/6','Elizabeth Proctor',''),(429,'12/13','Onur Dagliyan',''),(430,'12/20','Gurpreet Kaur',''),(431,'11/29','Jhuma Das',''),(432,'12/6','Arpit Tandon',''),(433,'12/13','Elizabeth Proctor',''),(434,'12/20','Onur Dagliyan',''),(435,'12/27','Gurpreet Kaur',''),(436,'12/22','No JC',''),(437,'11/29','Arpit Tandon',''),(438,'12/6','Jhuma Das',''),(439,'12/27','Onur Dagliyan',''),(440,'12/20','Gurpreet Kaur',''),(441,'12/6','Nikolay Dokholyan',''),(442,'12/13','Jhuma Das',''),(443,'12/20','Elizabeth Proctor',''),(444,'12/27','Gurpreet Kaur',''),(445,'12/27','',''),(446,'1/3','Gurpreet Kaur',''),(447,'1/10','Rachel Redler',''),(448,'1/17','Onur Dagliyan',''),(449,'1/24','David Shirvanyants',''),(450,'1/31','Jhuma Das',''),(451,'2/7','Arpit Tandon',''),(452,'2/14','Gurpreet Kaur',''),(453,'2/14','Elizabeth Proctor',''),(454,'2/21','Marino Convertino',''),(455,'2/28','Gurpreet Kaur',''),(456,'2/28','Mahmoud Shobair',''),(457,'3/7','Gurpreet Kaur',''),(458,'3/14','Rachel Redler',''),(459,'3/21','Onur Dagliyan',''),(460,'3/28','David Shirvanyants',''),(461,'4/4','Jhuma Das',''),(462,'3/28','Raul Mendez',''),(463,'4/4','David Shirvanyants',''),(464,'2/14','Elizabeth Proctor',''),(465,'2/21','Marino Convertino',''),(466,'2/28','Mahmoud Shobair',''),(467,'4/11','Jhuma Das',''),(468,'4/18','Arpit Tandon',''),(469,'4/25','Elizabeth Proctor',''),(470,'3/7','Gurpreet Kaur',''),(471,'3/14','Rachel Redler',''),(472,'3/21','Onur Dagliyan',''),(473,'3/28','Raul Mendez',''),(474,'4/4','David Shirvanyants',''),(475,'4/11','Jhuma Das',''),(476,'3/7','Nikolay Dokholyan',''),(477,'3/14','Arpit Tandon',''),(478,'3/21','Arpit Tandon',''),(479,'3/28','Arpit Tandon',''),(480,'3/14','Gurpreet Kaur',''),(481,'3/21','Rachel Redler',''),(482,'3/28','Onur Dagliyan',''),(483,'4/4','Raul Mendez',''),(484,'4/11','David Shirvanyants',''),(485,'4/18','Reed Jacob',''),(486,'4/25','Jhuma Das',''),(487,'5/2','Arpit Tandon',''),(488,'5/9','Elizabeth Proctor',''),(489,'5/2','Elizabeth Proctor',''),(490,'5/9','Jimmy Fay',''),(491,'5/16','Arpit Tandon',''),(492,'5/23','Marino Convertino',''),(493,'5/30','Andrey Krokhotin',''),(494,'5/2','Arpit Tandon',''),(495,'5/9','Elizabeth Proctor',''),(496,'5/23','Jimmy Fay',''),(497,'5/30','Arpit Tandon',''),(498,'6/6','Marino Convertino',''),(499,'6/13','Arpit Tandon',''),(500,'6/20','Andrey Krokhotin',''),(501,'6/27','Arpit Tandon',''),(502,'5/16','Arpit Tandon',''),(503,'6/4','Rachel Redler',''),(504,'6/11','Arpit Tandon',''),(505,'7/11','Arpit Tandon',''),(506,'7/4','Rachel Redler',''),(507,'6/27','Rachel Redler',''),(508,'7/4','Arpit Tandon',''),(509,'7/11','Onur Dagliyan',''),(510,'6/13','Arpit Tandon',''),(511,'6/20','Andrey Krokhotin',''),(512,'6/27','Mahmoud Shobair',NULL),(513,'7/4','Rachel Redler',NULL),(514,'6/27','Rachel Redler',NULL),(515,'7/4','Mahmoud Shobair',NULL),(516,'7/11','Rachel Redler',NULL),(517,'7/11','Onur Dagliyan',NULL),(518,'7/11','Rachel Redler',NULL),(519,'7/18','Onur Dagliyan',NULL),(520,'7/18','Arpit Tandon',NULL),(521,'7/4','',NULL),(522,'7/25','Mahmoud Shobair',NULL),(523,'8/1','Raul Mendez',NULL),(524,'8/8','Rachel Redler',NULL),(525,'8/1','David Shirvanyants',NULL),(526,'8/13','James Fay',NULL),(527,'8/13','Arpit Tandon',NULL),(528,'8/13','James Fay',NULL),(529,'8/20','Arpit Tandon',NULL),(530,'8/13','James Fay',NULL),(531,'8/1','Jhuma Das',NULL),(532,'8/15','Marino Convertino',NULL),(533,'8/15','Onur Dagliyan',NULL),(534,'8/15','Marino Convertino',NULL),(535,'8/22','Onur Dagliyan',NULL),(536,'8/29','David Shirvanyants',NULL),(537,'7/26','Mahmoud Shobair',NULL),(538,'8/8','Rachel Redler',NULL),(539,'7/25','Mahmoud Shobair',NULL),(540,'8/8','Marino Convertino',NULL),(541,'8/15','Reed Jacob',NULL),(542,'8/22','Rachel Redler',NULL),(543,'8/29','Onur Dagliyan',NULL),(544,'9/5','David Shirvanyants',NULL),(545,'9/12','James Fay',NULL),(546,'9/19','Raul Mendez',NULL),(547,'9/26','Elizabeth Proctor',NULL),(548,'10/3','Andrey Krokhotin',NULL),(549,'10/10','Arpit Tandon',NULL),(550,'10/17','Mahmoud Shobair',NULL),(551,'10/24','Jhuma Das',NULL),(552,'10/31','Reed Jacob',NULL),(553,'11/7','Marino Convertino',NULL),(554,'10/17','Arpit Tandon',NULL),(555,'10/10','Mahmoud Shobair',NULL),(556,'11/14','Rachel Redler',NULL),(557,'11/21','Onur Dagliyan',NULL),(558,'11/28','David Shirvanyants',NULL),(559,'10/31','Jhuma Das',NULL),(560,'11/7','Reed Jacob',NULL),(561,'11/14','Marino Convertino',NULL),(562,'11/21','Rachel Redler',NULL),(563,'11/28','Onur Dagliyan',NULL),(564,'12/5','James Fay',NULL),(565,'12/12','Raul Mendez',NULL),(566,'12/19','Elizabeth Proctor',NULL),(567,'12/5','Onur Dagliyan',NULL),(568,'11/28','No JC',NULL),(569,'12/12','James Fay',NULL),(570,'12/19','Raul Mendez',NULL),(571,'12/26','No JC',NULL),(572,'1/2','Andrey Krokhotin',NULL),(573,'1/9','Cheng Zhu',NULL),(574,'1/16','Arpit Tandon',NULL),(575,'1/23','Jhuma Das',NULL),(576,'1/30','Mahmoud Shobair',NULL),(577,'2/6','Reed Jacob',NULL),(578,'2/13','Marino Convertino',NULL),(579,'2/20','Raul Mendez',NULL),(580,'3/6','Onur Dagliyan',NULL),(581,'3/13','Jimmy Fay',NULL),(582,'3/20','Dominique Soroka',NULL),(583,'3/20','Joe Kousouros',NULL),(584,'3/27','Dominique Soroka',NULL),(585,'2/27','Elevator talk',NULL),(586,'4/3','Andrey Krokhotin',NULL),(587,'4/10','Cheng Zhu',NULL),(588,'4/17','Arpit Tandon',NULL),(589,'4/24','Jhuma Das',NULL),(590,'4/24','Dominique Soroka',NULL),(591,'5/1','Jhuma Das',NULL),(592,'5/8','Mahmoud Shobair',NULL),(593,'5/15','Reed Jacob',NULL),(594,'5/22','Marino Convertino',NULL),(595,'5/29','Elevator talk',NULL),(596,'5/1','Dominique Soroka',NULL),(597,'5/8','Jhuma Das',NULL),(598,'5/15','Mahmoud Shobair',NULL),(599,'5/22','Reed Jacob',NULL),(600,'5/29','Marino Convertino',NULL),(601,'4/24','No JC',NULL),(602,'5/22','Jhuma Das',NULL),(603,'5/29','Reed Jacob',NULL),(604,'5/8','Mahmoud Shobair',NULL),(605,'6/5','Marino Convertino',NULL),(606,'6/12','Onur Dagliyan',NULL),(607,'6/19','Jimmy Fay',NULL),(608,'6/19','Raul Mendez',NULL),(609,'6/26','Raul Mendez',NULL),(610,'6/19','Jimmy Fay',NULL),(611,'6/5','Jhuma Das',NULL),(612,'6/12','Marino Convertino',NULL),(613,'6/19','Jhuma Das',NULL),(614,'6/26','Onur Dagliyan',NULL),(615,'5/29','Reed Jacob',NULL),(616,'7/3','Jimmy Fay',NULL),(617,'7/10','Raul Mendez',NULL),(618,'7/17','Jhuma Das',NULL),(619,'7/24','Benfeard Williams',NULL),(620,'7/31','Regina Politi',NULL),(621,'7/17','Benfeard Williams',NULL),(622,'7/24','Dominique Soroka',NULL),(623,'7/31','Benfeard Williams',NULL),(624,'7/10','Arpit Tandon',NULL),(625,'7/17','Raul Mendez',NULL),(626,'7/17','Jhuma Das',NULL),(627,'7/24','Raul Mendez',NULL),(628,'7/31','Dominique Soroka',NULL),(629,'7/17','Regina Politi',NULL),(630,'8/7','Benfeard Williams',NULL),(631,'8/14','Andrey Krokhotin',NULL),(632,'8/21','Regina Politi',NULL),(633,'8/28','Cheng Zhu',NULL),(634,'8/21','Cheng Zhu',NULL),(635,'8/28','Regina Politi',NULL),(636,'9/4','Jhuma Das',NULL),(637,'9/11','Mahmoud Shobair',NULL),(638,'9/18','Reed Jacob',NULL),(639,'9/25','Marino Convertino',NULL),(640,'9/18','Marino Convertino',NULL),(641,'9/25','Reed Jacob',NULL),(642,'9/4','Yazhong Tao',NULL),(643,'9/11','Jhuma Das',NULL),(644,'9/18','Mahmoud Shobair',NULL),(645,'9/25','Marino Convertino',NULL),(646,'8/28','Regina Politi',NULL),(647,'10/2','Reed Jacob',NULL),(648,'9/25','Reed Jacob',NULL),(649,'10/2','Onur Dagliyan',NULL),(650,'10/9','Raul Mendez',NULL),(651,'10/16','Jimmy Fay',NULL),(652,'10/23','Dominique Soroka',NULL),(653,'10/30','Marino Convertino',NULL),(654,'10/30','Andrey Krokhotin',NULL),(655,'9/18','Mahmoud Shobair',NULL),(656,'11/6','Marino Convertino',NULL),(657,'11/13','Cheng Zhu',NULL),(658,'11/20','Jhuma Das',NULL),(659,'11/27','Mahmoud Shobair',NULL),(660,'11/27','Thanksgiving',NULL),(661,'12/4','Mahmoud Shobair',NULL),(662,'12/11','Reed Jacob',NULL),(663,'12/18','Raul Mendez',NULL),(664,'12/11','Raul Mendez',NULL),(665,'12/18','David Mowrey',NULL),(666,'1/8','Onur Dagliyan',NULL),(667,'1/15','Dominique Soroka',NULL),(668,'1/22','Marino Convertino',NULL),(669,'1/29','Andrey Krokhotin',NULL),(670,'1/15','Mahmoud Shobair',NULL),(671,'1/22','Andrey Krokhotin',NULL),(672,'1/29','Marino Convertino',NULL),(673,'2/5','Jimmy Fay',NULL),(674,'2/12','Cheng Zhu',NULL),(675,'2/19','Jhuma Das',NULL),(676,'2/26','Konstantin Popov',NULL),(677,'2/12','Jhuma Das',NULL),(678,'2/19','Cheng Zhu',NULL),(679,'2/5','Mahmoud Shobair',NULL),(680,'2/12','Beibei Wang',NULL),(681,'2/26','Jhuma Das',NULL),(682,'2/26','Cheng Zhu',NULL),(683,'2/19','No JC weather issue',NULL),(684,'3/5','Jhuma Das',NULL),(685,'3/12','Konstantin Popov',NULL),(686,'3/19','Benfeard Williams',NULL),(687,'3/26','Reed Jacob',NULL),(688,'3/5','Cheng Zhu',NULL),(689,'3/12','Jhuma Das',NULL),(690,'3/19','Konstantin Popov',NULL),(691,'3/26','Benfeard Williams',NULL),(692,'4/2','Marino Convertino',NULL),(693,'4/2','Jimmy Fay',NULL),(694,'4/9','Marino Convertino',NULL),(695,'4/16','Onur Dagliyan',NULL),(696,'4/23','Reed Jacob',NULL),(697,'4/30','David Mowrey',NULL),(698,'5/7','Andrey Krokhotin',NULL),(699,'5/14','Mahmoud Shobair',NULL),(700,'5/21','Jhuma Das',NULL),(701,'5/28','Konstantin Popov',NULL),(702,'6/4','Benfeard Williams',NULL),(703,'6/11','Jimmy Fay',NULL),(704,'6/11','Onur Dagliyan',NULL),(705,'6/11','Jimmy Fay',NULL),(706,'6/18','Onur Dagliyan',NULL),(707,'6/25','Reed Jacob',NULL),(708,'5/26','David Mowrey',NULL),(709,'5/26','David Mowrey',NULL),(710,'6/25','Onur Dagliyan',NULL),(711,'5/28','Onur Dagliyan',NULL),(712,'6/4','Konstantin Popov',NULL),(713,'6/11','Benfeard Williams',NULL),(714,'6/18','Jimmy Fay',NULL),(715,'7/2','Reed Jacob',NULL),(716,'7/9','Marino Convertino',NULL),(717,'7/16','Aspen Gutgsell',NULL),(718,'7/23','David Mowrey',NULL),(719,'7/30','Andrey Krokhotin',NULL),(720,'8/6','Mahmoud Shobair',NULL),(721,'8/13','Jhuma Das',NULL),(722,'8/20','Konstantin Popov',NULL),(723,'8/27','Benfeard Williams',NULL),(724,'8/13','Konstantin Popov',NULL),(725,'8/20','Jhuma Das',NULL),(726,'8/20','Benfeard Williams',NULL),(727,'8/27','Jhuma Das',NULL),(728,'9/3','Onur Dagliyan',NULL),(729,'9/10','Reed Jacob',NULL),(730,'9/17','Aspen Gutgsell',NULL),(731,'9/24','Marino Convertino',NULL),(732,'10/1','David Mowrey',NULL),(733,'10/8','Andrey Krokhotin',NULL),(734,'10/15','Mahmoud Shobair',NULL),(735,'10/22','Konstantin Popov',NULL),(736,'10/29','Benfeard Williams',NULL),(737,'10/22','3 minutes talk',NULL),(738,'10/29','Konstantin Popov',NULL),(739,'11/5','Benfeard Williams',NULL),(740,'11/12','Jhuma Das',NULL),(741,'11/19','Onur Dagliyan',NULL),(742,'11/26','Reed Jacob',NULL),(743,'11/19','Reed Jacob',NULL),(744,'11/26','Onur Dagliyan',NULL),(745,'11/19','Jhuma Das',NULL),(746,'11/26','Reed Jacob',NULL),(747,'11/26','Thanksgiving',NULL),(748,'12/3','Reed Jacob',NULL),(749,'12/10','Onur Dagliyan',NULL),(750,'12/17','Aspen Gutgsell',NULL),(751,'1/7','David Mowrey',NULL),(752,'1/14','Marino Convertino',NULL),(753,'1/21','Mahmoud Shobair',NULL),(754,'1/28','Andrey Krokhotin',NULL),(755,'2/4','Konstantin Popov',NULL),(756,'2/11','Benfeard Williams',NULL),(757,'2/18','Jhuma Das',NULL),(758,'2/25','Onur Dagliyan',NULL),(759,'2/4','Andrey Krokhotin',NULL),(760,'2/11','Konstantin Popov',NULL),(761,'2/18','Benfeard Williams',NULL),(762,'2/25','Jhuma Das',NULL),(763,'1/28','Practice talks',NULL),(764,'3/3','Onur Dagliyan',NULL),(765,'3/10','Reed Jacob',NULL),(766,'3/10','Aspen Gutgsell',NULL),(767,'3/10','Reed Jacob',NULL),(768,'3/17','Aspen Gutgsell',NULL),(769,'3/24','Marino Convertino',NULL),(770,'3/31','David Mowrey',NULL),(771,'2/18','Jimmy Fay',NULL),(772,'2/25','Benfeard Williams',NULL),(773,'3/3','Jhuma Das',NULL),(774,'3/10','Onur Dagliyan',NULL),(775,'3/17','Reed Jacob',NULL),(776,'3/24','Aspen Gutgsell',NULL),(777,'3/31','Marino Convertino',NULL),(778,'5/7','David Mowrey',NULL),(779,'5/12','Andrey Krokhotin',NULL),(780,'5/19','Onur Dagliyan',NULL),(781,'5/26','Reed Jacob',NULL),(782,'4/7','David Mowrey',NULL),(783,'4/12','Andrey Krokhotin',NULL),(784,'4/14','Andrey Krokhotin',NULL),(785,'4/21','Mahmoud Shobair',NULL),(786,'4/28','Konstantin Popov',NULL),(787,'3/17','Reed Jacob',NULL),(788,'3/24','Aspen Gutgsell',NULL),(789,'3/31','Marino Convertino',NULL),(790,'3/24','Cancelled',NULL),(791,'3/31','Aspen Gutgsell',NULL),(792,'4/7','Marino Convertino',NULL),(793,'4/14','David Mowrey',NULL),(794,'4/21','Andrey Krokhotin',NULL),(795,'4/28','Mahmoud Shobair',NULL),(796,'5/5','Jhuma Das',NULL),(797,'5/12','Konstantin Popov',NULL),(798,'5/19','Benfeard Williams',NULL),(799,'5/26','Reed Jacob',NULL),(800,'5/12','Mahmoud Shobair',NULL),(801,'5/19','Konstantin Popov',NULL),(802,'5/26','Benfeard Williams',NULL),(803,'4/28','BCBP Retreat',NULL),(804,'6/2','Onur Dagliyan',NULL),(805,'6/9','Aspen Gutgsell',NULL),(806,'6/16','Marino Convertino',NULL),(807,'6/23','David Mowrey',NULL),(808,'6/30','Andrey Krokhotin',NULL),(809,'5/19','Benfeard Williams',NULL),(810,'5/26','Konstantin Popov',NULL),(811,'5/26','Aspen Gutgsell',NULL),(812,'6/9','Konstantin Popov',NULL),(813,'7/7','Jhuma Das',NULL),(814,'7/14','Mahmoud Shobair',NULL),(815,'7/21','Konstantin Popov',NULL),(816,'7/28','Benfeard Williams',NULL),(817,'6/16','Marino Convertino',NULL),(818,'6/23','David Mowrey',NULL),(819,'6/30','Andrey Krokhotin',NULL),(820,'7/21','Benfeard Williams',NULL),(821,'7/28','Konstantin Popov',NULL),(822,'8/4','Cheng Zhu',NULL),(823,'8/11','Marino Convertino',NULL),(824,'8/18','David Mowrey',NULL),(825,'8/25','Andrey Krokhotin',NULL),(826,'09/01','Jhuma Das',NULL),(827,'09/08','Mahmoud Shobair',NULL),(828,'09/15','Konstantin Popov',NULL),(829,'09/22','Benfeard Williams',NULL),(830,'09/29','Cheng Zhu',NULL),(831,'09/08','Ria Das',NULL),(832,'09/15','Mahmoud Shobair',NULL),(833,'09/22','Konstantin Popov',NULL),(834,'09/29','Benfeard Williams',NULL),(835,'10/6','Cheng Zhu',NULL),(836,'10/13','Marino Convertino',NULL),(837,'10/20','David Mowrey',NULL),(838,'10/27','Andrey Krokhotin',NULL),(839,'10/6','Marino Convertino',NULL),(840,'10/13','Cheng Zhu',NULL),(841,'10/27','Brandon Price',NULL),(842,'11/3','Biophysics Symposium',NULL),(843,'11/10','Andrey Krokhotin',NULL),(844,'11/17','Mahmoud Shobair',NULL),(845,'11/24','Jhuma Das',NULL),(846,'11/24','Thanksgiving',NULL),(847,'12/1','Jhuma Das',NULL),(848,'12/8','Konstantin Popov',NULL),(849,'12/15','Benfeard Williams',NULL),(850,'12/22','Cheng Zhu',NULL),(851,'12/8','Jhuma Das',NULL),(852,'12/15','Konstantin Popov',NULL),(853,'12/22','Benfeard Williams',NULL),(854,'12/22','postponed',NULL),(855,'12/29','university holiday',NULL),(856,'1/5','Benfeard Williams',NULL),(857,'1/12','Cheng Zhu',NULL),(858,'1/19','Marino Convertino',NULL),(859,'1/26','David Mowrey',NULL),(860,'2/3','Andrey Krokhotin',NULL),(861,'2/10','Mahmoud Shobair',NULL),(862,'2/2','Andrey Krokhotin',NULL),(863,'2/9','Mahmoud Shobair',NULL),(864,'2/16','Jhuma Das',NULL),(865,'2/23','Konstantin Popov',NULL),(866,'3/2','Benfeard Williams',NULL),(867,'3/9','Cheng Zhu',NULL),(868,'3/16','Marino Convertino',NULL),(869,'3/23','David Mowrey',NULL),(870,'3/30','Andrey Krokhotin',NULL),(871,'1/9','Benfeard Williams',NULL),(872,'1/16','Cheng Zhu',NULL),(873,'1/23','Marino Convertino',NULL),(874,'3/9','Benfeard Williams',NULL),(875,'3/16','Cheng Zhu',NULL),(876,'3/23','Marino Convertino',NULL),(877,'3/23','Enrico Ragni',NULL),(878,'3/30','Marino Convertino',NULL),(879,'4/6','Andrey Krokhontin',NULL),(880,'4/13','Mahmoud Shobair',NULL),(881,'4/20','Konstantin Popov',NULL),(882,'4/27','Jhuma Das',NULL),(883,'3/30','Andrey Krokhotin',NULL),(884,'4/6','Jhuma Das',NULL),(885,'4/27','Benfeard Williams',NULL),(886,'3/23','Marino Convertino',NULL),(887,'5/4','Cheng Zhu',NULL),(888,'5/4','Cheng Zhu',NULL),(889,'5/11','Marino Convertino',NULL),(890,'5/18','Andrey Krokhotin',NULL),(891,'5/25','Jhuma Das',NULL),(892,'5/11','Cheng Zhu',NULL),(893,'5/18','Konstantin Popov',NULL),(894,'5/25','Andrey Krokhotin',NULL),(895,'5/4','Cancelled',NULL),(896,'6/1','Jhuma Das',NULL),(897,'6/8','Cheng Zhu',NULL),(898,'6/15','Konstantin Popov',NULL),(899,'6/22','Andrey Krokhontin',NULL),(900,'6/1','no JC',NULL),(901,'6/8','Jhuma Das',NULL),(902,'6/15','Cheng Zhu',NULL),(903,'6/22','Konstantin Popov',NULL),(904,'6/29','Andrey Krokhotin',NULL),(905,'6/15','No JC',NULL),(906,'6/22','Cheng Zhu',NULL),(907,'6/29','No JC',NULL),(908,'7/20','Ashhar Khan',NULL),(909,'7/20','Jhuma Das',NULL),(910,'8/3','Jhuma Das',NULL),(911,'8/10','Cheng Zhu',NULL),(912,'8/17','Konstantin Popov',NULL),(913,'8/24','Andrey Krokhotin',NULL),(914,'8/31','Ashaar Khan',NULL),(915,'7/20','Ashhar Khan',NULL),(916,'8/31','Konstantin Popov',NULL),(917,'8/17','Jhuma Das',NULL),(918,'9/7','Ashhar Khan',NULL),(919,'9/14','Jian Wang',NULL),(920,'9/14','Venkat Reddy',NULL),(921,'9/14','Jian Wang',NULL),(922,'9/28','Venkat Reddy',NULL),(923,'9/28','Jhuma Das',NULL),(924,'9/21','Venkat Reddy',NULL),(925,'8/24','Andrey Krokhotin',NULL);
/*!40000 ALTER TABLE `JournalClub` ENABLE KEYS */;
UNLOCK TABLES;

--
-- Table structure for table `events`
--

DROP TABLE IF EXISTS `events`;
/*!40101 SET @saved_cs_client     = @@character_set_client */;
/*!40101 SET character_set_client = utf8 */;
CREATE TABLE `events` (
  `eid` int(10) unsigned NOT NULL,
  `edescription` varchar(125) NOT NULL,
  `etype` char(1) NOT NULL,
  `efrequency` varchar(10) NOT NULL,
  `evisibility` varchar(10) NOT NULL,
  `evenue` varchar(50) NOT NULL,
  `estart_date` date NOT NULL,
  `eend_date` date NOT NULL,
  `elast_occurrence` date DEFAULT '0000-00-00',
  `estart_time` time NOT NULL,
  `eend_time` time NOT NULL,
  `enotes` text,
  `ecreated_by` varchar(60) NOT NULL,
  `ecreated_on` timestamp NOT NULL DEFAULT CURRENT_TIMESTAMP,
  PRIMARY KEY (`eid`)
) ENGINE=MyISAM DEFAULT CHARSET=utf8;
/*!40101 SET character_set_client = @saved_cs_client */;

--
-- Dumping data for table `events`
--

LOCK TABLES `events` WRITE;
/*!40000 ALTER TABLE `events` DISABLE KEYS */;
INSERT INTO `events` VALUES (1241822265,'Group Meeting','R','weekly','private','rm3007','2009-05-05','2038-01-19','2038-01-19','17:00:00','18:00:00','Weekly Group Meeting','Nikolay Dokholyan','2009-05-08 22:37:39'),(1241932447,'Proposal Defense','O','Once','private','rm3095','2009-05-29','2009-05-29','2038-01-19','15:00:00','17:00:00','I will be defending my PhD proposal','Pradeep Kota','2009-05-10 05:14:07'),(1241931953,'Journal Club','R','weekly','private','rm3007','2009-05-07','2038-01-19','2038-01-19','17:00:00','18:00:00','Weekly Journal Club','Nikolay Dokholyan','2009-05-10 05:05:53');
/*!40000 ALTER TABLE `events` ENABLE KEYS */;
UNLOCK TABLES;

--
-- Table structure for table `mem_info`
--

DROP TABLE IF EXISTS `mem_info`;
/*!40101 SET @saved_cs_client     = @@character_set_client */;
/*!40101 SET character_set_client = utf8 */;
CREATE TABLE `mem_info` (
  `id` int(10) unsigned NOT NULL AUTO_INCREMENT,
  `info` text,
  `education` text,
  `awards` text,
  `mem_id` int(11) NOT NULL,
  PRIMARY KEY (`id`)
) ENGINE=MyISAM AUTO_INCREMENT=157 DEFAULT CHARSET=utf8;
/*!40101 SET character_set_client = @saved_cs_client */;

--
-- Dumping data for table `mem_info`
--

LOCK TABLES `mem_info` WRITE;
/*!40000 ALTER TABLE `mem_info` DISABLE KEYS */;
INSERT INTO `mem_info` VALUES (1,'<p>Dr. Dokholyan began studying physics and mathematics at the Vecua Physics and Mathematics High School in Tbilisi, Georgia (former USSR). He graduated in 1988 with honors and attained specialty computer programming certification. He then entered the Moscow Institute of Physics and Technology, Department of General and Applied Physics. From 1988-1990, he was also affiliated with the Physics Institute of the Academy of Sciences (Moscow), and from 1990-1994, he was affiliated with the Institute for High Energy Physics (Protvino, Russia). He received a B.S. with honors in 1992. His bachelor&#39;s thesis was entitled &#34;Methods of Acceleration of Charged Particles&#34; and was supervised by Dr. Semeon S. Gershtein (IHEP, Russia). Two years later he received a M.S. with honors in 1994. His master&#39;s thesis, entitled &#34;e<sup>+</sup>e<sup>-</sup> Anihilation and tb CKM Matrix Element Measurement&#34;, was supervised by Dr. Semeon S. Gershtein and Dr. George Jikia (IHEP, Russia), and was published in Physics Letters B.</p><p>Dr. Dokholyan joined the Ph.D. program at the Boston University Department of Physics in the fall of 1994. In 1995 he became interested in Biophysics and joined the Center for Polymer Studies of the Boston University Physics Department under the direction of Dr. H. Eugene Stanley. From 1995 he was working with Dr. Sergey V. Buldyrev and Dr. Shlomo Havlin (Bar-Ilan University) on applying statistical mechanics tools to study repetitive patterns of coding and noncoding regions of DNA. In September 1997, Dr. Dokholyan began a joint work with the company British Petroleum (in collaboration with Dr. Peter King) to study transport phenomena in porous media, and, around the same time he started a collaboration with the research group of Dr. Eugene I. Shakhnovich (Harvard University) on the problem of protein design and folding. Dr. Dokholyan received his PhD in 1999. The thesis, entitled &#34;Applications of statistical mechanics to biological macromolecules&#34;, was supervised by Dr. H. Eugene Stanley.</p><p>In 1999, Dr. Dokholyan received National Institutes of Health NRSA award to work for three years as a postdoctoral fellow with Dr. Eugene Shakhnovich. In 2002, he joined Department of Biochemistry and Biophysics in the University of North Carolina at Chapel Hill, School of Medicine as an Assistant Professor. In 2008, Dr. Dokholyan was promoted as Associate Professor. Dr. Dokholyan is currently the Director of the Center for Computational and Systems Biology and the Graduate Director of the Program in Molecular and Cellular Biophysics at UNC.</p>','1999-2002 National Institutes of Health Postdoctoral Fellow, Department of Chemistry and Chemical Biology, Harvard University, USA. Area of research: Biophysics<br>1999 Ph.D., Boston University, USA. Physics<br>1994 M.S., Moscow Institute of Physics and Technology, Russia. Physics<br>1992 B.S., Moscow Institute of Physics and Technology, Russia. Physics','2004-2006 March of Dimes Basil O&#39;Connor Starter Scholar Research Award<br>2004 The University of North Carolina at Chapel Hill IBM Junior Faculty Development Award<br>2003 Recipient of a UNC Research Council Award<br>1999-2002 NIH postdoctoral fellowship<br>1998-1999 NIH Molecular Biophysics Predoctoral Traineeship<br>1995,1998,2001 NSF Young Scientist Travel Award<br>1994 &#34;Red Diploma&#34;<br>1990-1994 Recipient of Honorary Stipend, Moscow Institute of Physics and Technology',1),(2,'<p>Dr. Ding obtained his B.S. from the department of Intensive Instruction at the Nanjing University, China. After obtaining his B.S. he spent two years in the department of Physics at the same university as a graduate student. During this period, he studied photonic band material under the direction of Dr. Naiben Ming and Dr. Xinya Lei.</p>Dr. Ding then joined the Department of Physics at the Boston University to obtain his PhD. He finished his thesis titled &#34;Protein folding and aggregation - Molecular dynamics studies&#34; under the supervision of Dr. H. Eugene Stanley.</p><p>Dr. Ding is currently a research assistant professor in the Department of Biochemistry and Biophysics at the University of North Carolina, Chapel Hill. He joined the Dokholyan group to study the structure and dynamics of biomolecules including proteins, DNA and RNA, by applying principles from Physics (especially statistical physics) to develop tools that can be used to address many problems in biology.</p>',NULL,NULL,2),(3,NULL,NULL,NULL,3),(4,'<p>Dr. Sharma began high school studies at the Delhi Public School, Bhilai, India. He secured merit certification in physics and mathematics and graduated in 2000. He then entered the Indian Institute of Technology, Kanpur for undergraduate education. In 2004, he received a Bachelor of Technology degree in Computer Science & Engineering. His bachelor&#39;s thesis was entitled &#34;A novel approach to structural comparison of proteins&#34; and was supervised by Dr. Somenath Biswas (IIT, Kanpur).</p><p>In Fall, 2004, Dr. Sharma joined the Ph.D. program at the University of North Carolina Department of Biochemistry and Biophysics. In 2004 he became interested in Biophysics and joined the Molecular and Cellular Biophysics Training Program under the guidance of Prof. Nikolay V. Dokholyan. From 2005 he was working with Dr. Feng Ding on applying computational tools to study biomolecular dynamics. Dr. Sharma received his Ph.D. in 2009. His Ph.D. thesis, entitled &#34;Computational modeling and automation techniques to study biomolecular dynamics&#34;, was supervised by Prof. Nikolay V. Dokholyan.</p>','2004-2009 Doctor of Philosophy, Biochemistry & Biophysics, University of North Carolina at Chapel Hill, NC, USA<br>2000-2004 Bachelor of Technology, Computer Science & Engineering, Indian Institute of Technology, Kanpur, India','2007 National Biomedical Computation Resource Summer Fellowship.<br>2007 TeraGrid Resource Award for Multiscale Simulations of Biomolecular Dynamics.<br>2007 Worldwide Software Developers Conference Scholarship 2007 (WWDC07), Apple Inc.<br>2007 American Society for Biochemistry & Molecular Biology Travel Fellowship to present research at the Experimental Biology 2007 and ASBMB Annual Meeting 2007.<br>2007: American Association for the Advancement of Science, Nominated Member, Program for Excellence in Science.<br>2007-2009: National Institutes of Health Integrated Biomedical Research Training Program (IBRTP) Fellowship.<br>2004-2006: National Institutes of Health Molecular & Cellular Biophysics Graduate Research Fellowship.<br>2004 Honorarium equivalent to 1,000 USD by Department of Computer Science & Engineering, Indian Institute of Technology Kanpur for lecturing in IIT Kanpur Algorithms',4),(5,'<p>Dr. Serohijos received his undergraduate degrees in Physics and Computer Engineering from the Ateneo de Manila University in Metro Manila, Philippines. Until high school, he lived and studied in Leyte, an island in central Philippines.</p><p>Dr. Serohijos came to the United States to pursue his graduate studies in the Department of Physics and Astronomy and the Program in Molecular and Cellular Biophysics at the University of North Carolina at Chapel Hill. His research interests inclue computational and theoretical biophysics, structural biology and protein engineering. He obtained his Ph.D. degree under the supervision of Dr. Dokholyan in 2009. He also extensively collaborated with Dr. Timothy C. Elston in the department of Pharmacology at UNC on developing multiscale models of the structure of dynein and with Dr. John R. Riordan on studying the structure and misfolding of the CFTR choride channel.</p>','2004-2009 Ph.D. Physics, University of North Carolina at Chapel Hill<br>2002-2003 B.S. Computer Engineering, Ateneo de Manila University<br>1998-2002 B.S. Physics Cum Laude, Ateneo de Manila University','2009 Travel Award, Biophysics Society Annual Meeting<br>2009 Travel Award, Society for Biological Engineering<br>2008 North America Student Grant, FEBS-ABC2008 Meeting<br>2007-2009 American Heart Association Predoctoral Fellowship<br>2006 Travel Award, Biophysics Society Discussions<br>2005-2006 UNC Biophysics Graduate Trainee Award<br>2004-2005 UNC Board of Governors Fellowship<br>2002 Bank of Philippine Islands Science Award for Outstanding Research<br>2002 Award for Most Outstanding Scholar of Ateneo de Manila<br>2000-2003 Intel Corporation Scholar<br>1998-2002 Department of Science and Technology Scholar, Philippines',5),(6,'Dr. Wilcox began studying biochemistry at the university of Iowa, where he performed undergraduate research in the laboratory of Dr. Lori Wallrath studying heterochromatic gene silencing in Drosophila. He became interested in biophysics through his coursework during his junior and senior years and joined the Molecular and Cellular Biophysics Training Program at UNC Chapel Hill through the Biochemistry and Biophysics Department after graduating with honors. His undergraduate bachelor&#39;s thesis was entitled &#34;Mutational Analysis of Heterochromatin Protein 1.&#34; His interests are broad, but center around the generalizable aspects of protein interactions, folding and misfolding. Dr. Wilcox joined the Dokholyan group and undertook the study of Cu/Zn superoxide dismutase (SOD1) misfolding in familial amyotrophic lateral sclerosis. His work focused on elucidating the mechanism of SOD1 aggregation - particularly early initiating transitions and the effects of post-translational modifications. He obtained his Ph.D. from the Department of Biochemistry and Biophysics in 2009.','2009 Ph.D., University of North Carolina at Chapel Hill. Biochemistry and Biophysics<br>2003 B.S. with Honors, University of Iowa. Biochemistry',NULL,6),(7,'Srinivas did his undergraduate studies at Anna University, Chennai, India in Industrial Biotechnology. During his undergraduate studies, Srinivas did internships  at Hindustan Lever Research Center, Bangalore and National Center for Biological Sciences, Bangalore, India. Srinivas&#39; undergraduate research experience is mainly in raman spectroscopy and quantum mechanical calculations. His current research interests as a graduate student in Dokholyan lab are in understanding ion-channel structure-function relationships (Ryanodine receptors) and the effect of covalent platinum adducts on DNA structure and dynamics.','2006-2011 Ph.D. Biochemistry, University of North Carolina, Chapel Hill<br>2002-2006 Bachelor of Technology, Industrial Biotechnology, Anna University, Chennai, India.','2009-2011 American Heart Association Predoctoral Fellowship<br>2009 Travel Award from Graduate and Professional Student Federation, UNC<br>2008 Graduate Student and Postdoctoral Fellow Mentor Award, UNC<br>2006-2007 Molecular & Cellular Biophysics Graduate Research Fellowship<br>2005 99.37 percentile in Graduate Aptitude Test in Engineering (GATE; GATE score is the basis for selection to post graduate engineering fellowships in India)',9),(8,'Pradeep obtained his Bachelor of Technology (B.Tech) degree, double majoring in Industrial Biotechnology and Information Technology, from Anna University, India. While he was still an undergraduate student, Pradeep developed a keen interest in protein structure-function relationships and worked with Dr. Jayaram at the Indian Institute of Technology, New Delhi, India. In 2006, he worked as a Research Scholar with Dr. Gebhard Schertler at the Structural Studies Division, Medical Research Council Laboratory of Molecular Biology (MRC-LMB) at Cambridge, UK. While at MRC, Pradeep worked on understanding the physical properties and the energetic basis of activation of G-protein coupled receptors. In 2007, he joined Dr. Dokholyan&rsquo;s group and currently works on the structural basis of regulation of epithelial sodium channels (ENaC).','2007-present Ph.D. Biochemistry, University of North Carolina at Chapel Hill<br>2002-2006 Bachelor of Technology, Information Technology, Anna University, Chennai, India<br>2001-2005 Bachelor of Technology, Industrial Biotechnology, Anna University, Chennai, India','2007-2008 Molecular &amp; Cellular Biophysics Graduate Research Fellowship',10),(9,'After graduating from high school in Round Rock, TX, Rachel received a B.S. with honors in biochemistry from Texas A&M University in 2006. She taught chemistry for two years at Hearne High School in Hearne, TX before entering UNC&#39;s Biological and Biomedical Sciences Program and joining the Dokholyan lab. Her current research interests include the relationship between posttranslational modification of superoxide dismutase and cellular redox signaling, and implications for familial amyotrophic lateral sclerosis.','2006 B.S., Texas A&M University, College Station, TX. Biochemistry','2011-2014 Ruth L. Kirschstein National Research Service Award Predoctoral Fellowship, National Institute of Neurological Disorders and Stroke, National Institutes of Health<br>2006 Graduation <i>Summa Cum Laude</i>, Texas A&amp;M University<br>2002-2006 National Merit Scholar Finalist, Texas A&amp;M University<br>2002-2006 Minnie Stevens Piper Foundation Scholarship<br>2002-2006 IBM Thomas J. Watson Memorial Scholarship<br>2002-2006 Terry Foundation Scholarship',19),(10,NULL,'2008 B.S. Honors Physics, Purdue University<br>2008 B.A. Honors Russian Language and Literature, Purdue University','2008-2009 Program for Molecular and Cellular Biophysics Trainee at University of North Carolina - Chapel Hill<br>2008 Director&#39;s Award for top candidates upon acceptance to the Biological and Biomedical Sciences Program at the University of North Carolina in Chapel Hill, spring/summer<br>2008 Chosen as Student Response Speaker for Purdue University Graduation, Section II<br>Dobro Slovo Slavic Language Honors Society member<br>Outstanding Senior Award, Purdue University Department of Foreign Language and Literature (Russian section)<br>2004 NCAA Division I Big Ten Conference Track and Field All-Academic Team<br>2003-2004 Ascarelli Fellowship: research fellowship awarded to outstanding freshmen in the Purdue University Physics Department<br>2003-2007 National Merit Scholar, Purdue University',20),(11,'<p>Douglas Tsao began his research career at East Carolina University under the supervision of Dr. Paul J. Gemperline in 2003.  His primary focus was investigating the rate of reaction between butanol and acetic anhydride under various conditions using fiber-optic Raman spectroscopy.  From 2004 till 2006, Douglas worked with Dr. Alex G. Georgakilas on quantifying levels of DNA damage produced from different sources of radiation.  His honor&#39;s thesis was entitled, &#34;Detection of Oxidative Clustered DNA Lesions and Double Strand Breaks Induced by High LET 56Fe Space Radiation in Human Monocytes.&#34;  Douglas also worked with Dr. Andrew T. Morehead from 2005 to 2006 on synthesizing chiral-2-substituted indanones using rhodium-catalyzed asymmetric hydroacylation chemistry.</p><p>Douglas is currently pursuing his Ph.D. in chemistry at the University of North Carolina at Chapel Hill.  His main projects involve the study of pain perception and are co-advised by Dr. Nikolay Dokholyan and Dr. Luda Diatchenko.  Computational tools are used to make predictions that can then be tested experimentally.  Of specific interest are the molecular mechanisms underlying differences in pain sensitivity among individuals.  Current investigations include studying isoforms of catechol-O-methyltransferase as well as variations in the mRNA transcript structure due to polymorphisms.  Douglas is also involved in a project in collaboration with Dr. Allen P. Minton at NIH studying the effects of macromolecular crowding via discrete molecular dynamics.</p>','2006-Present PhD candidate, University of North Carolina at Chapel Hill, Chemistry<br>2006 B.S.  East Carolina University, Biochemistry and Chemistry','2008 UNC Department of Chemistry Outstanding Teaching Assistant Award<br>2006-2007 University of North Carolina at Chapel Hill Venable Fellowship<br>2006 Graduated with University Honors at East Carolina University<br>2006 East Carolina University Undergraduate Research Stipend<br>2004,2005,2006 Ledonia S. Wright Cultural Center Certificate of High Academic Achievement',23),(12,'<p>Onur obtained his BS and MS degrees in Chemical and Biological Engineering. When he was sophomore, he started to work at Molecular Biology and Biochemistry Laboratory of Dr. Halil Kavakli as an undergraduate researcher. His undergraduate and master research interests include the use of experimental and computational methods to understand the proteins and diseases.</p><p>He is also a member of Klaus Hahn&#39;s Lab, and currently works on the visualization and manipulation of protein activity in living cells.</p>','2010-Present Ph.D. candidate, University of North Carolina at Chapel Hill Biochemistry and Biophysics<br>2008-2010 M.S., Ko&ccedil; University, Istanbul, Turkey. Chemical and Biological Engineering<br>2004-2008 B.S., Ko&ccedil; University, Istanbul, Turkey. Chemical and Biological Engineering','2010 Director&#39;s Award from Biological and Biomedical Sciences Program at the University of North Carolina at Chapel Hill<br>2010 Doctoral Merit Assistantship for study in Biological and Biomedical Sciences, University of North Carolina at Chapel Hill<br>2008-2010 TUBITAK (The Scientific and Technological Research Council of Turkey) National Scholarship for master study<br>2008-2010 Fellowship for Graduate Study, Ko&ccedil; University, Turkey<br>2004-2008 Scholarship for Undergraduate Study, Ko&ccedil; University, Turkey',47),(13,'Elizabeth graduated with honors from Libertyville High School (Libertyville, IL) in 2003, after which she attended Purdue University with a dual major in Physics and Russian Language and Linguistics. At Purdue, Elizabeth competed in the pole vault for the Varsity Track and Field team; was an official tutor for the Departments of Mathematics, Physics, and Foreign Language; and held research positions in the departments of Physics, Chemistry, and Biology. Upon graduating from Purdue in 2008, Elizabeth came to the University of North Carolina at Chapel Hill to study biophysics, where she joined the Dokholyan laboratory. Her current research investigates the stability, dynamics, and aggregation of proteins relevant to human disease.','2008 B.S. Honors Physics, Purdue University<br>2008 B.A. Honors Russian Language and Literature, Purdue University','2012 Ruth L. Kirschstein National Research Service Award Predoctoral Fellowship, National Institute of Health<br>2012 F1000Prime Associate Faculty Member Travel Grant, Faculty of 1000<br>2012 NSF-MCC Travel Award, National Science Foundation and the Materials Computation Center, University of Illinois Urbana-Champaign<br>2011 Second place poster prize, Colorado Protein Stability Conference<br>2009-2010 Predoctoral Training Fellowship, Curriculum in Bioinformatics and Computational Biology, University of North Carolina at Chapel Hill<br>2008-2009 Program for Molecular and Cellular Biophysics Trainee at University of North Carolina - Chapel Hill<br>2008 Director&#39;s Award for top candidates upon acceptance to the Biological and Biomedical Sciences Program at the University of North Carolina in Chapel Hill, spring/summer<br>2008 Chosen as Student Response Speaker for Purdue University Graduation, Section II<br>Dobro Slovo Slavic Language Honors Society member<br>Outstanding Senior Award, Purdue University Department of Foreign Language and Literature (Russian section)<br>2004 NCAA Division I Big Ten Conference Track and Field All-Academic Team<br>2003-2004 Ascarelli Fellowship: research fellowship awarded to outstanding freshmen in the Purdue University Physics Department<br>2003-2007 National Merit Scholar, Purdue University',0),(14,'Elizabeth graduated with honors from Libertyville High School (Libertyville, IL) in 2003, after which she attended Purdue University with a dual major in Physics and Russian Language and Linguistics. At Purdue, Elizabeth competed in the pole vault for the Varsity Track and Field team; was an official tutor for the Departments of Mathematics, Physics, and Foreign Language; and held research positions in the departments of Physics, Chemistry, and Biology. Upon graduating from Purdue in 2008, Elizabeth came to the University of North Carolina at Chapel Hill to study biophysics, where she joined the Dokholyan laboratory. Her current research investigates the stability, dynamics, and aggregation of proteins relevant to human disease.','2008 B.S. Honors Physics, Purdue University<br>2008 B.A. Honors Russian Language and Literature, Purdue University','2012 Ruth L. Kirschstein National Research Service Award Predoctoral Fellowship, National Institute of Health<br>2012 F1000Prime Associate Faculty Member Travel Grant, Faculty of 1000<br>2012 NSF-MCC Travel Award, National Science Foundation and the Materials Computation Center, University of Illinois Urbana-Champaign<br>2011 Second place poster prize, Colorado Protein Stability Conference<br>2009-2010 Predoctoral Training Fellowship, Curriculum in Bioinformatics and Computational Biology, University of North Carolina at Chapel Hill<br>2008-2009 Program for Molecular and Cellular Biophysics Trainee at University of North Carolina - Chapel Hill<br>2008 Director&#39;s Award for top candidates upon acceptance to the Biological and Biomedical Sciences Program at the University of North Carolina in Chapel Hill, spring/summer<br>2008 Chosen as Student Response Speaker for Purdue University Graduation, Section II<br>Dobro Slovo Slavic Language Honors Society member<br>Outstanding Senior Award, Purdue University Department of Foreign Language and Literature (Russian section)<br>2004 NCAA Division I Big Ten Conference Track and Field All-Academic Team<br>2003-2004 Ascarelli Fellowship: research fellowship awarded to outstanding freshmen in the Purdue University Physics Department<br>2003-2007 National Merit Scholar, Purdue University',0),(15,'Elizabeth graduated with honors from Libertyville High School (Libertyville, IL) in 2003, after which she attended Purdue University with a dual major in Physics and Russian Language and Linguistics. At Purdue, Elizabeth competed in the pole vault for the Varsity Track and Field team; was an official tutor for the Departments of Mathematics, Physics, and Foreign Language; and held research positions in the departments of Physics, Chemistry, and Biology. Upon graduating from Purdue in 2008, Elizabeth came to the University of North Carolina at Chapel Hill to study biophysics, where she joined the Dokholyan laboratory. Her current research investigates the stability, dynamics, and aggregation of proteins relevant to human disease.','2008 B.S. Honors Physics, Purdue University<br>2008 B.A. Honors Russian Language and Literature, Purdue University','2012 Ruth L. Kirschstein National Research Service Award Predoctoral Fellowship, National Institute of Health<br>2012 F1000Prime Associate Faculty Member Travel Grant, Faculty of 1000<br>2012 NSF-MCC Travel Award, National Science Foundation and the Materials Computation Center, University of Illinois Urbana-Champaign<br>2011 Second place poster prize, Colorado Protein Stability Conference<br>2009-2010 Predoctoral Training Fellowship, Curriculum in Bioinformatics and Computational Biology, University of North Carolina at Chapel Hill<br>2008-2009 Program for Molecular and Cellular Biophysics Trainee at University of North Carolina - Chapel Hill<br>2008 Director&#39;s Award for top candidates upon acceptance to the Biological and Biomedical Sciences Program at the University of North Carolina in Chapel Hill, spring/summer<br>2008 Chosen as Student Response Speaker for Purdue University Graduation, Section II<br>Dobro Slovo Slavic Language Honors Society member<br>Outstanding Senior Award, Purdue University Department of Foreign Language and Literature (Russian section)<br>2004 NCAA Division I Big Ten Conference Track and Field All-Academic Team<br>2003-2004 Ascarelli Fellowship: research fellowship awarded to outstanding freshmen in the Purdue University Physics Department<br>2003-2007 National Merit Scholar, Purdue University',0),(16,'Elizabeth graduated with honors from Libertyville High School (Libertyville, IL) in 2003, after which she attended Purdue University with a dual major in Physics and Russian Language and Linguistics. At Purdue, Elizabeth competed in the pole vault for the Varsity Track and Field team; was an official tutor for the Departments of Mathematics, Physics, and Foreign Language; and held research positions in the departments of Physics, Chemistry, and Biology. Upon graduating from Purdue in 2008, Elizabeth came to the University of North Carolina at Chapel Hill to study biophysics, where she joined the Dokholyan laboratory. Her current research investigates the stability, dynamics, and aggregation of proteins relevant to human disease.','2008 B.S. Honors Physics, Purdue University<br>2008 B.A. Honors Russian Language and Literature, Purdue University','2012 Ruth L. Kirschstein National Research Service Award Predoctoral Fellowship, National Institute of Health<br>2012 F1000Prime Associate Faculty Member Travel Grant, Faculty of 1000<br>2012 NSF-MCC Travel Award, National Science Foundation and the Materials Computation Center, University of Illinois Urbana-Champaign<br>2011 Second place poster prize, Colorado Protein Stability Conference<br>2009-2010 Predoctoral Training Fellowship, Curriculum in Bioinformatics and Computational Biology, University of North Carolina at Chapel Hill<br>2008-2009 Program for Molecular and Cellular Biophysics Trainee at University of North Carolina - Chapel Hill<br>2008 Director&#39;s Award for top candidates upon acceptance to the Biological and Biomedical Sciences Program at the University of North Carolina in Chapel Hill, spring/summer<br>2008 Chosen as Student Response Speaker for Purdue University Graduation, Section II<br>Dobro Slovo Slavic Language Honors Society member<br>Outstanding Senior Award, Purdue University Department of Foreign Language and Literature (Russian section)<br>2004 NCAA Division I Big Ten Conference Track and Field All-Academic Team<br>2003-2004 Ascarelli Fellowship: research fellowship awarded to outstanding freshmen in the Purdue University Physics Department<br>2003-2007 National Merit Scholar, Purdue University',0),(17,'Elizabeth graduated with honors from Libertyville High School (Libertyville, IL) in 2003, after which she attended Purdue University with a dual major in Physics and Russian Language and Linguistics. At Purdue, Elizabeth competed in the pole vault for the Varsity Track and Field team; was an official tutor for the Departments of Mathematics, Physics, and Foreign Language; and held research positions in the departments of Physics, Chemistry, and Biology. Upon graduating from Purdue in 2008, Elizabeth came to the University of North Carolina at Chapel Hill to study biophysics, where she joined the Dokholyan laboratory. Her current research investigates the stability, dynamics, and aggregation of proteins relevant to human disease.','2008 B.S. Honors Physics, Purdue University<br>2008 B.A. Honors Russian Language and Literature, Purdue University','2012 Ruth L. Kirschstein National Research Service Award Predoctoral Fellowship, National Institute of Health<br>2012 F1000Prime Associate Faculty Member Travel Grant, Faculty of 1000<br>2012 NSF-MCC Travel Award, National Science Foundation and the Materials Computation Center, University of Illinois Urbana-Champaign<br>2011 Second place poster prize, Colorado Protein Stability Conference<br>2009-2010 Predoctoral Training Fellowship, Curriculum in Bioinformatics and Computational Biology, University of North Carolina at Chapel Hill<br>2008-2009 Program for Molecular and Cellular Biophysics Trainee at University of North Carolina - Chapel Hill<br>2008 Director&#39;s Award for top candidates upon acceptance to the Biological and Biomedical Sciences Program at the University of North Carolina in Chapel Hill, spring/summer<br>2008 Chosen as Student Response Speaker for Purdue University Graduation, Section II<br>Dobro Slovo Slavic Language Honors Society member<br>Outstanding Senior Award, Purdue University Department of Foreign Language and Literature (Russian section)<br>2004 NCAA Division I Big Ten Conference Track and Field All-Academic Team<br>2003-2004 Ascarelli Fellowship: research fellowship awarded to outstanding freshmen in the Purdue University Physics Department<br>2003-2007 National Merit Scholar, Purdue University',20),(18,'Elizabeth graduated with honors from Libertyville High School (Libertyville, IL) in 2003, after which she attended Purdue University with a dual major in Physics and Russian Language and Linguistics. At Purdue, Elizabeth competed in the pole vault for the Varsity Track and Field team; was an official tutor for the Departments of Mathematics, Physics, and Foreign Language; and held research positions in the departments of Physics, Chemistry, and Biology. Upon graduating from Purdue in 2008, Elizabeth came to the University of North Carolina at Chapel Hill to study biophysics, where she joined the Dokholyan laboratory. Her current research investigates the stability, dynamics, and aggregation of proteins relevant to human disease.','2008 B.S. Honors Physics, Purdue University<br>2008 B.A. Honors Russian Language and Literature, Purdue University','2010-2013  Ruth L. Kirschstein National Research Service Award Predoctoral Fellowship, National Institute of Health<br>2012 F1000Prime Associate Faculty Member Travel Grant, Faculty of 1000<br>2012 NSF-MCC Travel Award, National Science Foundation and the Materials Computation Center, University of Illinois Urbana-Champaign<br>2011 Second place poster prize, Colorado Protein Stability Conference<br>2009-2010 Predoctoral Training Fellowship, Curriculum in Bioinformatics and Computational Biology, University of North Carolina at Chapel Hill<br>2008-2009 Program for Molecular and Cellular Biophysics Trainee at University of North Carolina - Chapel Hill<br>2008 Director&#39;s Award for top candidates upon acceptance to the Biological and Biomedical Sciences Program at the University of North Carolina in Chapel Hill, spring/summer<br>2008 Chosen as Student Response Speaker for Purdue University Graduation, Section II<br>2008 Dobro Slovo Slavic Language Honors Society member<br>2008 Outstanding Senior Award, Purdue University Department of Foreign Language and Literature (Russian section)<br>2004 NCAA Division I Big Ten Conference Track and Field All-Academic Team<br>2003-2004 Ascarelli Fellowship: research fellowship awarded to outstanding freshmen in the Purdue University Physics Department<br>2003-2007 National Merit Scholar, Purdue University',20),(19,'Marino received his MSc summa cum laude in Pharmacy from the University of Bari (IT) in 2005. For his doctoral studies, he moved to the University of Zurich (CH), where he worked on the identification of small molecules with a therapeutic potential in the treatment of Alzheimer&#39;s disease. In 2011, he received a double PhD in Biochemistry and Medicinal Chemistry, under the supervision of Prof. A. Caflisch and Prof. A. Carotti. After one year spent in the Drug Discovery and Development Unit of the Italian Institute of Technology in Genoa (IT), working on the development of new anti-pain drugs, he joined the Dokholyan lab, in January 2013. His current research is focusing on the identification of a new class of compounds able to target different isoforms of the &#956;-opioid receptor with the specific aim to develop novel analgesic drugs.','2011 PhD, University of Zurich, CH. Biochemistry <br>2011 PhD, University of Bari, IT. Medicinal Chemistry<br>2005 MSc, University of Bari, IT. Pharmacy','2005 Seal of University of Bari as best graduated student of the academic year 2005',105),(20,'Jhuma received her Masters in Physics from University of Pune in 2003. Then, she worked with Prof. Bhattacharyaa as a research assistant at Saha Institute of Nuclear Physics until 2005. For her doctoral studies, she moved to University of Missouri (MU). Her thesis was based on computational and theoretical study of water and lipid dynamics in biomolecular systems. She earned her PhD in Physics, under the supervision of Prof. Ioan Kosztin in 2010. During her Ph.D., she also worked as a graduate teaching assistant of Classical Mechanics and Electromagnetics. Then, she joined Prof. Berkowitzs lab at the department of Chemistry, University of North Carolina at Chapel Hill as a postdoctoral fellow and studied the origin of long-range hydrophobic interaction between two inhomogeneously charged surfactant coated model surfaces. She joined the Dokholyan lab in August 2012. Her current research is focused on the computational modeling of the cystic fibrosis transmembrane conductance regulator channel to understand the structure-function relationship of this channel and develop effective therapeutics to treat cystic fibrosis.','2010 Ph.D. Physics University of Missouri <br> 2003 Physics, University of Pune, Maharashtra, India <br>2001 B.Sc., Physics, Lady Brabourne College, Calcutta University, India','2009 Honorable mention, 25th Annual Missouri Life Sciences Week University of Missouri<br>2008 Physics Leader&#39;ss Meeting Talks 2nd Prize, Department of Physics and Astronomy University of Missouri <br>2008 Donald K. Anderson Teaching Award Nomination, University of Missouri<br>2008 Ernest W. Landen Fellowship, Department of Physics and Astronomy University of Missouri<br> 2007 O. M. Stewart Scholarship, Department of Physics and Astronomy University of Missouri<br> 2005 Joint CSIR-UGC Junior Research Fellowship (JRF) and Eligibility for lectureship in Physical Sciences, National Eligibility Test, India<br> 2002 Book-grant, Department of Physics University of Pune',89),(21,'Jhuma received her Masters in Physics from University of Pune in 2003. Then, she worked with Prof. Bhattacharyaa as a research assistant at Saha Institute of Nuclear Physics until 2005. For her doctoral studies, she moved to University of Missouri (MU). Her thesis was based on computational and theoretical study of water and lipid dynamics in biomolecular systems. She earned her PhD in Physics, under the supervision of Prof. Ioan Kosztin in 2010. During her Ph.D., she also worked as a graduate teaching assistant of Classical Mechanics and Electromagnetics. Then, she joined Prof. Berkowitzs lab at the department of Chemistry, University of North Carolina at Chapel Hill as a postdoctoral fellow and studied the origin of long-range hydrophobic interaction between two inhomogeneously charged surfactant coated model surfaces. She joined the Dokholyan lab in August 2012. Her current research is focused on the computational modeling of the cystic fibrosis transmembrane conductance regulator channel to understand the structure-function relationship of this channel and to develop effective therapeutics to treat cystic fibrosis .','2010 Ph.D. Physics University of Missouri<br>2003 Physics, University of Pune, Maharashtra, India<br>2001 B.Sc., Physics, Lady Brabourne College, Calcutta University, India','2009 Honorable mention, 25th Annual Missouri Life Sciences Week University of Missouri<br>2008 Physics Leader&#39;ss Meeting Talks 2nd Prize, Department of Physics and Astronomy University of Missouri <br>2008 Donald K. Anderson Teaching Award Nomination, University of Missouri<br>2008 Ernest W. Landen Fellowship, Department of Physics and Astronomy University of Missouri<br>2007 O. M. Stewart Scholarship, Department of Physics and Astronomy University of Missouri<br>2005 Joint CSIR-UGC Junior Research Fellowship (JRF) and Eligibility for lectureship in Physical Sciences, National Eligibility Test, India<br>2002 Book-grant, Department of Physics University of Pune',89),(22,'Jhuma received her Masters in Physics from University of Pune in 2003. Then, she worked with Prof. Bhattacharyaa as a research assistant at Saha Institute of Nuclear Physics until 2005. For her doctoral studies, she moved to University of Missouri (MU). Her thesis was based on computational and theoretical study of water and lipid dynamics in biomolecular systems. She earned her PhD in Physics, under the supervision of Prof. Ioan Kosztin in 2010. During her Ph.D., she also worked as a graduate teaching assistant of Classical Mechanics and Electromagnetics. Then, she joined Prof. Berkowitzs lab at the department of Chemistry, University of North Carolina at Chapel Hill as a postdoctoral fellow and studied the origin of long-range hydrophobic interaction between two inhomogeneously charged surfactant coated model surfaces. She joined the Dokholyan lab in August 2012. Her current research is focused on the computational modeling of the cystic fibrosis transmembrane conductance regulator channel to understand the structure-function relationship of this channel and to develop effective therapeutics to treat cystic fibrosis.','2010 Ph.D. Physics University of Missouri<br>2003 Physics, University of Pune, Maharashtra, India<br>2001 B.Sc., Physics, Lady Brabourne College, Calcutta University, India','2009 Honorable mention, 25th Annual Missouri Life Sciences Week University of Missouri<br>2008 Physics Leader&#39;ss Meeting Talks 2nd Prize, Department of Physics and Astronomy University of Missouri <br>2008 Donald K. Anderson Teaching Award Nomination, University of Missouri<br>2008 Ernest W. Landen Fellowship, Department of Physics and Astronomy University of Missouri<br>2007 O. M. Stewart Scholarship, Department of Physics and Astronomy University of Missouri<br>2005 Joint CSIR-UGC Junior Research Fellowship (JRF) and Eligibility for lectureship in Physical Sciences, National Eligibility Test, India<br>2002 Book-grant, Department of Physics University of Pune',89),(23,'Onur obtained his BS and MS degrees in Chemical and Biological Engineering. His undergraduate and master research interests include machine learning and its application to biological problems; and investigation of structure-dynamics-function relationship of several proteins. He is also a member of Klaus Hahn&#39;ss lab at Pharmacology Department, and currently works on the visualization and manipulation of protein activity in living cells.','2010-Present Ph.D. candidate, University of North Carolina at Chapel Hill Biochemistry and Biophysics<br>2008-2010 M.S., Ko&ccedil; University, Istanbul, Turkey. Chemical and Biological Engineering<br>2004-2008 B.S., Ko&ccedil; University, Istanbul, Turkey. Chemical and Biological Engineering','2012-2015 Howard Hughes Medical Institute (HHMI) International Student Research Fellowship<br>2012 NSF-MCC Travel Award, National Science Foundation and the Materials Computation Center, University of Illinois Urbana-Champaign<br>2010 Director&#39;s Award from Biological and Biomedical Sciences Program at the University of North Carolina at Chapel Hill<br>2010 Doctoral Merit Assistantship for study in Biological and Biomedical Sciences, University of North Carolina at Chapel Hill<br>2008-2010 TUBITAK (The Scientific and Technological Research Council of Turkey) National Scholarship for master study<br>2008-2010 Fellowship for Graduate Study, Ko&ccedil; University, Turkey<br>2004-2008 Scholarship for Undergraduate Study, Ko&ccedil; University, Turkey',47),(24,'Arpit did his undergraduate studies at Amity University, New Delhi, India with a major in Industrial Biotechnology, and minors in pharmaceutical chemistry and computational biology. During his undergraduate studies, he did internships in Sanofi Aventis development center, Goa, India and Biotech Park, Lucknow, India. His undergraduate research interests include scaling molecular dynamics simulations to study pharmaceutically relevant large protein complexes. His research interests as a graduate student in Dokholyan lab involve developing computational tools to study RNA structure and dynamics.','2005-2009 Bachelors of Technology, Industrial Biotechnology, Amity University<br>2011-present Doctor of Philosophy, Biochemistry & Biophysics, University of North Carolina at Chapel Hill, NC, USA','2008 Department of Biotechnology, Government of India Travel Grant, to present work at Miptec 2008, drug discovery conference at University of Basel, Switzerland',80),(25,'Raul received his B. Sc. in Chemistry from the University of Barcelona (1997), Raul oriented his career towards Biology and obtained a Masters in Biotechnology (1999). Initially he spent two years in different Molecular Biology labs, including a Esteve Laboratories, a pharmaceutical company and the Genetics Lab at the Faculty of Veterinary, Universitat Autonoma de Barcelona. In 2002 he joined the lab of Prof. Shoshana Wodak in Brussels, Belgium for a Marie Curie 1-year fellowship, where he was trained on bioinformatics. Lately Raul got his Ph. D. in Professor Wodak Lab in 2007. His thesis consisted in the automatic evaluation of the protein - protein docking models within the international experiment called CAPRI. Since then he has always been interested in the protein conformational change problem. In 2008 he joined Dr. Ugo Bastollai&#39;s bioinformatics lab, at the Centre for Molecular Biology in Madrid, Spain, where he has been developing methods to study protein conformational changes using normal modes, studying disorder in and evolution of proteins and analyzing next generation sequencing data on DNA replication in A. Thaliana. During this period he has taught bioinformatics within the Master in Biophysics program, at the Universidad Autonoma de Madrid. He has now joined the Dokholyan lab to computationally design proteins with antigenic activity mimicking HIV gp41-gp120 proteins.','2007 Ph.D. in Bioinformatics, Universite Libre de Bruxelles, Belgium<br>1999 Master Biotechnology, Dept. Genetics. University of Barcelona, Spain<br>1997 B. Sc. in Chemistry. University of Barcelona, Spain','2001 Marie Curie Fellowship',109),(26,'Raul received his B. Sc. in Chemistry from the University of Barcelona (1997), Raul oriented his career towards Biology and obtained a Masters in Biotechnology (1999). Initially he spent two years in different Molecular Biology labs, including a Esteve Laboratories, a pharmaceutical company and the Genetics Lab at the Faculty of Veterinary, Universitat Autonoma de Barcelona. In 2002 he joined the lab of Prof. Shoshana Wodak in Brussels, Belgium for a Marie Curie 1-year fellowship, where he was trained on bioinformatics. Lately Raul got his Ph. D. in Professor Wodak Lab in 2007. His thesis consisted in the automatic evaluation of the protein - protein docking models within the international experiment called CAPRI. Since then he has always been interested in the protein conformational change problem. In 2008 he joined Dr. Ugo Bastolla&#39;s bioinformatics lab, at the Centre for Molecular Biology in Madrid, Spain, where he has been developing methods to study protein conformational changes using normal modes, studying disorder in and evolution of proteins and analyzing next generation sequencing data on DNA replication in A. Thaliana. During this period he has taught bioinformatics within the Master in Biophysics program, at the Universidad Autonoma de Madrid. He has now joined the Dokholyan lab to computationally design proteins with antigenic activity mimicking HIV gp41-gp120 proteins.','2007 Ph.D. in Bioinformatics, Universite Libre de Bruxelles, Belgium<br>1999 Master Biotechnology, Dept. Genetics. University of Barcelona, Spain<br>1997 B. Sc. in Chemistry. University of Barcelona, Spain','2001 Marie Curie Fellowship',109),(27,'Jimmy graduated from Servite High School in Anaheim California before receiving a BS in Biology and a BA in Chemistry from The University of North Carolina at Chapel Hill. His current research interests include developing immunogens to stimulate an antibody response against the HIV virus, elucidating the structure of certain superoxide dismutase mutants through crystallography, and the biochemical characterization of designed proteins.','2012  BS Biology, BA Chemistry, University of North Carolina at Chapel Hill Chapel Hill, NC','',69),(28,'Lanette was born and raised in Reno, Nevada. She earned her bachelors degree in Biochemistry from the University of Nevada, Reno. She earned her Ph.D. with Nick Pace and Texas A&M University working on the contribution that aromatic amino acids make to the stability and spectral properties of proteins. In her post-doctoral position with Gerald Carlson at University of Tennessee in Memphis, she worked on the structure of phosphorylase kinase. After taking some time off with her young children, she returned to science as a research technician. She has worked in various areas of biochemistry and cell biology including studies of tubulin in yeast, the regulation of RNAs in developing Drosophila, and making transgenic Xenopus. Her current work in Dr. Dokholyans lab focuses on the formation of small soluble oligomers of superoxide dismutase and the role they play in ALS.','','',40),(29,'Raul received his B. Sc. in Chemistry from the University of Barcelona (1997). Later he oriented his career towards Biology and obtained a Masters in Biotechnology (1999). Initially he spent two years in different Molecular Biology labs, including Dr. Esteve Laboratories (a pharmaceutical company) and the Genetics Lab at the Faculty of Veterinary, Universitat Autonoma de Barcelona. In 2002 he joined the lab of Prof. Shoshana Wodak in Brussels, Belgium for a Marie Curie 1-year fellowship, where he was trained on bioinformatics. Raul got his Ph. D. in 2007 at the same lab under the supervision of Professor Wodak in 2007. His thesis consisted in the automatic evaluation of the protein - protein docking models within the international experiment called CAPRI. Since then he has always been interested in the protein conformational change problem. In 2008 he joined Dr. Ugo Bastolla&#39;s bioinformatics lab, at the Centre for Molecular Biology in Madrid, Spain, where he has been developing methods to study protein conformational changes using normal modes, study disorder and evolution of proteins, and analyzing next generation sequencing data on DNA replication in A. Thaliana. During this period he has also taught bioinformatics within the Master in Biophysics program, at the Universidad Autonoma de Madrid. He has now joined the Dokholyan lab to computationally design proteins with antigenic activity mimicking HIV gp41-gp120 proteins.','2007 Ph.D. in Bioinformatics, Universite Libre de Bruxelles, Belgium<br>1999 Master Biotechnology, Dept. Genetics. University of Barcelona, Spain<br>1997 B. Sc. in Chemistry. University of Barcelona, Spain','2001 Marie Curie Fellowship',109),(30,'Onur obtained his BS and MS degrees in Chemical and Biological Engineering. His undergraduate and master research interests include machine learning and its application to biological problems; and investigation of structure-dynamics-function relationship of several proteins. He is also a member of Klaus Hahn&#39;s lab at Pharmacology Department, and currently works on the visualization and manipulation of protein activity in living cells.','2010-Present Ph.D. candidate, University of North Carolina at Chapel Hill Biochemistry and Biophysics<br>2008-2010 M.S., Ko&ccedil; University, Istanbul, Turkey. Chemical and Biological Engineering<br>2004-2008 B.S., Ko&ccedil; University, Istanbul, Turkey. Chemical and Biological Engineering','2012-2015 Howard Hughes Medical Institute (HHMI) International Student Research Fellowship<br>2012 NSF-MCC Travel Award, National Science Foundation and the Materials Computation Center, University of Illinois Urbana-Champaign<br>2010 Director&#39;s Award from Biological and Biomedical Sciences Program at the University of North Carolina at Chapel Hill<br>2010 Doctoral Merit Assistantship for study in Biological and Biomedical Sciences, University of North Carolina at Chapel Hill<br>2008-2010 TUBITAK (The Scientific and Technological Research Council of Turkey) National Scholarship for master study<br>2008-2010 Fellowship for Graduate Study, Ko&ccedil; University, Turkey<br>2004-2008 Scholarship for Undergraduate Study, Ko&ccedil; University, Turkey',47),(31,'Gurpreet obtained her B.Sc. in Biotechnology (2007) from Khalsa College, Amritsar, India and Masters in Bioinformatics (2009) from Department of Biotechnology, Guru Nanak Dev University (GNDU), Amritsar. During her Masters, she did 6 months dissertation on Population Modeling at Laboratory for Conservation of Endangered Species (LaCONES), Centre for Cellular and Molecular Biology (CCMB), Hyderabad, India. After completing Masters, she joined as a lecturer in Department of Biotechnology, GNDU and taught M.Sc. Bioinformatics students. Then, she worked as Project Trainee (Bioinformatics) in Distributed Informatics Sub-Centre (DISC), Department of Biotechnology, GNDU. She received INSPIRE (Innovation in Science Pursuit for Inspired Research) Fellowship (2010-2015) by Department of Science and Technology, Govt. of India to pursue her Ph.D. In 2012, she was awarded Fulbright-Nehru Doctoral and Professional Research Fellowship by United States-India Educational Foundation (USIEF), New Delhi, India to work as Visiting Scholar in Dokholyan laboratory. Her current research involves structure-function analysis of plant NADPH oxidase.','2010-Present Ph.D. Scholar, Plant Biotechnology Wing, Department of Biotechnology, Guru Nanak Dev University, Amritsar.<br>2007-2009 M.Sc. Bioinformatics, Department of Biotechnology, Guru Nanak Dev University, Amritsar.<br>2004-2007 B.Sc. Biotechnology, Khalsa College, Amritsar.','2012-Present  Fulbright-Nehru Doctoral and Professional Research Fellowship.<br>2012-Present INSPIRE Senior Research Fellowship, Department of Science and Technology, Govt. of India.<br>2012 Fulbright Summer program, Monterey Institute of International Studies, Monterey, California.<br>2012 Qualified BioInformatics National Certification (BINC) Examination at all India level conducted by Department of Biotechnology, Govt. of India.<br>2010-2012 INSPIRE Junior Research Fellowship, Department of Science and Technology, Govt. of India.<br>2009  University Gold Medalist in M.Sc. Bioinformatics.<br>2007 Gold Medalist in B.Sc. Biotechnology.<br>2006 Certificate of Merit and scholarship for First Rank Holder in B.Sc. Biotechnology 2nd year.<br>2005 Certificate of Merit and scholarship for First Rank Holder in B.Sc. Biotechnology 1st year.<br>2002 Merit Holder in Matriculation (10th grade).',92),(32,'Dr. Shirvanyants has graduated from the department of Chemistry at Tver State University in Russia. After graduation he spent 2 years working at the department of Organic Chemistry at Univerisity of Ulm. During this period he studied branched polymers on solid surfaces using methods of polymer theory and simulations. Dr. Shirvanynts has joined the Department of Chemistry at the University of North Carolina at Chapel Hill to obtain PhD degree. He finished his thesis titled &#34Conformations and dynamics of macromolecules in solution and on the surface&#34 under the supervision of Prof. Michael Rubinsteinand Prof. Sergey Sheyko. Dr. Shirvanyants is currently a research assistant in the Department of Biochemistry and Biophysics at the University of North Carolina at Chapel Hill. He joined the Dokholyan group to study the structure and dynamics of biomolecules, by applying principles from Physics (especially statistical physics) to develop tools that can be used toaddress many problems in biology.','','',43),(33,'After finishing BA and MA in chemistry, he embarked on doing a Ph. D. in theoretical physical chemistry. During his Ph.D., he studied reaction rate theory. After his postdoctoral research associateship from University of Utah on high energy density matter simulation, he moved to Seoul National University (Korea) as a researcher. There he continued  working on biomolecular simulations. In 2001, He was also a visiting scholar at IAMS, Taiwan. Currently, he is an Associate Professor at the Department of Chemistry, Sejong University, Seoul , Korea.','1983-1987 BA, Yonsei Univiserity, Seoul, Korea<br>1987-1989 MA, Yonsei University, Seoul, Korea<br>1990-1996 Ph.D., University of Chicago','',0),(34,'After finishing BA and MA in chemistry, he embarked on doing a Ph. D. in theoretical physical chemistry. During his Ph.D., he studied reaction rate theory. After his postdoctoral research associateship from University of Utah on high energy density matter simulation, he moved to Seoul National University (Korea) as a researcher. There he continued  working on biomolecular simulations. In 2001, He was also a visiting scholar at IAMS, Taiwan. Currently, he is an Associate Professor at the Department of Chemistry, Sejong University, Seoul , Korea.','1983-1987 BA, Yonsei Univiserity, Seoul, Korea<br>1987-1989 MA, Yonsei University, Seoul, Korea<br>1990-1996 Ph.D., University of Chicago','',111),(35,'Andrey graduated from Moscow Institute of Physics and Technology (Moscow, Russia). After graduation, he started research in the field of high energy physics in the Institute for Theoretical and Experimental Physics (Moscow, Russia). He was a member of CMS experimental collaboration at the Large Hadron Collider (LHC) in CERN (Geneva, Switzerland). There he was engaged in preparation of the experiment and in the analysis of collected experimental data after start of the LHC operation. In 2010 he drastically changed the area of his scientific interests to the field of biological physics. I worked in the group of Prof. Antti Niemi (Uppsala University) in the field of protein folding. They developed a novel theoretical approach to tackle the problem of protein folding. The originality of the our approach has attracted attention and was described in several scientific blogs. Some notable examples are \'http://www.technologyreview.com\' (MIT) and \'http://waterinbiology.blogspot.com\'. His current research project is concentrated around modeling of RNA and protein design.','1999 Master of Science from Moscow Institute for Physics and Technology (MIPT), Moscow, Russia<br>2008 Ph.D. from Institute for Theoretical and Experimental Physics (ITEP), Moscow, Russia','',110),(36,'Reed Jacob began his university education in 1999 and graduated with a BA from Boise State University in Music Composition. He then spent a few years exploring the continental United States. When his father, Steve A. Jacob, became ill with the diagnosis of stage IV pancreatic cancer, Reed returned home. He brought with him his lovely wife Brooke Jacob, and with her support returned to school at BSU. His journey into the academic world was marked by a shift from music to science. Realizing a tremendous aptitude for scientific discovery and computer-aided molecular docking, Reed entered the IDS program. Within three years, Reed has presented at a number of prestigious scientific conferences, contributed to major software development projects and contributed to the publication of five manuscripts to date. He is currently working toward a PhD in Bioinformatics and Computational Biology in Nikolay Dokholyan\'s lab at University of North Carolina Chapel Hill','','',96),(37,'Raul received his B. Sc. in Chemistry from the University of Barcelona (1997). Later he oriented his career towards Biology and obtained a Masters in Biotechnology (1999). Initially he spent two years in different Molecular Biology labs, including Dr. Esteve Laboratories (a pharmaceutical company) and the Genetics Lab at the Faculty of Veterinary, Universitat Autonoma de Barcelona. In 2002 he joined the lab of Prof. Shoshana Wodak in Brussels, Belgium for a Marie Curie 1-year fellowship, where he was trained on bioinformatics. Raul got his Ph. D. in 2007 at the same lab under the supervision of Professor Wodak in 2007. His thesis consisted in the automatic evaluation of the protein - protein docking models within the international experiment called CAPRI. Since then he has always been interested in the protein conformational change problem. In 2008 he joined Dr. Ugo Bastolla&#39;s bioinformatics lab, at the Centre for Molecular Biology in Madrid, Spain, where he has been developing methods to study protein conformational changes using normal modes, study disorder and evolution of proteins, and analyzing next generation sequencing data on DNA replication in A. Thaliana. During this period he has also taught bioinformatics within the Master in Biophysics program, at the Universidad Autonoma de Madrid. He has now joined the Dokholyan lab to computationally design proteins with antigenic activity mimicking HIV gp41-gp120 proteins.','2007 Ph.D. in Bioinformatics, Universite Libre de Bruxelles, Belgium<br>1999 Master Biotechnology, Dept. Genetics. University of Barcelona, Spain<br>1997 B. Sc. in Chemistry. University of Barcelona, Spain','2008-2009  ',114),(38,'After graduating from high school in Alexandria, Egypt, Mahmoud  studied Electronics Engineering at The American University in Cairo.  He continued his undergraduate education at the University of Florida, studying physics, with an emphasis in biophysics. He graduated with a B.S. in Physics in 2012. During his studies in the department of Physics, his emphasis was on optogenetics research and its applications in neurodynamics modeling. His current research includes RNA-modeling and the application of protein-engineering in cell-signaling.','2012 B.S. Physics, University of Florida','2012-2013 Program for Molecular and Cellular Biophysics Trainee at University of North Carolina - Chapel Hill<br>2011 Scholarship to attend Biophysical Society summer course in Biophysics held at UNC-Chapel Hill<br>2009-2010 Math Open the Doors Scholarship (MODS)<br>Merit academic award for top applicants to The American  University in Cairo',114),(39,'After graduating from high school in Alexandria, Egypt, Mahmoud  studied Electronics Engineering at The American University in Cairo.  He continued his undergraduate education at the University of Florida, studying physics, with an emphasis in biophysics. He graduated with a B.S. in Physics in 2012. During his studies in the department of Physics, his emphasis was on optogenetics research and its applications in neurodynamics modeling. His current research includes RNA-modeling and the application of protein-engineering in cell-signaling.','2012 B.S. Physics, University of Florida','2012-2013 Program for Molecular and Cellular Biophysics Trainee at University of North Carolina - Chapel Hill<br>2011 Scholarship to attend Biophysical Society summer course in Biophysics held at UNC-Chapel Hill<br>2009-2010 Math Open the Doors Scholarship (MODS)<br>2008-2009 Merit academic award for top applicants to The American  University in Cairo',114),(40,'Ardeshir finished his doctorate degree in pharmacy (Pharm.D.) in 2006 from Tehran University of Medical Sciences. His thesis was focused on molecular understanding of cancer cells and their markers. After graduation he started to practice in both retail and hospital pharmacies. He also worked as consultant pharmacist in Iran\'s drug and poison information center. Since his interests were in drug and protein/peptide design he decided to continue his education by pursuing a master\'s degree in structural biochemistry and bioinformatics. He graduated in 2012 from The University of Western Ontario, Canada. This experience introduced him to the field of computer aided structure prediction and design. He is now in the Biochemistry and Biophysics PhD program with focus on computer aided drug and protein design.','2012 M.Sc., The University of Western Ontario, London, Ontario, Canada<br>2006 Pharm.D., Tehran University of Medical Sciences, Tehran, Iran','2010-2012 Western Graduate Research Scholarships (WGRS) awarded by the University of Western Ontario<br>2000-2006 Full competitive scholarship for six years, awarded by Iran\'s Ministry of Health',112),(41,'After graduating from high school in Alexandria, Egypt, Mahmoud  studied Electronics Engineering at The American University in Cairo.  He continued his undergraduate education at the University of Florida, studying physics, with an emphasis in biophysics. He graduated with a B.S. in Physics in 2012. During his studies in the department of Physics, his emphasis was on optogenetics research and its applications in neurodynamics modeling. His current research includes RNA-modeling and the application of protein-engineering in cell-signaling.','2012 B.S. Physics, University of Florida','2012-2013 Program for Molecular and Cellular Biophysics Trainee at University of North Carolina - Chapel Hill<br>2011 Scholarship to attend Biophysical Society summer course in Biophysics held at UNC-Chapel Hill<br>2009-2010 Math Opens the Doors Scholarship (MODS)<br>2008-2009 Merit academic award for top applicants to The American  University in Cairo',114),(42,'Cheng attended Nanjing University (Nanjing, China) with a major in chemistry. During his undergraduate studies, Cheng joined a research group on membrane chemistry and gained experiences on organic synthesis of amphiphilic molecular and IR reflection absorption spectroscopy of monolayer membranes. Then he moved to Peking University (Beijing, China) as a Ph. D. student and studied protein design and protein folding in Dr. Luhua Lai`s lab. During 2007-2013, cheng had experimentally studied protein folding mechanisms and designed metal binding proteins and protein-protein interactions. In 2010, he joined University of Pennsylvania as a visiting student and worked on protein folding problems. After graduation in 2013, cheng joined the Dokholyan lab. His current research is focused on computational protein design to obtain efficient vaccine for HIV.','2013 Ph.D.in Physical Chemistry and Biophysics, Peking University, Beijing, China<br>2007 B.S. in Chemistry, Nanjing University, Nanjing, China','',119),(122,'Dominique received a Bachelor of Science in Biochemistry with a minor in Biology from the University of North Carolina at Chapel Hill. Since then she has worked in several labs, including LabCorp, the Center for Aids Research at Duke University Medical Center, and the Center for Excellence in Post-Harvest Technologies for North Carolina Agricultural and Technical State University.  Her technical background is a diverse mix of biological, organic, and analytical chemistry, virology, cell biology, having particular emphasis on molecular biology. She has just joined the Dokhoylan lab and is honored to be working with Rachel Redler on an exciting project involving amyotrophic lateral sclerosis.','2005 BS University of North Carolina at Chapel Hill, Biochemistry with a minor in Biology','',122),(124,'Dominique received a Bachelor of Science in Biochemistry with a minor in Biology from the University of North Carolina at Chapel Hill. Since then she has worked in several labs, including LabCorp, the Center for Aids Research at Duke University Medical Center, and the Center for Excellence in Post-Harvest Technologies for North Carolina Agricultural and Technical State University.  Her technical background is a diverse mix of biological, organic, and analytical chemistry, virology, cell biology, having particular emphasis on molecular biology. She has just joined the Dokhoylan lab and is honored to be working with Rachel Redler on an exciting project involving amyotrophic lateral sclerosis.','2005 BS University of North Carolina at Chapel Hill, Biochemistry with a minor in Biology','',122),(125,'Dominique received a Bachelor of Science in Biochemistry with a minor in Biology from the University of North Carolina at Chapel Hill. Since then she has worked in several labs, including LabCorp, the Center for Aids Research at Duke University Medical Center, and the Center for Excellence in Post-Harvest Technologies for North Carolina Agricultural and Technical State University.  Her technical background is a diverse mix of biological, organic, and analytical chemistry, virology, cell biology, having particular emphasis on molecular biology. She has just joined the Dokhoylan lab and is honored to be working with Rachel Redler on an exciting project involving amyotrophic lateral sclerosis.','2005 BS University of North Carolina at Chapel Hill, Biochemistry with a minor in Biology','',124),(126,'Dominique received a Bachelor of Science in Biochemistry with a minor in Biology from the University of North Carolina at Chapel Hill. Since then she has worked in several labs, including LabCorp, the Center for Aids Research at Duke University Medical Center, and the Center for Excellence in Post-Harvest Technologies for North Carolina Agricultural and Technical State University.  Her technical background is a diverse mix of biological, organic, and analytical chemistry, virology, cell biology, having particular emphasis on molecular biology. She has just joined the Dokhoylan lab and is honored to be working with Rachel Redler on an exciting project involving amyotrophic lateral sclerosis.','2005 BS University of North Carolina at Chapel Hill, Biochemistry with a minor in Biology','',126),(127,'Marino received his MSc summa cum laude in Pharmacy from the University of Bari (IT) in 2005. For his doctoral studies, he moved to the University of Zurich (CH), where he worked on the identification of small molecules with a therapeutic potential in the treatment of Alzheimer&#39;s disease. In 2011, he received a double PhD in Biochemistry and Medicinal Chemistry, under the supervision of Prof. A. Caflisch and Prof. A. Carotti. After one year spent in the Drug Discovery and Development Unit of the Italian Institute of Technology in Genoa (IT), working on the development of new anti-pain drugs, he joined the Dokholyan lab, in January 2013. His current research is focusing on the identification of a new class of compounds able to target different isoforms of the &#956;-opioid receptor with the specific aim to develop novel analgesic drugs.','2011 PhD, University of Zurich, CH. Biochemistry <br>2011 PhD, University of Bari, IT. Medicinal Chemistry<br>2005 MSc, University of Bari, IT. Pharmacy','2014 Best Young Researcher Award and Best Poster Award Winner at IV Study in Multidisciplinary Pain Research - SIMPAR 2014<br>2013 Directors Discretionary Program of Oak Ridge Leadership Computing Facility (USA) 6 million CPU-hours allocation on Titan supercomputer (Cray/XK7)<br>2013 Directors Discretionary Program of Argonne National Laboratory (USA) 5 million CPU-hours allocation on Intrepid supercomputer (Blue Gene/P)<br>2013 Directors Discretionary Program of Argonne National Laboratory (USA) 5 million CPU-hours allocation on Vesta supercomputer (Blue Gene/Q)<br>2006 University of Bari Bronze Seal as best graduated student of the Faculty of Pharmacy for the academic year',105),(128,'Benfeard did his undergraduate studies at Duke University with a major in Biophysics and a minor in chemistry. His undergraduate research interests included molecular dynamics simulations of exotic DNA structures and understanding cell cycle control mechanisms in yeast using time-lapse fluorescence microscopy. His current research interests include understanding different variants of Apolipoprotein E and their role in the development of Azheimer&#39;s disease.','2013 B.A., Duke University, Biophysics with a minor in Chemistry','',128),(129,'Jhuma received her Masters in Physics from University of Pune in 2003. Then, she worked with Prof. Bhattacharyaa as a research assistant at Saha Institute of Nuclear Physics until 2005. For her doctoral studies, she moved to University of Missouri (MU). Her thesis was based on computational and theoretical study of water and lipid dynamics in biomolecular systems. She earned her PhD in Physics, under the supervision of Prof. Ioan Kosztin in 2010. During her Ph.D., she also worked as a graduate teaching assistant of Classical Mechanics and Electromagnetics. Then, she joined Prof. Berkowitzs lab at the department of Chemistry, University of North Carolina at Chapel Hill as a postdoctoral fellow and studied the origin of long-range hydrophobic interaction between two inhomogeneously charged surfactant coated model surfaces. She joined the Dokholyan lab in August 2012. Her current research is focused on the computational modeling of the cystic fibrosis transmembrane conductance regulator channel to understand the structure-function relationship of this channel and to develop effective therapeutics to treat cystic fibrosis .','2010 Ph.D. Physics University of Missouri<br>2003 Physics, University of Pune, Maharashtra, India<br>2001 B.Sc., Physics, Lady Brabourne College, Calcutta University, India','2013 Directors Discretionary Program of Oak Ridge Leadership Computing Facility (USA) 6 million CPU-hours allocation on Titan supercomputer (Cray/XK7)<br>2013 Directors Discretionary Program of Argonne National Laboratory (USA) 5 million CPU-hours allocation on Intrepid supercomputer (Blue Gene/P)<br>2013 Directors Discretionary Program of Argonne National Laboratory (USA) 5 million CPU-hours allocation on Vesta supercomputer (Blue Gene/Q)<br>2009 Honorable mention, 25th Annual Missouri Life Sciences Week University of Missouri<br>2008 Physics Leader&#39;ss Meeting Talks 2nd Prize, Department of Physics and Astronomy University of Missouri <br>2008 Donald K. Anderson Teaching Award Nomination, University of Missouri<br>2008 Ernest W. Landen Fellowship, Department of Physics and Astronomy University of Missouri<br>2007 O. M. Stewart Scholarship, Department of Physics and Astronomy University of Missouri<br>2005 Joint CSIR-UGC Junior Research Fellowship (JRF) and Eligibility for lectureship in Physical Sciences, National Eligibility Test, India<br>2002 Book-grant, Department of Physics University of Pune',89),(130,'Dr. Dokholyan received his PhD in Physics at the Boston University under the supervision of Dr. H. Eugene Stanley in 1999. His work spanned the area of statistical mechanics and its applications to biological macromolecules. Upon graduation Dr. Dokholyan joined Dr. Eugene Shakhnovich at Harvard University Department of Chemistry and Chemical Biology as a NIH NRSA Fellow. His work was focused on protein folding, design, and evolution. Dr. Dokholyan joined Department of Biochemistry and Biophysics in the University of North Carolina at Chapel Hill, School of Medicine as an Assistant Professor in 2002, promoted to an Associate Professor in 2008, and to Full Professor in 2011. Dr. Dokholyan is currently the Director of the Center for Computational and Systems Biology and the Graduate Director of the Program in Molecular and Cellular Biophysics at UNC. Dr. Dokholyan has published 180 per review articles and 16 book chapters. Dr. Dokholyan was named American Physical Society Fellow in 2013. In 2014, Dr. Dokholyan was named Michael Hooker Distinguished Professor. Over the past decade Dr. Dokholyan has been working in two principal directions: (i) methodology development to understand and engineer molecular structure and function and (ii) application of these methodologies to understand the etiologies of human diseases and develop therapeutic strategies.','1999-2002 National Institutes of Health Postdoctoral Fellow, Department of Chemistry and Chemical Biology, Harvard University, USA. Area of research: Biophysics <br>1999 Ph.D., Boston University, USA. Physics<br>1994 M.S., Moscow Institute of Physics and Technology, Russia. Physics<br>1992 B.S., Moscow Institute of Physics and Technology, Russia. Physics','2004-2006 March of Dimes Basil O\'Connor Starter Scholar Research Award<br>2004 The University of North Carolina at Chapel Hill IBM Junior Faculty Development Award<br>2003 Recipient of a UNC Research Council Award<br>1999-2002 NIH postdoctoral fellowship<br>1998-1999 NIH Molecular Biophysics Predoctoral Traineeship<br>1995,1998,2001 NSF Young Scientist Travel Award<br>1994 Red Diploma<br>1990-1994 Recipient of Honorary Stipend, Moscow Institute of Physics and Technology',1),(131,'Dr. Dokholyan received his PhD in Physics at the Boston University under the supervision of Dr. H. Eugene Stanley in 1999. His work spanned the area of statistical mechanics and its applications to biological macromolecules. Upon graduation Dr. Dokholyan joined Dr. Eugene Shakhnovich at Harvard University Department of Chemistry and Chemical Biology as a NIH NRSA Fellow. His work was focused on protein folding, design, and evolution. Dr. Dokholyan joined Department of Biochemistry and Biophysics in the University of North Carolina at Chapel Hill, School of Medicine as an Assistant Professor in 2002, promoted to an Associate Professor in 2008, and to Full Professor in 2011. Dr. Dokholyan is currently the Director of the Center for Computational and Systems Biology and the Graduate Director of the Program in Molecular and Cellular Biophysics at UNC. Dr. Dokholyan has published 180 per review articles and 16 book chapters. Dr. Dokholyan was named American Physical Society Fellow in 2013. In 2014, Dr. Dokholyan was named Michael Hooker Distinguished Professor. Over the past decade Dr. Dokholyan has been working in two principal directions: (i) methodology development to understand and engineer molecular structure and function and (ii) application of these methodologies to understand the etiologies of human diseases and develop therapeutic strategies.','1999-2002 National Institutes of Health Postdoctoral Fellow, Department of Chemistry and Chemical Biology, Harvard University, USA. Area of research: Biophysics <br>1999 Ph.D., Boston University, USA. Physics<br>1994 M.S., Moscow Institute of Physics and Technology, Russia. Physics<br>1992 B.S., Moscow Institute of Physics and Technology, Russia. Physics','2014-present Michael Hooker Distinguished Professorship<br>2013 American Physical Society Fellow<br>2011-present Book Series Editor, Series in Computational Biophysics<br>2011 Editor-in-Chief, Research and Reports in Biochemistry<br>2004-2006 March of Dimes Basil O\'Connor Starter Scholar Research Award<br>2004 The University of North Carolina at Chapel Hill IBM Junior Faculty Development Award<br>2003 Recipient of a UNC Research Council Award<br>1999-2002 NIH postdoctoral fellowship<br>1998-1999 NIH Molecular Biophysics Predoctoral Traineeship<br>1995,1998,2001 NSF Young Scientist Travel Award<br>1994 Red Diploma<br>1990-1994 Recipient of Honorary Stipend, Moscow Institute of Physics and Technology',1),(132,'Aspen Gutgsell graduated with a Bachelors of Science in Biology and minored in Biochemistry and Chemistry from New Mexico State University. She is a first year BBSP rotation student looking to join the Biochemistry and Biophysics program. She has previous research experience using site direct mutagenesis to understand drug binding in an important protein for cell division. Aspen was also accepted into the HHMI EXROP program and completed a summer internship at the University of Washington investigating the role of neuronal signaling in tissue regeneration. Her current rotation project is to analyze post translation modifications in human SOD1 using proteomic techniques.','','',133),(133,'David graduated from Wittenberg University in 2008 with a B. Sc. in Biochemistry/Molecular Biology and a minor in Computational Science. Following graduation he enrolled in the Joint Carnegie Mellon University-University of Pittsburgh Ph. D. Program in Computational Biology. At the University of Pittsburgh he studied  structures and dynamics of pentameric ligand-gated ion channels and their interactions with anesthetics under the advisement of Prof. Pei Tang. In November 2014, he joined the Dokholyan lab where he works with computational and experimental approaches to design novel proteins for various biological systems.','','',134),(134,'Elizabeth graduated with honors from Libertyville High School (Libertyville, IL) in 2003, after which she attended Purdue University with a dual major in Physics and Russian Language and Linguistics. At Purdue, Elizabeth competed in the pole vault for the Varsity Track and Field team; was an official tutor for the Departments of Mathematics, Physics, and Foreign Language; and held research positions in the departments of Physics, Chemistry, and Biology. Upon graduating from Purdue in 2008, Elizabeth came to the University of North Carolina at Chapel Hill to study biophysics, where she joined the Dokholyan laboratory. Her research was to investigate the stability, dynamics, and aggregation of proteins relevant to human disease.<br>Elizabeth is currently a post-doctoral research associate at MIT in the lab of Douglas Lauffenburger, with a joint appointment at Harvard Medical School in the lab of Kevin Haigis, where she studies the effect of Alzheimers disease on networks of inflammatory pathways in the brain.','2008 B.S. Honors Physics, Purdue University<br>2008 B.A. Honors Russian Language and Literature, Purdue University','2010-2013  Ruth L. Kirschstein National Research Service Award Predoctoral Fellowship, National Institute of Health<br>2012 F1000Prime Associate Faculty Member Travel Grant, Faculty of 1000<br>2012 NSF-MCC Travel Award, National Science Foundation and the Materials Computation Center, University of Illinois Urbana-Champaign<br>2011 Second place poster prize, Colorado Protein Stability Conference<br>2009-2010 Predoctoral Training Fellowship, Curriculum in Bioinformatics and Computational Biology, University of North Carolina at Chapel Hill<br>2008-2009 Program for Molecular and Cellular Biophysics Trainee at University of North Carolina - Chapel Hill<br>2008 Director&#39;s Award for top candidates upon acceptance to the Biological and Biomedical Sciences Program at the University of North Carolina in Chapel Hill, spring/summer<br>2008 Chosen as Student Response Speaker for Purdue University Graduation, Section II<br>2008 Dobro Slovo Slavic Language Honors Society member<br>2008 Outstanding Senior Award, Purdue University Department of Foreign Language and Literature (Russian section)<br>2004 NCAA Division I Big Ten Conference Track and Field All-Academic Team<br>2003-2004 Ascarelli Fellowship: research fellowship awarded to outstanding freshmen in the Purdue University Physics Department<br>2003-2007 National Merit Scholar, Purdue University',20),(135,'Frank Teets graduated with a Bachelors of Science in Bioinformatics and Molecular Biology from Rensselaer Polytechnic Institute. He is a first year BBSP rotation student looking to join the Bioinformatics and Computational Biology program. He has previous research experience in the development of Monte Carlo protein design methods, viral phylogenetics, and the design of split-GFP biosensors for epidemeological applications. His current rotation will explore the use of split-strand cpGFP sensors for RhoA activity sensing.','','',135),(136,'Anthony Wu is currently a physics major with an interest in biophysics. His previous lab experience has consisted of short projects in environmental biology labs at both UNC & Duke, working mainly with bacterial cell cultures. Currently, he is aiming for graduate school in a biophysics program.','','',137),(137,'Alex graduated with a Bachelors of Science in Biology-Chemistry from Point Loma Nazarene University in San Diego, CA. He is a first year BBSP rotation student looking to join the Neurobiology curriculum. He has done research in small molecule organic synthesis, neurocell biology and human neuroimaging. His current rotation project is investigating proteins that associate with SOD1 trimers.','','',138),(138,'Dr. Yiwen Chen has recently accepted full-time faculty position as an Assistant Professor at the MD Anderson Cancer Center.','','',8),(139,'Kenan Michaels is an junior undergraduate student at the University of North Carolina, pursing a BS in Chemistry with a Biochemistry concentration, with minors in Business Administration and Biology. He was born and raised in Wake Forest, North Carolina where he completed his secondary education at North Raleigh Christian Academy, graduating 3rd in his class. Since a young age, Kenan has professed a profound love for the math and sciences, and has always had a passion for medicine. His current plan is to attend medical school upon graduation in May of 2017. He is working under Reed Jacob who is pursuing a PhD in Bioinformatics and Computational Biology in Nikolay Dokholyans Lab at UNC.','','',130),(140,'Kenan Michaels is an junior undergraduate student at the University of North Carolina, pursing a BS in Chemistry with a Biochemistry concentration, with minors in Business Administration and Biology. He was born and raised in Wake Forest, North Carolina where he completed his secondary education at North Raleigh Christian Academy, graduating 3rd in his class. Since a young age, Kenan has professed a profound love for the math and sciences, and has always had a passion for medicine. His current plan is to attend medical school upon graduation in May of 2017. He is working under Reed Jacob who is pursuing a PhD in Bioinformatics and Computational Biology in Nikolay Dokholyans Lab at UNC.','','',141),(141,'Hanna graduated top three of the graduating class from Romeo High School (Romeo, MI) in 2012, where she was a part of the Advanced Math, Science, and Technology Program all four years.  After, she attended Oakland University (Rochester Hill, MI) for three years majoring in Biochemistry and minoring in Spanish Language.  During her time at Oakland University, Hanna was an undergraduate research assistant in a biochemistry laboratory investigating neurodegenerative diseases (specifically researching the tau protein), elected Vice President and then President of the Oakland University student chapter of the American Chemical Society, elected Treasurer of the Academic Writing for Advanced Spanish Students,Degree in Three and Honors College student. After graduating in 2015 with high academic honors (Summa Cum Laude) and departmental honors, she joined the Biological and Biomedical Sciences Program at the University of North Carolina at Chapel Hill to study biochemistry, where she is doing a summer rotation in the Dokholyan laboratory.  Her research interests are on the topic of neurodegenerative diseases specifically Alzheimers disease and dementia.','2015 Bachelor of Science in Biochemistry and Minor in Spanish Language, Oakland University','2015 The Honors College Thesis Award with Distinction<br>2015 Invited panellist for The Women & Gender Studies Program Women in STEM<br>2014 American Chemical Society Chemistry Ambassador<br>2014 Oakland University Undergraduate Distinguished Achievement Award<br>2014 Invited speaker for Oakland University All-University Fund Drive Kickoff<br>2014 Polish American Cultural Society Scholarship<br>2014 Oakland University Undergraduate Research in Biological Science, Biochemistry, and Chemistry 2014 Program Travel Award<br>2014 Dr. Nasir Haque Endowment for Biological Science Scholarship<br>2014 Best Students for the Modern Language (Spanish) Award<br>2014 Lisa M. Flynn Endowed Fund for Chemistry<br>2014 College of Arts and Sciences Oakland University Credit Union Deans Choice Award Learning Beyond the Classroom<br>2014 Oakland Universitys Barry M. Goldwater Scholarship nominee<br>2013 Chrysler Undergraduate Student Research Award<br>2013 Polish American Cultural Society Scholarship<br>2013 Michael P. and Elizabeth A. Kenny Merit Scholarship for the Sciences<br>2013 Writing Excellence Award in the First-year Writing Research Essay from the Department of Writing and Rhetoric at Oakland University<br>2013 Poem published in Nuestras Voces:  A Journal of Poetry and Prose<br>2012 Distinguished Scholar Award from Oakland University<br>2012 Kiwanis/Steben Fritz, Sr. Memorial Scholarship<br>2012 Realtors Association Scholarship Fund<br>2012 Romeo Masonic Lodge Scholarship<br>2012 Rudolph C. Heino (Michigan Schools & Government Credit Union) Scholarship<br>2012 Presidents Volunteer Service Award from Barack Obama',143),(142,'Stephan Kudlacek graduated from the University of California  Irvine with a Bachelors of Science in Chemistry with an emphasis in Biochemistry. He is a first year rotation student in the Molecular and Cellular Biophysics training program looking to join the Biochemistry and Biophysics program. He has a diverse research experience, ranging from discovering synergistic FDA-approved drugs to reverse antibiotic resistance in MRSA, developing virus-based biosensors for the early detection of cancer and was a part of the UCI team that developed a device to refold aggregated proteins, which was featured in the Unboiling an Egg story. His current rotation project aims to de-orphanize the Bombesin (BB3) receptor developing a homology model and performing molecular docking.','','',145),(143,'Jack Maguire is a BBSP rotation student with plans to join the Bioinformatics and Computational Biology curriculum. He researched computational design of RNA secondary structures at the University of Rochester where he majored in Chemistry. His current project involves analyzing interactions between specific miRNAs and Toll-Like Receptor 8.','','',144),(144,'Marino received his MSc summa cum laude in Pharmacy from the University of Bari (IT) in 2005. For his doctoral studies, he moved to the University of Zurich (CH), where he worked on the identification of small molecules with a therapeutic potential in the treatment of Alzheimer&#39;s disease. In 2011, he received a double PhD in Biochemistry and Medicinal Chemistry, under the supervision of Prof. A. Caflisch and Prof. A. Carotti. After one year spent in the Drug Discovery and Development Unit of the Italian Institute of Technology in Genoa (IT), working on the development of new anti-pain drugs, he joined the Dokholyan lab, in January 2013. His current research is focusing on the identification of a new class of compounds able to target different isoforms of the &#956;-opioid receptor with the specific aim to develop novel analgesic drugs.','2011 PhD, University of Zurich, CH. Biochemistry <br>2011 PhD, University of Bari, IT. Medicinal Chemistry<br>2005 MSc, University of Bari, IT. Pharmacy','2014 Best Young Researcher Award and Best Poster Award Winner at IV Study in Multidisciplinary Pain Research - SIMPAR 2014<br>2013 Directors Discretionary Program of Oak Ridge Leadership Computing Facility (USA) 6 million CPU-hours allocation on Titan supercomputer (Cray/XK7)<br>2013 Directors Discretionary Program of Argonne National Laboratory (USA) 5 million CPU-hours allocation on Intrepid supercomputer (Blue Gene/P)<br>2013 Directors Discretionary Program of Argonne National Laboratory (USA) 5 million CPU-hours allocation on Vesta supercomputer (Blue Gene/Q)<br>2006 University of Bari Bronze Seal as best graduated student of the Faculty of Pharmacy for the academic year',146),(145,'Jessica graduated with a Bachelor of Science in biology and minor in chemistry from Towson University in 2012. She went on to receive a Master of Science in biology from Towson University in 2015. During her time at Towson Jessica investigated small proteins, defined as containing 50 or fewer amino acids, of Escherichia coli. For her masters thesis she investigated the sequence requirements of the small protein CydX. Jessicas current rotation project employs rational design and site-directed mutagenesis to generate mutants of dehydroepiandrosterone sulfotransferase that have an altered substrate specificity.','2012 B.S. Biology, Towson University','2014 Wilfred B. Hathaway Scholarship for Outstanding Graduate Student',148),(146,'Jessica graduated with a Bachelor of Science in biology and minor in chemistry from Towson University in 2012. She went on to receive a Master of Science in biology from Towson University in 2015. During her time at Towson Jessica investigated small proteins, defined as containing 50 or fewer amino acids, of Escherichia coli. For her masters thesis she investigated the sequence requirements of the small protein CydX. Jessicas current rotation project employs rational design and site-directed mutagenesis to generate mutants of dehydroepiandrosterone sulfotransferase that have an altered substrate specificity.','2015 M.S. Biology, Towson University<br>2012 B.S. Biology, Towson University','2014 Wilfred B. Hathaway Scholarship for Outstanding Graduate Student<br>2013 Lois D. ODell Scholarship for Outstanding Transfer Student<br>2012 Lois D. ODell Scholarship for Outstanding Transfer Student',148),(147,'Cathy Anderson is a senior undergraduate student at the University of North Carolina, currently completing a BS in chemistry with a biochemistry concentration, a BS in biology, and a minor in physics. Originally from Maryland, Cathy came to Chapel Hill for her undergraduate career because of her desire to study chemistry and biology. She has since discovered an interest in biophysics and biochemistry, and wants to explore how these fields overlap with biomedical research. During her junior year, she worked with graduate student, Reed Jacob, on a project to identify and assay novel bioscavengers against organophosphate nerve agents. She is now studying amyotrophic lateral sclerosis (ALS), and her current projects include looking at the role of Cu, Zn superoxide dismutase (SOD1) in disease progression and determining the effects of vitamin D3 on motor neuron death in ALS cell models . Cathy is planning on attending graduate school next year to do research in the biomedical sciences.','2012 B.S. Chemistry and Biology, University of North Carolina at Chapel Hill','2016 Senior Honors Thesis (Biology)<br>2016 Senior Honors Thesis (Chemistry)<br>2015 UNC Summer Undergraduate Research Fellowship<br>2015 Undergraduate Teaching Assistant Award, UNC Department of Physics<br>2014 James Maguire Memorial Award, UNC Department of Chemistry<br>2014 Phi Beta Kappa Award',129),(148,'Brandon is a first year Biological and Biomedical Sciences (BBSP) student working towards a PhD in Bioinformatics and Computational Biology.  While earning his bachelor\'s degrees in Quantitative Biology and Computer Science from UNC-Chapel Hill, he conducted research in genomic data mining and visualization focusing on high-resolution DNA-protein interactions.','','',152),(149,'Muhmad Alzpair is a Biomedical Engineering major in the Class of 2020 at the University of North Carolina-Chapel Hill. He is interested in disease etiology and neurodegenerative diseases.','','',155),(150,'Bharath Rama is a member of the Class of 2018 majoring in biochemistry & mathematics. He is interested in chemoinformatics as it pertains to medicine.','','',156),(151,'Jiyoung did her undergraduate studies in Seoul National University (SNU), Korea, majoring in chemistry. During her M.S. in SNU, she studied on laser-induced fluorescence spectroscopy of clusters in free Jet. She spent two years in the Fleming group of UC-Berkeley working on heterodyne-detected two-dimensional fifth-order Raman spectroscopy in liquid carbon disulfide. In 2001, she started her PhD at Caltech under the supervision of Prof. William A. Goddard III. Her PhD research included predition of atomistic 3D models for orphan GPCRs & molecular docking and functional assay for experimental validation. She continued her postdoctoral research at Caltech, working on structure prediction of glucagon-like peptide 1 receptor, a class B GPCR. Since 2011, she has been affiliated with Sangmyung University (Cheonan, Korea) as a tenure-tracked professor. Her current research interest is the computational studies of induced circular dichroism in host-guest complex system.','2006 PhD in Chemistry, California Institute of Technology, Pasadena<br>2001 MS in Chemistry, University of California, Berkeley<br>1997 MS in Chemistry, Seoul National University, Seoul, Korea<br>1995 BS in Chemistry, Seoul National University, Seoul, Korea','',158),(152,'Jiyoung did her undergraduate studies in Seoul National University (SNU), Korea, majoring in chemistry. During her M.S. in SNU, she studied on laser-induced fluorescence spectroscopy of clusters in free Jet. She spent two years in the Fleming group of UC-Berkeley working on heterodyne-detected two-dimensional fifth-order Raman spectroscopy in liquid carbon disulfide. In 2001, she started her PhD at Caltech under the supervision of Prof. William A. Goddard III. Her Ph.D. research included prediction of atomistic 3D models for orphan GPCRs & molecular docking and functional assay for experimental validation. She continued her postdoctoral research at Caltech, working on structure prediction of glucagon-like peptide 1 receptor, a class B GPCR. Since 2011, she has been affiliated with Sangmyung University (Cheonan, Korea) as a tenure-tracked professor. Her current research interest is the computational studies of induced circular dichroism in host-guest complex system.','2006 PhD in Chemistry, California Institute of Technology, Pasadena<br>2001 MS in Chemistry, University of California, Berkeley<br>1997 MS in Chemistry, Seoul National University, Seoul, Korea<br>1995 BS in Chemistry, Seoul National University, Seoul, Korea','',158),(153,'Ashhar received his Masters degree from Jamia Hamdard University (India, 2008) with major in Biochemistry. Then, he joined Prof. Shashank Deeps lab for his doctoral degree in the Department of Chemistry at Indian Institute Of Technology-Delhi in 2009. His doctoral work was directed towards understanding of the aggregation propensity of Superoxide dismutase 1(SOD1) protein and its variants involve in ALS disease. He made a systematic study of SOD1 fibrillation by tuning some of the structural features (through protein engineering) and environmental factors which could impact its fibrillation behaviors using various biochemical & biophysical techniques. In 2012, he visited CMPS, Lund University as a Visiting PhD Scholar under Erasmus Mundus Exchange Scholarship by European Union to do part of doctoral research under Prof Mikael Akke and Prof. Sara Snogerup Linse (co-mentor). There he elucidated the events in early stages of the aggregation of SOD1 with residue-specific information that drives the protein to interact readily, leading to aberrant oligomerization in SOD1 under physiologically relevant conditions. His current research is focussed on elucidation of toxicity pathways of SOD1 trimers.','2016 Ph.D., Indian Institute Of Technology-Delhi, New Delhi, India<br>2008 Masters in Science in Biochemistry, Jamia Hamdard University, New Delhi, India','2016 Postdoctoral Scholarship by the Carl Trygger Foundation, Sweden<br>2012 International Erasmus Mundus Scholarship (EMEA 2) by European Union to conduct research work (2012-2014) at CMPS, Lund University under Prof. Mikael Akke andProf. Sara Snogerup Linse (co-Mentor)<br>2011 Senior Research Fellowship (SRF), 2011-2012 by UGC/CSIR, India<br>2009 Life time member of the Biotech Research Society of India (BRSI), India<br>2008 Joint CSIR-UGC Junior Research Fellowship (JRF) and Eligibility for lectureship in Life Sciences, National Eligibility Test, India',163),(154,'Ashhar received his Masters degree from Jamia Hamdard University (India, 2008) with major in Biochemistry. Then, he joined Prof. Shashank Deeps lab for his doctoral degree at the Department of Chemistry, Indian Institute Of Technology-Delhi in 2009. His doctoral work was directed towards understanding the aggregation propensity of Superoxide dismutase 1(SOD1) protein and its variants involved in ALS disease. He performed a systematic study of SOD1 fibrillation by tuning some of the structural features (through protein engineering) and environmental factors which could impact its fibrillation behaviors using various biochemical & biophysical techniques. In 2012, he visited CMPS, Lund University as a Visiting PhD Scholar under Erasmus Mundus Exchange Scholarship by European Union to execute part of his doctoral research under Prof Mikael Akke and Prof. Sara Snogerup Linse (co-mentor). There he elucidated the events in early stages of the aggregation of SOD1 with residue-specific information that drives the protein to interact readily, leading to aberrant oligomerization in SOD1 under physiologically relevant conditions. His current research is focused on the elucidation of toxicity pathways of SOD1 trimers.','2016 Ph.D., Indian Institute Of Technology-Delhi, New Delhi, India<br>2008 Masters in Science in Biochemistry, Jamia Hamdard University, New Delhi, India','2016 Postdoctoral Scholarship by the Carl Trygger Foundation, Sweden<br>2012 International Erasmus Mundus Scholarship (EMEA 2) by European Union to conduct research work (2012-2014) at CMPS, Lund University under Prof. Mikael Akke andProf. Sara Snogerup Linse (co-Mentor)<br>2011 Senior Research Fellowship (SRF), 2011-2012 by UGC/CSIR, India<br>2009 Life time member of the Biotech Research Society of India (BRSI), India<br>2008 Joint CSIR-UGC Junior Research Fellowship (JRF) and Eligibility for lectureship in Life Sciences, National Eligibility Test, India',163),(155,'Ashhar received his Masters degree from Jamia Hamdard University (India, 2008) with major in Biochemistry. Then, he joined Prof. Shashank Deeps lab for his doctoral degree at the Department of Chemistry, Indian Institute Of Technology-Delhi in 2009. His doctoral work was directed towards understanding the aggregation propensity of Superoxide dismutase 1(SOD1) protein and its variants involved in ALS disease. He performed a systematic study of SOD1 fibrillation by tuning some of the structural features (through protein engineering) and environmental factors which could impact its fibrillation behaviors using various biochemical & biophysical techniques. In 2012, he visited CMPS, Lund University as a Visiting PhD Scholar under Erasmus Mundus Exchange Scholarship by European Union to execute part of his doctoral research under Prof Mikael Akke and Prof. Sara Snogerup Linse (co-mentor). There he elucidated the events in early stages of the aggregation of SOD1 with residue-specific information that drives the protein to interact readily, leading to aberrant oligomerization in SOD1 under physiologically relevant conditions. His current research is focused on the elucidation of toxicity pathways of SOD1 trimers.','2016 Ph.D., Indian Institute Of Technology-Delhi, New Delhi, India<br>2008 Masters in Science in Biochemistry, Jamia Hamdard University, New Delhi, India','2016 Postdoctoral Scholarship by the Carl Trygger Foundation, Sweden<br>2012 International Erasmus Mundus Scholarship (EMEA 2) by European Union to conduct research work (2012-2014) at CMPS, Lund University under Prof. Mikael Akke and Prof. Sara Snogerup Linse (co-Mentor)<br>2011 Senior Research Fellowship (SRF), 2011-2012 by UGC/CSIR, India<br>2009 Life time member of the Biotech Research Society of India (BRSI), India<br>2008 Joint CSIR-UGC Junior Research Fellowship (JRF) and Eligibility for lectureship in Life Sciences, National Eligibility Test, India',163),(156,'David recieved his Masters in Molecular Biology and Genetics from Masaryk University in Brno, Czech Republic. At Masaryk University, he continued his doctoral studies and recieved his Ph.D. in Molecular and Cellular Biology in 2017 under the supervision of prof. Jiri Damborsky. His thesis was focused on the developement of a method for in silico prediction of stable proteins and characterization of haloalkane dehalogenases utilizing molecular modeling approaches. For During his Ph.D., he attended Dr. Khare\'s lab in Rutgers University for three months as a visiting scholar, where he pursued desiging enzymes using Rosetta software. Currently, he is visiting Dr. Dokholyan\'s lab as a research scientist to study domain swapping in protein dimers using molecular dynamics simulations.','','',165);
/*!40000 ALTER TABLE `mem_info` ENABLE KEYS */;
UNLOCK TABLES;

--
-- Table structure for table `members`
--

DROP TABLE IF EXISTS `members`;
/*!40101 SET @saved_cs_client     = @@character_set_client */;
/*!40101 SET character_set_client = utf8 */;
CREATE TABLE `members` (
  `mem_id` int(10) NOT NULL AUTO_INCREMENT,
  `f_name` varchar(30) DEFAULT '',
  `m_init` varchar(5) DEFAULT '',
  `l_name` varchar(30) DEFAULT '',
  `has_publ` varchar(3) DEFAULT 'no',
  `current` varchar(3) DEFAULT 'yes',
  `designation` varchar(30) DEFAULT '',
  `memfrom` smallint(6) DEFAULT '2004',
  `memtill` varchar(10) DEFAULT 'Present',
  PRIMARY KEY (`mem_id`)
) ENGINE=MyISAM AUTO_INCREMENT=29 DEFAULT CHARSET=utf8;
/*!40101 SET character_set_client = @saved_cs_client */;

--
-- Dumping data for table `members`
--

LOCK TABLES `members` WRITE;
/*!40000 ALTER TABLE `members` DISABLE KEYS */;
INSERT INTO `members` VALUES (1,'Nikolay','V.','Dokholyan','yes','yes','Principal Investigator',2002,'Present'),(2,'Feng','','Ding','yes','yes','Research Assistant Professor',2004,'Present'),(3,'Shuangye','','Yin','yes','yes','Postdoctoral Fellow',2006,'Present'),(4,'Shantanu','','Sharma','yes','no','Graduate Student',2004,'2009'),(5,'Adrian','W.R.','Serohijos','yes','no','Graduate Student',2006,'2009'),(6,'Kyle','C.','Wilcox','yes','yes','Graduate Student',2004,'Present'),(7,'Sagar','D.','Khare','yes','no','Graduate Student',2002,'2005'),(8,'Yiwen','','Chen','yes','no','Graduate Student',2003,'2007'),(9,'Srinivas','','Ramachandran','yes','yes','Graduate Student',2006,'Present'),(10,'Pradeep','','Kota','yes','yes','Graduate Student',2007,'Present'),(13,'Huifen','','Nie','yes','no','Postdoctoral Fellow',2005,'2007'),(14,'Brittany','M.','Fotsch','no','yes','Undergraduate Student',2007,'Present'),(15,'Jainab','','Khatun','yes','no','Postdoctoral Fellow',2003,'2004'),(17,'Peter','','Itskowitz','no','no','Postdoctoral Fellow',2004,'2004'),(18,'Joshua','K.','Jordon','yes','no','Technician',2007,'Present'),(19,'Rachel','L.','Redler','yes','yes','Graduate Student',2008,'Present'),(20,'Elizabeth','A.','Proctor','yes','yes','Graduate Student',2008,'Present'),(21,'Bryan','','Der','no','no','Rotation Student',2009,'Present'),(22,'Barry','','Kesner','yes','no','Graduate Student',2008,'2009'),(23,'Douglas','C.','Tsao','yes','yes','Graduate Student',2007,'Present'),(27,'Kalada','','Kienka','no','yes','Undergraduate Student',2010,'Present'),(28,'Shayna','','Atkins','no','yes','Undergraduate Student',2010,'Present');
/*!40000 ALTER TABLE `members` ENABLE KEYS */;
UNLOCK TABLES;

--
-- Table structure for table `mems`
--

DROP TABLE IF EXISTS `mems`;
/*!40101 SET @saved_cs_client     = @@character_set_client */;
/*!40101 SET character_set_client = utf8 */;
CREATE TABLE `mems` (
  `mem_id` int(10) NOT NULL AUTO_INCREMENT,
  `f_name` varchar(30) DEFAULT '',
  `m_init` varchar(5) DEFAULT '',
  `l_name` varchar(30) DEFAULT '',
  `email` varchar(100) DEFAULT NULL,
  `has_publ` varchar(3) DEFAULT 'no',
  `current` varchar(3) DEFAULT 'no',
  `designation` varchar(30) DEFAULT '',
  `memfrom` smallint(6) DEFAULT '2004',
  `memtill` varchar(10) DEFAULT '',
  PRIMARY KEY (`mem_id`)
) ENGINE=MyISAM AUTO_INCREMENT=168 DEFAULT CHARSET=utf8;
/*!40101 SET character_set_client = @saved_cs_client */;

--
-- Dumping data for table `mems`
--

LOCK TABLES `mems` WRITE;
/*!40000 ALTER TABLE `mems` DISABLE KEYS */;
INSERT INTO `mems` VALUES (1,'Nikolay','V.','Dokholyan','dokh@email.unc.edu','yes','yes','Principal Investigator',2002,'Present'),(2,'Feng','','Ding','fding@unc.edu','yes','no','Research Assistant Professor',2004,'2012'),(3,'Shuangye','','Yin','syin@email.unc.edu','yes','no','Postdoctoral Fellow',2006,'2011'),(4,'Shantanu','','Sharma','shantanu@unc.edu','yes','no','Graduate Student',2004,'2009'),(5,'Adrian','W.R.','Serohijos','awrs@email.unc.edu','yes','no','Graduate Student',2006,'2009'),(6,'Kyle','C.','Wilcox','iowajones@gmail.com','yes','no','Graduate Student',2004,'2009'),(7,'Sagar','D.','Khare',NULL,'yes','no','Graduate Student',2002,'2005'),(8,'Yiwen','','Chen','YChen26@mdanderson.org','yes','no','Graduate Student',2003,'2007'),(9,'Srinivas','','Ramachandran','ramachan@email.unc.edu','yes','no','Graduate Student',2006,'2011'),(10,'Pradeep','','Kota','pkota@email.unc.edu','yes','no','Graduate Student',2007,'2012'),(13,'Huifen','','Nie',NULL,'yes','no','Postdoctoral Fellow',2005,'2007'),(14,'Brittany','M.','Fotsch','bfotsch@email.unc.edu','no','no','Undergraduate Student',2007,'2010'),(15,'Jainab','','Khatun',NULL,'yes','no','Postdoctoral Fellow',2003,'2004'),(17,'Peter','','Itskowitz',NULL,'no','no','Postdoctoral Fellow',2004,'2004'),(18,'Joshua','K.','Jordon',NULL,'yes','no','Technician',2007,'2009'),(19,'Rachel','L.','Redler','rachelredler@gmail.com','yes','no','Graduate Student',2008,'2013'),(20,'Elizabeth','A.','Proctor','eproctor@mit.edu','yes','no','Graduate Student',2008,'2013'),(21,'Bryan','','Der','bder@email.unc.edu','no','no','Rotation Student',2009,'2009'),(22,'Barry','','Kesner','kesner@med.unc.edu','yes','no','Graduate Student',2008,'2009'),(23,'Douglas','C.','Tsao','tsao@email.unc.edu','yes','no','Graduate Student',2007,'2011'),(39,'Vinal','V.','Lakhani','vlakhani@email.unc.edu','yes','no','Rotation Student',2009,'2009'),(40,'Lanette','R.','Fee',NULL,'yes','no','Technician',2009,'2013'),(41,'William','','Monteith',NULL,'no','no','Rotation Student',2010,'2010'),(42,'Jeremy','','Hahn','jhahn01@mit.edu','no','no','Research Associate',2010,'2010'),(43,'David','G.','Shirvanyants','shir@email.unc.edu','yes','no','Research Associate',2010,'2013'),(44,'Shayna','','Atkins',NULL,'no','no','Undergraduate Student',2010,'2010'),(45,'Kalada','','Kienka',NULL,'no','no','Undergraduate Student',2010,'2010'),(48,'Greggory','','Rice','gmr@unc.edu','no','no','Rotation Student',2010,'2010'),(50,'Srinivas Niranj','','Chandrasekaran','sniranj@email.unc.edu','no','no','Rotation Student',2010,'2010'),(46,'James','','Prevo','jprevo@email.unc.edu','no','no','Undergraduate Student',2010,'2011'),(47,'Onur','','Dagliyan','dagliyan@email.unc.edu','yes','no','Graduate Student',2010,'2016'),(49,'Timothy','','Jacobs','tjacobs2@email.unc.edu','no','no','Rotation Student',2010,'2010'),(51,'Tamas','','Hegedus','tamas@hegelab.org','yes','no','Visiting Scientist',2010,'2010'),(52,'Gergely','','Gyimesi','ggyimesi@hegelab.org','no','no','Visiting Scientist',2010,'2010'),(53,'Lada','','Biedermannova','','yes','no','Visiting Scientist',2006,'2007'),(54,'Mingfeng','','Yang','','no','no','Visiting Scientist',2007,'2007'),(55,'Justin','','Caravella','','no','no','Visiting Scientist',2009,'2009'),(56,'Peng','','Gong','','yes','no','Graduate (MS) Student',2004,'2006'),(57,'Joshua','J.','Larocque','','yes','no','Undergraduate Student',2005,'2005'),(58,'Vinal','V.','Lakhani','vlakhani@email.unc.edu','no','no','Undergraduate Student',2006,'2009'),(59,'Kenneth','N.','Gwanmesia','','yes','no','Undergraduate Student',2004,'2004'),(60,'Justin','T.','Low','','no','no','Undergraduate Student',2004,'2004'),(61,'Tamara','','James','','no','no','Undergraduate Student',2005,'2005'),(62,'Sunjay','','Barton','','yes','no','Undergraduate Student',2006,'2006'),(63,'Sunny','','Darji','','no','no','Undergraduate Student',2007,'2007'),(64,'Elvira','','Jasarevic','','no','no','Undergraduate Student',2007,'2007'),(65,'Kevin','','DAuria','','no','no','Undergraduate Student',2008,'2008'),(66,'Andrew','D. A.','Marshall','','no','no','Undergraduate Student',2009,'2009'),(67,'Regis','A.','James','','no','no','Undergraduate Student',2009,'2009'),(69,'James','M.','Fay','jimmyfay3737@email.unc.edu','yes','no','Technician',2012,'2016'),(70,'Manuel','','Sparta','sparta@chem.ucla.edu','no','no','Visiting Scientist',2010,'2010'),(71,'Anastassia','N.','Alexandrova','ana@chem.ucla.edu','no','no','Visiting Scientist',2010,'2010'),(72,'Adam','D.','Friedman','friedmad@email.unc.edu','no','no','Rotation Student',2011,'2011'),(73,'Shura','','Hayryan','shura@phys.sinica.edu.tw','no','no','Visiting Scientist',2011,'2011'),(74,'Taeho','','Kim','tkim@ifrec.osaka-u.ac.jp','no','no','Visiting Scientist',2011,'2011'),(75,'Zimeng','','Li','lizimeng@unc.edu','no','no','Rotation Student',2011,'2011'),(76,'Mahmoud','','Shobair','infinityrelativity@hotmail.com','no','no','Undergraduate Student',2011,'2011'),(77,'Mary','','Dandulakis','mdandula@live.unc.edu','no','no','Undergraduate Student',2011,'2011'),(78,'Chris','','Kao','ckao@email.unc.edu','no','no','Undergraduate Student',2011,'2012'),(79,'Srinivas','','Saripalli','ssar@live.unc.edu','no','no','Undergraduate Student',2013,'2013'),(80,'Arpit','','Tandon','atandon@email.unc.edu','no','no','Graduate (MS) Student',2011,'2014'),(81,'Christine','S','Kim','cskk622@email.unc.edu','no','no','Rotation Student',2011,'2012'),(82,'Patrick','','McCarter','pcmccart@email.unc.edu','no','no','Rotation Student',2011,'2012'),(83,'Travis','','Hoppe','travis.hoppe@nih.gov','no','no','Visiting Scientist',2011,'2011'),(84,'Doo Nam','','Kim','kimdn@email.unc.edu','no','no','Rotation Student',2012,'2012'),(85,'Cameron','','Pinnock','bw7722@wayne.edu','no','no','Summer student',2012,'2012'),(86,'Juan','R','Diaz','juan.diaz3@upr.edu','no','no','Summer student',2012,'2012'),(87,'Jordan','','Texier','texier@email.unc.edu','no','no','Undergraduate Student',2013,'2013'),(89,'Jhuma','','Das','jdas@email.unc.edu','no','yes','Research Associate',2012,'Present'),(109,'Raul','','Mendez-Giraldez','rmendez@med.unc.edu','yes','no','Postdoctoral Fellow',2013,'2015'),(88,'Dylan','','Glatt-Dowd','glattdow@email.unc.edu','no','no','Rotation student',2012,'2012'),(90,'Rachel','','Cohen','cohenr@email.unc.edu','no','no','Rotation Student',2012,'2012'),(91,'Chanin','B.','Tolson','ctolson@email.unc.edu','no','no','Rotation Student',2012,'2012'),(92,'Gurpreet','','Kaur','gurpreet.bioinfo@gmail.com','no','no','Fulbright Foreign Student',2012,'Present'),(93,'Kevin','','Houlihan','khouli@gmail.com','no','no','Rotation Student',2012,'2012'),(95,'Mahmoud','','Shobair','shobair@email.unc.edu','no','no','Rotation Student',2012,'2012'),(107,'Sainath','','Asokan','sasokan@live.unc.edu','no','no','Undergraduate Student',2013,'2013'),(96,'Reed','B.','Jacob','rbjacob@email.unc.edu','yes','no','Graduate Student',2013,'2017'),(97,'Mayukh','','Chakrabarti','mchakrab@email.unc.edu','no','no','Rotation Student',2012,'2012'),(98,'Pooja','','Patel','pooja117@live.unc.edu','no','no','Undergraduate Student',2012,'2014'),(100,'Cameron','','Pinnock',NULL,'no','no','Undergraduate Student',2012,'2012'),(101,'Juan','R.','Diaz','','no','no','Undergraduate Student',2012,'2012'),(102,'Simon','','Menaker','','no','no','Undergraduate Student',2012,'2012'),(108,'Ian','','Mercer','imercer@live.unc.edu','no','no','Undergraduate Student',2013,'2013'),(105,'Marino','','Convertino','mconvert@email.unc.edu','yes','no','Research Assistant Professor',2013,'2017'),(106,'Chris','H.','McMahon','mcmahonc@live.unc.edu','no','no','Undergraduate Student',2013,'2013'),(110,'Andrey','','Krokhotin','krohotin@med.unc.edu','yes','yes','Postdoctoral Fellow',2013,'Present'),(111,'Soonmin','','Jang','sjang@sejong.ac.kr','no','no','Visiting Scientist',2013,'2014'),(112,'Ardeshir',' ','Goliaei','goliaei@email.unc.edu','no','no','Rotation Student',2013,'2013'),(113,'Kossi',' ','Agbeve','kaagbeve@email.unc.edu','no','no','Rotation Student',2013,'2013'),(114,'Mahmoud','','Shobair','shobair@email.unc.edu','yes','no','Graduate Student',2013,'2017'),(115,'Joseph','N.','Kousouros','joekousouros@yahoo.com','no','no','Summer student',2013,'2013'),(116,'Joseph',' ','Park','parkj4522@student.bucks.edu','no','no','Summer student',2013,'2013'),(117,'Benfeard','','Williams','benfeard@email.unc.edu','no','no','Rotation Student',2013,'2013'),(156,'Bharath','','Rama','bharathr@live.unc.edu','yes','yes','Undergraduate Student',2016,'Present'),(119,'Cheng','','Zhu','cheng_zhu@med.unc.edu','yes','yes','Postdoctoral Fellow',2013,'Present'),(120,'Joseph','N.','Kousouros','kousouro@email.unc.edu','no','no','Technician',2013,'2015'),(121,'Yazhong','','Tao','yazhongtao@gmail.com','yes','no','Technician',2014,'2016'),(123,'Rachel','L.','Redler','rachelredler@gmail.com','no','no','Postdoctoral Fellow',2014,'2014'),(124,'Dominique','N.','Soroka','sodo@email.unc.edu','no','no','Rotation Student',2013,'2014'),(125,'Matt','','Satusky','satusky@live.unc.edu','no','no','Rotation Student',2014,'2014'),(144,'Jack','','Maguire','jackmag@email.unc.edu','no','no','Rotation Student',2015,'2015'),(127,'Regina','','Politi','reginap@email.unc.edu','yes','no','Postdoctoral Fellow',2014,'2016'),(128,'Benfeard','','Williams','benfeard@email.unc.edu','yes','no','Graduate Student',2014,'2017'),(129,'Cathy','J.','Anderson','catson4@live.unc.edu','no','no','Undergraduate Student',2013,'2016'),(130,'Kenan','','Michaels','kcmichae@live.unc.edu','yes','no','Undergraduate Student',2014,'2016'),(131,'Olivia','','Dickens','olivia.dickens13@gmail.com','no','no','Summer Student',2014,'2014'),(132,'Olivia','','Dickens','olivia.dickens13@gmail.com','no','no','Summer student',2014,'2014'),(133,'Aspen','R.','Gutgsell','aspeng@email.unc.edu ','no','no','Rotation Student',2015,'2016'),(134,'David','D.','Mowrey','david_mowrey@med.unc.edu','yes','no','Postdoctoral Fellow',2014,'2017'),(135,'Frank','D.','Teets','frankdt@email.unc.edu','no','no','Rotation Student',2014,'2015'),(136,'Konstantin','I.','Popov','kpopov@email.unc.edu','no','yes','Research Assistant Professor',2014,'Present'),(137,'Anthony','','Wu','wua2015@live.unc.edu','no','no','Undergraduate Student',2015,'2016'),(138,'Alex','','Carlson','alxndrc@email.unc.edu','no','no','Rotation Student',2015,'2015'),(140,'Edgar','M.','Faison','emfaison@live.unc.edu','no','yes','Technician',2015,'Present'),(149,'Faith','','Jarvis','floj521@live.unc.edu','yes','yes','Undergraduate Student',2016,'Present'),(142,'Stephen','L.','Upton','slupton@live.unc.edu','no','no','Undergraduate Student',2015,'2016'),(143,'Hanna','','Trzeciakiewicz','htrzecia@email.unc.edu','no','no','Rotation Student',2015,'2015'),(145,'Stephan','T.','Kudlacek','skudlacek@email.unc.edu','no','no','Rotation Student',2015,'2015'),(159,'Madeleine','J.','Nieto','mnieto24@live.unc.edu','no','yes','Undergraduate Student',2017,'Present'),(148,'Jessica','J.','Hobson','jjean@email.unc.edu','no','no','Rotation Student',2015,'2016'),(151,'Ria','','Das','rajeshreedas2019@u.northwestern.edu','no','no','Summer student',2016,'2016'),(152,'Brandon','','Price','baprice@email.unc.edu','no','no','Rotation Student',2016,'2016'),(153,'Kasey','','Skinner','krskinne@email.unc.edu','no','no','Rotation Student',2016,'2016'),(154,'Alice','','Gabrielov','alicgab@email.unc.edu','no','no','Rotation Student',2016,'2016'),(155,'Muhmad','','Alzpair','muhmad@live.unc.edu','no','no','Undergraduate Student',2016,'2016'),(157,'Jake','R.','Strauss','straussj@live.unc.edu','no','no','Undergraduate Student',2016,'2017'),(158,'Jiyoung','','Heo','jyheo72@email.unc.edu','no','yes','Visiting Scientist',2017,'Present'),(160,'Ernesto','','Alva Sevilla','ernesto.alva@me.com','no','yes','Summer Student',2017,'Present'),(162,'Madeleine','','Nieto','mnieto24@live.unc.edu','no','yes','Summer Student',2017,'Present'),(163,'M.Ashhar','','Iqbal','ashhar_khan@med.unc.edu','no','yes','Postdoctoral Fellow',2017,'Present'),(164,'Skanda','K.','Sastry','skanda@email.unc.edu','no','yes','Summer Student',2017,'Present'),(165,'David','','Bednar','davidbednar1208@gmail.com','no','yes','Visiting Scientist',2017,'Present'),(166,'Shu','','Zhang','szhang01@unc.edu','no','yes','Rotation Student',2017,'Present'),(167,'Venkat','','Reddy','venkat4bt@gmail.com','no','yes','Postdoctoral Fellow',2017,'Present');
/*!40000 ALTER TABLE `mems` ENABLE KEYS */;
UNLOCK TABLES;

--
-- Table structure for table `publications`
--

DROP TABLE IF EXISTS `publications`;
/*!40101 SET @saved_cs_client     = @@character_set_client */;
/*!40101 SET character_set_client = utf8 */;
CREATE TABLE `publications` (
  `pub_id` int(10) unsigned NOT NULL AUTO_INCREMENT,
  `code` varchar(60) DEFAULT '',
  `authors` varchar(500) DEFAULT '',
  `title` varchar(500) DEFAULT '',
  `journal` varchar(250) DEFAULT '',
  `volume` int(5) DEFAULT '0',
  `fpage` varchar(10) DEFAULT '',
  `lpage` varchar(10) DEFAULT '',
  `year` int(5) DEFAULT '2009',
  `topic` varchar(100) DEFAULT '',
  `cauthors` varchar(100) DEFAULT 'Dokholyan, N. V.',
  `suppinfo` varchar(3) DEFAULT 'yes',
  `cover` varchar(3) DEFAULT 'no',
  `type` varchar(15) DEFAULT 'article',
  `pmid` int(10) unsigned DEFAULT '0',
  `pmcid` varchar(20) DEFAULT '',
  `editors` varchar(200) DEFAULT '',
  `series` varchar(100) DEFAULT '',
  `publisher` varchar(100) DEFAULT '',
  `abstract` text,
  PRIMARY KEY (`pub_id`)
) ENGINE=MyISAM AUTO_INCREMENT=707 DEFAULT CHARSET=utf8;
/*!40101 SET character_set_client = @saved_cs_client */;

--
-- Dumping data for table `publications`
--

LOCK TABLES `publications` WRITE;
/*!40000 ALTER TABLE `publications` DISABLE KEYS */;
INSERT INTO `publications` VALUES (687,'rtcd_nar09','Ramachandran, S., Temple, B. R., Chaney, S. G., and Dokholyan, N. V., ','Structural basis for the sequence dependent effects of platinum-DNA adducts','Nucleic Acids Research',37,'2434','2448',2009,'DNA','Chaney, S. G., Dokholyan, N. V.','yes','no','article',19255091,'',NULL,NULL,NULL,'The differences in efficacy and molecular mechanisms of platinum based anti-cancer drugs cisplatin (CP) and oxaliplatin (OX) have been hypothesized to be in part due to the differential binding affinity of cellular and damage recognition proteins to CP and OX adducts formed on adjacent guanines in genomic DNA. HMGB1a in particular exhibits higher binding affinity to CP-GG adducts, and the extent of discrimination between CP- and OX-GG adducts is dependent on the bases flanking the adducts. However, the structural basis for this differential binding is not known. Here, we show that the conformational dynamics of CP- and OX-GG adducts are distinct and depend on the sequence context of the adduct. Molecular dynamics simulations of the Pt-GG adducts in the TGGA sequence context revealed that even though the major conformations of CP- and OX-GG adducts were similar, the minor conformations were distinct. Using the pattern of hydrogen bond formation between the Pt-ammines and the adjacent DNA bases, we identified the major and minor conformations sampled by Pt-DNA. We found that the minor conformations sampled exclusively by the CP-GG adduct exhibit structural properties that favor binding by HMGB1a, which may explain its higher binding affinity to CP-GG adducts, while these conformations are not sampled by OX-GG adducts because of the constraints imposed by its cyclohexane ring, which may explain the negligible binding affinity of HMGB1a for OX-GG adducts in the TGGA sequence context. Based on these results, we postulate that the constraints imposed by the cyclohexane ring of OX affect the DNA conformations explored by OX-GG adduct compared to those of CP-GG adduct, which may influence the binding affinities of HMG-domain proteins for Pt-GG adducts, and that these conformations are further influenced by the DNA sequence context of the Pt-GG adduct.'),(688,'tldpkdd_jbc09','Torres, M. P., Lee, M. J., Ding, F., Purbeck, C., Kuhlman, B., Dokholyan, N. V., and Dohlman, H. G., ','G protein mono-ubiquitination by the RSP5 ubiquitin ligase','Journal of Biological Chemistry',284,'8940','8950',2009,'Proteins: Mechanism','Dohlman, H. G.','no','no','article',19176477,'',NULL,NULL,NULL,'Emerging evidence suggests that ubiquitination serves as a protein trafficking signal in addition to its well characterized role in promoting protein degradation. The yeast G protein &alpha; subunit Gpa1 represents a rare example of a protein that undergoes both mono- and poly-ubiquitination. Whereas mono-ubiquitinated Gpa1 is targeted to the vacuole, poly-ubiquitinated Gpa1 is directed instead to the proteasome. Here we investigate the structural requirements for mono- and poly-ubiquitination of Gpa1. We find that variants of Gpa1 engineered to be unstable are more likely to be poly-ubiquitinated and less likely to be mono-ubiquitinated. In addition, mutants that cannot be myristoylated are no longer mono-ubiquitinated but are still polyubiquitinated. Finally, we show that the ubiquitin ligase Rsp5 is necessary for Gpa1 mono-ubiquitination <i>in vivo</i> and that the purified enzyme is sufficient to catalyze Gpa1 mono-ubiquitination <i>in vitro</i>. Taken together, these data indicate that mono- and poly-ubiquitination have distinct enzyme and substrate recognition requirements; whereas poly-ubiquitination targets misfolded protein for degradation, a distinct ubiquitination apparatus targets the fully mature, fully myristoylated G protein for mono-ubiquitination and delivery to the vacuole.'),(682,'pxccdb_mcp09','Petrotchenko, E. V., Xiao, K., Cable, J., Chen, Y., Dokholyan, N. V., and Borchers, C. H., ','Bimane Bisthiopropionic Acid N-Succinimidyl Ester (BiPS), a photo-cleavable, isotopically-coded, fluorescent crosslinker for structural proteomics','Molecular and Cellular Proteomics',8,'273','286',2009,'Biotechnology','Borchers, C. H.','no','yes','article',18838738,'',NULL,NULL,NULL,'Cross-linking combined with mass spectrometry is an emerging approach for studying protein structure and protein-protein interactions. However, unambiguous mass spectrometric identification of cross-linked peptides derived from proteolytically digested cross-linked proteins is still challenging. Here we describe the use of a novel cross-linker, bimane bisthiopropionic acid Nsuccinimidyl ester (BiPS), that overcomes many of the challenges associated with other cross-linking reagents. BiPS is distinguished from other cross-linkers by a unique combination of properties: it is photocleavable, fluorescent, homobifunctional, amine-reactive, and isotopically coded. As demonstrated with a model protein complex, RNase S, the fluorescent moiety of BiPS allows for sensitive and specific monitoring of the different cross-linking steps, including detection and isolation of cross-linked proteins by gel electrophoresis, determination of in-gel digestion completion, and fluorescence-based separation of cross-linked peptides by HPLC. The isotopic coding of BiPS results in characteristic ion signal &quot;doublets&quot; in mass spectra, thereby permitting ready detection of cross-linker-containing peptides. Under MALDI-MS conditions, partial photocleavage of the cross-linker occurs, releasing the crosslinked peptides. This allows differentiation between dead-end, intra-, and interpeptide cross-links based on losses of specific mass fragments. It also allows the use of the isotope doublets as mass spectrometric &quot;signatures.&quot; A software program was developed that permits automatic cross-link identification and assignment of the cross-link type. Furthermore photocleavage of BiPS assists in cross-link identification by allowing separate tandem mass spectrometry sequencing of each peptide comprising the original cross-link. By combining the use of BiPS with MS, we have provided the first direct evidence for the docking site of a phosphorylated G-protein coupled receptor C terminus on the multifunctional adaptor protein &beta;-arrestin, clearly demonstrating the broad potential and application of this novel cross-linker in structural and cellular biology.'),(683,'mpyrdt_psfb09','Meissner, G., Pasek, D. A., Yamaguchi, N., Ramachandran, S., Dokholyan, N. V., and Tripathy, A.,','Thermodynamics of calmodulin binding to cardiac and skeletal muscle ryanodine receptor ion channels','Proteins: Structure, Function, and Bioinformatics',74,'207','211',2009,'Proteins: Transmembrane','Meissner, G.','no','no','article',18618700,'',NULL,NULL,NULL,'The skeletal muscle (RyR1) and cardiac muscle (RyR2) ryanodine receptor calcium release channels contain a single, conserved calmodulin (CaM) binding domain, yet are differentially regulated by CaM. Here, we report that high-affinity [<sup>35</sup>S]CaM binding to RyR1 is driven by favorable enthalpic and entropic contributions at Ca<sup>2+</sup> concentrations from &lt;0.01 to 100 &micro;M. At 0.15 &micro;M Ca<sup>2+</sup>, [<sup>35</sup>S]CaM bound to RyR2 with decreased affinity and binding enthalpy compared with RyR1. The rates of [<sup>35</sup>S]CaM dissociation from RyR1 increased as the temperature was raised, whereas at 0.15 &micro;M Ca<sup>2+</sup> the rate from RyR2 was little affected. The results suggest major differences in the energetics of CaM binding to and dissociation from RyR1 and RyR2.'),(684,'tsde_jcp09','Tsygankov, D., Serohijos, A. W. R., Dokholyan, N. V., and Elston, T. C., ','Kinetic models for the coordinated stepping of cytoplasmic dynein','Journal of Chemical Physics',130,'25101','',2009,'Proteins: Mechanism','Elston, T. C.','no','no','article',19154055,'',NULL,NULL,NULL,'To generate processive motion along a polymer track requires that motor proteins couple their ATP hydrolysis cycle with conformational changes in their structural subunits. Numerous experimental and theoretical efforts have been devoted to establishing how this chemomechanical coupling occurs. However, most processive motors function as dimers. Therefore a full understanding of the motor&#146;s performance also requires knowledge of the coordination between the chemomechanical cycles of the two heads. We consider a general two-headed model for cytoplasmic dynein that is built from experimental measurements on the chemomechanical states of monomeric dynein. We explore different possible scenarios of coordination that simultaneously satisfy two main requirements of the dimeric protein: high processivity (long run length) and high motor velocity (fast ATP turnover). To demonstrate the interplay between these requirements and the necessity for coordination, we first develop and analyze a simple mechanical model for the force-induced stepping in the absence of ATP. Next we use a simplified model of dimeric dynein&#146;s chemomechanical cycle to establish the kinetic rules that must be satisfied for the model to be consistent with recent data for the motor&#146;s performance from single molecule experiments. Finally, we use the results of these investigations to develop a full model for dimeric dynein&#146;s chemomechanical cycle and analyze this model to make experimentally testable predictions.'),(618,'ddbss_jmb02','Ding, F., Dokholyan, N. V., Buldyrev, S. V., Stanley, H. E., and Shakhnovich, E. I., ','Molecular dynamic simulation of the SH3 domain aggregation suggests a generic amyloidogenesis mechanism','Journal of Molecular Biology',324,'851','857',2002,'Proteins: Aggregation','Dokholyan, N. V.','no','no','article',12460582,'',NULL,NULL,NULL,'We use molecular dynamics simulation to study the aggregation of Src SH3 domain proteins. For the case of two proteins, we observe two possible aggregation conformations: the closed form dimer and the open aggregation state. The closed dimer is formed by &quot;domain swapping&quot; &#8212; the two proteins exchange their RT-loops. All the hydrophobic residues are buried inside the dimer so proteins cannot further aggregate into elongated amyloid fibrils. We find that the open structure &#8212; stabilized by backbone hydrogen bond interactions &#8212; packs the RT-loops together by swapping the two strands of the RT-loop. The packed RT-loops form a &beta;-sheet structure and expose the backbone to promote further aggregation. We also simulate more than two proteins, and find that the aggregate adopts a fibrillar double &beta;-sheet structure, which is formed by packing the RT-loops from different proteins. Our simulations are consistent with a possible generic amyloidogenesis scenario.'),(619,'sdds_jmb03','Shakhnovich, B. E., Dokholyan, N. V., DeLisi, C., and Shakhnovich, E. I., ','Functional fingerprints of folds: Evidence for correlated structure-function evolution','Journal of Molecular Biology',326,'1','9',2003,'Proteins: Evolution','Shakhnovich, E. I.','no','no','article',12547186,'',NULL,NULL,NULL,'Using structural similarity clustering of protein domains: protein domain universe graph (PDUG), and a hierarchical functional annotation: gene ontology (GO) as two evolutionary lenses, we find that each structural cluster (domain fold) exhibits a distribution of functions that is unique to it. These functional distributions are functional fingerprints that are specific to characteristic structural clusters and vary from cluster to cluster. Furthermore, as structural similarity threshold for domain clustering in the PDUG is relaxed we observe an influx of earlier-diverged domains into clusters. These domains join clusters without destroying the functional fingerprint. These results can be understood in light of a divergent evolution scenario that posits correlated divergence of structural and functional traits in protein domains from one or few progenitors.'),(620,'dbbsd_psfg03','Ding, F., Borreguero, J. M., Buldyrev, S. V., Stanley, H. E., and Dokholyan, N. V., ','Mechanism for the &alpha;-Helix to &beta;-Hairpin Transition','Proteins: Structure, Function, and Genetics',53,'220','228',2003,'Proteins: Folding','Ding, F.','no','no','article',14517973,'',NULL,NULL,NULL,'The aggregation of &alpha;-helix-rich proteins into &beta;-sheet-rich amyloid fibrils is associated with fatal diseases, such as Alzheimer&#39;s disease and prion disease. During an aggregation process, protein secondary structure elements &#8212; &alpha;-helices &#8212; undergo conformational changes to &beta;-sheets. The fact that proteins with different sequences and structures undergo a similar transition on aggregation suggests that the sequence nonspecific hydrogen bond interaction among protein backbones is an important factor. We perform molecular dynamics simulations of a polyalanine model, which is an &alpha;-helix in its native state and observe a metastable &beta;-hairpin intermediate. Although a &beta;-hairpin has larger potential energy than an &alpha;-helix, the entropy of a &beta;-hairpin is larger because of fewer constraints imposed by the hydrogen bonds. In the vicinity of the transition temperature, we observe the interconversion of the &alpha;-helix and &beta;-sheet states via a random coil state. We also study the effect of the environment by varying the relative strength of side-chain interactions for a designed peptide &#8212; an &alpha;-helix in its native state. For a certain range of side-chain interaction strengths, we find that the intermediate &beta;-hairpin state is destabilized and even disappears, suggesting an important role of the environment in the aggregation propensity of a peptide. Proteins 2003;53:220228.'),(621,'dds_bj03','Deeds, E. J., Dokholyan, N. V., and Shakhnovich, E. I., ','Protein evolution within a structural space','Biophysical Journal',85,'2962','2972',2003,'Proteins: Evolution','Shakhnovich, E. I.','no','no','article',14581198,'',NULL,NULL,NULL,'Understanding of the evolutionary origins of protein structures represents a key component of the understanding of molecular evolution as a whole. Here we seek to elucidate how the features of an underlying protein structural &quot;space&quot; might impact protein structural evolution. We approach this question using lattice polymers as a completely characterized model of this space. We develop a measure of structural comparison of lattice structures that is analogous to the one used to understand structural similarities between real proteins. We use this measure of structural relatedness to create a graph of lattice structures and compare this graph (in which nodes are lattice structures and edges are defined using structural similarity) to the graph obtained for real protein structures. We find that the graph obtained from all compact lattice structures exhibits a distribution of structural neighbors per node consistent with a random graph. We also find that subgraphs of 3500 nodes chosen either at random or according to physical constraints also represent random graphs. We develop a divergent evolution model based on the lattice space which produces graphs that, within certain parameter regimes, recapitulate the scale-free behavior observed in similar graphs of real protein structures.'),(623,'pdsg_pnas03','Pei, J., Dokholyan, N. V., Shakhnovich, E. I., and Grishin, N. V., ','Using protein design for homology detection and active site searches','Proceedings of the National Academy of Sciences USA',100,'11361','11366',2003,'Proteins: Evolution','Grishin, N. V.','yes','no','article',12975528,'',NULL,NULL,NULL,'We describe a method of designing artificial sequences that resemble naturally occurring sequences in terms of their compatibility with a template structure and its functional constraints. The design procedure is a Monte Carlo simulation of amino acid substitution process. The selective fixation of substitutions is dictated by a simple scoring function derived from the template structure and a multiple alignment of its homologs. Designed sequences represent an enlargement of sequence space around native sequences. We show that the use of designed sequences improves the performance of profile-based homology detection. The difference in position-specific conservation between designed sequences and native sequences is helpful for prediction of functionally important residues. Our sequence selection criteria in evolutionary simulations introduce amino acid substitution rate variation among sites in a natural way, providing a better model to test phylogenetic methods.'),(624,'kdd_jmb03','Khare, S. D., Ding, F., and Dokholyan, N. V., ','Folding of Cu, Zn superoxide dismutase and Familial Amyotrophic Lateral Sclerosis','Journal of Molecular Biology',334,'515','525',2003,'Proteins: Folding','Dokholyan, N. V.','no','no','article',14623191,'',NULL,NULL,NULL,'Cu, Zn superoxide dismutase (SOD1) has been implicated in the familial form of the neurodegenerative disease amyotrophic lateral sclerosis (ALS). It has been suggested that mutant mediated SOD1 misfolding/aggregation is an integral part of the pathology of ALS. We study the folding thermodynamics and kinetics of SOD1 using a hybrid molecular dynamics approach. We reproduce the experimentally observed SOD1 folding thermodynamics and find that the residues which contribute the most to SOD1 thermal stability are also crucial for apparent two-state folding kinetics. Surprisingly, we find that these residues are located on the surface of the protein and not in the hydrophobic core. Mutations in some of the identified residues are found in patients with the disease. We argue that the identified residues may play an important role in aggregation. To further characterize the folding of SOD1, we study the role of cysteine residues in folding and find that non-native disulfide bond formation may significantly alter SOD1 folding dynamics and aggregation propensity.'),(625,'d_psfb04','Dokholyan, N. V., ','What is the protein design alphabet?','Proteins: Structure, Function, and Bioinformatics',54,'622','628',2004,'Proteins: Design','Dokholyan, N. V.','no','no','article',14997558,'',NULL,NULL,NULL,'Selecting a protein sequence that corresponds to a specific three-dimensional protein structure is known as the protein design problem. One principal bottleneck in solving this problem is our lack of knowledge of precise atomic interactions. Using a simple model of amino acid interactions, we determine three crucial factors that are important for solving the <i>protein design problem</i>. Among these factors is the <i>protein alphabet</i> &#8212; a set of sequence elements that encodes protein structure. Our model predicts that alphabet size is independent of protein length, suggesting the possibility of designing a protein of arbitrary length with the natural protein alphabet. We also find that protein alphabet size is governed by protein structural properties and the energetic properties of the protein alphabet units. We discover that the usage of average types of amino acid in proteins is less than expected if amino acids were chosen randomly with naturally occurring frequencies. We propose three possible scenarios that account for amino acid underusage in proteins. These scenarios suggest the possibility that amino acids themselves might not constitute the alphabet of natural proteins. Proteins 2004;54:622628.'),(626,'tsds_pnas04','Tiana, G., Shakhnovich, B. E., Dokholyan, N. V., and Shakhnovich, E. I., ','Imprint of evolution on protein structures','Proceedings of the National Academy of Sciences USA',101,'2846','2851',2004,'Proteins: Evolution','Shakhnovich, E. I.','yes','no','article',14970345,'',NULL,NULL,NULL,'We attempt to understand the evolutionary origin of protein folds by simulating their divergent evolution with a three-dimensional lattice model. Starting from an initial seed lattice structure, evolution of model proteins progresses by sequence duplication and subsequent point mutations. A new gene&#39;s ability to fold into a stable and unique structure is tested each time through direct kinetic folding simulations. Where possible, the algorithm accepts the new sequence and structure and thus a &quot;new protein structure&quot; is born. During the course of each run, this model evolutionary algorithm provides several thousand new proteins with diverse structures. Analysis of evolved structures shows that later evolved structures are more designable than seed structures as judged by recently developed structural determinant of protein designability, as well as direct estimate of designability for selected structures by thermodynamic sampling of their sequence space. We test the significance of this trend predicted on lattice models on real proteins and show that protein domains that are found in eukaryotic organisms only feature statistically significant higher designability than their prokaryotic counterparts. These results present a fundamental view on protein evolution highlighting the relative roles of structural selection and evolutionary dynamics on genesis of modern proteins.'),(627,'kkd_jmb04','Khatun, J., Khare, S. D., and Dokholyan, N. V., ','Can contact potentials reliably predict stability of proteins?','Journal of Molecular Biology',336,'1223','1238',2004,'Proteins: Design','Dokholyan, N. V.','no','no','article',15037081,'',NULL,NULL,NULL,'The simplest approximation of interaction potential between amino acid residues in proteins is the contact potential, which defines the effective free energy of a protein conformation by a set of amino acid contacts formed in this conformation. Finding a contact potential capable of predicting free energies of protein states across a variety of protein families will aid protein folding and engineering in silico on a computationally tractable time-scale. We test the ability of contact potentials to accurately and transferably (across various protein families) predict stability changes of proteins upon mutations. We develop a new methodology to determine the contact potentials in proteins from experimental measurements of changes in proteins thermodynamic stabilities (&Delta;&Delta;G) upon mutations. We apply our methodology to derive sets of contact interaction parameters for a hierarchy of interaction models including solvation and multi-body contact parameters. We test how well our models reproduce experimental measurements by statistical tests. We evaluate the maximum accuracy of predictions obtained by using contact potentials and the correlation between parameters derived from different data-sets of experimental (&Delta;&Delta;G) values. We argue that it is impossible to reach experimental accuracy and derive fully transferable contact parameters using the contact models of potentials. However, contact parameters may yield reliable predictions of &Delta;&Delta;G for datasets of mutations confined to the same amino acid positions in the sequence of a single protein.'),(628,'pdubcsd_pre04','Peng, S., Ding, F., Urbanc, B., Buldyrev, S. V., Cruz, L., Stanley, H. E., and Dokholyan, N. V., ','Discrete molecular dynamics simulations of peptide aggregation','Physical Review E',69,'41908','',2004,'Proteins: Aggregation','Peng, S.','no','no','article',15169044,'',NULL,NULL,NULL,'We study the aggregation of peptides using the discrete molecular dynamics simulations. Specifically, at temperatures above the &alpha;-helix melting temperature of a single peptide, the model peptides aggregate into a multilayer parallel &beta;-sheet structure. This structure has an interstrand distance of 4.8 A and an intersheet distance of 10 A, which agree with experimental observations. Our model explains these results as follows: hydrogen-bond interactions give rise to the interstrand spacing in &beta; sheets, while G&otilde; interactions between side chains make &beta; strands parallel to each other and allow &beta; sheets to pack into layers. An important feature of our results is that the aggregates contain free edges, which may allow for further aggregation of model peptides to form elongated fibrils.'),(629,'bdbsd_bj04','Borreguero, J. M., Ding, F., Buldyrev, S. V., Stanley, H. E., and Dokholyan, N. V., ','Multiple folding pathways of the SH3 domain','Biophysical Journal',87,'521','533',2004,'Proteins: Folding','Borreguero, J. M.','no','no','article',15240485,'',NULL,NULL,NULL,'Experimental observations suggest that proteins follow different folding pathways under different environmental conditions. We perform molecular dynamics simulations of a model of the c-Crk SH3 domain over a broad range of temperatures, and identify distinct pathways in the folding transition. We determine the kinetic partition temperature &#8212; the temperature for which the c-Crk SH3 domain undergoes a rapid folding transition with minimal kinetic barriers &#8212; and observe that below this temperature the model protein may undergo a folding transition by multiple folding pathways via only one or two intermediates. Our findings suggest the hypothesis that the SH3 domain, a protein fold for which only two-state folding kinetics was observed in previous experiments, may exhibit intermediate states under conditions that strongly stabilize the native state.'),(630,'tdbs_jcp04','Tiana, G., Dokholyan, N. V., Broglia, R. A., and Shakhnovich, E. I., ','The evolution dynamics of model proteins','Journal of Chemical Physics',121,'2381','2389',2004,'Proteins: Evolution','','no','no','article',15260793,'',NULL,NULL,NULL,'Explicit simulations of protein evolution, where protein chains are described at a molecular, although simplified, level provide important information to understand the similarities found to exist between known proteins. The results of such simulations suggest that a number of evolutionary-related quantities, such as the distribution of sequence similarity for structurally similar proteins, are controlled by evolutionary kinetics and do not reflect an equilibrium state. An important result for phylogeny is that a subset of the residues of each protein evolve on a much larger time scale than the other residues.'),(631,'ucdskbsd_bj04','Urbanc, B., Cruz, L., Ding, F., Sammond, D., Khare, S. D., Buldyrev, S. V., Stanley, H. E., and Dokholyan, N. V., ','Molecular dynamics simulation of Amyloid beta dimer formation','Biophysical Journal',87,'2310','2321',2004,'Proteins: Aggregation','Urbanc, B.','no','no','article',15454432,'',NULL,NULL,NULL,'Recent experiments with amyloid &beta; (A&beta;) peptide indicate that formation of toxic oligomers may be an important contribution to the onset of Alzheimer&#39;s disease. The toxicity of A&beta; oligomers depends on their structure, which is governed by assembly dynamics. Due to limitations of current experimental techniques, a detailed knowledge of oligomer structure at the atomic level is missing. We introduce a molecular dynamics approach to study A&beta; dimer formation. 1), We use discrete molecular dynamics simulations of a coarse-grained model to identify a variety of dimer conformations; and 2), we employ all-atom molecular mechanics simulations to estimate thermodynamic stability of all dimer conformations. Our simulations of a coarse-grained A&beta; peptide model predicts 10 different planar &beta;-strand dimer conformations. We then estimate the free energies of all dimer conformations in all-atom molecular mechanics simulations with explicit water. We compare the free energies of A&beta;(1-42) and A&beta;(1-40) dimers. We find that 1), dimer conformations have higher free energies compared to their corresponding monomeric states; and 2), the free-energy difference between the A&beta;(1-42) and the corresponding A&beta;(1-40) dimer conformation is not significant. Our results suggest that A&beta; oligomerization is not accompanied by the formation of thermodynamically stable planar &beta;-strand dimers.'),(632,'kcd_pnas04','Khare, S. D., Caplow, M., and Dokholyan, N. V., ','The rate and equilibrium constants for a multi-step reaction sequence for the aggregation of superoxide dismutase in ALS','Proceedings of the National Academy of Sciences USA',101,'15094','15099',2004,'Proteins: Aggregation','Dokholyan, N. V., Caplow, M.','yes','no','article',15475574,'',NULL,NULL,NULL,'Mutation-induced aggregation of the dimeric enzyme Cu, Zn superoxide dismutase 1 (SOD1) has been implicated in the familial form of the disease amyotrophic lateral sclerosis, but the mechanism of aggregation is not known. Here, we show that <i>in vitro</i> SOD1 aggregation is a multistep reaction that minimally consists of dimer dissociation, metal loss from the monomers, and oligomerization of the apo-monomers:<p align=center><img width=300 height=80 border=0 src=images/kcd_pnas04.png></p> where <i>D<sub>holo</sub>,M<sub>holo</sub>,M<sub>apo</sub>,</i> and <i>A</i> are the holo-dimer, holo-monomer, apo-monomer, and aggregate, respectively. Under aggregation-promoting conditions (pH 3.5), the rate and equilibrium constants corresponding to each step are: (i) dimer dissociation, <i>K<sub>d</sub></i> &#8776; 1 &micro;M; <i>k<sub>off</sub></i> &#8776; 1 &#215; 10<sup>&#8722;3</sup> s<sup>&#8722;1</sup>, <i>k<sub>on</sub></i> &#8776; 1 &#215; 10<sup>3</sup> M<sup>&#8722;1</sup> &#8901; s<sup>&#8722;1</sup>; (ii) metal loss, <i>K<sub>m</sub></i> &#8776; 0.1 &micro;M, <i>k<sub>m</sub><sup>&#8722;</sup></i> &#8776; 1 &#215; 10<sup>&#8722;3</sup> s<sup>&#8722;1</sup>, <i>k<sub>m</sub><sup>+</sup></i> &#8776; 1 &#215; 10<sup>4</sup> M<sup>&#8722;1</sup> &#8901; s<sup>&#8722;1</sup>; and (iii) assembly (rate-limiting step), <i>k<sub>agg</i> &#8776; 1 &#215; 10<sup>3</sup> M<sup>&#8722;1</sup> &#8901; s<sup>&#8722;1</sup>. In contrast, under near-physiological conditions (pH 7.8), where aggregation is drastically reduced, dimer dissociation is less thermodynamically favorable: <i>K<sub>d</sub></i> &#8776; 0.1 nM, and extremely slow: <i>k<sub>off</sub></i> &#8776; 3 &#215; 10<sup>&#8722;5</sup> s<sup>&#8722;1</sup>, <i>k<sub>on</sub></i> &#8776; 3 &#215; 10<sup>5</sup> M<sup>&#8722;1</sup> &#8901; s<sup>&#8722;1</sup>. Our results suggest that familial amyotrophic lateral sclerosis-linked SOD1 aggregation occurs by a mutation-induced increase in dimer dissociation and/or increase in apomonomer formation.'),(633,'dcdkpscd_s04','Dixon, R. D. S., Chen, Y., Ding, F., Khare, S. D. , Prutzman, K. C., Schaller, M. D., Campbell, S. L., and Dokholyan, N. V., ','New insights into FAK signaling and localization based on detection of a FAT domain folding intermediate','Structure',12,'2161','2171',2004,'Proteins: Folding','Dokholyan, N. V., Campbell, S. L.','yes','no','article',15576030,'',NULL,NULL,NULL,'Mounting evidence suggests that the focal adhesion targeting (FAT) domain, an antiparallel four-helix bundle, exists in alternative conformations that may modulate phosphorylation, ligand binding, and the subcellular localization of focal adhesion kinase (FAK). In order to characterize the conformational dynamics of the FAT domain, we have developed a novel method for reconstructing the folding pathway of the FAT domain by using discrete molecular dynamics (DMD) simulations, with free energy constraints derived from NMR hydrogen exchange data. The DMD simulations detect a folding intermediate, in which a cooperative unfolding event causes helix 1 to lose helical character while separating from the helix bundle. The conformational dynamic features of helix 1 in the intermediate state of the FAT domain are likely to facilitate Y926 phosphorylation , yet interfere with paxillin binding. The presence of this intermediate state in vivo may promote FAK signaling via the ERK/MAPK pathway and by release of FAK from focal adhesions.'),(634,'dbd_bj05','Ding, F., Buldyrev, S. V., and Dokholyan, N. V., ','Folding Trp-cage to NMR resolution native structure using a coarse-grained protein model','Biophysical Journal',88,'147','155',2005,'Proteins: Folding','Dokholyan, N. V., Ding, F.','yes','no','article',15533926,'',NULL,NULL,NULL,'We develop a coarse-grained protein model with a simplified amino acid interaction potential. Using this model, we perform discrete molecular dynamics folding simulations of a small 20-residue proteinTrp-cagefrom a fully extended conformation. We demonstrate the ability of the Trp-cage model to consistently reach conformations within 2-A backbone root-mean-square distance from the corresponding NMR structures. The minimum root-mean-square distance of Trp-cage conformations in simulations can be &lt;1 A . Our findings suggest that, at least in the case of Trp-cage, a detailed all-atom protein model with a molecular mechanics force field is not necessary to reach the native state of a protein. Our results also suggest that the success of folding Trp-cage in our simulations and in the reported all-atom molecular mechanics simulation studies may be mainly due to the special stabilizing features specific to this miniprotein.'),(635,'d_g05','Dokholyan, N. V., ','The architecture of the protein domain universe','Gene',347,'199','205',2005,'Proteins: Evolution','Dokholyan, N. V.','no','no','article',15777630,'',NULL,NULL,NULL,'Understanding the design of the universe of protein structures may provide insights into protein evolution. We study the architecture of the protein domain universe, which has been found to poses peculiar scale-free properties [Dokholyan, N.V., Shakhnovich, B., Shakhnovich, E.I., 2002b. Expanding protein universe and its origin from the biological Big Bang. Proceedings of the National Academy of Sciences of the United States of America 99, 1413214136]. We examine the origin of these scale-free properties of the graph of protein domain structures (PDUG) and determine that that the PDUG is not modular, i.e. it does not consist of modules with uniform properties. Instead, we find the PDUG to be self-similar at all scales. We further characterize the PDUG architecture by studying the properties of the hub nodes that are responsible for the scale-free connectivity of the PDUG. We introduce a measure of the betweenness centrality of protein domains in the PDUG and find a power-law distribution of the betweenness centrality values. The scale-free distribution of hubs in the protein universe suggests that a set of specific statistical mechanics models, such as the self-organized criticality model, can potentially identify the principal driving forces of protein evolution. We also find a gatekeeper protein domain, removal of which partitions the largest cluster into two large sub-clusters. We suggest that the loss of such gatekeeper protein domains in the course of evolution is responsible for the creation of new fold families.'),(636,'djd_s05','Ding, F., Jha, R. K., and Dokholyan, N. V., ','Scaling behavior and structure of denatured proteins','Structure',13,'1047','1054',2005,'Proteins: Folding','Ding, F.','no','yes','article',16004876,'',NULL,NULL,NULL,'An ensemble of random-coil conformations with no persistent structures has long been accepted as the classical model of denatured proteins due to its consistency with the experimentally determined scaling of protein sizes. However, recent NMR spectroscopy studies on proteins at high chemical denaturant concentrations suggest the presence of significant amounts of native-like structures, in contrast to the classical random-coil picture. To reconcile these seemingly controversial observations, we examine thermally denatured states of experimentally characterized proteins by using molecular dynamics simulations. For all studied proteins, we find that denatures states indeed have strong local conformational bias toward native states while a random-coil power law scaling of protein sizes is preserved. In addition, we explain why experimentally determined size of the protein creatine kinase does not follow general scaling. In simulations, we observe that this protein exhibits a stable intermediate state, the size of which is consistent with the reported experimental observation.'),(637,'dd_tb05','Ding, F., and Dokholyan, N. V., ','Simple but predictive protein models','Trends in Biotechnology',23,'450','455',2005,'Proteins: Folding','Dokholyan, N. V.','no','no','article',16038997,'',NULL,NULL,NULL,'The traditional approach to computational biophysics studies of molecular systems is brute force molecular dynamics simulations under the conditions of interest. The disadvantages of this approach are that the time and length scales that are accessible to computer simulations often do not reach biologically relevant scales. An alternative approach, which we call intuitive modeling, is hypothesis-driven and based on tailoring simplified protein models to the systems of interest. Using intuitive modeling, the length and time scales that can be achieved using simplified protein models exceed those of traditional molecular-dynamic simulations. Here, we describe several recent studies that signify the predictive power of simplified protein models within the intuitive-modeling approach.'),(638,'kwgd_psfb05','Khare, S. D., Wilcox, K. C., Gong, P., and Dokholyan, N. V., ','Sequence and structural determinants of Cu, Zn superoxide dismutase aggregation','Proteins: Structure, Function, and Bioinformatics',61,'617','632',2005,'Proteins: Aggregation','Dokholyan, N. V.','no','no','article',16152647,'',NULL,NULL,NULL,'Diverse point mutations in the enzyme Cu, Zn superoxide dismutase (SOD1) are linked to its aggregation in the familial form of the disease amyotrophic lateral sclerosis. The disease-associated mutations are known to destabilize the protein, but the structural basis of the aggregation of the destabilized protein and the structure of aggregates are not well understood. Here, we investigate in silico the sequence and structural determinants of SOD1 aggregation: (1) We identify sequence fragments in SOD1 that have a high aggregation propensity, using only the sequence of SOD1, and (2) we perform molecular dynamics simulations of the SOD1 dimer folding and misfolding. In both cases, we identify identical regions of the protein as having high propensity to form intermolecular interactions. These regions correspond to the N- and C-termini, and two crossover loops and two beta-strands in the Greek-key native fold of SOD1. Our results suggest that the high aggregation propensity of mutant SOD1 may result from a synergy of two factors: the presence of highly amyloidogenic sequence fragments (&quot;hot spots&quot;), and the presence of these fragments in regions of the protein that are structurally most likely to form intermolecular contacts under destabilizing conditions. Therefore, we postulate that the balance between the self-association of aggregation-prone sequences and the specific structural context of these sequences in the native state determines the aggregation propensity of proteins.'),(639,'dgdss_jmb05','Ding, F., Guo, W. H., Dokholyan, N. V., Shakhnovich, E. I., and Shea, J.-E., ','Reconstruction of src SH3 transition state ensemble using multiscale molecular dynamics simulations','Journal of Molecular Biology',350,'1035','1050',2005,'Proteins: Folding','J.-E. Shea','no','no','article',15982666,'',NULL,NULL,NULL,'We use an integrated computational approach to reconstruct accurately the transition state ensemble (TSE) for folding of the src-SH3 protein domain. We first identify putative TSE conformations from free energy surfaces generated by importance sampling molecular dynamics for a fully atomic, solvated model of the src-SH3 protein domain. These putative TSE conformations are then subjected to a folding analysis using a coarse-grained representation of the protein and rapid discrete molecular dynamics simulations. Those conformations that fold to the native conformation with a probability (<i>P<sub>fold</sub></i>) of approximately 0.5, constitute the true transition state. Approximately 20% of the putative TSE structures were found to have a <i>P<sub>fold</sub></i> near 0.5, indicating that, although correct TSE conformations are populated at the free energy barrier, there is a critical need to refine this ensemble. Our simulations indicate that the true TSE conformations are compact, with a well-defined central &beta; sheet, in good agreement with previous experimental and theoretical studies. A structured central &beta; sheet was found to be present in a number of pre-TSE conformations, however, indicating that this element, although required in the transition state, does not define it uniquely. An additional tight cluster of contacts between highly conserved residues belonging to the diverging turn and second &beta;-sheet of the protein emerged as being critical elements of the folding nucleus. A number of commonly used order parameters to identify the transition state for folding were investigated, with the number of native C<sup>&beta;</sup> contacts displaying the most satisfactory correlation with <i>P<sub>fold</sub></i> values.'),(640,'kdgd_plscb05','Khare, S. D., Ding, F., Gwanmesia, K. N., and Dokholyan, N. V., ','Molecular origin of polyglutamine-mediated protein aggregation in neurodegenerative diseases','Public Library of Science Computational Biology',1,'e30','',2005,'Proteins: Aggregation','Dokholyan, N. V.','no','yes','article',16158094,'',NULL,NULL,NULL,'Expansion of polyglutamine (polyQ) tracts in proteins results in protein aggregation and is associated with cell death in at least nine neurodegenerative diseases. Disease age of onset is correlated with the polyQ insert length above a critical value of 35-40 glutamines. The aggregation kinetics of isolated polyQ peptides in vitro also shows a similar critical-length dependence. While recent experimental work has provided considerable insights into polyQ aggregation, the molecular mechanism of aggregation is not well understood. Here, using computer simulations of isolated polyQ peptides, we show that a mechanism of aggregation is the conformational transition in a single polyQ peptide chain from random coil to a parallel &beta;-helix. This transition occurs selectively in peptides longer than 37 glutamines. In the &beta;-helices observed in simulations, all residues adopt &beta;-strand backbone dihedral angles, and the polypeptide chain coils around a central helical axis with 18.5 &#177; 2 residues per turn. We also find that mutant polyQ peptides with proline-glycine inserts show formation of antiparallel b-hairpins in their ground state, in agreement with experiments. The lower stability of mutant &beta;-helices explains their lower aggregation rates compared to wild type. Our results provide a molecular mechanism for polyQ-mediated aggregation.'),(641,'cd_jmb05','Chen, Y., and Dokholyan, N. V., ','A single disulfide bond differentiates aggregation pathways of &beta;2-microglobulin','Journal of Molecular Biology',354,'473','482',2005,'Proteins: Aggregation','Dokholyan, N. V.','no','no','article',16242719,'',NULL,NULL,NULL,'Deposition of wild-type &beta;2-microglobulin (&beta;2m) into amyloid fibrils is a complication in patients undergoing long-term hemodialysis. The native &beta;-sandwich fold of &beta;2m has a highly conserved disulfide bond linking Cys25 and Cys80. Oxidized &beta;2m forms needle-like amyloid fibrils at pH 2.5 in vitro, whereas reduced &beta;2m, at acid pH, in which the intra-chain disulfide bond is disrupted, cannot form typical fibrils. Instead, reduced &beta;2mforms thinner and more flexible filaments. To uncover the difference in molecular mechanisms underlying the aggregation of the oxidized and reduced &beta;2m, we performed molecular dynamics simulations of &beta;2m oligomerization under oxidized and reduced conditions. We show that, consistent with experimental observations, the oxidized &beta;2m forms domain-swapped dimer, in which the two proteins exchange their N-terminal segments complementing each other. In contrast, both dimers and trimers, formed by reduced &beta;2m, are comprised of parallel &beta;-sheets between monomers and stabilized by the hydrogen bond network along the backbone. The oligomerized monomers are in extended conformations, capable of further aggregation. We find that both reduced and oxidized dimers are thermodynamically less stable than their corresponding monomers, indicating that &beta;2m oligomerization is not accompanied by the formation of a thermodynamically stable dimer. Our studies suggest that the different aggregation pathways of oxidized and reduced &beta;2m are dictated by the formation of distinct precursor oligomeric species that are modulated by Cys25-Cys80 disulfide-bonds. We propose that the propagation of domain swapping is the aggregation mechanism for the oxidized &beta;2m, while &quot;parallel stacking&quot; of partially unfolded &beta;2m is the aggregation mechanism for the reduced &beta;2m.'),(642,'dld_jbc05','Ding, F., Larocque, J. J., and Dokholyan, N. V., ','Direct observation of protein folding, aggregation, and a prion-like conformational conversion','Journal of Biological Chemistry',280,'40235','40240',2005,'Proteins: Aggregation','Dokholyan, N. V.','no','no','article',16204250,'',NULL,NULL,NULL,'Protein conformational transition from &alpha;-helices to &beta;-sheets precedes aggregation of proteins implicated in many diseases, including Alzheimer and prion diseases. Direct characterization of such transitions is often hindered by the complicated nature of the interaction network among amino acids. A recently engineered small protein-like peptide with a simple amino acid composition features a temperature-driven &alpha;-helix to &beta;-sheet conformational change. Here we studied the conformational transition of this peptide by molecular dynamics simulations. We observed a critical temperature, below which the peptide folds into a.dd &alpha;-helical coiled-coil state and above which the peptide misfolds into &beta;-rich structures with a high propensity to aggregate. The structures adopted by this peptide during low temperature simulations have a backbone root mean square deviation less than 2 A from the crystal structure. At high temperatures, this peptide adopts an amyloid-like structure, which is mainly composed of coiled anti-parallel &beta;-sheets with the cross-&beta;-signature of amyloid fibrils. Most strikingly, we observed conformational conversions in which an &alpha;-helix is converted into a &beta;-strand by proximate stable &beta;-sheets with exposed hydrophobic surfaces and unsaturated hydrogen bonds. Our study suggested a possible generic molecular mechanism of the template-mediated aggregation process, originally proposed by Prusiner (Prusiner, S. B. (1998) Proc. Natl. Acad. Sci. U. S. A. 95, 13363-13383) to account for prion infectivity.'),(643,'dpcd_s06','Ding, F., Prutzman, K. C., Campbell, S. L., and Dokholyan, N. V., ','Topological determinants of protein domain swapping','Structure',14,'5','14',2006,'Proteins: Aggregation','Dokholyan, N. V.','no','no','article',16407060,'',NULL,NULL,NULL,'Protein domain swapping has been repeatedly observed in a variety of proteins and is believed to result from destabilization due to mutations or changes in environment. Based on results from our studies and others, we propose that structures of the domain-swapped proteins are mainly determined by their native topologies. We performed molecular dynamics simulations of seven different proteins, known to undergo domain swapping experimentally, under mildly denaturing conditions and found in all cases that the domain-swapped structures can be recapitulated by using protein topology in a simple protein model. Our studies further indicated that, in many cases, domain swapping occurs at positions around which the protein tends to unfold prior to complete unfolding. This, in turn, enabled prediction of protein structural elements that are responsible for domain swapping. In particular, two distinct domain-swapped dimer conformations of the focal adhesion targeting domain of focal adhesion kinase were predicted computationally and were supported experimentally by data obtained from NMR analyses.'),(644,'d_cosb06','Dokholyan, N. V., ','Studies of folding and misfolding using simplified models','Current Opinions in Structural Biology',16,'79','85',2006,'Proteins: Folding','Dokholyan, N. V.','no','no','article',16413773,'',NULL,NULL,NULL,'Computer simulations are as vital to our studies of biological systems as experiments. They bridge and rationalize experimental observations, extend the experimental &#39;field of view&#39;, which is often limited to a specific time or length scale, and, most importantly, provide novel insights into biological systems, offering hypotheses about yet-to-be uncovered phenomena. These hypotheses spur further experimental discoveries. Simplified molecular models have a special place in the field of computational biology. Branded as less accurate than all-atom protein models, they have offered what all-atom molecular dynamics simulations could not &#8212; the resolution of the length and time scales of biological phenomena. Not only have simplified models proven to be accurate in explaining or reproducing several biological phenomena, they have also offered a novel multiscale computational strategy for accessing a broad range of time and length scales upon integration with traditional all-atom simulations. Recent computer simulations of simplified models have shaken or advanced the established understanding of biological phenomena. It was demonstrated that simplified models can be as accurate as traditional molecular dynamics approaches in identifying native conformations of proteins. Their application to protein structure prediction yielded phenomenal accuracy in recapitulating native protein conformations. New studies that utilize the synergy of simplified protein models with all-atom models and experiments yielded novel insights into complex biological processes, such as protein folding, aggregation and the formation of large protein complexes.'),(645,'ddwcydycb_pnas06','Demidov, V. V., Dokholyan, N. V., Witte-Hoffman, C., Chalasani, P., Yiu, H.-W., Ding, F., Yu, Y., Cantor, C. R., and Broude, N. E., ','Fast complementation of split fluorescent protein triggered by DNA hybridization','Proceedings of the National Academy of Sciences USA',103,'2052','2056',2006,'Biotechnology','Broude, N. E. & Demidov, V. V.','yes','no','article',16461889,'',NULL,NULL,NULL,'Fluorescent proteins have proven to be excellent reporters and biochemical sensors with a wide range of applications. In a split form, they are not fluorescent, but their fluorescence can be restored by supplementary proteinprotein or proteinnucleic acid interactions that reassemble the split polypeptides. However, in prior studies, it took hours to restore the fluorescence of a split fluorescent protein because the formation of the protein chromophore slowly occurred de novo concurrently with reassembly. Here we provide evidence that a fluorogenic chromophore can self-catalytically form within an isolated N-terminal fragment of the enhanced green fluorescent protein (EGFP). We show that restoration of the split protein fluorescence can be driven by nucleic acid complementary interactions. In our assay, fluorescence development is fast (within a few minutes) when complementary oligonucleotide-linked fragments of the split EGFP are combined. The ability of our EGFP system to respond quickly to DNA hybridization should be useful for detecting the kinetics of many other types of pairwise interactions both in vitro and in living cells.'),(646,'kd_pnas06','Khare, S. D., and Dokholyan, N. V., ','Common dynamical signatures of FALS-associated structurally-diverse Cu, Zn superoxide dismutase mutants','Proceedings of the National Academy of Sciences USA',103,'3147','3152',2006,'Proteins: Aggregation','Dokholyan, N. V.','yes','no','article',16488975,'',NULL,NULL,NULL,'More than 100 structurally diverse point mutations leading to aggregation in the dimeric enzyme Cu, Zn superoxide dismutase (SOD1) are implicated in familial amyotrophic lateral sclerosis (FALS). Although SOD1 dimer dissociation is a known requirement for its aggregation, the common structural basis for diverse FALS mutations resulting in aggregation is not fully understood. In molecular dynamics simulations of wild-type SOD1 and three structurally diverse FALS mutants (A4V, G37R, and H46R), we find that a common effect of mutations on SOD1 dimer is the mutation-induced disruption of dynamic coupling between monomers. In the wild-type dimer, the principal coupled motion corresponds to a &quot;breathing motion&quot; of the monomers around an axis parallel to the dimer interface, and an opening-closing motion of the distal metal-binding loops. These coupled motions are disrupted in all three mutants independent of the mutation location. Loss of coupled motions in mutant dimers occurs with increased disruption of a key stabilizing structural element (the &beta;-plug) leading to the de-protection of edge strands. To rationalize disruption of coupling, which is independent of the effect of the mutation on global SOD1 stability, we analyze the residueresidue interaction network formed in SOD1. We find that the dimer interface and metal-binding loops, both involved in coupled motions, are regions of high connectivity in the network. Our results suggest that independent of the effect on protein stability, altered protein dynamics, due to long-range communication within its structure, may underlie the aggregation of mutant SOD1 in FALS.'),(647,'bsd_cb06','Bloom, K., Sharma, S., and Dokholyan, N. V., ','The path of DNA in the kinetochore','Current Biology',16,'R276','R278',2006,'Chromatin: Replication','Bloom, K. & Dokholyan, N. V.','yes','no','article',16631569,'',NULL,NULL,NULL,'Not applicable for this article type. Please see PDF.'),(648,'cd_tg06','Chen, Y., and Dokholyan, N. V., ','The coordinated evolution of yeast proteins constrained by functional modularity','Trends in Genetics',22,'416','419',2006,'Proteins: Evolution','Dokholyan, N. V.','yes','no','article',16797778,'',NULL,NULL,NULL,'Functional modularity is a key attribute of cellular systems and has important roles in evolution. However, the extent to which functional modularity affects protein evolution is largely unknown. Here, we analyzed the evolution of both sequence and expression level of proteins in the yeast <i>Saccharomyces cerevisiae</i> and found that proteins within the same functional modules evolve at more similar rates than those between different modules. We also found stronger co-evolution of expression levels between proteins within functional modules than between them. These results suggest that a coordinated evolution of both the sequence and expression level of proteins is constrained by functional modularity.'),(649,'dd_plscb06','Ding, F., and Dokholyan, N. V., ','Emergence of protein fold families through rational design','Public Library of Science Computational Biology',2,'e85','',2006,'Proteins: Design','Dokholyan, N. V.','yes','yes','article',16839198,'',NULL,NULL,NULL,'Diverse proteins with similar structures are grouped into families of homologs and analogs, if their sequence similarity is higher or lower, respectively, than 20%-30%. It was suggested that protein homologs and analogs originate from a common ancestor and diverge in their distinct evolutionary time scales, emerging as a consequence of the physical properties of the protein sequence space. Although a number of studies have determined key signatures of protein family organization, the sequence-structure factors that differentiate the two evolution-related protein families remain unknown. Here, we stipulate that subtle structural changes, which appear due to accumulating mutations in the homologous families, lead to distinct packing of the protein core and, thus, novel compositions of core residues. The latter process leads to the formation of distinct families of homologs. We propose that such differentiation results in the formation of analogous families. To test our postulate, we developed a molecular modeling and design toolkit, Medusa, to computationally design protein sequences that correspond to the same fold family. We find that analogous proteins emerge when a backbone structure deviates only 1-2 A root-mean-square deviation from the original structure. For close homologs, core residues are highly conserved. However, when the overall sequence similarity drops to &#8764;25%-30%, the composition of core residues starts to diverge, thereby forming novel families of protein homologs. This direct observation of the formation of protein homologs within a specific fold family supports our hypothesis. The conservation of amino acids in designed sequences recapitulates that of the naturally occurring sequences, thereby validating our computational design methodology.'),(650,'kcd_ajpfd06','Khare, S. D., Caplow, M., and Dokholyan, N. V., ','FALS mutations in Cu, Zn superoxide dismutase destabilize the dimer and increase dimer dissociation propensity: a large-scale thermodynamic analysis','Amyloid: the Journal of Protein Folding Disorders ',13,'226','235',2006,'Proteins: Aggregation','Dokholyan, N. V.','no','no','article',17107883,'',NULL,NULL,NULL,'Mutations in the dimeric enzyme Cu, Zn superoxide dismutase (SOD1) leading to its aggregation are implicated in the toxicity in familial amyotrophic lateral sclerosis (FALS). We and others have previously shown that aggregation occurs by a pathway involving dimer dissociation, metal-loss from monomers and multimeric assembly of apo-SOD1 monomers. We postulate that FALS mutations cause enhanced aggregation by affecting one or more steps in the pathway, and computationally test this postulate for 75 known mis-sense FALS mutants of SOD1. Based on an extensive thermodynamic analysis of the stability of apo-dimer and apo-monomer forms of these mutants, we classify the mutations into the following groups: 70 out of 75 mutations in SOD1 lead to (i) decreased dimer stability, and/or (ii) increased dimer dissociation, compared to wild type, and four mutations lead to (iii) decreased monomer stability compared to wild type. Our results suggest that enhanced aggregation of SOD1 in FALS occurs due to an increased population of mutant SOD1 apomonomers compared to wild type. The dissociation of multimeric proteins induced by diverse mutations may be a common theme in several human diseases.'),(651,'cd_jbc06','Chen, Y., and Dokholyan, N. V., ','Insights into allosteric control of vinculin function from its large-scale conformational dynamics','Journal of Biological Chemistry',281,'29148','29154',2006,'Proteins: Folding','Dokholyan, N. V.','no','no','article',16891659,'',NULL,NULL,NULL,'Vinculin is an important constituent of both cell-cell and cellmatrix junctions, where it plays crucial roles in the regulation of cell adhesion and migration. When activated, it mediates the linkage between cadherins (cell-cell) or integrins (cell-matrix) and the actin cytoskeleton through interactions with various proteins. The activation of vinculin requires structural conversions from an autoinhibited conformation to the &quot;open&quot; conformations in which the occluded binding sites of its different ligands become exposed, while the structural dynamics underlying the vinculin activation remains largely unknown. Here we report the first computational study of large scale conformational dynamics of full-length vinculin. We find that the &quot;holding&quot; and &quot;releasing&quot; motions between vinculin tail and pincerlike structure formed by first three domains of vinculin are the dominant motions near the native state of vinculin, indicating that an inherent flexibility of vinculin has a large influence on its allostery. We also find a cooperative dissociation between the head and tail domains of vinculin with increasing temperature in both thermodynamic and kinetic simulations, implying that vinculin may function as an allosteric switch in response to external signals. We show that the kinetics of vinculin unfolding exhibits specific sequential patterns, suggesting that a sophisticated interplay between domains may synergistically contribute to vinculin activation. We further find that the interaction between vinculin-binding site peptide from talin and vinculin significantly destabilizes the intramolecular head-tail interactions, suggesting a direct role of talin binding in vinculin activation.'),(652,'sdnwulpd_b06','Sharma, S., Ding, F., Nie, H., Watson, D., Unnithan, A., Lopp, J., Pozefsky, D., and Dokholyan, N. V., ','iFold: A platform for interactive folding simulations of proteins','Bioinformatics',22,'2693','2694',2006,'Biotechnology','Dokholyan, N. V.','yes','no','article',16940324,'',NULL,NULL,NULL,'<strong>Summary:</strong> We built a novel web-based platform for performing discrete molecular dynamics simulations of proteins. In silico protein folding involves searching for minimal frustration in the vast conformational landscape. Conventional approaches for simulating protein folding insufficiently address the problem of simulations in relevant time and length scales necessary for a mechanistic understanding of underlying biomolecular phenomena. Discrete molecular dynamics (DMD) offers an opportunity to bridge the size and timescale gaps and uncover the structural and biological properties of experimentally undetectable protein dynamics. The iFold server supports large-scale simulations of protein folding, thermal denaturation, thermodynamic scan, simulated annealing and <i>p<sub>fold</sub></i> analysis using DMD and coarse-grained protein model with structure-based G&otilde;-interactions between amino acids.<br><strong>Availability:</strong> <a id=maintext href=http://ifold.dokhlab.org target=_blank>http://ifold.dokhlab.org</a>'),(653,'ppedmb_ac06','Petrotchenko, E. V., Pasek, D., Elms, P., Dokholyan, N. V., Meissner, G., and Borchers, C. H., ','Combining fluorescence detection and mass spectrometric analysis for comprehensive and quantitative analysis of redox-sensitive cysteines in native membrane proteins','Analytical Chemistry',78,'7959','7966',2006,'Biotechnology','Borchers, C. H.','no','no','article',17134128,'',NULL,NULL,NULL,'Monobromobimane (MBB) is a lipophilic reagent that selectively modifies free cysteine residues in proteins. Because of its lipophilic character, MBB is capable of labeling cysteine residues in membrane proteins under native conditions. Reaction of MBB with the sulfhydryl groups of free cysteines leads to formation of highly fluorescent derivatives. Here we describe a procedure for the detection and relative quantitation of MBB-labeled cysteines using fluorescence and mass spectrometric analyses, which allow determination of free cysteine content and unambiguous identification of MBB-modified cysteine residues. We have applied this approach to the analysis of the free and redox-sensitive cysteine residues of a large membrane protein, the sarcoplasmic reticulum Ca<sup>2+</sup> release channel with a molecular mass of 2.2 million Da. Labeling was performed under physiologic conditions where the channel complex is in its native environment and is functionally active. The purified MBB-labeled channel complex was enzymatically digested, and the resulting peptides were separated by reversed-phase high performance chromatography. MBB-labeled peptides were detected by fluorescence and identified by matrix-assisted laser desorption/ionization (MALDI) mass spectrometry. Under MALDI conditions, partial photolytic fragmentation of the MBB-peptide bound occurred, thus allowing convenient screening for the MBB-modified peptides in the MS spectrum by detection of the specific mass increment of 190.07 Da for MBB-modified cysteine residues. Modification of the peptides was further confirmed by tandem mass spectrometric analysis, utilizing sequencing information and the presence of the specific immonium ion for the MBB-modified cysteine residues at <i>m/z</i> 266.6. Quantitative information was obtained by comparison of both fluorescence and MS signal intensities of MBB-modified peptides. Combination of fluorescence with MS detection and analysis of MBB-labeled peptides supported by a customized software program provides a convenient method for identifying and quantifying redoxsensitive cysteines in membrane proteins of native biological systems. Identification of one redox-sensitive cysteine (2327) in the native membrane-bound sarcoplasmic reticulum Ca<sup>2+</sup> release channel is described.'),(654,'scded_pnas06','Serohijos, A. W., Chen, Y., Ding, F., Elston, T. C., and Dokholyan, N. V., ','A structural model reveals energy transduction in dynein','Proceedings of the National Academy of Sciences USA',103,'18540','18545',2006,'Proteins: Assembly','Dokholyan, N. V. & Elston, T. C.','yes','no','article',17121997,'',NULL,NULL,NULL,'Intracellular active transport is driven by ATP-hydrolyzing motor proteins that move along cytoskeletal filaments. In particular, the microtubule-associated dynein motor is involved in the transport of organelles and vesicles, the maintenance of the Golgi, and mitosis. However, unlike kinesin and myosin, the mechanism by which dynein converts chemical energy into mechanical force remains largely a mystery, due primarily to the lack of a highresolution molecular structure. Using homology modeling and normal mode analysis, we propose a complete atomic structure and a mechanism for force generation by the motor protein dynein. In agreement with very recent electron microscopy (EM) reconstructions showing dynein as a ring-shaped heptamer, our model consists of six ATPases of the AAA (ATPases associated with various cellular activities) superfamily and a C-terminal domain, which is experimentally known to control motor function. Our model shows a coiled coil spanning the diameter of the motor that accounts for previously unidentified structures in EM studies and provides a potential mechanism for long-range communication between the AAA domains. Furthermore, normal mode analysis reveals that the subunits of the motor that contain the nucleotide binding sites exhibit minimal movement, whereas the rest of the motor is very mobile. Our analysis suggests the likely domain rearrangements of the motor unit that generate its power stroke. This study provides insights into the structure and function of dynein that can guide further experimental investigations into energy transduction in dynein.'),(655,'sdd_bj07','Sharma, S., Ding, F., and Dokholyan, N. V., ','Multi-scale modeling of nucleosome dynamics','Biophysical Journal',92,'1457','1470',2007,'Chromatin: Structure','Dokholyan, N. V.','yes','no','article',17142268,'',NULL,NULL,NULL,'Nucleosomes form the fundamental building blocks of chromatin. Subtle modifications of the constituent histone tails mediate chromatin stability and regulate gene expression. For this reason, it is important to understand structural dynamics of nucleosomes at atomic levels. We report a novel multiscale model of the fundamental chromatin unit, a nucleosome, using a simplified model for rapid discrete molecular dynamics simulations and an all-atom model for detailed structural investigation. Using a simplified structural model, we perform equilibrium simulations of a single nucleosome at various temperatures. We further reconstruct all-atom nucleosome structures from simulation trajectories. We find that histone tails bind to nucleosomal DNA via strong salt-bridge interactions over a wide range of temperatures, suggesting a mechanism of chromatin structural organization whereby histone tails regulate inter- and intranucleosomal assemblies via binding with nucleosomal DNA. We identify specific regions of the histone core H2A/H2B-H4/H3-H3/H4-H2B/H2A, termed &quot;cold sites&quot;, which retain a significant fraction of contacts with adjoining residues throughout the simulation, indicating their functional role in nucleosome organization. Cold sites are clustered around H3-H3, H2A-H4 and H4-H2A interhistone interfaces, indicating the necessity of these contacts for nucleosome stability. Essential dynamics analysis of simulation trajectories shows that bending across the H3-H3 is a prominent mode of intranucleosomal dynamics. We postulate that effects of salts on mononucleosomes can be modeled in discrete molecular dynamics by modulating histone-DNA interaction potentials. Local fluctuations in nucleosomal DNA vary significantly along the DNA sequence, suggesting that only a fraction of histone-DNA contacts make strong interactions dominating mononucleosomal dynamics. Our findings suggest that histone tails have a direct functional role in stabilizing higher-order chromatin structure, mediated by salt-bridge interactions with adjacent DNA.'),(656,'psdzsg_pa07','Podobnik, B., Shao, J., Dokholyan, N. V., Zlatic, V., Stanley, H. E., and Grosse, I., ','Similarity and dissimilarity in correlations of genomic DNA','Physica A',373,'497','502',2007,'Bioinformatics','Shao, J.','no','no','article',0,'',NULL,NULL,NULL,'We analyze auto-correlations of human chromosomes 1-22 and rice chromosomes 1-12 for seven binary mapping rules and find that the correlation patterns are different for different rules but almost identical for all of the chromosomes, despite their varying lengths and GC contents. We propose a simple stochastic process for modeling these correlations, and we find that the proposed process can reproduce, quantitatively and qualitatively, the correlation patterns found in the genomes of human and rice.'),(657,'dnd_pre07','Davis, C. H., Nie, H., and Dokholyan, N. V., ','Insights into thermophilic archaebacterial membrane stability from simplified models of lipid membranes','Physical Review E',75,'51921','',2007,'Lipids','Dokholyan, N. V.','no','no','article',17677112,'',NULL,NULL,NULL,'Lipid aggregation into fluid bilayers is an essential process for sustaining life. Simplified models of lipid structure, which allow for long time scales or large length scales not obtainable with all-atom simulations, have recently been developed and show promise for describing lipid dynamics in biological systems. Here, we describe two simplified models, a reduced-lipid model and a bola-lipid model for thermophilic bacterial membranes, developed for use with the rapid discrete molecular dynamics simulation method. In the reducedlipid model, we represent the lipid chain by a series of three beads interacting through pairwise discrete potentials that model hydrophobic attractions between hydrocarbon tails in implicit solvent. Our phase diagram recapitulates those produced by continuous potential models with similar coarse-grained lipid representations. We also find that phase transition temperatures for our reduced-lipid model are dependent upon the flexibility of the lipid chain, giving an insight into archaebacterial membrane stability and prompting development of a bola-lipid model specific for archaebacteria lipids. With both the reduced-lipid and bola-lipid model, we find that the reduced flexibility inherent in archaebacteria lipids yields more stable bilayers as manifested by increased phase transition temperatures. The results of these studies provide a simulation methodology for lipid molecules in biological systems and show that discrete molecular dynamics is applicable to lipid aggregation and dynamics.'),(658,'ydd_nm07','Yin, S., Ding, F., and Dokholyan, N. V., ','Eris: An automated estimator of protein stability','Nature Methods',4,'466','467',2007,'Biotechnology','Dokholyan, N. V.','yes','no','article',17538626,'',NULL,NULL,NULL,'Not applicable for this article type. Please see PDF.'),(659,'cdd_jpcb07','Chen, Y., Ding, F., and Dokholyan, N. V., ','Fidelity of the protein structure reconstruction from inter-residue proximity constraints','Journal of Physical Chemistry B',111,'7432','7438',2007,'Proteins: Folding','Dokholyan, N. V.','yes','no','article',17542631,'',NULL,NULL,NULL,'Inter-residue proximity constraints obtained in such experiments as cross-linking/mass spectrometry are important sources of information for protein structure determination. A central question in structure determination using these constraints is, What is the minimal number of inter-residue constraints needed to determine the fold of a protein? It is also unknown how the different structural aspects of constraints differentiate their ability in determining the native fold and whether there is a rational strategy for selecting constraints that feature higher fidelity in structure determination. To shed light on these questions, we study the fidelity of protein fold determination using theoretical inter-residue proximity constraints derived from protein native structures and the effect of various subsets of such constraints on fold determination. We show that approximately 70% randomly selected constraints are sufficient for determining the fold of a domain (with an average root-mean-square deviation of &le;3.4 A from their native structures). We find that random constraint selection often outperforms the rational strategy that predominantly favors the constraints representing global structural features. To uncover a strategy for constraint selection for the optimal structure determination, we study the role of the topological properties of these constraints. Interestingly, we do not observe any correlation between various simple topological properties of the selected constraints, emphasizing different global and local structural features, and the performance of these constraints, suggesting that accurate protein structure determination relies on a composite of global and local structural information.'),(660,'ccd_bj07','Chen, Y., Campbell, S. L., and Dokholyan, N. V., ','Deciphering protein dynamics from NMR data using explicit structure sampling and selection','Biophysical Journal',93,'2300','2306',2007,'Proteins: Folding','Dokholyan, N. V.','no','no','article',17557784,'',NULL,NULL,NULL,'Perhaps one of the most prominent realizations of recent years is the critical role that protein dynamics plays in many facets of cellular function. While characterization of protein dynamics is fundamental to our understanding of protein function, the ability to explicitly detect an ensemble of protein conformations from dynamics data is a paramount challenge in structural biology. Here, we report a new computational method, Sample and Select, for determining the ensemble of protein conformations consistent with NMR dynamics data. This method can be generalized and extended to different sources of dynamics data, enabling broad applicability in deciphering protein dynamics at different timescales. The structural ensemble derived from Sample and Select will provide structural and dynamic information that should aid us in understanding and manipulating protein function.'),(661,'bjkdd_jbc07','Barton, S., Jacak, R., Khare, S. D., Ding, F., and Dokholyan, N. V., ','The length dependence of the polyQ-mediated protein aggregation','Journal of Biological Chemistry',282,'25487','25492',2007,'Proteins: Aggregation','Dokholyan, N. V.','no','no','article',17591778,'',NULL,NULL,NULL,'Polyglutamine (polyQ) repeat disorders are caused by the expansion of CAG tracts in certain genes, resulting in transcription of proteins with abnormally long polyQ inserts.Whenthese inserts expand beyond 3545 glutamines, affected proteins form toxic aggregates, leading to neuron death. Chymotrypsin inhibitor 2 (CI2) with an inserted glutamine repeat has previously been used to model polyQ-mediated aggregation in vitro. However, polyQ insertion lengths in these studies have been kept below the pathogenic threshold. We perform molecular dynamics simulations to study monomer folding dynamics and dimer formation in CI2-polyQ chimeras with insertion lengths of up to 80 glutamines. Our model recapitulates the experimental results of previous studies of chimeric CI2 proteins, showing high folding cooperativity of monomers as well as protein association via domain swapping. Surprisingly, for chimeras with insertion lengths above the pathogenic threshold, monomer folding cooperativity decreases and the dominant mode for dimer formation becomes interglutamine hydrogen bonding. These results support a mechanism for pathogenic polyQ-mediated aggregation, in which expanded polyQ tracts destabilize affected proteins and promote the formation of partially unfolded intermediates. These unfolded intermediates form aggregates through associations by interglutamine interactions.'),(662,'kd_cpps07','Khare, S. D., and Dokholyan, N. V., ','Molecular mechanisms of polypeptide aggregation in human diseases','Current Protein & Peptide Science',8,'573','579',2007,'Proteins: Aggregation','Dokholyan, N. V.','no','no','article',18220844,'',NULL,NULL,NULL,'Protein aggregation is implicated in a plethora of neurodegenerative diseases. The proteins found to aggregate in these diseases are unrelated to their native structures and amino acid sequences, but form similar insoluble fibrils with characteristic cross-&beta; sheet morphologies called amyloid in the aggregated state. While both the mechanism of aggregation and the structure of the aggregates are not fully understood at the molecular level, recent studies provide strong support for the idea that protein aggregation into highly stable, insoluble amyloid structures is a general property of the polypeptide chain. For proteins with a unique native state, it is known that aggregation occurs under conditions that promote native-state destabilization <i>in vitro</i> and <i>in vivo</i>. Taken together, the results of several important recent investigations suggest three broad molecular frameworks that may underlie the conversion of normally soluble peptides and proteins into insoluble amyloid fibrils: (1) edge-strand hydrogen bonding, (2) domain-swapping, and (3) self-association of amyloidogenic fragments. We argue that these underlying scenarios are not mutually exclusive and may be protein-dependent - i.e., a protein with a high content of hinge-regions may aggregate <i>via</i> a runaway domain-swap, whereas a protein with a high content of amyloidogenic fragments may aggregat primarily by the self-association of these fragments. These different scenarios provide frameworks to understand the molecular mechanism of polypeptide aggregation.'),(663,'sgtbdc_jmb07','Sharma, S., Gong, P., Temple, B., Bhattacharyya, D., Dokholyan, N. V., and Chaney, S. G., ','Molecular dynamic simulations of cisplatin- and oxaliplatin-d(GG) intrastand cross-links reveal differences in their conformational dynamics','Journal of Molecular Biology',373,'1123','1140',2007,'DNA: Drugs','Chaney, S. G.','no','no','article',17900616,'',NULL,NULL,NULL,'Mismatch repair proteins, DNA damage-recognition proteins and translesion DNA polymerases discriminate between Pt-GG adducts containing cisdiammine ligands (formed by cisplatin (CP) and carboplatin) and <i>trans-RR</i>-diaminocyclohexane ligands (formed by oxaliplatin (OX)) and this discrimination is thought to be important in determining differences in the efficacy, toxicity and mutagenicity of these platinum anticancer agents. We have postulated that these proteins recognize differences in conformation and/or conformational dynamics of the DNA containing the adducts. We have previously determined the NMR solution structure of OX-DNA, CP-DNA and undamaged duplex DNA in the 5&#8242;-d(CCTCA<u>GG</u>CCTCC)-3&#8242; sequence context and have shown the existence of several conformational differences in the vicinity of the Pt-GG adduct. Here we have used molecular dynamics simulations to explore differences in the conformational dynamics between OX-DNA, CP-DNA and undamaged DNA in the same sequence context. Twenty-five 10 ns unrestrained fully solvated molecular dynamics simulations were performed starting from two different DNA conformations using AMBER v8.0. All 25 simulations reached equilibrium within 4 ns, were independent of the starting structure and were in close agreement with previous crystal and NMR structures. Our data show that the cis-diammine (CP) ligand preferentially forms hydrogen bonds on the 5&#8242; side of the Pt-GG adduct, while the <i>trans-RR</i>-diaminocyclohexane (OX) ligand preferentially forms hydrogen bonds on the 3&#8242; side of the adduct. In addition, our data show that these differences in hydrogen bond formation are strongly correlated with differences in conformational dynamics, specifically the fraction of time spent in different DNA conformations in the vicinity of the adduct, for CP- and OX-DNA adducts. We postulate that differential recognition of CP- and OX-GG adducts by mismatch repair proteins, DNA damage-recognition proteins and DNA polymerases may be due, in part, to differences in the fraction of time that the adducts spend in a conformation favorable for protein binding.'),(664,'lbddbss_jmb07','Lam, A. R., Borreguero, J. M., Ding, F., Dokholyan, N. V., Buldyrev, S. V., Shakhnovich, E. I., and Stanley, H. E., ','Parallel folding pathways in the SH3 domain protein','Journal of Molecular Biology',373,'1348','1360',2007,'Proteins: Folding','Lam, A. R.','no','no','article',17900612,'',NULL,NULL,NULL,'The transition-state ensemble (TSE) is the set of protein conformations with an equal probability to fold or unfold. Its characterization is crucial for an understanding of the folding process. We determined the TSE of the src-SH3 domain protein by using extensive molecular dynamics simulations of the G&otilde; model and computing the folding probability of a generated set of TSE candidate conformations. We found that the TSE possesses a well-defined hydrophobic core with variable enveloping structures resulting from the superposition of three parallel folding pathways. The most preferred pathway agrees with the experimentally determined TSE, while the two least preferred pathways differ significantly. The knowledge of the different pathways allows us to design the interactions between amino acids that guide the protein to fold through the least preferred pathway. This particular design is akin to a circular permutation of the protein. The finding motivates the hypothesis that the different experimentally observed TSEs in homologous proteins and circular permutants may represent potentially available pathways to the wild-type protein.'),(665,'ydd_s07','Yin, S., Ding, F., and Dokholyan, N. V., ','Modeling backbone flexibility improves protein stability estimation','Structure',15,'1567','1576',2007,'Biotechnology','Dokholyan, N. V.','yes','no','article',18073107,'',NULL,NULL,NULL,'In designing mutagenesis experiments, it is often crucial to know how certain mutations will affect the structure and thermodynamic stability of the protein. Here, we present a methodology, Eris, to efficiently and accurately compute the stability changes of proteins upon mutations using our protein-modeling suite, Medusa. We evaluate the stability changes upon mutations for 595 mutants from five structurally unrelated proteins, and find significant correlations between the predicted and experimental results. For cases when the high-resolution protein structure is not available, we find that better predictions are obtained by backbone structure prerelaxation. The advantage of our approach is that it is based on physical descriptions of atomic interactions, and does not rely on parameter training with available experimental protein stability data. Unlike other methods, Eris also models the backbone flexibility, thereby allowing for determination of the mutation-induced backbone conformational changes. Eris is freely available via the web server at <a id=maintext href=http://eris.dokhlab.org target=_blank>http://eris.dokhlab.org</a>'),(666,'sdd_fb08','Sharma, S., Ding, F., and Dokholyan, N. V., ','Probing protein aggregation using discrete molecular dynamics','Frontier in Bioscience',13,'4795','4808',2008,'Proteins: Aggregation','Dokholyan, N. V.','no','no','article',18508545,'',NULL,NULL,NULL,'Understanding the role of biomolecular dynamics in cellular processes leading to human diseases and the ability to rationally manipulate these processes is of fundamental importance in scientific research. The last decade has witnessed significant progress in probing biophysical behavior of proteins. However, we are still limited in understanding how changes in protein dynamics and inter-protein interactions occurring in short length- and time-scales lead to aberrations in their biological function. Bridging this gap in biology probed using computer simulations marks a challenging frontier in computational biology. Here we examine hypothesis-driven simplified protein models in conjunction with discrete molecular dynamics in the study protein aggregations, implicated in series of neurodegenerative diseases, such as Alzheimer&#39;s and Huntington&#39;s diseases. Discrete molecular dynamics simulations of simplified protein models have emerged as a powerful methodology with its ability to bridge the gap in time and length scales from protein dynamics to aggregation, and provide an indispensable tool for probing protein aggregation.'),(667,'cdnsswyd_abb08','Chen, Y., Ding, F., Nie, H., Serohijos, A. W., Sharma, S., Wilcox, K. C., Yin, S., and Dokholyan, N. V., ','Protein folding: then and now','Archives of Biochemistry and Biophysics',469,'4','19',2008,'Proteins: Folding','Dokholyan, N. V.','no','no','article',17585870,'',NULL,NULL,NULL,'Over the past three decades the protein folding field has undergone monumental changes. Originally a purely academic question, how a protein folds has now become vital in understanding diseases and our abilities to rationally manipulate cellular life by engineering protein folding pathways. We review and contrast past and recent developments in the protein folding field. Specifically, we discuss the progress in our understanding of protein folding thermodynamics and kinetics, the properties of evasive intermediates, and unfolded states. We also discuss how some abnormalities in protein folding lead to protein aggregation and human diseases.'),(668,'sd_bj08','Sharma, S., and Dokholyan, N. V., ','DNA sequence mediates nucleosome structure and stability','Biophysical Journal',94,'1','3',2008,'Chromatin','Dokholyan, N. V.','no','no','article',17933872,'',NULL,NULL,NULL,'Not applicable for this article type. Please see PDF.'),(669,'shrd_plscb08','Serohijos, A. W. R., Hegedus, T., Riordan, J. R., and Dokholyan, N. V., ','Diminished self-chaperoning activity of the &Delta;F508 mutant CFTR results in protein misfolding','Public Library of Science Computational Biology',4,'e1000008','',2008,'Proteins: Folding','Dokholyan, N. V.','no','no','article',18463704,'',NULL,NULL,NULL,'The absence of a functional ATP Binding Cassette (ABC) protein called the Cystic Fibrosis Transmembrane Conductance Regulator (CFTR) from apical membranes of epithelial cells is responsible for cystic fibrosis (CF). Over 90% of CF patients carry at least one mutant allele with deletion of phenylalanine at position 508 located in the N-terminal nucleotide binding domain (NBD1). Biochemical and cell biological studies show that the &Delta;F508 mutant exhibits inefficient biosynthetic maturation and susceptibility to degradation probably due to misfolding of NBD1 and the resultant misassembly of other domains. However, little is known about the direct effect of the Phe508 deletion on the NBD1 folding, which is essential for rational design strategies of cystic fibrosis treatment. Here we show that the deletion of Phe508 alters the folding dynamics and kinetics of NBD1, thus possibly affecting the assembly of the complete CFTR. Using molecular dynamics simulations, we find that meta-stable intermediate states appearing on wild type and mutant folding pathways are populated differently and that their kinetic accessibilities are distinct. The structural basis of the increased misfolding propensity of the &Delta;F508 NBD1 mutant is the perturbation of interactions in residue pairs Q493/P574 and F575/F578 found in loop S7-H6. As a proof of- principle that the S7-H6 loop conformation can modulate the folding kinetics of NBD1, we virtually design rescue mutations in the identified critical interactions to force the S7-H6 loop into the wild type conformation. Two redesigned NBD1-&Delta;F508 variants exhibited significantly higher folding probabilities than the original NBD1-&Delta;F508, thereby partially rescuing folding ability of the NBD1-&Delta;F508 mutant. We propose that these observed defects in folding kinetics of mutant NBD1 may also be modulated by structures separate from the 508 site. The identified structural determinants of increased misfolding propensity of NBD1-&Delta;F508 are essential information in correcting this pathogenic mutant.'),(670,'shahcdr_pnas08','Serohijos, A. W. R., Hegedus, T., Aleksandrov, A. A., He, L., Cui, L., Dokholyan, N. V., and Riordan, J. R., ','Phenylalanine-508 mediates a cytoplasmic-membrane domain contact in the CFTR 3D structure crucial to assembly and channel function','Proceedings of the National Academy of Sciences USA',105,'3256','3261',2008,'Proteins: Folding','Riordan, J. R.; Dokholyan, N. V.','yes','no','article',18305154,'',NULL,NULL,NULL,'Deletion of phenylalanine-508 (Phe-508) from the N-terminal nucleotide- binding domain (NBD1) of the cystic fibrosis transmembrane conductance regulator (CFTR), a member of the ATP-binding cassette (ABC) transporter family, disrupts both its folding and function and causes most cystic fibrosis. Most mutant nascent chains do not pass quality control in the ER, and those that do remain thermally unstable, only partially functional, and are rapidly endocytosed and degraded. Although the lack of the Phe-508 peptide backbone diminishes the NBD1 folding yield, the absence of the aromatic side chain is primarily responsible for defective CFTR assembly and channel gating. However, the site of interdomain contact by the side chain is unknown as is the high-resolution 3D structure of the complete protein. Here we present a 3D structure of CFTR, constructed by molecular modeling and supported biochemically, in which Phe-508 mediates a tertiary interaction between the surface of NBD1 and a cytoplasmic loop (CL4) in the C-terminal membrane-spanning domain (MSD2). This crucial cytoplasmic membrane interface, which is dynamically involved in regulation of channel gating, explains the known sensitivity of CFTR assembly to many disease-associated mutations in CL4 as well as NBD1 and provides a sharply focused target for small molecules to treat CF. In addition to identifying a key intramolecular site to be repaired therapeutically, our findings advance understanding of CFTR structure and function and provide a platform for focused biochemical studies of other features of this unique ABC ion channel.'),(671,'dscdbd_r08','Ding, F., Sharma, S., Chalasani, P., Demidov, V. V., Broude, N. E., and Dokholyan, N. V., ','Ab initio RNA folding by discrete molecular dynamics: from structure prediction to folding mechanisms','RNA',14,'1164','1173',2008,'RNA','Dokholyan, N. V.','yes','no','article',18456842,'',NULL,NULL,NULL,'RNA molecules with novel functions have revived interest in the accurate prediction of RNA three-dimensional (3D) structure and folding dynamics. However, existing methods are inefficient in automated 3D structure prediction. Here, we report a robust computational approach for rapid folding of RNA molecules. We develop a simplified RNA model for discrete molecular dynamics (DMD) simulations, incorporating base-pairing and base-stacking interactions. We demonstrate correct folding of 150 structurally diverse RNA sequences. The majority of DMD-predicted 3D structures have &lt;4 A deviations from experimental structures. The secondary structures corresponding to the predicted 3D structures consist of 94% native base-pair interactions. Folding thermodynamics and kinetics of tRNA<sup>Phe</sup>, pseudoknots, and mRNA fragments in DMD simulations are in agreement with previous experimental findings. Folding of RNA molecules features transient, non-native conformations, suggesting nonhierarchical RNA folding. Our method allows rapid conformational sampling of RNA folding, with computational time increasing linearly with RNA length. We envision this approach as a promising tool for RNA structural and functional analyses.'),(672,'hsdhr_jmb08','Hegedus, T. Z., Serohijos, A. W. R., Dokholyan, N. V., He, L., and Riordan, J. R., ','Computational studies reveal phosphorylation dependent changes in the unstructured R domain of CFTR','Journal of Molecular Biology',378,'1052','1063',2008,'Proteins: Folding','Riordan, J. R.','no','no','article',18423665,'',NULL,NULL,NULL,'The cystic fibrosis transmembrane conductance regulator (CFTR) is a cAMP-dependent chloride channel that is mutated in cystic fibrosis, an inherited disease of high morbidity and mortality. The phosphorylation of its &#8764;200 amino acid R domain by protein kinase A is obligatory for channel gating under normal conditions. The R domain contains more than ten PKA phosphorylation sites. No individual site is essential but phosphorylation of increasing numbers of sites enables progressively greater channel activity. In spite of numerous studies of the role of the R domain in CFTR regulation, its mechanism of action remains largely unknown. This is because neither its structure nor its interactions with other parts of CFTR have been completely elucidated. Studies have shown that the R domain lacks welldefined secondary structural elements and is an intrinsically disordered region of the channel protein. Here, we have analyzed the disorder pattern and employed computational methods to explore low-energy conformations of the R domain. The specific disorder and secondary structure patterns detected suggest the presence of molecular recognition elements (MoREs) that may mediate phosphorylation-regulated intra- and interdomain interactions. Simulations were performed to generate an ensemble of accessible R domain conformations. Although the calculated structures may represent more compact conformers than occur in vivo, their secondary structure propensities are consistent with predictions and published experimental data. Equilibrium simulations of a mimic of a phosphorylated R domain showed that it exhibited an increased radius of gyration. In one possible interpretation of these findings, by changing its size, the globally unstructured R domain may act as an entropic spring to perturb the packing of membrane-spanning sequences that constitute the ion permeability pathway and thereby activate channel gating.'),(673,'dtnd_s08','Ding, F., Tsao, D., Nie, H., and Dokholyan, N. V., ','Ab initio folding of proteins with all-atom discrete molecular dynamics','Structure',16,'1010','1018',2008,'Proteins: Folding','Dokholyan, N. V.','yes','no','article',18611374,'',NULL,NULL,NULL,'Discrete molecular dynamics (DMD) is a rapid sampling method used in protein folding and aggregation studies. Until now, DMD was used to perform simulations of simplified protein models in conjunction with structure-based force fields. Here, we develop an all-atom protein model and a transferable force field featuring packing, solvation, and environmentdependent hydrogen bond interactions. We performed folding simulations of six small proteins (20-60 residues) with distinct native structures by the replica exchange method. In all cases, native or near-native states were reached in simulations. For three small proteins, multiple folding transitions are observed, and the computationally characterized thermodynamics are in qualitative agreement with experiments. The predictive power of all-atom DMD highlights the importance of environment-dependent hydrogen bond interactions in modeling protein folding. The developed approach can be used for accurate and rapid sampling of conformational spaces of proteins and protein-protein complexes and applied to protein engineering and design of protein-protein interactions.'),(674,'tfddtr_plscb08','Teotico, D. G., Frazier, M. L., Ding, F., Dokholyan, N. V., Temple, B. R. S., and Redinbo, M. R., ','Active nuclear receptors exhibit highly correlated AF-2 domain motions','Public Library of Science Computational Biology',4,'e1000111','',2008,'Proteins: Folding','Redinbo, M. R.','no','yes','article',18617990,'',NULL,NULL,NULL,'Nuclear receptor ligand binding domains (LBDs) convert ligand binding events into changes in gene expression by recruiting transcriptional coregulators to a conserved activation function-2 (AF-2) surface. While most nuclear receptor LBDs form homo- or heterodimers, the human nuclear receptor pregnane X receptor (PXR) forms a unique and essential homodimer and is proposed to assemble into a functional heterotetramer with the retinoid X receptor (RXR). How the homodimer interface, which is located 30 A from the AF-2, would affect function at this critical surface has remained unclear. By using 20- to 30-ns molecular dynamics simulations on PXR in various oligomerization states, we observed a remarkably high degree of correlated motion in the PXRRXR heterotetramer, most notably in the four helices that create the AF-2 domain. The function of such correlation may be to create &quot;active-capable&quot; receptor complexes that are ready to bind to transcriptional coactivators. Indeed, we found in additional simulations that active-capable receptor complexes involving other orphan or steroid nuclear receptors also exhibit highly correlated AF-2 domain motions. We further propose a mechanism for the transmission of long-range motions through the nuclear receptor LBD to the AF-2 surface. Taken together, our findings indicate that long-range motions within the LBD scaffold are critical to nuclear receptor function by promoting a mobile AF-2 state ready to bind coactivators.'),(675,'cd_mbe08','Chen, Y., and Dokholyan, N. V., ','Natural selection against protein aggregation on self-interacting and essential proteins in yeast, fly and worm','Molecular Biology and Evolution',25,'1530','1533',2008,'Proteins: Evolution','Dokholyan, N. V.','no','no','article',18503047,'',NULL,NULL,NULL,'Protein aggregation is the phenomenon of protein self-association potentially leading to detrimental effects on physiology, which is closely related to numerous human diseases such as Alzheimer&#39;s and Parkinson&#39;s disease. Despite progress in understanding the mechanism of protein aggregation, how natural selection against protein aggregation acts on subunits of protein complexes and on proteins with different contributions to organism fitness remains largely unknown. Here, we perform a proteome-wide analysis by using an experimentally validated algorithm TANGO and utilizing sequence, interactomic and phenotype-based functional genomic data from yeast, fly, and nematode. We find that proteins that are capable of forming homooligomeric complex have lower aggregation propensity compared with proteins that do not function as homooligomer. Further, proteins that are essential to the fitness of an organism have lower aggregation propensity compared with nonessential ones. Our finding suggests that the selection force against protein aggregation acts across different hierarchies of biological system.'),(676,'hsntwsdb_plscb08','Hao, J., Serohijos, A. W. R., Newton, G., Tassone, G., Wang, Z., Sgroi, D. C., Dokholyan, N. V., and Basilion, J. P., ','Identification of peptide ligands to CRIP1, a novel biomarker for cancers','Public Library of Science Computational Biology',4,'e1000138','',2008,'Biotechnology','Dokholyan, N. V., Basilion, J. P.','yes','no','article',18670594,'',NULL,NULL,NULL,'Cysteine-rich intestinal protein 1 (CRIP1) has been identified as a novel marker for early detection of cancers. Here we report on the use of phage display in combination with molecular modeling to identify a high-affinity ligand for CRIP1. Panning experiments using a circularized C7C phage library yielded several consensus sequences with modest binding affinities to purified CRIP1. Two sequence motifs, A1 and B5, having the highest affinities for CRIP1, were chosen for further study. With peptide structure information and the NMR structure of CRIP1, the higher-affinity A1 peptide was computationally redesigned, yielding a novel peptide, A1M, whose affinity was predicted to be much improved. Synthesis of the peptide and saturation and competitive binding studies demonstrated approximately a 10-fold improvement in the affinity of A1M compared to that of either A1 or B5 peptide. These techniques have broad application to the design of novel ligand peptides.'),(677,'sdd_b08','Sharma, S., Ding, F., and Dokholyan, N. V., ','iFoldRNA: Three-dimensional RNA structure prediction and folding','Bioinformatics',24,'1951','1952',2008,'RNA','Dokholyan, N. V.','yes','no','article',18579566,'',NULL,NULL,NULL,'<strong>Summary:</strong> Three-dimensional RNA structure prediction and folding is of significant interest in the biological research community. Here, we present iFoldRNA, a novel web-based methodology for RNA structure prediction with near atomic resolution accuracy and analysis of RNA folding thermodynamics. iFoldRNA rapidly explores RNA conformations using discrete molecular dynamics simulations of input RNA sequences. Starting from simplified linear chain conformations, RNA molecules (&lt;50 nt) fold to native-like structures within half an hour of simulation, facilitating rapid RNA structure prediction. All-atom reconstruction of energetically stable conformations generates iFoldRNA predicted RNA structures. The predicted RNA structures are within 25 A root mean squre deviations (RMSDs) from corresponding experimentally derived structures. RNA folding parameters including specific heat, contact maps, simulation trajectories, gyration radii, RMSDs from native state, fraction of native-like contacts are accessible from iFoldRNA. We expect iFoldRNA will serve as a useful resource for RNA structure prediction and folding thermodynamic analyses.<br><strong>Availability:</strong> <a id=maintext href=http://iFoldRNA.dokhlab.org target=_blank>http://iFoldRNA.dokhlab.org</a>'),(678,'ybvd_jcim08','Yin, S., Biedermannova, L., Vondrasek, J., and Dokholyan, N. V., ','MedusaScore: An accurate force-field based scoring function for virtual drug screening','Journal of Chemical Information and Modeling',48,'1656','1662',2008,'Drugs','Dokholyan, N. V.','yes','no','article',18672869,'',NULL,NULL,NULL,'Virtual screening is becoming an important tool for drug discovery. However, the application of virtual screening has been limited by the lack of accurate scoring functions. Here, we present a novel scoring function, MedusaScore, for evaluating protein-ligand binding. MedusaScore is based on models of physical interactions that include van der Waals, solvation, and hydrogen bonding energies. To ensure the best transferability of the scoring function, we do not use any protein-ligand experimental data for parameter training. We then test the MedusaScore for docking decoy recognition and binding affinity prediction and find superior performance compared to other widely used scoring functions. Statistical analysis indicates that one source of inaccuracy of MedusaScore may arise from the unaccounted entropic loss upon ligand binding, which suggests avenues of approach for further MedusaScore improvement.'),(679,'olhlvghdsvs_jbc08','Ozvegy-Laczka, C., Laczko, R., Hegedus, C., Litman, T., Varady, G., Goda, K., Hegedus, T. Z., Dokholyan, N. V., Sorrentino, B. P., Varadi, A., Sarkadi, B., ','Interaction with the 5D3 monoclonal antibody is regulated by intramolecular rearrangements but not by covalent dimer formation of the human ABCG2 multidrug transporter','Journal of Biological Chemistry',283,'26059','26070',2008,'Proteins: Transmembrane','Sarkadi, B.','no','no','article',18644784,'',NULL,NULL,NULL,'Human ABCG2 is a plasma membrane glycoprotein working as a homodimer or homo-oligomer. The protein plays an important role in the protection/detoxification of various tissues and may also be responsible for the multidrug-resistant phenotype of cancer cells. In our previous study we found that the 5D3 monoclonal antibody shows a function-dependent reactivity to an extracellular epitope of the ABCG2 transporter. In the current experiments we have further characterized the 5D3- ABCG2interaction. The effect of chemical cross-linking and the modulation of extracellular SS bridges on the transporter function and 5D3 reactivity of ABCG2 were investigated in depth. We found that several protein cross-linkers greatly increased 5D3 labeling in ABCG2 expressing HEK cells; however, there was no correlation between covalent dimer formation, the inhibition of transport activity, and the increase in 5D3 binding. Dithiothreitol treatment, which reduced the extracellular SS bridge-forming cysteines of ABCG2, had no effect on transport function but caused a significant decrease in 5D3 binding. When analyzing ABCG2 mutants carrying Cys-to-Ala changes in the extracellular loop, we found that the mutant C603A (lacking the intermolecular SS bond) showed comparable transport activity and 5D3 reactivity to the wild-type ABCG2. However, disruption of the intramolecular SS bridge (in C592A, C608A, or C592A/C608A mutants) in this loop abolished 5D3 binding, whereas the function of the protein was preserved. Based on these results and ab initio folding simulations, we propose a model for the large extracellular loop of the ABCG2 protein.'),(685,'sshlbjbbddd_s09','Schwede, T., Sali, A., Honig, B., Levitt, M., Berman, H. M., Jones, B., Brenner, S., Burley, S. K., Das, R., Dokholyan, N. V., Dunbrack, R. L. Jr., Fidelis, K., Fiser, A., Godzik, A., Humblet, C., Jacobson, M. P., Joachimiak, A., Krystek, S., Kortemme, T., Kryshtafovych, A., Montelione, G., Moult, J., Murray, D., Sanchez, R., Sosnick, T. R., Standley, D. M., Stouch, T., Vajda, S., Westbrook, J. D., and Wilson, I. A., ','Outcome of a workshop on applications of protein models in biomedical research','Structure',17,'151','159',2009,'Proteins','Schwede, T.','no','no','article',19217386,'',NULL,NULL,NULL,'We describe the proceedings and conclusions from the &quot;Workshop on Applications of Protein Models in Biomedical Research&quot; (the Workshop) that was held at the University of California, San Francisco on 11 and 12 July, 2008. At the Workshop, international scientists involved with structure modeling explored (i) how models are currently used in biomedical research, (ii) the requirements and challenges for different applications, and (iii) how the interaction between the computational and experimental research communities could be strengthened to advance the field.'),(686,'glddw_jacs09','Gherghe, C. M., Leonard, C. W., Ding, F., Dokholyan, N. V., and Weeks, K. M., ','Native-like RNA tertiary structures using a sequence-encoded cleavage agent and refinement by discrete molecular dynamics','Journal of the American Chemical Society',131,'2541','2546',2009,'RNA','Dokholyan, N. V., Weeks, K. M.','yes','no','article',19193004,'',NULL,NULL,NULL,'The difficulty of analyzing higher order RNA structure, especially for folding intermediates and for RNAs whose functions require domains that are conformationally flexible, emphasizes the need for new approaches for modeling RNA tertiary structure accurately. Here, we report a concise approach that makes use of facile RNA structure probing experiments that are then interpreted using a computational algorithm, carefully tailored to optimize both the resolution and refinement speed for the resulting structures, without requiring user intervention. The RNA secondary structure is first established using SHAPE chemistry. We then use a sequence-directed cleavage agent, which can be placed arbitrarily in many helical motifs, to obtain high quality inter-residue distances. We interpret this in-solution chemical information using a fast, coarse grained, discrete molecular dynamics engine in which each RNA nucleotide is represented by pseudoatoms for the phosphate, ribose, and nucleobase groups. By this approach, we refine base paired positions in yeast tRNA<sup>Asp</sup> to 4 A rmsd without any preexisting information or assumptions about secondary or tertiary structures. This blended experimental and computational approach has the potential to yield native-like models for the diverse universe of functionally important RNAs whose structures cannot be characterized by conventional structural methods.'),(681,'dd_pnas08','Ding, F., and Dokholyan, N. V., ','Dynamical roles of metal ions and the disulfide bond in Cu, Zn superoxide dismutase folding and aggregation','Proceedings of the National Academy of Sciences USA',105,'19696','19701',2008,'Proteins: Aggregation','Dokholyan, N. V.','yes','no','article',19052230,'',NULL,NULL,NULL,'Misfolding and aggregation of Cu, Zn superoxide dismutase (SOD1) is implicated in neuronal death in amyotrophic lateral sclerosis. Each SOD1 monomer binds to 1 copper and 1 zinc ion and maintains its disulfide bond (Cys-57-Cys-146) in the reducing cytoplasm of cell. Mounting experimental evidence suggests that metal loss and/or disulfide reduction are important for initiating misfolding and aggregation of SOD1. To uncover the role of metals and the disulfide bond in the SOD1 folding, we systemically study the folding thermodynamics and structural dynamics of SOD1 monomer and dimer with and without metal binding and under disulfide-intact or disulfide reduced environments in computational simulations. We use all-atom discrete molecular dynamics for sampling. Our simulation results provide dynamical evidence to the stabilizing role of metal ions in both dimer and monomer SOD1. The disulfide bond anchors a loop (Glu-49 to Asn-53) that contributes to the dimer interface. The reduction of the disulfide bond in SOD1 with metal ions depleted results in a flexible Glu-49-Asn-53 loop, which, in turn, disrupts dimer formation. Interestingly, the disulfide bond reduction does not affect the thermostability of monomer SOD1 as significantly as the metal ions do. We further study the structural dynamics of metal-free SOD1 monomers, the precursor for aggregation, in simulations and find inhomogeneous local unfolding of &beta;-strands. The strands protected by the metal-binding and electrostatic loops are found to unfold first after metal loss, leading to a partially unfolded &beta;-sheet prone to aggregation. Our simulation study sheds light on the critical role of metals and disulfide bond in SOD1 folding and aggregation.'),(680,'hashcadr_jbc08','He, L., Aleksandrov, L. A., Serohijos, A. W. R., Hegedus, T., Cui, L., Aleksandrov, A. A., Dokholyan, N. V., and Riordan, J. R.,','Multiple membrane-cytoplasmic domain contacts in CFTR','Journal of Biological Chemistry',283,'26383','26390',2008,'Proteins: Folding','Riordan, J. R.','yes','no','article',18658148,'',NULL,NULL,NULL,'The cystic fibrosis transmembrane conductance regulator (CFTR) is a unique ATP-binding cassette (ABC) ion channel mutated in patients with cystic fibrosis. The most common mutation, deletion of phenylalanine 508 (F508) and many other disease-associated mutations occur in the nucleotide binding domains (NBD) and the cytoplasmic loops (CL) of the membrane-spanning domains (MSD). A recently constructed computational model of the CFTR three-dimensional structure, supported by experimental data (Serohijos, A. W., Hegedus, T., Aleksandrov, A. A., He, L., Cui, L., Dokholyan, N. V., and Riordan, J. R. (2008) Proc. Natl. Acad. Sci. U. S. A. 105, 3256-3261) revealed that several of these mutations including F508 disrupted interfaces between these domains. Here we have used cysteine cross-linking experiments to verify all NBD/CL interfaces predicted by the structural model and observed that their cross-linking has a variety of different effects on channel gating. The interdomain contacts comprise aromatic clusters important for stabilization of the interfaces and also involve the Q-loops and X-loops that are in close proximity to the ATP binding sites. Cross-linking of all domain-swapping contacts between NBDs andMSDcytoplasmic loops in opposite halves of the protein rapidly and reversibly arrest single channel gating while those in the same halves have lesser impact. These results reinforce the idea that mediation of regulatory signals between cytoplasmic- and membrane-integrated domains of the CFTR channel apparently relies on an array of precise but highly dynamic interdomain structural joints.'),(616,'dss_pnas02','Dokholyan, N. V., Shakhnovich, B. E., and Shakhnovich, E. I., ','Expanding protein universe and its origin from the biological Big Bang','Proceedings of the National Academy of Sciences USA',99,'','14136',2002,'Proteins: Evolution','Dokholyan, N. V.','no','no','article',12384571,'',NULL,NULL,NULL,'The bottom-up approach to understanding the evolution of organisms is by studying molecular evolution. With the large number of protein structures identified in the past decades, we have discovered peculiar patterns that nature imprints on protein structural space in the course of evolution. In particular, we have discovered that the universe of protein structures is organized hierarchically into a scale-free network. By understanding the cause of these patterns, we attempt to glance at the very origin of life.'),(617,'ddbss_bj02','Ding, F., Dokholyan, N. V., Buldyrev, S. V., Stanley, H. E., and Shakhnovich, E. I., ','Direct molecular dynamics observation of protein folding transition state ensemble','Biophysical Journal',83,'3525','3532',2002,'Proteins: Folding','Ding, F.','no','no','article',12496119,'',NULL,NULL,NULL,'The concept of the protein transition state ensemble (TSE), a collection of the conformations that have 50% probability to convert rapidly to the folded state and 50% chance to rapidly unfold, constitutes the basis of the modern interpretation of protein engineering experiments. It has been conjectured that conformations constituting the TSE in many proteins are the expanded and distorted forms of the native state built around a specific folding nucleus. This view has been supported by a number of on-lattice and off-lattice simulations. Here we report a direct observation and characterization of the TSE by molecular dynamic folding simulations of the C-Src SH3 domain, a small protein that has been extensively studied experimentally. Our analysis reveals a set of key interactions between residues, conserved by evolution, that must be formed to enter the kinetic basin of attraction of the native state.'),(614,'dlds_pnas02','Dokholyan, N. V., Li, L., Ding, F., and Shakhnovich, E. I., ','Topological determinants of protein folding','Proceedings of the National Academy of Sciences USA',99,'8637','8641',2002,'Proteins: Folding','Dokholyan, N. V.','no','no','article',12084924,'',NULL,NULL,NULL,'The folding of many small proteins is kinetically a two-state process that represents overcoming the major free-energy barrier. A kinetic characteristic of a conformation, its probability to descend to the native state domain in the amount of time that represents a small fraction of total folding time, has been introduced to determine to which side of the free-energy barrier a conformation belongs. However, which features make a protein conformation on the folding pathway become committed to rapidly descending to the native state has been a mystery. Using two small, well characterized proteins, CI2 and C-Src SH3, we show how topological properties of protein conformations determine their kinetic ability to fold. We use a macroscopic measure of the protein contact network topology, the average graph connectivity, by constructing graphs that are based on the geometry of protein conformations. We find that the average connectivity is higher for conformations with a high folding probability than for those with a high probability to unfold. Other macroscopic measures of protein structural and energetic properties such as radius of gyration, rms distance, solvent-accessible surface area, contact order, and potential energy fail to serve as predictors of the probability of a given conformation to fold.'),(615,'vdpk_pre02','Vendruscolo, M., Dokholyan, N. V., Paci, E., and Karplus, M., ','A small-world view of the amino acids that play a key role in protein folding','Physical Review E',65,'61910','',2002,'Proteins: Folding','','no','no','article',12188762,'',NULL,NULL,NULL,'We use geometrical considerations to provide a different perspective on the fact that a few selected amino acids, the so-called &quot;key residues,&quot; act as nucleation centers for protein folding. By constructing graphs corresponding to protein structures we show that they have the &quot;small-world&quot; feature of having a limited set of vertices with large connectivity. These vertices correspond to the key residues that play the role of &quot;hubs&quot; in the network of interactions that stabilize the structure of the transition state.'),(613,'dms_pa02','Dokholyan, N. V., Mirny, L. A., and Shakhnovich, E. I., ','Understanding conserved amino acids in proteins','Physica A',314,'600','606',2002,'Proteins: Evolution','Dokholyan, N. V.','no','no','article',0,'',NULL,NULL,NULL,'It has been conjectured that evolution exerted pressure to preserve amino acids bearing thermodynamic, kinetic, and functional roles. In this letter we show that the physical requirement to maintain protein stability gives rise to a sequence conservatism pattern that is in remarkable agreement with that found in natural proteins. Based on the physical properties of amino acids, we propose a model of evolution that explains conserved amino acids across protein families sharing the same fold.'),(612,'bdbss_jmb02','Borreguero, J. M., Dokholyan, N. V., Buldyrev, S. V., Stanley, H. E., and Shakhnovich, E. I., ','Thermodynamics and folding kinetics analysis of the SH3 domain from Discrete Molecular Dynamics','Journal of Molecular Biology',318,'863','876',2002,'Proteins: Folding','Borreguero, J. M.','no','no','article',12054829,'',NULL,NULL,NULL,'We perform a detailed analysis of the thermodynamics and folding kinetics of the SH3 domain fold with discrete molecular dynamic simulations. We propose a protein model that reproduces some of the experimentally observed thermodynamic and folding kinetic properties of proteins. Specifically, we use our model to study the transition state ensemble of the SH3 fold family of proteins, a set of unstable conformations that fold to the protein native state with probability 1/2. We analyze the participation of each secondary structure element formed at the transition state ensemble. We also identify the folding nucleus of the SH3 fold and test extensively its importance for folding kinetics. We predict that a set of amino acid contacts between the RT-loop and the distal hairpin are the critical folding nucleus of the SH3 fold and propose a hypothesis that explains this result.'),(611,'dpbs_pre02','Dokholyan, N. V., Pitard, E., Buldyrev, S. V., and Stanley, H. E., ','Glassy behavior of a homopolymer from molecular dynamics simulations','Physical Review E',65,'030801(R)','',2002,'Proteins: Polymers','','no','no','article',11909019,'',NULL,NULL,NULL,'We study at- and out-of-equilibrium dynamics of a single homopolymer chain at low temperature using molecular dynamics. The main quantities of interest are the average root mean square displacement of the monomers below the &theta; point, and the structure factor, as a function of time. The observation of these quantities show a close resemblance to those measured in structural glasses and suggest that the polymer chain in its low temperature phase is in a glassy phase, with its dynamics dominated by traps. In equilibrium, at low temperature, we observe the trapping of the monomers and a slowing down of the overall motion of the polymer as well as non-exponential relaxation of the structure factor. Out of equilibrium, at low temperatures, we compute the two-time quantities and observe breaking of ergodicity in a range of waiting times, with the onset of aging.'),(610,'ds_jmb01','Dokholyan, N. V., and Shakhnovich, E. I., ','Understanding hierarchical protein evolution from first principles','Journal of Molecular Biology',312,'289','307',2001,'Proteins: Evolution','Dokholyan, N. V.','no','no','article',11545603,'',NULL,NULL,NULL,'We propose a model that explains the hierarchical organization of proteins in fold families. The model, which is based on the evolutionary selection of proteins by their native state stability, reproduces patterns of amino acids conserved across protein families. Due to its dynamic nature, the model sheds light on the evolutionary time-scales. By studying the relaxation of the correlation function between consecutive mutations at a given position in proteins, we observe separation of the evolutionary time-scales: at short time intervals families of proteins with similar sequences and structures are formed, while at long time intervals the families of structurally similar proteins that have low sequence similarity are formed. We discuss the evolutionary implications of our model. We provide a &quot;profile&quot; solution to our model and find agreement between predicted patterns of conserved amino acids and those actually observed in nature.'),(606,'dbss_fd98','Dokholyan, N. V., Buldyrev, S. V., Stanley, H. E., and Shakhnovich, E. I., ','Molecular dynamics studies of folding of a protein-like model','Folding and Design',3,'577','587',1998,'Proteins: Folding','Dokholyan, N. V.','no','no','article',9889167,'',NULL,NULL,NULL,'<strong>Background:</strong> Many attempts have been made to resolve in time the folding of model proteins in computer simulations. Different computational approaches have emerged. Some of these approaches suffer from insensitivity to the geometrical properties of the proteins (lattice models), whereas others are computationally heavy (traditional molecular dynamics).<br><br><strong>Results:</strong> We used the recently proposed approach of Zhou and Karplus to study the folding of a protein model based on the discrete time molecular dynamics algorithm. We show that this algorithm resolves with respect to time the folding <img src=papers/dbss_fd98.png width=50 height=10> unfolding transition. In addition, we demonstrate the ability to study the core of the model protein.<br><br><strong>Conclusions:</strong> The algorithm along with the model of interresidue interactions can serve as a tool for studying the thermodynamics and kinetics of protein models.'),(607,'dbss_jmb00','Dokholyan, N. V., Buldyrev, S. V., Stanley, H. E., and Shakhnovich, E. I., ','Identifying the protein folding nucleus using molecular dynamics','Journal of Molecular Biology',296,'1183','1188',2000,'Proteins: Folding','Dokholyan, N. V.','no','no','article',10698625,'',NULL,NULL,NULL,'Molecular dynamics simulations of folding in an off-lattice protein model reveal a nucleation scenario, in which a few well-defined contacts are formed with high probability in the transition state ensemble of conformations. Their appearance determines folding cooperativity and drives the model protein into its folded conformation. Amino acid residues participating in those contacts may serve as &quot;accelerator pedals&quot; used by molecular evolution to control protein folding rate.'),(608,'d_rrdsp00','Dokholyan, N. V., ','The protein folding problem','Recent Research Developments in Statistical Physics',1,'77','84',2000,'Proteins: Folding','Dokholyan, N. V.','no','no','article',0,'',NULL,NULL,NULL,'We present an overview of the protein folding problem. We discuss difficulties associated with this problem, its importance, the various approaches to its solution, and recent advances in the field. We discuss two important aspects of protein folding properties: its folding thermodynamics and kinetics. We also describe a popular protein folding scenario &#8211; folding via nucleation.'),(609,'sdbs_pre01','Scala, A., Dokholyan, N. V., Buldyrev, S. V., and Stanley, H. E., ','Thermodynamically important contacts in folding of model proteins','Physical Review E',63,'32901','',2001,'Proteins: Folding','Dokholyan, N. V.','no','no','article',11308693,'',NULL,NULL,NULL,'We introduce a quantity, the entropic susceptibility, that measures the thermodynamic importance &#8211; for the folding transition &#8211; of the contacts between amino acids in model proteins. Using this quantity, we find that only one equilibrium run of a computer simulation of a model protein is sufficient to select a subset of contacts that give rise to the peak in the specific heat observed at the folding transition. To illustrate the method, we identify thermodynamically important contacts in a model 46-mer. We show that only about 50% of all contacts present in the protein native state are responsible for the sharp peak in the specific heat at the folding transition temperature, while the remaining 50% of contacts do not affect the specific heat.'),(605,'bdehkms_pa03','Buldyrev, S. V., Dokholyan, N. V., Erramilli, S., Hong, M. K., Kim, J. Y., Malescio, G., and Stanley, H. E., ','Hierarchy in social organization','Physica A',330,'659','665',2003,'Complex systems','Buldyrev, S. V.','no','no','article',0,'',NULL,NULL,NULL,'We find that area and population distributions of nations follow an inverse power-law, as is known for cities, but with a different exponent. To interpret this result, we develop a growth model based on the geometrical properties of partitioning of the plane. The substantial agreement between the model and the actual nation distributions motivates the hypothesis that the distribution of aggregates of organisms is related to land partitioning. To test this hypothesis we follow the development of bacterial colonies of <i>Escherichia coli</i>, which, compared to humans, are on a completely different level of complexity. We find that the distributions of <i>E. coli</i> colonies follow an inverse power law with exponent similar to that of nations.'),(603,'kbdhlps_pa02','King, P. R., Buldyrev, S. V., Dokholyan, N. V., Havlin, S., Lopez, E., Paul, G., and Stanley, H.E.,','Uncertainty in oil production predicted by percolation theory','Physica A',306,'376','380',2002,'Random systems: Percolation','King, P. R.','no','no','article',0,'',NULL,NULL,NULL,'In this paper, we apply scaling laws from percolation theory to the problem of estimating the time for a fluid injected into an oil field to breakthrough into a production well. The main contribution is to show that when these previously published results are used on realistic data they are in good agreement with results calculated in a more conventional way but they can be obtained significantly more quickly. As a result they may be used in practical engineering circumstances and aid decision making for real field problems.'),(602,'kbdhlpsv_pg01','King, P. R., Buldyrev, S. V., Dokholyan, N. V., Havlin, S., Lee, Y., Paul, G., Stanley, H. E., and Vandesteeg, N.,','Predicting oil recovery using percolation theory','Petroleum Geoscience',7,'S105','S107',2001,'Random systems: Percolation','King, P. R.','no','no','article',0,'',NULL,NULL,NULL,'In this paper we apply scaling laws from percolation theory to the problem of estimating the time for a fluid injected into an oil field to breakthrough into a production well. The main contribution is to show that, when these previously published results are used on realistic data, they are in good agreement with results calculated in a more conventional way but they can be obtained significantly more quickly. As a result they may be used in practical engineering circumstances and aid decision-making for real field problems.'),(601,'abdhklps_pre00','Andrade, J. S. Jr., Buldyrev, S. V., Dokholyan, N. V., Havlin, S., King, P. R., Lee, Y., Paul, G., and Stanley, H. E., ','Flow between two sites on a percolation cluster','Physical Review E',62,'8270','8281',2000,'Random systems: Percolation','','no','no','article',11138125,'',NULL,NULL,NULL,'We study the flow of fluid in porous media in dimensions <i>d</i>=2 and 3. The medium is modeled by bond percolation on a lattice of <i>L<sup>d</sup></i> sites, while the flow front is modeled by tracer particles driven by a pressure difference between two fixed sites (&quot;wells&quot;) separated by Euclidean distance <i>r</i>. We investigate the distribution function of the shortest path connecting the two sites, and propose a scaling ansatz that accounts for the dependence of this distribution (i) on the size of the system <i>L</i> and (ii) on the bond occupancy probability <i>p</i>. We confirm by extensive simulations that the ansatz holds for d=2 and 3. Further, we study two dynamical quantities: (i) the minimal traveling time of a tracer particle between the wells when the total flux is constant and (ii) the minimal traveling time when the pressure difference is constant. A scaling ansatz for these dynamical quantities also includes the effect of finite system size <i>L</i> and off-critical bond occupation probability <i>p</i>. We find that the scaling form for the distribution functions for these dynamical quantities for d=2 and 3 is similar to that for the shortest path, but with different critical exponents. Our results include estimates for all parameters that characterize the scaling form for the shortest path and the minimal traveling time in two and three dimensions; these parameters are the fractal dimension, the power law exponent, and the constants and exponents that characterize the exponential cutoff functions.'),(600,'pbdhkls_pre00','Paul, G., Buldyrev, S. V., Dokholyan, N. V., Havlin, S., King, P. R., Lee, Y.,  and Stanley, H. E.,','Dependence of conductance on percolation backbone mass','Physical Review E',61,'3435','3440',2000,'Random systems: Percolation','Paul, G.','no','no','article',11088120,'',NULL,NULL,NULL,'We study &lang;&sigma;(<i>M<sub>B</sub>, r</i>)&rang; the average conductance of the backbone, defined by two points separated by Euclidean distance <i>r</i>, of mass <i>M<sub>B</sub></i> on two-dimensional percolation clusters at the percolation threshold. We find that with increasing <i>M<sub>B</sub></i> and for fixed <i>r</i>, &lang;(&sigma;<i>M<sub>B</sub> ,r</i>)&rang; asymptotically <i>decreases</i> to a constant, in contrast with the behavior of homogeneous systems and nonrandom fractals (such as the Sierpinski gasket) in which conductance increases with increasing <i>M<sub>B</sub></i>. We explain this behavior by studying the distribution of shortest paths between the two points on clusters with a given <i>M<sub>B</sub></i>. We also study the dependence of conductance on <i>M<sub>B</sub></i> above the percolation threshold and find that (i) slightly above <i>p<sub>c</sub></i>, the conductance first decreases and then increases with increasing <i>M<sub>B</sub></i> and (ii) further above <i>p<sub>c</sub></i>, the conductance increases monotonically for all values of <i>M<sub>B</sub></i>, as is the case for homogeneous systems.'),(599,'kbdhlps_pa99','King, P. R., Buldyrev, S. V., Dokholyan, N. V., Havlin, S., Lee, Y., Paul, G., and Stanley, H. E.,','Application of statistical physics to the oil industry: Predicting oil recovery using percolation theory','Physica A',274,'60','66',1999,'Random systems: Percolation','King, P. R.','no','no','article',0,'',NULL,NULL,NULL,'In this paper we apply scaling laws from percolation theory to the problem of estimating the time for a fluid injected into an oil field (for the purposes of recovering the oil) to breakthrough into a production well. The main contribution is to show that percolation theory, when applied to a realistic model, can be used to obtain the same results as calculated in a more conventional way but significantly more quickly. Specifically, we found that a previously proposed scaling form for the breakthrough time distribution when applied to a real oil field is in good agreement with more time consuming simulation results. Consequently these methods can be used in practical engineering circumstances to aid decision making for real field problems.'),(598,'labdhkps_pre99','Lee, Y., Andrade, J. S. Jr., Buldyrev, S. V., Dokholyan, N. V., Havlin, S., King, P. R., Paul, G., and Stanley, H. E., ','Traveling time and traveling length in critical percolation clusters','Physical Review E',60,'3425','3428',1999,'Random systems: Percolation','Lee, Y.','no','no','article',11970164,'',NULL,NULL,NULL,'We study traveling time and traveling length for tracer dispersion in two-dimensional bond percolation, modeling flow by tracer particles driven by a pressure difference between two points separated by Euclidean distance <i>r</i>. We find that the minimal traveling time <i>t<sub>min</sub></i> scales as <i>t<sub>min</sub></i> &#8764; <i>r<sup>1.33</sup></i>, which is different from the scaling of the most probable traveling time, <i>t</i> &#8764; <i>r<sup>1.64</sup></i>. We also calculate the length of the path corresponding to the minimal traveling time and find <i>l<sub>min</sub></i> &#8764; <i>r<sup>1.13</sup></i> and that the most probable traveling length scales as <i>l</i> &#8764; <i>r<sup>1.21</sup>. We present the relevant distribution functions and scaling relations.'),(597,'kabdlhs_pa99','King, P. R., Andrade, J. S. Jr., Buldyrev, S. V., Dokholyan, N. V., Lee, Y., Havlin, S., and Stanley, H. E., ','Percolation and oil recovery','Physica A',266,'107','114',1999,'Random systems: Percolation','King, P. R.','no','no','article',0,'',NULL,NULL,NULL,'One particular practical problem in oil recovery is to predict the time to breakthrough of a fluid injected in one well and the subsequent decay in the production rate of oil at another well. Because we only have a stochastic view of the distribution of rock properties we need to predict the uncertainty in the breakthrough time and post-breakthrough behaviour in order to calculate the economic risk. In this paper we use percolation theory to predict (i) the distribution of the chemical path (shortest path) between two points (representing well pairs) at a given Euclidean separation and present a scaling hypothesis for this distribution which is confirmed by numerical simulation, (ii) the distribution of breakthrough times which can be calculated algebraically rather than by very time consuming direct numerical simulation of large numbers of realisations.'),(595,'dlbhks_jsp98','Dokholyan, N. V., Lee, Y., Buldyrev, S. V., Havlin, S., King, P. R., and Stanley, H. E.,','Scaling of the distribution of shortest paths in percolation','Journal of Statistical Physics',93,'603','613',1999,'Random systems: Percolation','Dokholyan, N. V.','no','no','article',0,'',NULL,NULL,NULL,'We present a scaling hypothesis for the distribution function of the shortest paths connecting any two points on a percolating cluster which accounts for (i) the effect of the finite size of the system and (ii) the dependence of this distribution on the site occupancy probability <i>p</i>. We test the hypothesis for the case of two-dimensional percolation.'),(596,'dbhkls_pa99','Dokholyan, N. V., Buldyrev, S. V., Havlin, S., King, P. R., Lee, Y., and Stanley, H. E., ','Distribution of shortest paths in percolation','Physica A',266,'53','59',1999,'Random systems: Percolation','Dokholyan, N. V.','no','no','article',0,'',NULL,NULL,NULL,'We present a scaling Ansatz for the distribution function of the shortest paths connecting any two points on a percolating cluster which accounts for (i) the effect of the finite size of the system, and (ii) the dependence of this distribution on the site occupancy probability <i>p</i>. We present evidence supporting the scaling Ansatz for the case of two-dimensional percolation.'),(592,'bdhss_pa99','Buldyrev, S. V., Dokholyan, N. V., Havlin, S., Stanley, H. E., and Stanley, H. R. H., ','Expansion of tandem repeats and oligomer clustering in coding and noncoding DNA sequences','Physica A',273,'19','32',1999,'Bioinformatics','Buldyrev, S. V.','no','no','article',0,'',NULL,NULL,NULL,'We review recent studies of distribution of dimeric tandem repeats and short oligomer clustering in DNA sequences. We find that distribution of dimeric tandem repeats in coding DNA is exponential, while in noncoding DNA it often has long power-law tails. We explain this phenomenon using mutation models based on random multiplicative processes. We also develop a clustering measure based on percolation theory that quantifies the degree of clustering of short oligomers. We find that mono-, di-, and tetramers cluster more in noncoding DNA than in coding DNA. However trimers have some degree of clustering in coding DNA and noncoding DNA. We relate this phenomena to modes of tandem repeat expansion.'),(594,'hjdsshe_nimprb98','Hong, M. K., Jeung, A. G., Dokholyan, N. V., Smith, T. I., Schwettman, H. A., Huie, P., and Erramilli, S., ','Imaging single living cells with a scanning near-field infrared microscope based on a free electron laser','Nuclear Instruments and Methods in Physics Research',0,'246','255',1999,'Cellular motility','Erramilli, S.','no','no','article',0,'',NULL,NULL,NULL,'We report the first sub-wavelength mid-infrared images under water, and describe an application to obtaining images of single living cells in water using the Stanford Free Electron Laser (FEL). Spatial resolution is enhanced at the peak of infrared absorption of water. Images were obtained of single motile fibroblasts with the FEL wavelength tuned to absorption peaks of both protein and lipid molecules. Analysis of the unexpectedly strong absorption due to lipid molecules in motile fibroblasts suggests that the concentration of lipid molecules in lamellopodia is consistent with membrane flow.'),(593,'dbhs_jtb00','Dokholyan, N. V., Buldyrev, S. V., Havlin, S., and Stanley, H. E., ','Distributions of dimeric tandem repeats in noncoding and coding DNA sequences','Journal of  Theoretical Biology',202,'273','282',2000,'Bioinformatics','Dokholyan, N. V.','no','no','article',10666360,'',NULL,NULL,NULL,'We study the length distribution functions for the 16 possible distinct dimeric tandem repeats in DNA sequences of diverse taxonomic partitions of GenBank (known human and mouse genomes, and complete genomes of <i>Caenorhabditis elegans</i> and yeast). For coding DNA, we find that all 16 distribution functions are exponential. For non-coding DNA, the distribution functions for most of the dimeric repeats have surprisingly long tails, that fit a power-law function. We hypothesize that: (i) the exponential distributions of dimeric repeats in protein coding sequences indicate strong evolutionary pressure against tandem repeat expansion in coding DNA sequences; and (ii) long tails in the distributions of dimers in non-coding DNA may be a result of various mutational mechanisms. These long, non-exponential tails in the distribution of dimeric repeats in non-coding DNA are hypothesized to be due to the higher tolerance of non-coding DNA to mutations. By comparing genomes of various phylogenetic types of organisms, we find that the shapes of the distributions are not universal, but rather depend on the specific class of species and the type of a dimer.'),(587,'dj_plb94','Dokholyan, N. V., and Jikia, G. V., ','Single top quark production and V<sub>tb</sub> CKM matrix element measurement in high energy e<sup>+</sup>e<sup>-</sup> collisions','Physics Letters B',336,'251','256',1994,'High energy physics','Dokholyan, N. V.','no','no','article',0,'',NULL,NULL,NULL,'A new method for the determination of the CKM mixing matrix element V<sub>tb</sub> has been proposed. It has been shown, that at future colliders one will measure the <i>tb</i>-mixing element with an accuracy 12-28 %'),(588,'dbhs_prl97','Dokholyan, N. V., Buldyrev, S. V., Havlin, S., and Stanley, H. E., ','Distribution of base pair repeats in coding and noncoding DNA sequences','Physical Review Letters',79,'5182','5185',1997,'Bioinformatics','Dokholyan, N. V.','no','no','article',0,'',NULL,NULL,NULL,'We analyze the histograms for the lengths of the 16 possible distinct repeats of identical dimers, known as dimeric tandem repeats, in DNA sequences. For coding regions, the probability of finding a repetitive sequence of l copies of a particular dimer decreases exponentially as l increases. For the noncoding regions, the distribution functions for most of the 16 dimers have long tails and can be approximated by power-law functions, while for coding DNA, they can be well fit by a first order Markov process. We propose a model, based on known biophysical processes, which leads to the observed probability distribution functions for noncoding DNA. We argue that this difference in the shape of the distribution functions between coding and noncoding DNA arises from the fact that noncoding DNA is more tolerant to evolutionary mutational alterations than coding DNA.'),(589,'dbhs_pa98','Dokholyan, N. V., Buldyrev, S. V., Havlin, S., and Stanley, H. E., ','Model of unequal chromosomal crossing over in DNA sequences','Physica A',249,'594','599',1998,'Bioinformatics','Dokholyan, N. V.','no','no','article',0,'',NULL,NULL,NULL,'It is known that some dimeric tandem repeats (DTR) are very abundant in noncoding DNA. We find that certain DTR length distribution functions in noncoding DNA can be fit by a power law function. We analyze a simplified model of unequal chromosomal crossing over and find that it produces a stable power law length distribution function, with the exponent  &micro; = 2. Although the exponent predicted by this model differs from those observed in nature, we argue that the biophysical process underlying this model provides the major contribution to the DTR length distribution function.'),(591,'sdbhs_jbsd99','Stanley, H. R. H., Dokholyan, N. V., Buldyrev, S. V., Havlin, S., and Stanley, H. E., ','Clustering of identical oligomers in coding and noncoding DNA sequences','Journal of Biomolecular Structure and Dynamics',17,'79','87',1999,'Bioinformatics','Stanley, H. R. H.','no','no','article',10496423,'',NULL,NULL,NULL,'We develop a quantitative method for analyzing repetitions of identical short oligomers in coding and noncoding DNA sequences. We analyze sequences presently available in the GenBank separately for primate, mammal, vertebrate, rodent, invertebrate and plant taxonomic partitions. We find that some oligomers &quot;cluster&quot; more than they would if randomly distributed, while other oligomers &quot;repel&quot; each other. To quantify this degree of clustering, we define clustering measures. We find that (i) clustering significantly differs in coding and noncoding DNA; (ii) in most cases, monomers, dimers and tetramers cluster in noncoding DNA but appear to repel each other in coding DNA. (iii) The degree of clustering for different sources (primates, invertebrates, and plants) is more conserved among these sources in the case of coding DNA than in the case of noncoding DNA. (iv) In contrast to other oligomers, we find that trimers always prefer to cluster. (v) Clustering of each particular oligomer is conserved within the same organism.'),(590,'bdghpsv_pa98','Buldyrev, S. V., Dokholyan, N. V., Goldberger, A. L., Havlin, S., Peng, C.-K., Stanley, H. E., and Viswanathan, G. M., ','Analysis of DNA sequences using methods of statistical physics','Physica A',249,'430','438',1998,'Bioinformatics','Buldyrev, S. V.','no','no','article',0,'',NULL,NULL,NULL,'We review the present status of the studies of DNA sequences using methods of statistical physics. We present evidence, based on systematic studies of the entire GenBank database, supporting the idea that the DNA sequence in genes containing noncoding regions is correlated, and that the correlation is remarkably long range, i.e., base pairs thousands of base pairs distant are correlated. We do not find such a long-range correlation in the coding regions of the DNA. We discuss the mechanisms of molecular evolution that may lead to the presence of long-range power-law correlations in noncoding DNA and their absence in coding DNA. One such mechanism is the simple repeat expansion, which recently has attracted the attention of the biological community in conjunction with genetic diseases. We also review new tools &#8212; e.g., detrended  fluctuation analysis &#8212; that are useful for studies of complex hierarchical DNA structure.'),(689,'rsxmd_plscb09','Ramachandran, S., Serohijos, A. W. R., Xu, L., Meissner, G., and Dokholyan, N. V.,','Ryanodine receptor pore structure and function','Public Library of Science Computational Biology',5,'e1000367','',2009,NULL,'Dokholyan, N. V.,','no','no','article',19390614,'',NULL,NULL,NULL,'Ryanodine Receptors (RyRs) are ion channels that regulate muscle contraction by releasing calcium ions from intracellular stores into the cytoplasm. Mutations in skeletal muscle RyR (RyR1) give rise to congenital diseases such as the Central Core Disease. The absence of high-resolution structures of RyR1 has limited our understanding of channel function and disease mechanisms at the molecular level. Here, we report a structural model of the pore-forming region of RyR1 and electrophysiological studies on a Central Core Disease mutant RyR1-G4898R. Molecular dynamics simulations on the structural model show preferential localization of Ca<sup>2+</sup> over K<sup>+</sup> in the selectivity filter. We observe high ion binding to the residues D4899, E4900, D4938 and D4945 along the pore, which are experimentally known to be critical for channel function and selectivity. Furthermore, simulations on the mutant RyR1-D4899Q show a loss of preference to Ca<sup>2+</sup> in the selectivity filter as seen experimentally. Electrophysiological experiments on RyR1-G4898R show constitutively open channels that conduct K<sup>+</sup> but not Ca<sup>2+</sup>. Our simulations with G4898R likewise show a decrease in preference of Ca<sup>2+</sup> over K<sup>+</sup> in the selectivity filter. Together, the computational and experimental results shed light on the functioning of the RyR1 pore.'),(690,'lbdghks_pre03','Lopez, E., Buldyrev, S. V., Dokholyan, N. V., Goldmakher, L., Havlin, S., King, P. R., and Stanley, H. E.,','Postbreakthrough behavior in flow through porous media','Physical Review E',67,'56314','',2003,NULL,'Lopez, E.,','no','no','article',0,'',NULL,NULL,NULL,'We numerically simulate the traveling time of a tracer in convective flow between two points (injection and extraction) separated by a distance <i>r</i> in a model of porous media, <i>d</i>=2 percolation. We calculate and analyze the traveling time probability density function for two values of the fraction of connecting bonds <i>p</i>: the homogeneous case <i>p</i>=1 and the inhomogeneous critical threshold case <i>p=p<sub>c</sub></i> . We analyze both constant current and constant pressure conditions at <i>p=p<sub>c</sub></i> . The homogeneous <i>p</i>=1 case serves as a comparison base for the more complicated <i>p=p<sub>c</sub></i> situation. We find several regions in the probability density of the traveling times for the homogeneous case (<i>p</i>=1) and also for the critical case (<i>p=p<sub>c</sub></i>) for both constant pressure and constant current conditions. For constant pressure, the first region <i>I<sub>P</sub></i> corresponds to the short times before the flow breakthrough occurs, when the probability distribution is strictly zero. The second region <i>II<sub>P</sub></i> corresponds to numerous fast flow lines reaching the extraction point, with the probability distribution reaching its maximum. The third region <i>III<sub>P</sub></i> corresponds to intermediate times and is characterized by a power-law decay. The fourth region <i>IV<sub>P</sub></i> corresponds to very long traveling times, and is characterized by a different power-law decaying tail. The power-law characterizing region <i>IV<sub>P</sub></i> is related to the multifractal properties of flow in percolation, and an expression for its dependence on the system size <i>L</i> is presented. The constant current behavior is different from the constant pressure behavior, and can be related analytically to the constant pressure case. We present theoretical arguments for the values of the exponents characterizing each region and crossover times. Our results are summarized in two scaling assumptions for the traveling time probability density; one for constant pressure and one for constant current. We also present the production curve associated with the probability of traveling times, which is of interest to oil recovery.'),(691,'mbsd_prl03','Marques, M. I., Borreguero, J. M., Stanley, H. E., and Dokholyan, N. V.,','Possible mechanism for cold denaturation of proteins at high pressure','Physical Review Letters',91,'138103','',2003,NULL,'Marques, M. I.,','no','no','article',0,'',NULL,NULL,NULL,'We study cold denaturation of proteins at high pressures. Using multicanonical Monte Carlo simulations of a model protein in a water bath, we investigate the effect of water density fluctuations on protein stability. We find that above the pressure where water freezes to the dense ice phase (&#8776; 2 kbars) the mechanism for cold denaturation with decreasing temperature is the loss of local low-density water structure. We find our results in agreement with data of bovine pancreatic ribonuclease A.'),(692,'kbdhlps_pa02a','King, P. R., Buldyrev, S. V., Dokholyan, N. V., Havlin, S., Lopez, E., Paul, G., and Stanley, H.E.,','Using percolation theory to predict oil field performance','Physica A',314,'103','108',2002,NULL,'King, P. R.,','no','no','article',0,'',NULL,NULL,NULL,'In this paper, we apply scaling laws from percolation theory to the problem of estimating the time for a fluid injected into an oil field to breakthrough into a production well. The main contribution is to show that when these previously published results are used on realistic data they are in good agreement with results calculated in a more conventional way but they can be obtained significantly more quickly. As a result they may be used in practical engineering circumstances and aid decision making for real field problems.'),(693,'dbbdss_me03','Dokholyan, N. V., Borreguero, J. M., Buldyrev, S. V., Ding, F., Stanley, H. E., and Shakhnovich, E. I.,','Identifying importance of amino acids for protein folding from crystal structures','Methods in Enzymology',374,'618','640',2003,NULL,'Dokholyan, N. V.,','no','no','book',0,'','Carter Jr. C. W., and Sweet, R. M.,','Macromolecular Crystallography D','Elsevier',''),(694,'ds_mbiu06','Dokholyan, N. V., and Shakhnovich, E. I.,','Scale-free evolution: from proteins to organisms','Power Laws, Scale-free Networks and Genome Biology',0,'86','105',2006,'Proteins: evolution','Dokholyan, N. V.,','no','no','book',0,'','Koonin, E. V., Wolf, Y. I., and Karev, G. P.,','Molecular Biology Intelligence Unit','Eurekah.com and Springer',''),(695,'ds_bmpbe07','Dokholyan, N. V., and Shakhnovich, E. I.,','Towards unifying protein evolution theory','Structural approaches to sequence evolution: Molecules, networks, populations',0,'113','126',2007,'Proteins: evolution','Dokholyan, N. V.,','no','no','book',0,'','Bastolla, U., Porto, M., Roman, H. E., and Vendruscolo, M.,','Biological and Medical Physics, Biomedical Engineering','Springer',''),(696,'d_08','Dokholyan, N. V.,','Protein designability and engineering','Structural Bioinformatics',0,'','',2008,'Proteins: Design','Dokholyan, N. V.,','no','no','book',0,'','Bourne. P., and Gu, J.,','','Wiley & Sons',''),(697,'crsdtbwc_pohmccc09','Chaney, S. G., Ramachandran, S., Sharma, S., Dokholyan, N. V., Temple, B., Bhattacharyya, D., Wu, Y., and Campbell, S.,','Differences in Conformation and Conformational Dynamics Between Cisplatin and Oxaliplatin DNA Adducts','Platinum and Other Heavy Metal Compounds in Cancer Chemotherapy',0,'157','169',2009,'DNA: Drugs','Chaney, S. G.,','no','no','book',0,'','Bonetti, A., Leone, R., Muggia, F. M., and Howell, S. B.,','Cancer Drug Discovery and Development','Humana Press',''),(703,'stled_pccp09','Serohijos, A. W. R., Tsygankov, D., Liu, S., Elston, T. C., and Dokholyan, N. V.','Multiscale approaches for studying energy transduction in dynein','Physical Chemistry Chemical Physics',11,'4840','4850',2009,'Proteins: Mechanism','Dokholyan, N. V.','no','no','article',19506759,'',NULL,NULL,NULL,'Cytoplasmic dynein is an important motor that drives all minus-end directed movement along microtubules. Dynein is a complex motor whose processive motion is driven by ATP-hydrolysis. Dynein&rsquo;s run length has been measured to be several millimetres with typical velocities in the order of a few nanometres per second. Therefore, the average time between steps is a fraction of a second. When this time scale is compared with typical time scales for protein side chain and backbone movements (&#126;10<sup>-9</sup> s and &#126;10<sup>-5</sup> s, respectively), it becomes clear that a multi-timescale modelling approach is required to understand energy transduction in this protein. Here, we reviewrecent efforts to use computational and mathematical modelling to understand various aspects of dynein&rsquo;s chemomechanical cycle. First, we describe a structural model of dynein&rsquo;s motor unit showing a heptameric organization of the motor subunits. Second, we describe our molecular dynamics simulations of the motor unit that are used to investigate the dynamics of the various motor domains. Third, we present a kinetic model of the coordination between the two dynein heads. Lastly, we investigate the various potential geometries of the dimer during its hydrolytic and stepping cycle.'),(704,'kdtd_psfb09','Kesner, B. A., Ding, F., Temple, B. S., and Dokholyan, N. V.','N-terminal strands of filamin Ig domains act as a conformational switch under biological forces','Proteins: Structure, Function, and Bioinformatics',0,'','',2009,'Proteins: Structure','Dokholyan, N. V.','no','no','article',19514078,'',NULL,NULL,NULL,'Conformational changes of filamin A under stress have been postulated to play crucial roles in signaling pathways of cell responses. Direct observation of conformational changes under stress is beyond the resolution of current experimental techniques. On the other hand, computational studies are mainly limited to either traditional molecular dynamics simulations of short durations and high forces or simulations of simplified models. Here we perform all-atom discrete molecular dynamics (DMD) simulations to study thermally and force-induced unfolding of filamin A. The high conformational sampling efficiency of DMD allows us to observe force-induced unfolding of filamin A Ig domains under physiological forces. The computationally identified critical unfolding forces agree well with experimental measurements. Despite a large heterogeneity in the population of force-induced intermediate states, we find a common initial unfolding intermediate in all the Ig domains of filamin, where the N-terminal strand unfolds. We also study the thermal unfolding of several filamin Ig-like domains. We find that thermally induced unfolding features an early-stage intermediate state similar to the one observed in force-induced unfolding and characterized by the N-terminal strand being unfurled. We propose that the N-terminal strand may act as a conformational switch that unfolds under physiological forces leading to exposure of cryptic binding sites, removal of native binding sites, and modulating the quaternary structure of domains.'),(705,'ksrcd_pnas09','Kota, P., Summers, D. W., Ren, H-Y., Cyr, D. M. and Dokholyan, N. V.','Identification of a consensus motif in substrates bound by a type I Hsp40','Proceedings of the National Academy of Sciences USA',106,'11073','11078',2009,'Proteins: Structure','Dokholyan, N. V.','yes','no','article',19549854,'',NULL,NULL,NULL,'Protein aggregation is a hallmark of a large and diverse number of conformational diseases. Molecular chaperones of the Hsp40 family (Escherichia coli DnaJ homologs) recognize misfolded disease proteins and suppress the accumulation of toxic protein species. Type I Hsp40s are very potent at suppressing protein aggregation and facilitating the refolding of damaged proteins. Yet, the molecular mechanism for the recognition of nonnative polypeptides by Type I Hsp40s such as yeast Ydj1 is not clear. Here we computationally identify a unique motif that is selectively recognized by Ydj1p. The motif is characterized by the consensus sequence GX[LMQ]{P}X{P}{CIMPVW}, where [XY] denotes either X or Y and {XY} denotes neither X nor Y. We further verify the validity of the motif by site-directed mutagenesis and show that substrate binding by Ydj1 requires recognition of this motif. A yeast proteome screen revealed that many proteins contain more than one stretch of residues that contain the motif and are separated by varying numbers of amino acids. In light of our results, we propose a 2-site peptide-binding model and a plausible mechanism of peptide presentation by Ydj1p to the chaperones of the Hsp70 family. Based on our results, and given that Ydj1p and its human ortholog Hdj2 are functionally interchangeable, we hypothesize that our results can be extended to understanding human diseases.'),(698,'d_nic08','Dokholyan, N. V.','Simplified approaches to complex biological systems','Proceedings of the From Computational Biophysics to Systems Biology (CBSB08) Symposium, May 19-21, 2008',40,'85','88',2008,'Proteins: Folding','Dokholyan, N. V.','no','no','conf',0,'','Hansmann, U. H. E., Meinke, J. H., Mohanty, S., Nadler, W. and Zimmermann, O.','','',NULL),(699,'dghklopttt_jad04','Dokholyan, N. V., Glabe, C., Holtzman, D., Koudinov, A., Lomakin, A., Oyler, N., Paganini, N., Teplow, D., Thirumalai, D., and Tycko, R.','Live discussion: Protein folding and neurodegeneration: Biophysics to the rescue&#63;','Journal of Alzheimer&#39;s Disease',6,'99','105',2004,'Proteins: Aggregation','','no','no','conf',0,'','','','',NULL),(700,'dbss_aip00','Dokholyan, N. V., Buldyrev, S. V., Stanley, H. E., and Shakhnovich, E. I.','Kinetics of the protein folding transition.','AIP Proceedings of the Third Tohwa University Conference on Statistical Physics, November 8-12, 1999',0,'419','425',2000,'Proteins: Folding','','no','no','conf',0,'','','','',NULL),(701,'ds_pisp00','Dokholyan, N. V., and Shakhnovich, E. I.','Two models of amino acid conservation in proteins.','Proceedings of the International School of Physics &quot;Enrico Fermi&quot; Course CXLV: &quot;Protein folding, evolution and design&quot;, Varenna, Italy',0,'227','245',2000,'Proteins: Evolution','','no','no','conf',0,'','','','',NULL),(702,'wzjhyrccd_jbc09','Wilcox, K. C., Zhou, L., Jordon, J., Huang, Y., Yu, Y., Redler, R. L., Chen, X., Caplow, M., and Dokholyan, N. V.','Modifications of SOD1 in human erythrocytes: A possible role in ALS','Journal of Biological Chemistry',284,'13940','13947',2009,'Proteins: Aggregation','','no','no','article',19299510,'','','','','Over 100 mutations in Cu/Zn superoxide dismutase (SOD1) result in familial amyotrophic lateral sclerosis (FALS). Dimer dissociation is the first step in SOD1 aggregation, and studies suggest nearly every amino acid residue in Cu/Zn superoxide dismutase (SOD1) is dynamically connected to the dimer interface. Post-translational modifications of SOD1 residues might be expected to have similar effects to mutations, but few have been identified. Here we show, using SOD1 isolated from human erythrocytes, that human SOD1 is phosphorylated at threonine 2 and glutathionylated at cysteine 111.  A second SOD1 phosphorylation was observed and mapped to either Thr-58 or Ser-59. Cysteine-111 glutathionylation promotes SOD1 monomer formation &#8211; a necessary initiating step in SOD1 aggregation &#8211; by causing a two-fold increase in the K<sub>d</sub>. This change in the dimer stability is expected to result in a 67% increase in monomer concentration &#8211; 315 nM rather than 212 nM at physiological SOD1 concentrations. Because protein glutathionylation is associated with redox regulation, our finding that glutathionylation promotes SOD1 monomer formation supports a model in which increased oxidative stress promotes SOD1 aggregation.'),(706,'ypld_pnas09','Yin, S., Proctor, E. A., Lugovskoy, A. A. and Dokholyan, N. V.','Fast screening of protein surfaces using geometric invariant fingerprints','Proceedings of the National Academy of Sciences USA',0,'','',2009,'Proteins: Methods','Dokholyan, N. V.','no','no','article',0,'',NULL,NULL,NULL,'We develop a rapid and efficient method for the comparison of protein local surface similarities utilizing geometric invariants (fingerprints). By combining fast fingerprint comparison with explicit alignment, we successfully screen the entire Protein Data Bank for proteins that possess local surface similarities. Our method is independent of sequence and fold similarities, and has potential application to protein structure annotation and protein-protein interface design.');
/*!40000 ALTER TABLE `publications` ENABLE KEYS */;
UNLOCK TABLES;

--
-- Table structure for table `pubs`
--

DROP TABLE IF EXISTS `pubs`;
/*!40101 SET @saved_cs_client     = @@character_set_client */;
/*!40101 SET character_set_client = utf8 */;
CREATE TABLE `pubs` (
  `pub_id` int(10) unsigned NOT NULL AUTO_INCREMENT,
  `code` varchar(60) DEFAULT '',
  `authors` varchar(2000) DEFAULT NULL,
  `title` varchar(500) DEFAULT '',
  `journal` varchar(250) DEFAULT '',
  `volume` int(5) DEFAULT '0',
  `fpage` varchar(10) DEFAULT '',
  `lpage` varchar(10) DEFAULT '',
  `year` int(11) DEFAULT '2010',
  `topic` varchar(100) DEFAULT '',
  `cauthors` varchar(100) DEFAULT 'Dokholyan, N. V.',
  `suppinfo` varchar(3) DEFAULT 'yes',
  `cover` varchar(3) DEFAULT 'no',
  `f1000` varchar(3) DEFAULT 'no',
  `type` varchar(15) DEFAULT 'article',
  `pmid` int(10) unsigned DEFAULT '0',
  `pmcid` varchar(20) DEFAULT '',
  `editors` varchar(200) DEFAULT '',
  `series` varchar(100) DEFAULT '',
  `publisher` varchar(100) DEFAULT '',
  `abstract` text,
  PRIMARY KEY (`pub_id`)
) ENGINE=MyISAM AUTO_INCREMENT=851 DEFAULT CHARSET=utf8;
/*!40101 SET character_set_client = @saved_cs_client */;

--
-- Dumping data for table `pubs`
--

LOCK TABLES `pubs` WRITE;
/*!40000 ALTER TABLE `pubs` DISABLE KEYS */;
INSERT INTO `pubs` VALUES (302,'rtcd_nar09','Ramachandran, S., Temple, B. R., Chaney, S. G., and Dokholyan, N. V., ','Structural basis for the sequence dependent effects of platinum-DNA adducts','Nucleic Acids Research',37,'2434','2448',2009,'DNA','Chaney, S. G., Dokholyan, N. V.','yes','no','no','article',19255091,'',NULL,NULL,NULL,'The differences in efficacy and molecular mechanisms of platinum based anti-cancer drugs cisplatin (CP) and oxaliplatin (OX) have been hypothesized to be in part due to the differential binding affinity of cellular and damage recognition proteins to CP and OX adducts formed on adjacent guanines in genomic DNA. HMGB1a in particular exhibits higher binding affinity to CP-GG adducts, and the extent of discrimination between CP- and OX-GG adducts is dependent on the bases flanking the adducts. However, the structural basis for this differential binding is not known. Here, we show that the conformational dynamics of CP- and OX-GG adducts are distinct and depend on the sequence context of the adduct. Molecular dynamics simulations of the Pt-GG adducts in the TGGA sequence context revealed that even though the major conformations of CP- and OX-GG adducts were similar, the minor conformations were distinct. Using the pattern of hydrogen bond formation between the Pt-ammines and the adjacent DNA bases, we identified the major and minor conformations sampled by Pt-DNA. We found that the minor conformations sampled exclusively by the CP-GG adduct exhibit structural properties that favor binding by HMGB1a, which may explain its higher binding affinity to CP-GG adducts, while these conformations are not sampled by OX-GG adducts because of the constraints imposed by its cyclohexane ring, which may explain the negligible binding affinity of HMGB1a for OX-GG adducts in the TGGA sequence context. Based on these results, we postulate that the constraints imposed by the cyclohexane ring of OX affect the DNA conformations explored by OX-GG adduct compared to those of CP-GG adduct, which may influence the binding affinities of HMG-domain proteins for Pt-GG adducts, and that these conformations are further influenced by the DNA sequence context of the Pt-GG adduct.'),(303,'tldpkdd_jbc09','Torres, M. P., Lee, M. J., Ding, F., Purbeck, C., Kuhlman, B., Dokholyan, N. V., and Dohlman, H. G., ','G protein mono-ubiquitination by the RSP5 ubiquitin ligase','Journal of Biological Chemistry',284,'8940','8950',2009,'Proteins: Mechanism','Dohlman, H. G.','yes','no','no','article',19176477,'',NULL,NULL,NULL,'Emerging evidence suggests that ubiquitination serves as a protein trafficking signal in addition to its well characterized role in promoting protein degradation. The yeast G protein &alpha; subunit Gpa1 represents a rare example of a protein that undergoes both mono- and poly-ubiquitination. Whereas mono-ubiquitinated Gpa1 is targeted to the vacuole, poly-ubiquitinated Gpa1 is directed instead to the proteasome. Here we investigate the structural requirements for mono- and poly-ubiquitination of Gpa1. We find that variants of Gpa1 engineered to be unstable are more likely to be poly-ubiquitinated and less likely to be mono-ubiquitinated. In addition, mutants that cannot be myristoylated are no longer mono-ubiquitinated but are still polyubiquitinated. Finally, we show that the ubiquitin ligase Rsp5 is necessary for Gpa1 mono-ubiquitination <i>in vivo</i> and that the purified enzyme is sufficient to catalyze Gpa1 mono-ubiquitination <i>in vitro</i>. Taken together, these data indicate that mono- and poly-ubiquitination have distinct enzyme and substrate recognition requirements; whereas poly-ubiquitination targets misfolded protein for degradation, a distinct ubiquitination apparatus targets the fully mature, fully myristoylated G protein for mono-ubiquitination and delivery to the vacuole.'),(297,'pxccdb_mcp09','Petrotchenko, E. V., Xiao, K., Cable, J., Chen, Y., Dokholyan, N. V., and Borchers, C. H., ','Bimane Bisthiopropionic Acid N-Succinimidyl Ester (BiPS), a photo-cleavable, isotopically-coded, fluorescent crosslinker for structural proteomics','Molecular and Cellular Proteomics',8,'273','286',2009,'Biotechnology','Borchers, C. H.','no','yes','no','article',18838738,'',NULL,NULL,NULL,'Cross-linking combined with mass spectrometry is an emerging approach for studying protein structure and protein-protein interactions. However, unambiguous mass spectrometric identification of cross-linked peptides derived from proteolytically digested cross-linked proteins is still challenging. Here we describe the use of a novel cross-linker, bimane bisthiopropionic acid Nsuccinimidyl ester (BiPS), that overcomes many of the challenges associated with other cross-linking reagents. BiPS is distinguished from other cross-linkers by a unique combination of properties: it is photocleavable, fluorescent, homobifunctional, amine-reactive, and isotopically coded. As demonstrated with a model protein complex, RNase S, the fluorescent moiety of BiPS allows for sensitive and specific monitoring of the different cross-linking steps, including detection and isolation of cross-linked proteins by gel electrophoresis, determination of in-gel digestion completion, and fluorescence-based separation of cross-linked peptides by HPLC. The isotopic coding of BiPS results in characteristic ion signal &quot;doublets&quot; in mass spectra, thereby permitting ready detection of cross-linker-containing peptides. Under MALDI-MS conditions, partial photocleavage of the cross-linker occurs, releasing the crosslinked peptides. This allows differentiation between dead-end, intra-, and interpeptide cross-links based on losses of specific mass fragments. It also allows the use of the isotope doublets as mass spectrometric &quot;signatures.&quot; A software program was developed that permits automatic cross-link identification and assignment of the cross-link type. Furthermore photocleavage of BiPS assists in cross-link identification by allowing separate tandem mass spectrometry sequencing of each peptide comprising the original cross-link. By combining the use of BiPS with MS, we have provided the first direct evidence for the docking site of a phosphorylated G-protein coupled receptor C terminus on the multifunctional adaptor protein &beta;-arrestin, clearly demonstrating the broad potential and application of this novel cross-linker in structural and cellular biology.'),(298,'mpyrdt_psfb09','Meissner, G., Pasek, D. A., Yamaguchi, N., Ramachandran, S., Dokholyan, N. V., and Tripathy, A.,','Thermodynamics of calmodulin binding to cardiac and skeletal muscle ryanodine receptor ion channels','Proteins: Structure, Function, and Bioinformatics',74,'207','211',2009,'Proteins: Transmembrane','Meissner, G.','no','no','no','article',18618700,'',NULL,NULL,NULL,'The skeletal muscle (RyR1) and cardiac muscle (RyR2) ryanodine receptor calcium release channels contain a single, conserved calmodulin (CaM) binding domain, yet are differentially regulated by CaM. Here, we report that high-affinity [<sup>35</sup>S]CaM binding to RyR1 is driven by favorable enthalpic and entropic contributions at Ca<sup>2+</sup> concentrations from &lt;0.01 to 100 &micro;M. At 0.15 &micro;M Ca<sup>2+</sup>, [<sup>35</sup>S]CaM bound to RyR2 with decreased affinity and binding enthalpy compared with RyR1. The rates of [<sup>35</sup>S]CaM dissociation from RyR1 increased as the temperature was raised, whereas at 0.15 &micro;M Ca<sup>2+</sup> the rate from RyR2 was little affected. The results suggest major differences in the energetics of CaM binding to and dissociation from RyR1 and RyR2.'),(299,'tsde_jcp09','Tsygankov, D., Serohijos, A. W. R., Dokholyan, N. V., and Elston, T. C., ','Kinetic models for the coordinated stepping of cytoplasmic dynein','Journal of Chemical Physics',130,'25101','',2009,'Proteins: Mechanism','Elston, T. C.','no','no','no','article',19154055,'',NULL,NULL,NULL,'To generate processive motion along a polymer track requires that motor proteins couple their ATP hydrolysis cycle with conformational changes in their structural subunits. Numerous experimental and theoretical efforts have been devoted to establishing how this chemomechanical coupling occurs. However, most processive motors function as dimers. Therefore a full understanding of the motor&#146;s performance also requires knowledge of the coordination between the chemomechanical cycles of the two heads. We consider a general two-headed model for cytoplasmic dynein that is built from experimental measurements on the chemomechanical states of monomeric dynein. We explore different possible scenarios of coordination that simultaneously satisfy two main requirements of the dimeric protein: high processivity (long run length) and high motor velocity (fast ATP turnover). To demonstrate the interplay between these requirements and the necessity for coordination, we first develop and analyze a simple mechanical model for the force-induced stepping in the absence of ATP. Next we use a simplified model of dimeric dynein&#146;s chemomechanical cycle to establish the kinetic rules that must be satisfied for the model to be consistent with recent data for the motor&#146;s performance from single molecule experiments. Finally, we use the results of these investigations to develop a full model for dimeric dynein&#146;s chemomechanical cycle and analyze this model to make experimentally testable predictions.'),(231,'ddbss_jmb02','Ding, F., Dokholyan, N. V., Buldyrev, S. V., Stanley, H. E., and Shakhnovich, E. I., ','Molecular dynamic simulation of the SH3 domain aggregation suggests a generic amyloidogenesis mechanism','Journal of Molecular Biology',324,'851','857',2002,'Proteins: Aggregation','Dokholyan, N. V.','no','no','no','article',12460582,'',NULL,NULL,NULL,'We use molecular dynamics simulation to study the aggregation of Src SH3 domain proteins. For the case of two proteins, we observe two possible aggregation conformations: the closed form dimer and the open aggregation state. The closed dimer is formed by &quot;domain swapping&quot; &#8212; the two proteins exchange their RT-loops. All the hydrophobic residues are buried inside the dimer so proteins cannot further aggregate into elongated amyloid fibrils. We find that the open structure &#8212; stabilized by backbone hydrogen bond interactions &#8212; packs the RT-loops together by swapping the two strands of the RT-loop. The packed RT-loops form a &beta;-sheet structure and expose the backbone to promote further aggregation. We also simulate more than two proteins, and find that the aggregate adopts a fibrillar double &beta;-sheet structure, which is formed by packing the RT-loops from different proteins. Our simulations are consistent with a possible generic amyloidogenesis scenario.'),(234,'sdds_jmb03','Shakhnovich, B. E., Dokholyan, N. V., DeLisi, C., and Shakhnovich, E. I., ','Functional fingerprints of folds: Evidence for correlated structure-function evolution','Journal of Molecular Biology',326,'1','9',2003,'Proteins: Evolution','Shakhnovich, E. I.','no','no','yes','article',12547186,'',NULL,NULL,NULL,'Using structural similarity clustering of protein domains: protein domain universe graph (PDUG), and a hierarchical functional annotation: gene ontology (GO) as two evolutionary lenses, we find that each structural cluster (domain fold) exhibits a distribution of functions that is unique to it. These functional distributions are functional fingerprints that are specific to characteristic structural clusters and vary from cluster to cluster. Furthermore, as structural similarity threshold for domain clustering in the PDUG is relaxed we observe an influx of earlier-diverged domains into clusters. These domains join clusters without destroying the functional fingerprint. These results can be understood in light of a divergent evolution scenario that posits correlated divergence of structural and functional traits in protein domains from one or few progenitors.'),(235,'dbbsd_psfg03','Ding, F., Borreguero, J. M., Buldyrev, S. V., Stanley, H. E., and Dokholyan, N. V., ','Mechanism for the &alpha;-Helix to &beta;-Hairpin Transition','Proteins: Structure, Function, and Genetics',53,'220','228',2003,'Proteins: Folding','Ding, F.','no','no','no','article',14517973,'',NULL,NULL,NULL,'The aggregation of &alpha;-helix-rich proteins into &beta;-sheet-rich amyloid fibrils is associated with fatal diseases, such as Alzheimer&#39;s disease and prion disease. During an aggregation process, protein secondary structure elements &#8212; &alpha;-helices &#8212; undergo conformational changes to &beta;-sheets. The fact that proteins with different sequences and structures undergo a similar transition on aggregation suggests that the sequence nonspecific hydrogen bond interaction among protein backbones is an important factor. We perform molecular dynamics simulations of a polyalanine model, which is an &alpha;-helix in its native state and observe a metastable &beta;-hairpin intermediate. Although a &beta;-hairpin has larger potential energy than an &alpha;-helix, the entropy of a &beta;-hairpin is larger because of fewer constraints imposed by the hydrogen bonds. In the vicinity of the transition temperature, we observe the interconversion of the &alpha;-helix and &beta;-sheet states via a random coil state. We also study the effect of the environment by varying the relative strength of side-chain interactions for a designed peptide &#8212; an &alpha;-helix in its native state. For a certain range of side-chain interaction strengths, we find that the intermediate &beta;-hairpin state is destabilized and even disappears, suggesting an important role of the environment in the aggregation propensity of a peptide. Proteins 2003;53:220228.'),(236,'dds_bj03','Deeds, E. J., Dokholyan, N. V., and Shakhnovich, E. I., ','Protein evolution within a structural space','Biophysical Journal',85,'2962','2972',2003,'Proteins: Evolution','Shakhnovich, E. I.','no','no','no','article',14581198,'',NULL,NULL,NULL,'Understanding of the evolutionary origins of protein structures represents a key component of the understanding of molecular evolution as a whole. Here we seek to elucidate how the features of an underlying protein structural &quot;space&quot; might impact protein structural evolution. We approach this question using lattice polymers as a completely characterized model of this space. We develop a measure of structural comparison of lattice structures that is analogous to the one used to understand structural similarities between real proteins. We use this measure of structural relatedness to create a graph of lattice structures and compare this graph (in which nodes are lattice structures and edges are defined using structural similarity) to the graph obtained for real protein structures. We find that the graph obtained from all compact lattice structures exhibits a distribution of structural neighbors per node consistent with a random graph. We also find that subgraphs of 3500 nodes chosen either at random or according to physical constraints also represent random graphs. We develop a divergent evolution model based on the lattice space which produces graphs that, within certain parameter regimes, recapitulate the scale-free behavior observed in similar graphs of real protein structures.'),(238,'pdsg_pnas03','Pei, J., Dokholyan, N. V., Shakhnovich, E. I., and Grishin, N. V., ','Using protein design for homology detection and active site searches','Proceedings of the National Academy of Sciences USA',100,'11361','11366',2003,'Proteins: Evolution','Grishin, N. V.','yes','no','no','article',12975528,'',NULL,NULL,NULL,'We describe a method of designing artificial sequences that resemble naturally occurring sequences in terms of their compatibility with a template structure and its functional constraints. The design procedure is a Monte Carlo simulation of amino acid substitution process. The selective fixation of substitutions is dictated by a simple scoring function derived from the template structure and a multiple alignment of its homologs. Designed sequences represent an enlargement of sequence space around native sequences. We show that the use of designed sequences improves the performance of profile-based homology detection. The difference in position-specific conservation between designed sequences and native sequences is helpful for prediction of functionally important residues. Our sequence selection criteria in evolutionary simulations introduce amino acid substitution rate variation among sites in a natural way, providing a better model to test phylogenetic methods.'),(239,'kdd_jmb03','Khare, S. D., Ding, F., and Dokholyan, N. V., ','Folding of Cu, Zn superoxide dismutase and Familial Amyotrophic Lateral Sclerosis','Journal of Molecular Biology',334,'515','525',2003,'Proteins: Folding','Dokholyan, N. V.','no','no','no','article',14623191,'',NULL,NULL,NULL,'Cu, Zn superoxide dismutase (SOD1) has been implicated in the familial form of the neurodegenerative disease amyotrophic lateral sclerosis (ALS). It has been suggested that mutant mediated SOD1 misfolding/aggregation is an integral part of the pathology of ALS. We study the folding thermodynamics and kinetics of SOD1 using a hybrid molecular dynamics approach. We reproduce the experimentally observed SOD1 folding thermodynamics and find that the residues which contribute the most to SOD1 thermal stability are also crucial for apparent two-state folding kinetics. Surprisingly, we find that these residues are located on the surface of the protein and not in the hydrophobic core. Mutations in some of the identified residues are found in patients with the disease. We argue that the identified residues may play an important role in aggregation. To further characterize the folding of SOD1, we study the role of cysteine residues in folding and find that non-native disulfide bond formation may significantly alter SOD1 folding dynamics and aggregation propensity.'),(240,'d_psfb04','Dokholyan, N. V., ','What is the protein design alphabet?','Proteins: Structure, Function, and Bioinformatics',54,'622','628',2004,'Proteins: Design','Dokholyan, N. V.','no','no','no','article',14997558,'',NULL,NULL,NULL,'Selecting a protein sequence that corresponds to a specific three-dimensional protein structure is known as the protein design problem. One principal bottleneck in solving this problem is our lack of knowledge of precise atomic interactions. Using a simple model of amino acid interactions, we determine three crucial factors that are important for solving the <i>protein design problem</i>. Among these factors is the <i>protein alphabet</i> &#8212; a set of sequence elements that encodes protein structure. Our model predicts that alphabet size is independent of protein length, suggesting the possibility of designing a protein of arbitrary length with the natural protein alphabet. We also find that protein alphabet size is governed by protein structural properties and the energetic properties of the protein alphabet units. We discover that the usage of average types of amino acid in proteins is less than expected if amino acids were chosen randomly with naturally occurring frequencies. We propose three possible scenarios that account for amino acid underusage in proteins. These scenarios suggest the possibility that amino acids themselves might not constitute the alphabet of natural proteins. Proteins 2004;54:622628.'),(241,'tsds_pnas04','Tiana, G., Shakhnovich, B. E., Dokholyan, N. V., and Shakhnovich, E. I., ','Imprint of evolution on protein structures','Proceedings of the National Academy of Sciences USA',101,'2846','2851',2004,'Proteins: Evolution','Shakhnovich, E. I.','yes','no','no','article',14970345,'',NULL,NULL,NULL,'We attempt to understand the evolutionary origin of protein folds by simulating their divergent evolution with a three-dimensional lattice model. Starting from an initial seed lattice structure, evolution of model proteins progresses by sequence duplication and subsequent point mutations. A new gene&#39;s ability to fold into a stable and unique structure is tested each time through direct kinetic folding simulations. Where possible, the algorithm accepts the new sequence and structure and thus a &quot;new protein structure&quot; is born. During the course of each run, this model evolutionary algorithm provides several thousand new proteins with diverse structures. Analysis of evolved structures shows that later evolved structures are more designable than seed structures as judged by recently developed structural determinant of protein designability, as well as direct estimate of designability for selected structures by thermodynamic sampling of their sequence space. We test the significance of this trend predicted on lattice models on real proteins and show that protein domains that are found in eukaryotic organisms only feature statistically significant higher designability than their prokaryotic counterparts. These results present a fundamental view on protein evolution highlighting the relative roles of structural selection and evolutionary dynamics on genesis of modern proteins.'),(242,'kkd_jmb04','Khatun, J., Khare, S. D., and Dokholyan, N. V., ','Can contact potentials reliably predict stability of proteins?','Journal of Molecular Biology',336,'1223','1238',2004,'Proteins: Design','Dokholyan, N. V.','no','no','no','article',15037081,'',NULL,NULL,NULL,'The simplest approximation of interaction potential between amino acid residues in proteins is the contact potential, which defines the effective free energy of a protein conformation by a set of amino acid contacts formed in this conformation. Finding a contact potential capable of predicting free energies of protein states across a variety of protein families will aid protein folding and engineering in silico on a computationally tractable time-scale. We test the ability of contact potentials to accurately and transferably (across various protein families) predict stability changes of proteins upon mutations. We develop a new methodology to determine the contact potentials in proteins from experimental measurements of changes in proteins thermodynamic stabilities (&Delta;&Delta;G) upon mutations. We apply our methodology to derive sets of contact interaction parameters for a hierarchy of interaction models including solvation and multi-body contact parameters. We test how well our models reproduce experimental measurements by statistical tests. We evaluate the maximum accuracy of predictions obtained by using contact potentials and the correlation between parameters derived from different data-sets of experimental (&Delta;&Delta;G) values. We argue that it is impossible to reach experimental accuracy and derive fully transferable contact parameters using the contact models of potentials. However, contact parameters may yield reliable predictions of &Delta;&Delta;G for datasets of mutations confined to the same amino acid positions in the sequence of a single protein.'),(243,'pdubcsd_pre04','Peng, S., Ding, F., Urbanc, B., Buldyrev, S. V., Cruz, L., Stanley, H. E., and Dokholyan, N. V., ','Discrete molecular dynamics simulations of peptide aggregation','Physical Review E',69,'41908','',2004,'Proteins: Aggregation','Peng, S.','no','no','no','article',15169044,'',NULL,NULL,NULL,'We study the aggregation of peptides using the discrete molecular dynamics simulations. Specifically, at temperatures above the &alpha;-helix melting temperature of a single peptide, the model peptides aggregate into a multilayer parallel &beta;-sheet structure. This structure has an interstrand distance of 4.8 A and an intersheet distance of 10 A, which agree with experimental observations. Our model explains these results as follows: hydrogen-bond interactions give rise to the interstrand spacing in &beta; sheets, while G&otilde; interactions between side chains make &beta; strands parallel to each other and allow &beta; sheets to pack into layers. An important feature of our results is that the aggregates contain free edges, which may allow for further aggregation of model peptides to form elongated fibrils.'),(244,'bdbsd_bj04','Borreguero, J. M., Ding, F., Buldyrev, S. V., Stanley, H. E., and Dokholyan, N. V., ','Multiple folding pathways of the SH3 domain','Biophysical Journal',87,'521','533',2004,'Proteins: Folding','Borreguero, J. M.','no','no','no','article',15240485,'',NULL,NULL,NULL,'Experimental observations suggest that proteins follow different folding pathways under different environmental conditions. We perform molecular dynamics simulations of a model of the c-Crk SH3 domain over a broad range of temperatures, and identify distinct pathways in the folding transition. We determine the kinetic partition temperature &#8212; the temperature for which the c-Crk SH3 domain undergoes a rapid folding transition with minimal kinetic barriers &#8212; and observe that below this temperature the model protein may undergo a folding transition by multiple folding pathways via only one or two intermediates. Our findings suggest the hypothesis that the SH3 domain, a protein fold for which only two-state folding kinetics was observed in previous experiments, may exhibit intermediate states under conditions that strongly stabilize the native state.'),(245,'tdbs_jcp04','Tiana, G., Dokholyan, N. V., Broglia, R. A., and Shakhnovich, E. I., ','The evolution dynamics of model proteins','Journal of Chemical Physics',121,'2381','2389',2004,'Proteins: Evolution','','no','no','no','article',15260793,'',NULL,NULL,NULL,'Explicit simulations of protein evolution, where protein chains are described at a molecular, although simplified, level provide important information to understand the similarities found to exist between known proteins. The results of such simulations suggest that a number of evolutionary-related quantities, such as the distribution of sequence similarity for structurally similar proteins, are controlled by evolutionary kinetics and do not reflect an equilibrium state. An important result for phylogeny is that a subset of the residues of each protein evolve on a much larger time scale than the other residues.'),(246,'ucdskbsd_bj04','Urbanc, B., Cruz, L., Ding, F., Sammond, D., Khare, S. D., Buldyrev, S. V., Stanley, H. E., and Dokholyan, N. V., ','Molecular dynamics simulation of Amyloid beta dimer formation','Biophysical Journal',87,'2310','2321',2004,'Proteins: Aggregation','Urbanc, B.','no','no','no','article',15454432,'',NULL,NULL,NULL,'Recent experiments with amyloid &beta; (A&beta;) peptide indicate that formation of toxic oligomers may be an important contribution to the onset of Alzheimer&#39;s disease. The toxicity of A&beta; oligomers depends on their structure, which is governed by assembly dynamics. Due to limitations of current experimental techniques, a detailed knowledge of oligomer structure at the atomic level is missing. We introduce a molecular dynamics approach to study A&beta; dimer formation. 1), We use discrete molecular dynamics simulations of a coarse-grained model to identify a variety of dimer conformations; and 2), we employ all-atom molecular mechanics simulations to estimate thermodynamic stability of all dimer conformations. Our simulations of a coarse-grained A&beta; peptide model predicts 10 different planar &beta;-strand dimer conformations. We then estimate the free energies of all dimer conformations in all-atom molecular mechanics simulations with explicit water. We compare the free energies of A&beta;(1-42) and A&beta;(1-40) dimers. We find that 1), dimer conformations have higher free energies compared to their corresponding monomeric states; and 2), the free-energy difference between the A&beta;(1-42) and the corresponding A&beta;(1-40) dimer conformation is not significant. Our results suggest that A&beta; oligomerization is not accompanied by the formation of thermodynamically stable planar &beta;-strand dimers.'),(247,'kcd_pnas04','Khare, S. D., Caplow, M., and Dokholyan, N. V., ','The rate and equilibrium constants for a multi-step reaction sequence for the aggregation of superoxide dismutase in ALS','Proceedings of the National Academy of Sciences USA',101,'15094','15099',2004,'Proteins: Aggregation','Dokholyan, N. V., Caplow, M.','yes','no','no','article',15475574,'',NULL,NULL,NULL,'Mutation-induced aggregation of the dimeric enzyme Cu, Zn superoxide dismutase 1 (SOD1) has been implicated in the familial form of the disease amyotrophic lateral sclerosis, but the mechanism of aggregation is not known. Here, we show that <i>in vitro</i> SOD1 aggregation is a multistep reaction that minimally consists of dimer dissociation, metal loss from the monomers, and oligomerization of the apo-monomers:<p align=center><img width=300 height=80 border=0 src=images/kcd_pnas04.png></p> where <i>D<sub>holo</sub>,M<sub>holo</sub>,M<sub>apo</sub>,</i> and <i>A</i> are the holo-dimer, holo-monomer, apo-monomer, and aggregate, respectively. Under aggregation-promoting conditions (pH 3.5), the rate and equilibrium constants corresponding to each step are: (i) dimer dissociation, <i>K<sub>d</sub></i> &#8776; 1 &micro;M; <i>k<sub>off</sub></i> &#8776; 1 &#215; 10<sup>&#8722;3</sup> s<sup>&#8722;1</sup>, <i>k<sub>on</sub></i> &#8776; 1 &#215; 10<sup>3</sup> M<sup>&#8722;1</sup> &#8901; s<sup>&#8722;1</sup>; (ii) metal loss, <i>K<sub>m</sub></i> &#8776; 0.1 &micro;M, <i>k<sub>m</sub><sup>&#8722;</sup></i> &#8776; 1 &#215; 10<sup>&#8722;3</sup> s<sup>&#8722;1</sup>, <i>k<sub>m</sub><sup>+</sup></i> &#8776; 1 &#215; 10<sup>4</sup> M<sup>&#8722;1</sup> &#8901; s<sup>&#8722;1</sup>; and (iii) assembly (rate-limiting step), <i>k<sub>agg</i> &#8776; 1 &#215; 10<sup>3</sup> M<sup>&#8722;1</sup> &#8901; s<sup>&#8722;1</sup>. In contrast, under near-physiological conditions (pH 7.8), where aggregation is drastically reduced, dimer dissociation is less thermodynamically favorable: <i>K<sub>d</sub></i> &#8776; 0.1 nM, and extremely slow: <i>k<sub>off</sub></i> &#8776; 3 &#215; 10<sup>&#8722;5</sup> s<sup>&#8722;1</sup>, <i>k<sub>on</sub></i> &#8776; 3 &#215; 10<sup>5</sup> M<sup>&#8722;1</sup> &#8901; s<sup>&#8722;1</sup>. Our results suggest that familial amyotrophic lateral sclerosis-linked SOD1 aggregation occurs by a mutation-induced increase in dimer dissociation and/or increase in apomonomer formation.'),(248,'dcdkpscd_s04','Dixon, R. D. S., Chen, Y., Ding, F., Khare, S. D. , Prutzman, K. C., Schaller, M. D., Campbell, S. L., and Dokholyan, N. V., ','New insights into FAK signaling and localization based on detection of a FAT domain folding intermediate','Structure',12,'2161','2171',2004,'Proteins: Folding','Dokholyan, N. V., Campbell, S. L.','yes','no','no','article',15576030,'',NULL,NULL,NULL,'Mounting evidence suggests that the focal adhesion targeting (FAT) domain, an antiparallel four-helix bundle, exists in alternative conformations that may modulate phosphorylation, ligand binding, and the subcellular localization of focal adhesion kinase (FAK). In order to characterize the conformational dynamics of the FAT domain, we have developed a novel method for reconstructing the folding pathway of the FAT domain by using discrete molecular dynamics (DMD) simulations, with free energy constraints derived from NMR hydrogen exchange data. The DMD simulations detect a folding intermediate, in which a cooperative unfolding event causes helix 1 to lose helical character while separating from the helix bundle. The conformational dynamic features of helix 1 in the intermediate state of the FAT domain are likely to facilitate Y926 phosphorylation , yet interfere with paxillin binding. The presence of this intermediate state in vivo may promote FAK signaling via the ERK/MAPK pathway and by release of FAK from focal adhesions.'),(249,'dbd_bj05','Ding, F., Buldyrev, S. V., and Dokholyan, N. V., ','Folding Trp-cage to NMR resolution native structure using a coarse-grained protein model','Biophysical Journal',88,'147','155',2005,'Proteins: Folding','Dokholyan, N. V., Ding, F.','yes','no','no','article',15533926,'',NULL,NULL,NULL,'We develop a coarse-grained protein model with a simplified amino acid interaction potential. Using this model, we perform discrete molecular dynamics folding simulations of a small 20-residue proteinTrp-cagefrom a fully extended conformation. We demonstrate the ability of the Trp-cage model to consistently reach conformations within 2-A backbone root-mean-square distance from the corresponding NMR structures. The minimum root-mean-square distance of Trp-cage conformations in simulations can be &lt;1 A . Our findings suggest that, at least in the case of Trp-cage, a detailed all-atom protein model with a molecular mechanics force field is not necessary to reach the native state of a protein. Our results also suggest that the success of folding Trp-cage in our simulations and in the reported all-atom molecular mechanics simulation studies may be mainly due to the special stabilizing features specific to this miniprotein.'),(250,'d_g05','Dokholyan, N. V., ','The architecture of the protein domain universe','Gene',347,'199','205',2005,'Proteins: Evolution','Dokholyan, N. V.','no','no','no','article',15777630,'',NULL,NULL,NULL,'Understanding the design of the universe of protein structures may provide insights into protein evolution. We study the architecture of the protein domain universe, which has been found to poses peculiar scale-free properties [Dokholyan, N.V., Shakhnovich, B., Shakhnovich, E.I., 2002b. Expanding protein universe and its origin from the biological Big Bang. Proceedings of the National Academy of Sciences of the United States of America 99, 1413214136]. We examine the origin of these scale-free properties of the graph of protein domain structures (PDUG) and determine that that the PDUG is not modular, i.e. it does not consist of modules with uniform properties. Instead, we find the PDUG to be self-similar at all scales. We further characterize the PDUG architecture by studying the properties of the hub nodes that are responsible for the scale-free connectivity of the PDUG. We introduce a measure of the betweenness centrality of protein domains in the PDUG and find a power-law distribution of the betweenness centrality values. The scale-free distribution of hubs in the protein universe suggests that a set of specific statistical mechanics models, such as the self-organized criticality model, can potentially identify the principal driving forces of protein evolution. We also find a gatekeeper protein domain, removal of which partitions the largest cluster into two large sub-clusters. We suggest that the loss of such gatekeeper protein domains in the course of evolution is responsible for the creation of new fold families.'),(251,'djd_s05','Ding, F., Jha, R. K., and Dokholyan, N. V., ','Scaling behavior and structure of denatured proteins','Structure',13,'1047','1054',2005,'Proteins: Folding','Ding, F.','no','yes','no','article',16004876,'',NULL,NULL,NULL,'An ensemble of random-coil conformations with no persistent structures has long been accepted as the classical model of denatured proteins due to its consistency with the experimentally determined scaling of protein sizes. However, recent NMR spectroscopy studies on proteins at high chemical denaturant concentrations suggest the presence of significant amounts of native-like structures, in contrast to the classical random-coil picture. To reconcile these seemingly controversial observations, we examine thermally denatured states of experimentally characterized proteins by using molecular dynamics simulations. For all studied proteins, we find that denatures states indeed have strong local conformational bias toward native states while a random-coil power law scaling of protein sizes is preserved. In addition, we explain why experimentally determined size of the protein creatine kinase does not follow general scaling. In simulations, we observe that this protein exhibits a stable intermediate state, the size of which is consistent with the reported experimental observation.'),(252,'dd_tb05','Ding, F., and Dokholyan, N. V., ','Simple but predictive protein models','Trends in Biotechnology',23,'450','455',2005,'Proteins: Folding','Dokholyan, N. V.','no','no','no','article',16038997,'',NULL,NULL,NULL,'The traditional approach to computational biophysics studies of molecular systems is brute force molecular dynamics simulations under the conditions of interest. The disadvantages of this approach are that the time and length scales that are accessible to computer simulations often do not reach biologically relevant scales. An alternative approach, which we call intuitive modeling, is hypothesis-driven and based on tailoring simplified protein models to the systems of interest. Using intuitive modeling, the length and time scales that can be achieved using simplified protein models exceed those of traditional molecular-dynamic simulations. Here, we describe several recent studies that signify the predictive power of simplified protein models within the intuitive-modeling approach.'),(253,'kwgd_psfb05','Khare, S. D., Wilcox, K. C., Gong, P., and Dokholyan, N. V., ','Sequence and structural determinants of Cu, Zn superoxide dismutase aggregation','Proteins: Structure, Function, and Bioinformatics',61,'617','632',2005,'Proteins: Aggregation','Dokholyan, N. V.','no','no','no','article',16152647,'',NULL,NULL,NULL,'Diverse point mutations in the enzyme Cu, Zn superoxide dismutase (SOD1) are linked to its aggregation in the familial form of the disease amyotrophic lateral sclerosis. The disease-associated mutations are known to destabilize the protein, but the structural basis of the aggregation of the destabilized protein and the structure of aggregates are not well understood. Here, we investigate in silico the sequence and structural determinants of SOD1 aggregation: (1) We identify sequence fragments in SOD1 that have a high aggregation propensity, using only the sequence of SOD1, and (2) we perform molecular dynamics simulations of the SOD1 dimer folding and misfolding. In both cases, we identify identical regions of the protein as having high propensity to form intermolecular interactions. These regions correspond to the N- and C-termini, and two crossover loops and two beta-strands in the Greek-key native fold of SOD1. Our results suggest that the high aggregation propensity of mutant SOD1 may result from a synergy of two factors: the presence of highly amyloidogenic sequence fragments (&quot;hot spots&quot;), and the presence of these fragments in regions of the protein that are structurally most likely to form intermolecular contacts under destabilizing conditions. Therefore, we postulate that the balance between the self-association of aggregation-prone sequences and the specific structural context of these sequences in the native state determines the aggregation propensity of proteins.'),(254,'dgdss_jmb05','Ding, F., Guo, W. H., Dokholyan, N. V., Shakhnovich, E. I., and Shea, J.-E., ','Reconstruction of src SH3 transition state ensemble using multiscale molecular dynamics simulations','Journal of Molecular Biology',350,'1035','1050',2005,'Proteins: Folding','J.-E. Shea','no','no','no','article',15982666,'',NULL,NULL,NULL,'We use an integrated computational approach to reconstruct accurately the transition state ensemble (TSE) for folding of the src-SH3 protein domain. We first identify putative TSE conformations from free energy surfaces generated by importance sampling molecular dynamics for a fully atomic, solvated model of the src-SH3 protein domain. These putative TSE conformations are then subjected to a folding analysis using a coarse-grained representation of the protein and rapid discrete molecular dynamics simulations. Those conformations that fold to the native conformation with a probability (<i>P<sub>fold</sub></i>) of approximately 0.5, constitute the true transition state. Approximately 20% of the putative TSE structures were found to have a <i>P<sub>fold</sub></i> near 0.5, indicating that, although correct TSE conformations are populated at the free energy barrier, there is a critical need to refine this ensemble. Our simulations indicate that the true TSE conformations are compact, with a well-defined central &beta; sheet, in good agreement with previous experimental and theoretical studies. A structured central &beta; sheet was found to be present in a number of pre-TSE conformations, however, indicating that this element, although required in the transition state, does not define it uniquely. An additional tight cluster of contacts between highly conserved residues belonging to the diverging turn and second &beta;-sheet of the protein emerged as being critical elements of the folding nucleus. A number of commonly used order parameters to identify the transition state for folding were investigated, with the number of native C<sup>&beta;</sup> contacts displaying the most satisfactory correlation with <i>P<sub>fold</sub></i> values.'),(255,'kdgd_plscb05','Khare, S. D., Ding, F., Gwanmesia, K. N., and Dokholyan, N. V., ','Molecular origin of polyglutamine-mediated protein aggregation in neurodegenerative diseases','Public Library of Science Computational Biology',1,'e30','',2005,'Proteins: Aggregation','Dokholyan, N. V.','no','yes','no','article',16158094,'',NULL,NULL,NULL,'Expansion of polyglutamine (polyQ) tracts in proteins results in protein aggregation and is associated with cell death in at least nine neurodegenerative diseases. Disease age of onset is correlated with the polyQ insert length above a critical value of 35-40 glutamines. The aggregation kinetics of isolated polyQ peptides in vitro also shows a similar critical-length dependence. While recent experimental work has provided considerable insights into polyQ aggregation, the molecular mechanism of aggregation is not well understood. Here, using computer simulations of isolated polyQ peptides, we show that a mechanism of aggregation is the conformational transition in a single polyQ peptide chain from random coil to a parallel &beta;-helix. This transition occurs selectively in peptides longer than 37 glutamines. In the &beta;-helices observed in simulations, all residues adopt &beta;-strand backbone dihedral angles, and the polypeptide chain coils around a central helical axis with 18.5 &#177; 2 residues per turn. We also find that mutant polyQ peptides with proline-glycine inserts show formation of antiparallel b-hairpins in their ground state, in agreement with experiments. The lower stability of mutant &beta;-helices explains their lower aggregation rates compared to wild type. Our results provide a molecular mechanism for polyQ-mediated aggregation.'),(256,'cd_jmb05','Chen, Y., and Dokholyan, N. V., ','A single disulfide bond differentiates aggregation pathways of &beta;2-microglobulin','Journal of Molecular Biology',354,'473','482',2005,'Proteins: Aggregation','Dokholyan, N. V.','no','no','no','article',16242719,'',NULL,NULL,NULL,'Deposition of wild-type &beta;2-microglobulin (&beta;2m) into amyloid fibrils is a complication in patients undergoing long-term hemodialysis. The native &beta;-sandwich fold of &beta;2m has a highly conserved disulfide bond linking Cys25 and Cys80. Oxidized &beta;2m forms needle-like amyloid fibrils at pH 2.5 in vitro, whereas reduced &beta;2m, at acid pH, in which the intra-chain disulfide bond is disrupted, cannot form typical fibrils. Instead, reduced &beta;2mforms thinner and more flexible filaments. To uncover the difference in molecular mechanisms underlying the aggregation of the oxidized and reduced &beta;2m, we performed molecular dynamics simulations of &beta;2m oligomerization under oxidized and reduced conditions. We show that, consistent with experimental observations, the oxidized &beta;2m forms domain-swapped dimer, in which the two proteins exchange their N-terminal segments complementing each other. In contrast, both dimers and trimers, formed by reduced &beta;2m, are comprised of parallel &beta;-sheets between monomers and stabilized by the hydrogen bond network along the backbone. The oligomerized monomers are in extended conformations, capable of further aggregation. We find that both reduced and oxidized dimers are thermodynamically less stable than their corresponding monomers, indicating that &beta;2m oligomerization is not accompanied by the formation of a thermodynamically stable dimer. Our studies suggest that the different aggregation pathways of oxidized and reduced &beta;2m are dictated by the formation of distinct precursor oligomeric species that are modulated by Cys25-Cys80 disulfide-bonds. We propose that the propagation of domain swapping is the aggregation mechanism for the oxidized &beta;2m, while &quot;parallel stacking&quot; of partially unfolded &beta;2m is the aggregation mechanism for the reduced &beta;2m.'),(257,'dld_jbc05','Ding, F., Larocque, J. J., and Dokholyan, N. V., ','Direct observation of protein folding, aggregation, and a prion-like conformational conversion','Journal of Biological Chemistry',280,'40235','40240',2005,'Proteins: Aggregation','Dokholyan, N. V.','no','no','no','article',16204250,'',NULL,NULL,NULL,'Protein conformational transition from &alpha;-helices to &beta;-sheets precedes aggregation of proteins implicated in many diseases, including Alzheimer and prion diseases. Direct characterization of such transitions is often hindered by the complicated nature of the interaction network among amino acids. A recently engineered small protein-like peptide with a simple amino acid composition features a temperature-driven &alpha;-helix to &beta;-sheet conformational change. Here we studied the conformational transition of this peptide by molecular dynamics simulations. We observed a critical temperature, below which the peptide folds into a.dd &alpha;-helical coiled-coil state and above which the peptide misfolds into &beta;-rich structures with a high propensity to aggregate. The structures adopted by this peptide during low temperature simulations have a backbone root mean square deviation less than 2 A from the crystal structure. At high temperatures, this peptide adopts an amyloid-like structure, which is mainly composed of coiled anti-parallel &beta;-sheets with the cross-&beta;-signature of amyloid fibrils. Most strikingly, we observed conformational conversions in which an &alpha;-helix is converted into a &beta;-strand by proximate stable &beta;-sheets with exposed hydrophobic surfaces and unsaturated hydrogen bonds. Our study suggested a possible generic molecular mechanism of the template-mediated aggregation process, originally proposed by Prusiner (Prusiner, S. B. (1998) Proc. Natl. Acad. Sci. U. S. A. 95, 13363-13383) to account for prion infectivity.'),(258,'dpcd_s06','Ding, F., Prutzman, K. C., Campbell, S. L., and Dokholyan, N. V., ','Topological determinants of protein domain swapping','Structure',14,'5','14',2006,'Proteins: Aggregation','Dokholyan, N. V.','no','no','no','article',16407060,'',NULL,NULL,NULL,'Protein domain swapping has been repeatedly observed in a variety of proteins and is believed to result from destabilization due to mutations or changes in environment. Based on results from our studies and others, we propose that structures of the domain-swapped proteins are mainly determined by their native topologies. We performed molecular dynamics simulations of seven different proteins, known to undergo domain swapping experimentally, under mildly denaturing conditions and found in all cases that the domain-swapped structures can be recapitulated by using protein topology in a simple protein model. Our studies further indicated that, in many cases, domain swapping occurs at positions around which the protein tends to unfold prior to complete unfolding. This, in turn, enabled prediction of protein structural elements that are responsible for domain swapping. In particular, two distinct domain-swapped dimer conformations of the focal adhesion targeting domain of focal adhesion kinase were predicted computationally and were supported experimentally by data obtained from NMR analyses.'),(259,'d_cosb06','Dokholyan, N. V., ','Studies of folding and misfolding using simplified models','Current Opinions in Structural Biology',16,'79','85',2006,'Proteins: Folding','Dokholyan, N. V.','no','no','no','article',16413773,'',NULL,NULL,NULL,'Computer simulations are as vital to our studies of biological systems as experiments. They bridge and rationalize experimental observations, extend the experimental &#39;field of view&#39;, which is often limited to a specific time or length scale, and, most importantly, provide novel insights into biological systems, offering hypotheses about yet-to-be uncovered phenomena. These hypotheses spur further experimental discoveries. Simplified molecular models have a special place in the field of computational biology. Branded as less accurate than all-atom protein models, they have offered what all-atom molecular dynamics simulations could not &#8212; the resolution of the length and time scales of biological phenomena. Not only have simplified models proven to be accurate in explaining or reproducing several biological phenomena, they have also offered a novel multiscale computational strategy for accessing a broad range of time and length scales upon integration with traditional all-atom simulations. Recent computer simulations of simplified models have shaken or advanced the established understanding of biological phenomena. It was demonstrated that simplified models can be as accurate as traditional molecular dynamics approaches in identifying native conformations of proteins. Their application to protein structure prediction yielded phenomenal accuracy in recapitulating native protein conformations. New studies that utilize the synergy of simplified protein models with all-atom models and experiments yielded novel insights into complex biological processes, such as protein folding, aggregation and the formation of large protein complexes.'),(260,'ddwcydycb_pnas06','Demidov, V. V., Dokholyan, N. V., Witte-Hoffman, C., Chalasani, P., Yiu, H.-W., Ding, F., Yu, Y., Cantor, C. R., and Broude, N. E., ','Fast complementation of split fluorescent protein triggered by DNA hybridization','Proceedings of the National Academy of Sciences USA',103,'2052','2056',2006,'Biotechnology','Broude, N. E. & Demidov, V. V.','yes','no','no','article',16461889,'',NULL,NULL,NULL,'Fluorescent proteins have proven to be excellent reporters and biochemical sensors with a wide range of applications. In a split form, they are not fluorescent, but their fluorescence can be restored by supplementary proteinprotein or proteinnucleic acid interactions that reassemble the split polypeptides. However, in prior studies, it took hours to restore the fluorescence of a split fluorescent protein because the formation of the protein chromophore slowly occurred de novo concurrently with reassembly. Here we provide evidence that a fluorogenic chromophore can self-catalytically form within an isolated N-terminal fragment of the enhanced green fluorescent protein (EGFP). We show that restoration of the split protein fluorescence can be driven by nucleic acid complementary interactions. In our assay, fluorescence development is fast (within a few minutes) when complementary oligonucleotide-linked fragments of the split EGFP are combined. The ability of our EGFP system to respond quickly to DNA hybridization should be useful for detecting the kinetics of many other types of pairwise interactions both in vitro and in living cells.'),(261,'kd_pnas06','Khare, S. D., and Dokholyan, N. V., ','Common dynamical signatures of FALS-associated structurally-diverse Cu, Zn superoxide dismutase mutants','Proceedings of the National Academy of Sciences USA',103,'3147','3152',2006,'Proteins: Aggregation','Dokholyan, N. V.','yes','no','no','article',16488975,'',NULL,NULL,NULL,'More than 100 structurally diverse point mutations leading to aggregation in the dimeric enzyme Cu, Zn superoxide dismutase (SOD1) are implicated in familial amyotrophic lateral sclerosis (FALS). Although SOD1 dimer dissociation is a known requirement for its aggregation, the common structural basis for diverse FALS mutations resulting in aggregation is not fully understood. In molecular dynamics simulations of wild-type SOD1 and three structurally diverse FALS mutants (A4V, G37R, and H46R), we find that a common effect of mutations on SOD1 dimer is the mutation-induced disruption of dynamic coupling between monomers. In the wild-type dimer, the principal coupled motion corresponds to a &quot;breathing motion&quot; of the monomers around an axis parallel to the dimer interface, and an opening-closing motion of the distal metal-binding loops. These coupled motions are disrupted in all three mutants independent of the mutation location. Loss of coupled motions in mutant dimers occurs with increased disruption of a key stabilizing structural element (the &beta;-plug) leading to the de-protection of edge strands. To rationalize disruption of coupling, which is independent of the effect of the mutation on global SOD1 stability, we analyze the residueresidue interaction network formed in SOD1. We find that the dimer interface and metal-binding loops, both involved in coupled motions, are regions of high connectivity in the network. Our results suggest that independent of the effect on protein stability, altered protein dynamics, due to long-range communication within its structure, may underlie the aggregation of mutant SOD1 in FALS.'),(262,'bsd_cb06','Bloom, K., Sharma, S., and Dokholyan, N. V., ','The path of DNA in the kinetochore','Current Biology',16,'R276','R278',2006,'Chromatin: Replication','Bloom, K. & Dokholyan, N. V.','yes','no','no','article',16631569,'',NULL,NULL,NULL,'Not applicable for this article type. Please see PDF.'),(263,'cd_tg06','Chen, Y., and Dokholyan, N. V., ','The coordinated evolution of yeast proteins constrained by functional modularity','Trends in Genetics',22,'416','419',2006,'Proteins: Evolution','Dokholyan, N. V.','yes','no','no','article',16797778,'',NULL,NULL,NULL,'Functional modularity is a key attribute of cellular systems and has important roles in evolution. However, the extent to which functional modularity affects protein evolution is largely unknown. Here, we analyzed the evolution of both sequence and expression level of proteins in the yeast <i>Saccharomyces cerevisiae</i> and found that proteins within the same functional modules evolve at more similar rates than those between different modules. We also found stronger co-evolution of expression levels between proteins within functional modules than between them. These results suggest that a coordinated evolution of both the sequence and expression level of proteins is constrained by functional modularity.'),(264,'dd_plscb06','Ding, F., and Dokholyan, N. V., ','Emergence of protein fold families through rational design','Public Library of Science Computational Biology',2,'e85','',2006,'Proteins: Design','Dokholyan, N. V.','yes','yes','no','article',16839198,'',NULL,NULL,NULL,'Diverse proteins with similar structures are grouped into families of homologs and analogs, if their sequence similarity is higher or lower, respectively, than 20%-30%. It was suggested that protein homologs and analogs originate from a common ancestor and diverge in their distinct evolutionary time scales, emerging as a consequence of the physical properties of the protein sequence space. Although a number of studies have determined key signatures of protein family organization, the sequence-structure factors that differentiate the two evolution-related protein families remain unknown. Here, we stipulate that subtle structural changes, which appear due to accumulating mutations in the homologous families, lead to distinct packing of the protein core and, thus, novel compositions of core residues. The latter process leads to the formation of distinct families of homologs. We propose that such differentiation results in the formation of analogous families. To test our postulate, we developed a molecular modeling and design toolkit, Medusa, to computationally design protein sequences that correspond to the same fold family. We find that analogous proteins emerge when a backbone structure deviates only 1-2 A root-mean-square deviation from the original structure. For close homologs, core residues are highly conserved. However, when the overall sequence similarity drops to &#8764;25%-30%, the composition of core residues starts to diverge, thereby forming novel families of protein homologs. This direct observation of the formation of protein homologs within a specific fold family supports our hypothesis. The conservation of amino acids in designed sequences recapitulates that of the naturally occurring sequences, thereby validating our computational design methodology.'),(265,'kcd_ajpfd06','Khare, S. D., Caplow, M., and Dokholyan, N. V., ','FALS mutations in Cu, Zn superoxide dismutase destabilize the dimer and increase dimer dissociation propensity: a large-scale thermodynamic analysis','Amyloid: the Journal of Protein Folding Disorders ',13,'226','235',2006,'Proteins: Aggregation','Dokholyan, N. V.','no','no','no','article',17107883,'',NULL,NULL,NULL,'Mutations in the dimeric enzyme Cu, Zn superoxide dismutase (SOD1) leading to its aggregation are implicated in the toxicity in familial amyotrophic lateral sclerosis (FALS). We and others have previously shown that aggregation occurs by a pathway involving dimer dissociation, metal-loss from monomers and multimeric assembly of apo-SOD1 monomers. We postulate that FALS mutations cause enhanced aggregation by affecting one or more steps in the pathway, and computationally test this postulate for 75 known mis-sense FALS mutants of SOD1. Based on an extensive thermodynamic analysis of the stability of apo-dimer and apo-monomer forms of these mutants, we classify the mutations into the following groups: 70 out of 75 mutations in SOD1 lead to (i) decreased dimer stability, and/or (ii) increased dimer dissociation, compared to wild type, and four mutations lead to (iii) decreased monomer stability compared to wild type. Our results suggest that enhanced aggregation of SOD1 in FALS occurs due to an increased population of mutant SOD1 apomonomers compared to wild type. The dissociation of multimeric proteins induced by diverse mutations may be a common theme in several human diseases.'),(266,'cd_jbc06','Chen, Y., and Dokholyan, N. V., ','Insights into allosteric control of vinculin function from its large-scale conformational dynamics','Journal of Biological Chemistry',281,'29148','29154',2006,'Proteins: Folding','Dokholyan, N. V.','no','no','no','article',16891659,'',NULL,NULL,NULL,'Vinculin is an important constituent of both cell-cell and cellmatrix junctions, where it plays crucial roles in the regulation of cell adhesion and migration. When activated, it mediates the linkage between cadherins (cell-cell) or integrins (cell-matrix) and the actin cytoskeleton through interactions with various proteins. The activation of vinculin requires structural conversions from an autoinhibited conformation to the &quot;open&quot; conformations in which the occluded binding sites of its different ligands become exposed, while the structural dynamics underlying the vinculin activation remains largely unknown. Here we report the first computational study of large scale conformational dynamics of full-length vinculin. We find that the &quot;holding&quot; and &quot;releasing&quot; motions between vinculin tail and pincerlike structure formed by first three domains of vinculin are the dominant motions near the native state of vinculin, indicating that an inherent flexibility of vinculin has a large influence on its allostery. We also find a cooperative dissociation between the head and tail domains of vinculin with increasing temperature in both thermodynamic and kinetic simulations, implying that vinculin may function as an allosteric switch in response to external signals. We show that the kinetics of vinculin unfolding exhibits specific sequential patterns, suggesting that a sophisticated interplay between domains may synergistically contribute to vinculin activation. We further find that the interaction between vinculin-binding site peptide from talin and vinculin significantly destabilizes the intramolecular head-tail interactions, suggesting a direct role of talin binding in vinculin activation.'),(267,'sdnwulpd_b06','Sharma, S., Ding, F., Nie, H., Watson, D., Unnithan, A., Lopp, J., Pozefsky, D., and Dokholyan, N. V., ','iFold: A platform for interactive folding simulations of proteins','Bioinformatics',22,'2693','2694',2006,'Biotechnology','Dokholyan, N. V.','yes','no','no','article',16940324,'',NULL,NULL,NULL,'<strong>Summary:</strong> We built a novel web-based platform for performing discrete molecular dynamics simulations of proteins. In silico protein folding involves searching for minimal frustration in the vast conformational landscape. Conventional approaches for simulating protein folding insufficiently address the problem of simulations in relevant time and length scales necessary for a mechanistic understanding of underlying biomolecular phenomena. Discrete molecular dynamics (DMD) offers an opportunity to bridge the size and timescale gaps and uncover the structural and biological properties of experimentally undetectable protein dynamics. The iFold server supports large-scale simulations of protein folding, thermal denaturation, thermodynamic scan, simulated annealing and <i>p<sub>fold</sub></i> analysis using DMD and coarse-grained protein model with structure-based G&otilde;-interactions between amino acids.<br><strong>Availability:</strong> <a id=maintext href=http://ifold.dokhlab.org target=_blank>http://ifold.dokhlab.org</a>'),(268,'ppedmb_ac06','Petrotchenko, E. V., Pasek, D., Elms, P., Dokholyan, N. V., Meissner, G., and Borchers, C. H., ','Combining fluorescence detection and mass spectrometric analysis for comprehensive and quantitative analysis of redox-sensitive cysteines in native membrane proteins','Analytical Chemistry',78,'7959','7966',2006,'Biotechnology','Borchers, C. H.','no','no','no','article',17134128,'',NULL,NULL,NULL,'Monobromobimane (MBB) is a lipophilic reagent that selectively modifies free cysteine residues in proteins. Because of its lipophilic character, MBB is capable of labeling cysteine residues in membrane proteins under native conditions. Reaction of MBB with the sulfhydryl groups of free cysteines leads to formation of highly fluorescent derivatives. Here we describe a procedure for the detection and relative quantitation of MBB-labeled cysteines using fluorescence and mass spectrometric analyses, which allow determination of free cysteine content and unambiguous identification of MBB-modified cysteine residues. We have applied this approach to the analysis of the free and redox-sensitive cysteine residues of a large membrane protein, the sarcoplasmic reticulum Ca<sup>2+</sup> release channel with a molecular mass of 2.2 million Da. Labeling was performed under physiologic conditions where the channel complex is in its native environment and is functionally active. The purified MBB-labeled channel complex was enzymatically digested, and the resulting peptides were separated by reversed-phase high performance chromatography. MBB-labeled peptides were detected by fluorescence and identified by matrix-assisted laser desorption/ionization (MALDI) mass spectrometry. Under MALDI conditions, partial photolytic fragmentation of the MBB-peptide bound occurred, thus allowing convenient screening for the MBB-modified peptides in the MS spectrum by detection of the specific mass increment of 190.07 Da for MBB-modified cysteine residues. Modification of the peptides was further confirmed by tandem mass spectrometric analysis, utilizing sequencing information and the presence of the specific immonium ion for the MBB-modified cysteine residues at <i>m/z</i> 266.6. Quantitative information was obtained by comparison of both fluorescence and MS signal intensities of MBB-modified peptides. Combination of fluorescence with MS detection and analysis of MBB-labeled peptides supported by a customized software program provides a convenient method for identifying and quantifying redoxsensitive cysteines in membrane proteins of native biological systems. Identification of one redox-sensitive cysteine (2327) in the native membrane-bound sarcoplasmic reticulum Ca<sup>2+</sup> release channel is described.'),(269,'scded_pnas06','Serohijos, A. W., Chen, Y., Ding, F., Elston, T. C., and Dokholyan, N. V., ','A structural model reveals energy transduction in dynein','Proceedings of the National Academy of Sciences USA',103,'18540','18545',2006,'Proteins: Assembly','Dokholyan, N. V. & Elston, T. C.','yes','no','yes','article',17121997,'',NULL,NULL,NULL,'Intracellular active transport is driven by ATP-hydrolyzing motor proteins that move along cytoskeletal filaments. In particular, the microtubule-associated dynein motor is involved in the transport of organelles and vesicles, the maintenance of the Golgi, and mitosis. However, unlike kinesin and myosin, the mechanism by which dynein converts chemical energy into mechanical force remains largely a mystery, due primarily to the lack of a highresolution molecular structure. Using homology modeling and normal mode analysis, we propose a complete atomic structure and a mechanism for force generation by the motor protein dynein. In agreement with very recent electron microscopy (EM) reconstructions showing dynein as a ring-shaped heptamer, our model consists of six ATPases of the AAA (ATPases associated with various cellular activities) superfamily and a C-terminal domain, which is experimentally known to control motor function. Our model shows a coiled coil spanning the diameter of the motor that accounts for previously unidentified structures in EM studies and provides a potential mechanism for long-range communication between the AAA domains. Furthermore, normal mode analysis reveals that the subunits of the motor that contain the nucleotide binding sites exhibit minimal movement, whereas the rest of the motor is very mobile. Our analysis suggests the likely domain rearrangements of the motor unit that generate its power stroke. This study provides insights into the structure and function of dynein that can guide further experimental investigations into energy transduction in dynein.'),(270,'sdd_bj07','Sharma, S., Ding, F., and Dokholyan, N. V., ','Multi-scale modeling of nucleosome dynamics','Biophysical Journal',92,'1457','1470',2007,'Chromatin: Structure','Dokholyan, N. V.','yes','no','no','article',17142268,'',NULL,NULL,NULL,'Nucleosomes form the fundamental building blocks of chromatin. Subtle modifications of the constituent histone tails mediate chromatin stability and regulate gene expression. For this reason, it is important to understand structural dynamics of nucleosomes at atomic levels. We report a novel multiscale model of the fundamental chromatin unit, a nucleosome, using a simplified model for rapid discrete molecular dynamics simulations and an all-atom model for detailed structural investigation. Using a simplified structural model, we perform equilibrium simulations of a single nucleosome at various temperatures. We further reconstruct all-atom nucleosome structures from simulation trajectories. We find that histone tails bind to nucleosomal DNA via strong salt-bridge interactions over a wide range of temperatures, suggesting a mechanism of chromatin structural organization whereby histone tails regulate inter- and intranucleosomal assemblies via binding with nucleosomal DNA. We identify specific regions of the histone core H2A/H2B-H4/H3-H3/H4-H2B/H2A, termed &quot;cold sites&quot;, which retain a significant fraction of contacts with adjoining residues throughout the simulation, indicating their functional role in nucleosome organization. Cold sites are clustered around H3-H3, H2A-H4 and H4-H2A interhistone interfaces, indicating the necessity of these contacts for nucleosome stability. Essential dynamics analysis of simulation trajectories shows that bending across the H3-H3 is a prominent mode of intranucleosomal dynamics. We postulate that effects of salts on mononucleosomes can be modeled in discrete molecular dynamics by modulating histone-DNA interaction potentials. Local fluctuations in nucleosomal DNA vary significantly along the DNA sequence, suggesting that only a fraction of histone-DNA contacts make strong interactions dominating mononucleosomal dynamics. Our findings suggest that histone tails have a direct functional role in stabilizing higher-order chromatin structure, mediated by salt-bridge interactions with adjacent DNA.'),(271,'psdzsg_pa07','Podobnik, B., Shao, J., Dokholyan, N. V., Zlatic, V., Stanley, H. E., and Grosse, I., ','Similarity and dissimilarity in correlations of genomic DNA','Physica A',373,'497','502',2007,'Bioinformatics','Shao, J.','no','no','no','article',0,'',NULL,NULL,NULL,'We analyze auto-correlations of human chromosomes 1-22 and rice chromosomes 1-12 for seven binary mapping rules and find that the correlation patterns are different for different rules but almost identical for all of the chromosomes, despite their varying lengths and GC contents. We propose a simple stochastic process for modeling these correlations, and we find that the proposed process can reproduce, quantitatively and qualitatively, the correlation patterns found in the genomes of human and rice.'),(272,'dnd_pre07','Davis, C. H., Nie, H., and Dokholyan, N. V., ','Insights into thermophilic archaebacterial membrane stability from simplified models of lipid membranes','Physical Review E',75,'51921','',2007,'Lipids','Dokholyan, N. V.','no','no','no','article',17677112,'',NULL,NULL,NULL,'Lipid aggregation into fluid bilayers is an essential process for sustaining life. Simplified models of lipid structure, which allow for long time scales or large length scales not obtainable with all-atom simulations, have recently been developed and show promise for describing lipid dynamics in biological systems. Here, we describe two simplified models, a reduced-lipid model and a bola-lipid model for thermophilic bacterial membranes, developed for use with the rapid discrete molecular dynamics simulation method. In the reducedlipid model, we represent the lipid chain by a series of three beads interacting through pairwise discrete potentials that model hydrophobic attractions between hydrocarbon tails in implicit solvent. Our phase diagram recapitulates those produced by continuous potential models with similar coarse-grained lipid representations. We also find that phase transition temperatures for our reduced-lipid model are dependent upon the flexibility of the lipid chain, giving an insight into archaebacterial membrane stability and prompting development of a bola-lipid model specific for archaebacteria lipids. With both the reduced-lipid and bola-lipid model, we find that the reduced flexibility inherent in archaebacteria lipids yields more stable bilayers as manifested by increased phase transition temperatures. The results of these studies provide a simulation methodology for lipid molecules in biological systems and show that discrete molecular dynamics is applicable to lipid aggregation and dynamics.'),(273,'ydd_nm07','Yin, S., Ding, F., and Dokholyan, N. V., ','Eris: An automated estimator of protein stability','Nature Methods',4,'466','467',2007,'Biotechnology','Dokholyan, N. V.','yes','no','no','article',17538626,'',NULL,NULL,NULL,'Not applicable for this article type. Please see PDF.'),(274,'cdd_jpcb07','Chen, Y., Ding, F., and Dokholyan, N. V., ','Fidelity of the protein structure reconstruction from inter-residue proximity constraints','Journal of Physical Chemistry B',111,'7432','7438',2007,'Proteins: Folding','Dokholyan, N. V.','yes','no','no','article',17542631,'',NULL,NULL,NULL,'Inter-residue proximity constraints obtained in such experiments as cross-linking/mass spectrometry are important sources of information for protein structure determination. A central question in structure determination using these constraints is, What is the minimal number of inter-residue constraints needed to determine the fold of a protein? It is also unknown how the different structural aspects of constraints differentiate their ability in determining the native fold and whether there is a rational strategy for selecting constraints that feature higher fidelity in structure determination. To shed light on these questions, we study the fidelity of protein fold determination using theoretical inter-residue proximity constraints derived from protein native structures and the effect of various subsets of such constraints on fold determination. We show that approximately 70% randomly selected constraints are sufficient for determining the fold of a domain (with an average root-mean-square deviation of &le;3.4 A from their native structures). We find that random constraint selection often outperforms the rational strategy that predominantly favors the constraints representing global structural features. To uncover a strategy for constraint selection for the optimal structure determination, we study the role of the topological properties of these constraints. Interestingly, we do not observe any correlation between various simple topological properties of the selected constraints, emphasizing different global and local structural features, and the performance of these constraints, suggesting that accurate protein structure determination relies on a composite of global and local structural information.'),(275,'ccd_bj07','Chen, Y., Campbell, S. L., and Dokholyan, N. V., ','Deciphering protein dynamics from NMR data using explicit structure sampling and selection','Biophysical Journal',93,'2300','2306',2007,'Proteins: Folding','Dokholyan, N. V.','no','no','no','article',17557784,'',NULL,NULL,NULL,'Perhaps one of the most prominent realizations of recent years is the critical role that protein dynamics plays in many facets of cellular function. While characterization of protein dynamics is fundamental to our understanding of protein function, the ability to explicitly detect an ensemble of protein conformations from dynamics data is a paramount challenge in structural biology. Here, we report a new computational method, Sample and Select, for determining the ensemble of protein conformations consistent with NMR dynamics data. This method can be generalized and extended to different sources of dynamics data, enabling broad applicability in deciphering protein dynamics at different timescales. The structural ensemble derived from Sample and Select will provide structural and dynamic information that should aid us in understanding and manipulating protein function.'),(276,'bjkdd_jbc07','Barton, S., Jacak, R., Khare, S. D., Ding, F., and Dokholyan, N. V., ','The length dependence of the polyQ-mediated protein aggregation','Journal of Biological Chemistry',282,'25487','25492',2007,'Proteins: Aggregation','Dokholyan, N. V.','no','no','no','article',17591778,'',NULL,NULL,NULL,'Polyglutamine (polyQ) repeat disorders are caused by the expansion of CAG tracts in certain genes, resulting in transcription of proteins with abnormally long polyQ inserts.Whenthese inserts expand beyond 3545 glutamines, affected proteins form toxic aggregates, leading to neuron death. Chymotrypsin inhibitor 2 (CI2) with an inserted glutamine repeat has previously been used to model polyQ-mediated aggregation in vitro. However, polyQ insertion lengths in these studies have been kept below the pathogenic threshold. We perform molecular dynamics simulations to study monomer folding dynamics and dimer formation in CI2-polyQ chimeras with insertion lengths of up to 80 glutamines. Our model recapitulates the experimental results of previous studies of chimeric CI2 proteins, showing high folding cooperativity of monomers as well as protein association via domain swapping. Surprisingly, for chimeras with insertion lengths above the pathogenic threshold, monomer folding cooperativity decreases and the dominant mode for dimer formation becomes interglutamine hydrogen bonding. These results support a mechanism for pathogenic polyQ-mediated aggregation, in which expanded polyQ tracts destabilize affected proteins and promote the formation of partially unfolded intermediates. These unfolded intermediates form aggregates through associations by interglutamine interactions.'),(277,'kd_cpps07','Khare, S. D., and Dokholyan, N. V., ','Molecular mechanisms of polypeptide aggregation in human diseases','Current Protein & Peptide Science',8,'573','579',2007,'Proteins: Aggregation','Dokholyan, N. V.','no','no','no','article',18220844,'',NULL,NULL,NULL,'Protein aggregation is implicated in a plethora of neurodegenerative diseases. The proteins found to aggregate in these diseases are unrelated to their native structures and amino acid sequences, but form similar insoluble fibrils with characteristic cross-&beta; sheet morphologies called amyloid in the aggregated state. While both the mechanism of aggregation and the structure of the aggregates are not fully understood at the molecular level, recent studies provide strong support for the idea that protein aggregation into highly stable, insoluble amyloid structures is a general property of the polypeptide chain. For proteins with a unique native state, it is known that aggregation occurs under conditions that promote native-state destabilization <i>in vitro</i> and <i>in vivo</i>. Taken together, the results of several important recent investigations suggest three broad molecular frameworks that may underlie the conversion of normally soluble peptides and proteins into insoluble amyloid fibrils: (1) edge-strand hydrogen bonding, (2) domain-swapping, and (3) self-association of amyloidogenic fragments. We argue that these underlying scenarios are not mutually exclusive and may be protein-dependent - i.e., a protein with a high content of hinge-regions may aggregate <i>via</i> a runaway domain-swap, whereas a protein with a high content of amyloidogenic fragments may aggregat primarily by the self-association of these fragments. These different scenarios provide frameworks to understand the molecular mechanism of polypeptide aggregation.'),(278,'sgtbdc_jmb07','Sharma, S., Gong, P., Temple, B., Bhattacharyya, D., Dokholyan, N. V., and Chaney, S. G., ','Molecular dynamic simulations of cisplatin- and oxaliplatin-d(GG) intrastand cross-links reveal differences in their conformational dynamics','Journal of Molecular Biology',373,'1123','1140',2007,'DNA: Drugs','Chaney, S. G.','no','no','no','article',17900616,'',NULL,NULL,NULL,'Mismatch repair proteins, DNA damage-recognition proteins and translesion DNA polymerases discriminate between Pt-GG adducts containing cisdiammine ligands (formed by cisplatin (CP) and carboplatin) and <i>trans-RR</i>-diaminocyclohexane ligands (formed by oxaliplatin (OX)) and this discrimination is thought to be important in determining differences in the efficacy, toxicity and mutagenicity of these platinum anticancer agents. We have postulated that these proteins recognize differences in conformation and/or conformational dynamics of the DNA containing the adducts. We have previously determined the NMR solution structure of OX-DNA, CP-DNA and undamaged duplex DNA in the 5&#8242;-d(CCTCA<u>GG</u>CCTCC)-3&#8242; sequence context and have shown the existence of several conformational differences in the vicinity of the Pt-GG adduct. Here we have used molecular dynamics simulations to explore differences in the conformational dynamics between OX-DNA, CP-DNA and undamaged DNA in the same sequence context. Twenty-five 10 ns unrestrained fully solvated molecular dynamics simulations were performed starting from two different DNA conformations using AMBER v8.0. All 25 simulations reached equilibrium within 4 ns, were independent of the starting structure and were in close agreement with previous crystal and NMR structures. Our data show that the cis-diammine (CP) ligand preferentially forms hydrogen bonds on the 5&#8242; side of the Pt-GG adduct, while the <i>trans-RR</i>-diaminocyclohexane (OX) ligand preferentially forms hydrogen bonds on the 3&#8242; side of the adduct. In addition, our data show that these differences in hydrogen bond formation are strongly correlated with differences in conformational dynamics, specifically the fraction of time spent in different DNA conformations in the vicinity of the adduct, for CP- and OX-DNA adducts. We postulate that differential recognition of CP- and OX-GG adducts by mismatch repair proteins, DNA damage-recognition proteins and DNA polymerases may be due, in part, to differences in the fraction of time that the adducts spend in a conformation favorable for protein binding.'),(279,'lbddbss_jmb07','Lam, A. R., Borreguero, J. M., Ding, F., Dokholyan, N. V., Buldyrev, S. V., Shakhnovich, E. I., and Stanley, H. E., ','Parallel folding pathways in the SH3 domain protein','Journal of Molecular Biology',373,'1348','1360',2007,'Proteins: Folding','Lam, A. R.','no','no','no','article',17900612,'',NULL,NULL,NULL,'The transition-state ensemble (TSE) is the set of protein conformations with an equal probability to fold or unfold. Its characterization is crucial for an understanding of the folding process. We determined the TSE of the src-SH3 domain protein by using extensive molecular dynamics simulations of the G&otilde; model and computing the folding probability of a generated set of TSE candidate conformations. We found that the TSE possesses a well-defined hydrophobic core with variable enveloping structures resulting from the superposition of three parallel folding pathways. The most preferred pathway agrees with the experimentally determined TSE, while the two least preferred pathways differ significantly. The knowledge of the different pathways allows us to design the interactions between amino acids that guide the protein to fold through the least preferred pathway. This particular design is akin to a circular permutation of the protein. The finding motivates the hypothesis that the different experimentally observed TSEs in homologous proteins and circular permutants may represent potentially available pathways to the wild-type protein.'),(280,'ydd_s07','Yin, S., Ding, F., and Dokholyan, N. V., ','Modeling backbone flexibility improves protein stability estimation','Structure',15,'1567','1576',2007,'Biotechnology','Dokholyan, N. V.','yes','no','no','article',18073107,'',NULL,NULL,NULL,'In designing mutagenesis experiments, it is often crucial to know how certain mutations will affect the structure and thermodynamic stability of the protein. Here, we present a methodology, Eris, to efficiently and accurately compute the stability changes of proteins upon mutations using our protein-modeling suite, Medusa. We evaluate the stability changes upon mutations for 595 mutants from five structurally unrelated proteins, and find significant correlations between the predicted and experimental results. For cases when the high-resolution protein structure is not available, we find that better predictions are obtained by backbone structure prerelaxation. The advantage of our approach is that it is based on physical descriptions of atomic interactions, and does not rely on parameter training with available experimental protein stability data. Unlike other methods, Eris also models the backbone flexibility, thereby allowing for determination of the mutation-induced backbone conformational changes. Eris is freely available via the web server at <a id=maintext href=http://eris.dokhlab.org target=_blank>http://eris.dokhlab.org</a>'),(281,'sdd_fb08','Sharma, S., Ding, F., and Dokholyan, N. V., ','Probing protein aggregation using discrete molecular dynamics','Frontiers in Bioscience',13,'4795','4808',2008,'Proteins: Aggregation','Dokholyan, N. V.','no','no','no','article',18508545,'',NULL,NULL,NULL,'Understanding the role of biomolecular dynamics in cellular processes leading to human diseases and the ability to rationally manipulate these processes is of fundamental importance in scientific research. The last decade has witnessed significant progress in probing biophysical behavior of proteins. However, we are still limited in understanding how changes in protein dynamics and inter-protein interactions occurring in short length- and time-scales lead to aberrations in their biological function. Bridging this gap in biology probed using computer simulations marks a challenging frontier in computational biology. Here we examine hypothesis-driven simplified protein models in conjunction with discrete molecular dynamics in the study protein aggregations, implicated in series of neurodegenerative diseases, such as Alzheimer&#39;s and Huntington&#39;s diseases. Discrete molecular dynamics simulations of simplified protein models have emerged as a powerful methodology with its ability to bridge the gap in time and length scales from protein dynamics to aggregation, and provide an indispensable tool for probing protein aggregation.'),(282,'cdnsswyd_abb08','Chen, Y., Ding, F., Nie, H., Serohijos, A. W., Sharma, S., Wilcox, K. C., Yin, S., and Dokholyan, N. V., ','Protein folding: then and now','Archives of Biochemistry and Biophysics',469,'4','19',2008,'Proteins: Folding','Dokholyan, N. V.','no','no','no','article',17585870,'',NULL,NULL,NULL,'Over the past three decades the protein folding field has undergone monumental changes. Originally a purely academic question, how a protein folds has now become vital in understanding diseases and our abilities to rationally manipulate cellular life by engineering protein folding pathways. We review and contrast past and recent developments in the protein folding field. Specifically, we discuss the progress in our understanding of protein folding thermodynamics and kinetics, the properties of evasive intermediates, and unfolded states. We also discuss how some abnormalities in protein folding lead to protein aggregation and human diseases.'),(283,'sd_bj08','Sharma, S., and Dokholyan, N. V., ','DNA sequence mediates nucleosome structure and stability','Biophysical Journal',94,'1','3',2008,'Chromatin','Dokholyan, N. V.','no','no','no','article',17933872,'',NULL,NULL,NULL,'Not applicable for this article type. Please see PDF.'),(284,'shrd_plscb08','Serohijos, A. W. R., Hegedus, T., Riordan, J. R., and Dokholyan, N. V., ','Diminished self-chaperoning activity of the &Delta;F508 mutant CFTR results in protein misfolding','Public Library of Science Computational Biology',4,'e1000008','',2008,'Proteins: Folding','Dokholyan, N. V.','no','no','no','article',18463704,'',NULL,NULL,NULL,'The absence of a functional ATP Binding Cassette (ABC) protein called the Cystic Fibrosis Transmembrane Conductance Regulator (CFTR) from apical membranes of epithelial cells is responsible for cystic fibrosis (CF). Over 90% of CF patients carry at least one mutant allele with deletion of phenylalanine at position 508 located in the N-terminal nucleotide binding domain (NBD1). Biochemical and cell biological studies show that the &Delta;F508 mutant exhibits inefficient biosynthetic maturation and susceptibility to degradation probably due to misfolding of NBD1 and the resultant misassembly of other domains. However, little is known about the direct effect of the Phe508 deletion on the NBD1 folding, which is essential for rational design strategies of cystic fibrosis treatment. Here we show that the deletion of Phe508 alters the folding dynamics and kinetics of NBD1, thus possibly affecting the assembly of the complete CFTR. Using molecular dynamics simulations, we find that meta-stable intermediate states appearing on wild type and mutant folding pathways are populated differently and that their kinetic accessibilities are distinct. The structural basis of the increased misfolding propensity of the &Delta;F508 NBD1 mutant is the perturbation of interactions in residue pairs Q493/P574 and F575/F578 found in loop S7-H6. As a proof of- principle that the S7-H6 loop conformation can modulate the folding kinetics of NBD1, we virtually design rescue mutations in the identified critical interactions to force the S7-H6 loop into the wild type conformation. Two redesigned NBD1-&Delta;F508 variants exhibited significantly higher folding probabilities than the original NBD1-&Delta;F508, thereby partially rescuing folding ability of the NBD1-&Delta;F508 mutant. We propose that these observed defects in folding kinetics of mutant NBD1 may also be modulated by structures separate from the 508 site. The identified structural determinants of increased misfolding propensity of NBD1-&Delta;F508 are essential information in correcting this pathogenic mutant.'),(285,'shahcdr_pnas08','Serohijos, A. W. R., Hegedus, T., Aleksandrov, A. A., He, L., Cui, L., Dokholyan, N. V., and Riordan, J. R., ','Phenylalanine-508 mediates a cytoplasmic-membrane domain contact in the CFTR 3D structure crucial to assembly and channel function','Proceedings of the National Academy of Sciences USA',105,'3256','3261',2008,'Proteins: Folding','Riordan, J. R.; Dokholyan, N. V.','yes','no','no','article',18305154,'',NULL,NULL,NULL,'Deletion of phenylalanine-508 (Phe-508) from the N-terminal nucleotide- binding domain (NBD1) of the cystic fibrosis transmembrane conductance regulator (CFTR), a member of the ATP-binding cassette (ABC) transporter family, disrupts both its folding and function and causes most cystic fibrosis. Most mutant nascent chains do not pass quality control in the ER, and those that do remain thermally unstable, only partially functional, and are rapidly endocytosed and degraded. Although the lack of the Phe-508 peptide backbone diminishes the NBD1 folding yield, the absence of the aromatic side chain is primarily responsible for defective CFTR assembly and channel gating. However, the site of interdomain contact by the side chain is unknown as is the high-resolution 3D structure of the complete protein. Here we present a 3D structure of CFTR, constructed by molecular modeling and supported biochemically, in which Phe-508 mediates a tertiary interaction between the surface of NBD1 and a cytoplasmic loop (CL4) in the C-terminal membrane-spanning domain (MSD2). This crucial cytoplasmic membrane interface, which is dynamically involved in regulation of channel gating, explains the known sensitivity of CFTR assembly to many disease-associated mutations in CL4 as well as NBD1 and provides a sharply focused target for small molecules to treat CF. In addition to identifying a key intramolecular site to be repaired therapeutically, our findings advance understanding of CFTR structure and function and provide a platform for focused biochemical studies of other features of this unique ABC ion channel.'),(286,'dscdbd_r08','Ding, F., Sharma, S., Chalasani, P., Demidov, V. V., Broude, N. E., and Dokholyan, N. V., ','Ab initio RNA folding by discrete molecular dynamics: from structure prediction to folding mechanisms','RNA',14,'1164','1173',2008,'RNA','Dokholyan, N. V.','yes','no','no','article',18456842,'',NULL,NULL,NULL,'RNA molecules with novel functions have revived interest in the accurate prediction of RNA three-dimensional (3D) structure and folding dynamics. However, existing methods are inefficient in automated 3D structure prediction. Here, we report a robust computational approach for rapid folding of RNA molecules. We develop a simplified RNA model for discrete molecular dynamics (DMD) simulations, incorporating base-pairing and base-stacking interactions. We demonstrate correct folding of 150 structurally diverse RNA sequences. The majority of DMD-predicted 3D structures have &lt;4 A deviations from experimental structures. The secondary structures corresponding to the predicted 3D structures consist of 94% native base-pair interactions. Folding thermodynamics and kinetics of tRNA<sup>Phe</sup>, pseudoknots, and mRNA fragments in DMD simulations are in agreement with previous experimental findings. Folding of RNA molecules features transient, non-native conformations, suggesting nonhierarchical RNA folding. Our method allows rapid conformational sampling of RNA folding, with computational time increasing linearly with RNA length. We envision this approach as a promising tool for RNA structural and functional analyses.'),(287,'hsdhr_jmb08','Hegedus, T. Z., Serohijos, A. W. R., Dokholyan, N. V., He, L., and Riordan, J. R., ','Computational studies reveal phosphorylation dependent changes in the unstructured R domain of CFTR','Journal of Molecular Biology',378,'1052','1063',2008,'Proteins: Folding','Riordan, J. R.','no','no','no','article',18423665,'',NULL,NULL,NULL,'The cystic fibrosis transmembrane conductance regulator (CFTR) is a cAMP-dependent chloride channel that is mutated in cystic fibrosis, an inherited disease of high morbidity and mortality. The phosphorylation of its &#8764;200 amino acid R domain by protein kinase A is obligatory for channel gating under normal conditions. The R domain contains more than ten PKA phosphorylation sites. No individual site is essential but phosphorylation of increasing numbers of sites enables progressively greater channel activity. In spite of numerous studies of the role of the R domain in CFTR regulation, its mechanism of action remains largely unknown. This is because neither its structure nor its interactions with other parts of CFTR have been completely elucidated. Studies have shown that the R domain lacks welldefined secondary structural elements and is an intrinsically disordered region of the channel protein. Here, we have analyzed the disorder pattern and employed computational methods to explore low-energy conformations of the R domain. The specific disorder and secondary structure patterns detected suggest the presence of molecular recognition elements (MoREs) that may mediate phosphorylation-regulated intra- and interdomain interactions. Simulations were performed to generate an ensemble of accessible R domain conformations. Although the calculated structures may represent more compact conformers than occur in vivo, their secondary structure propensities are consistent with predictions and published experimental data. Equilibrium simulations of a mimic of a phosphorylated R domain showed that it exhibited an increased radius of gyration. In one possible interpretation of these findings, by changing its size, the globally unstructured R domain may act as an entropic spring to perturb the packing of membrane-spanning sequences that constitute the ion permeability pathway and thereby activate channel gating.'),(288,'dtnd_s08','Ding, F., Tsao, D., Nie, H., and Dokholyan, N. V., ','Ab initio folding of proteins with all-atom discrete molecular dynamics','Structure',16,'1010','1018',2008,'Proteins: Folding','Dokholyan, N. V.','yes','no','no','article',18611374,'',NULL,NULL,NULL,'Discrete molecular dynamics (DMD) is a rapid sampling method used in protein folding and aggregation studies. Until now, DMD was used to perform simulations of simplified protein models in conjunction with structure-based force fields. Here, we develop an all-atom protein model and a transferable force field featuring packing, solvation, and environmentdependent hydrogen bond interactions. We performed folding simulations of six small proteins (20-60 residues) with distinct native structures by the replica exchange method. In all cases, native or near-native states were reached in simulations. For three small proteins, multiple folding transitions are observed, and the computationally characterized thermodynamics are in qualitative agreement with experiments. The predictive power of all-atom DMD highlights the importance of environment-dependent hydrogen bond interactions in modeling protein folding. The developed approach can be used for accurate and rapid sampling of conformational spaces of proteins and protein-protein complexes and applied to protein engineering and design of protein-protein interactions.'),(289,'tfddtr_plscb08','Teotico, D. G., Frazier, M. L., Ding, F., Dokholyan, N. V., Temple, B. R. S., and Redinbo, M. R., ','Active nuclear receptors exhibit highly correlated AF-2 domain motions','Public Library of Science Computational Biology',4,'e1000111','',2008,'Proteins: Folding','Redinbo, M. R.','no','yes','no','article',18617990,'',NULL,NULL,NULL,'Nuclear receptor ligand binding domains (LBDs) convert ligand binding events into changes in gene expression by recruiting transcriptional coregulators to a conserved activation function-2 (AF-2) surface. While most nuclear receptor LBDs form homo- or heterodimers, the human nuclear receptor pregnane X receptor (PXR) forms a unique and essential homodimer and is proposed to assemble into a functional heterotetramer with the retinoid X receptor (RXR). How the homodimer interface, which is located 30 A from the AF-2, would affect function at this critical surface has remained unclear. By using 20- to 30-ns molecular dynamics simulations on PXR in various oligomerization states, we observed a remarkably high degree of correlated motion in the PXRRXR heterotetramer, most notably in the four helices that create the AF-2 domain. The function of such correlation may be to create &quot;active-capable&quot; receptor complexes that are ready to bind to transcriptional coactivators. Indeed, we found in additional simulations that active-capable receptor complexes involving other orphan or steroid nuclear receptors also exhibit highly correlated AF-2 domain motions. We further propose a mechanism for the transmission of long-range motions through the nuclear receptor LBD to the AF-2 surface. Taken together, our findings indicate that long-range motions within the LBD scaffold are critical to nuclear receptor function by promoting a mobile AF-2 state ready to bind coactivators.'),(290,'cd_mbe08','Chen, Y., and Dokholyan, N. V., ','Natural selection against protein aggregation on self-interacting and essential proteins in yeast, fly and worm','Molecular Biology and Evolution',25,'1530','1533',2008,'Proteins: Evolution','Dokholyan, N. V.','no','no','no','article',18503047,'',NULL,NULL,NULL,'Protein aggregation is the phenomenon of protein self-association potentially leading to detrimental effects on physiology, which is closely related to numerous human diseases such as Alzheimer&#39;s and Parkinson&#39;s disease. Despite progress in understanding the mechanism of protein aggregation, how natural selection against protein aggregation acts on subunits of protein complexes and on proteins with different contributions to organism fitness remains largely unknown. Here, we perform a proteome-wide analysis by using an experimentally validated algorithm TANGO and utilizing sequence, interactomic and phenotype-based functional genomic data from yeast, fly, and nematode. We find that proteins that are capable of forming homooligomeric complex have lower aggregation propensity compared with proteins that do not function as homooligomer. Further, proteins that are essential to the fitness of an organism have lower aggregation propensity compared with nonessential ones. Our finding suggests that the selection force against protein aggregation acts across different hierarchies of biological system.'),(291,'hsntwsdb_plscb08','Hao, J., Serohijos, A. W. R., Newton, G., Tassone, G., Wang, Z., Sgroi, D. C., Dokholyan, N. V., and Basilion, J. P., ','Identification of peptide ligands to CRIP1, a novel biomarker for cancers','Public Library of Science Computational Biology',4,'e1000138','',2008,'Biotechnology','Dokholyan, N. V., Basilion, J. P.','yes','no','no','article',18670594,'',NULL,NULL,NULL,'Cysteine-rich intestinal protein 1 (CRIP1) has been identified as a novel marker for early detection of cancers. Here we report on the use of phage display in combination with molecular modeling to identify a high-affinity ligand for CRIP1. Panning experiments using a circularized C7C phage library yielded several consensus sequences with modest binding affinities to purified CRIP1. Two sequence motifs, A1 and B5, having the highest affinities for CRIP1, were chosen for further study. With peptide structure information and the NMR structure of CRIP1, the higher-affinity A1 peptide was computationally redesigned, yielding a novel peptide, A1M, whose affinity was predicted to be much improved. Synthesis of the peptide and saturation and competitive binding studies demonstrated approximately a 10-fold improvement in the affinity of A1M compared to that of either A1 or B5 peptide. These techniques have broad application to the design of novel ligand peptides.'),(292,'sdd_b08','Sharma, S., Ding, F., and Dokholyan, N. V., ','iFoldRNA: Three-dimensional RNA structure prediction and folding','Bioinformatics',24,'1951','1952',2008,'RNA','Dokholyan, N. V.','yes','no','no','article',18579566,'',NULL,NULL,NULL,'<strong>Summary:</strong> Three-dimensional RNA structure prediction and folding is of significant interest in the biological research community. Here, we present iFoldRNA, a novel web-based methodology for RNA structure prediction with near atomic resolution accuracy and analysis of RNA folding thermodynamics. iFoldRNA rapidly explores RNA conformations using discrete molecular dynamics simulations of input RNA sequences. Starting from simplified linear chain conformations, RNA molecules (&lt;50 nt) fold to native-like structures within half an hour of simulation, facilitating rapid RNA structure prediction. All-atom reconstruction of energetically stable conformations generates iFoldRNA predicted RNA structures. The predicted RNA structures are within 25 A root mean squre deviations (RMSDs) from corresponding experimentally derived structures. RNA folding parameters including specific heat, contact maps, simulation trajectories, gyration radii, RMSDs from native state, fraction of native-like contacts are accessible from iFoldRNA. We expect iFoldRNA will serve as a useful resource for RNA structure prediction and folding thermodynamic analyses.<br><strong>Availability:</strong> <a id=maintext href=http://iFoldRNA.dokhlab.org target=_blank>http://iFoldRNA.dokhlab.org</a>'),(293,'ybvd_jcim08','Yin, S., Biedermannova, L., Vondrasek, J., and Dokholyan, N. V., ','MedusaScore: An accurate force-field based scoring function for virtual drug screening','Journal of Chemical Information and Modeling',48,'1656','1662',2008,'Drugs','Dokholyan, N. V.','yes','no','no','article',18672869,'',NULL,NULL,NULL,'Virtual screening is becoming an important tool for drug discovery. However, the application of virtual screening has been limited by the lack of accurate scoring functions. Here, we present a novel scoring function, MedusaScore, for evaluating protein-ligand binding. MedusaScore is based on models of physical interactions that include van der Waals, solvation, and hydrogen bonding energies. To ensure the best transferability of the scoring function, we do not use any protein-ligand experimental data for parameter training. We then test the MedusaScore for docking decoy recognition and binding affinity prediction and find superior performance compared to other widely used scoring functions. Statistical analysis indicates that one source of inaccuracy of MedusaScore may arise from the unaccounted entropic loss upon ligand binding, which suggests avenues of approach for further MedusaScore improvement.'),(294,'olhlvghdsvs_jbc08','Ozvegy-Laczka, C., Laczko, R., Hegedus, C., Litman, T., Varady, G., Goda, K., Hegedus, T. Z., Dokholyan, N. V., Sorrentino, B. P., Varadi, A., Sarkadi, B., ','Interaction with the 5D3 monoclonal antibody is regulated by intramolecular rearrangements but not by covalent dimer formation of the human ABCG2 multidrug transporter','Journal of Biological Chemistry',283,'26059','26070',2008,'Proteins: Transmembrane','Sarkadi, B.','no','no','no','article',18644784,'',NULL,NULL,NULL,'Human ABCG2 is a plasma membrane glycoprotein working as a homodimer or homo-oligomer. The protein plays an important role in the protection/detoxification of various tissues and may also be responsible for the multidrug-resistant phenotype of cancer cells. In our previous study we found that the 5D3 monoclonal antibody shows a function-dependent reactivity to an extracellular epitope of the ABCG2 transporter. In the current experiments we have further characterized the 5D3- ABCG2interaction. The effect of chemical cross-linking and the modulation of extracellular SS bridges on the transporter function and 5D3 reactivity of ABCG2 were investigated in depth. We found that several protein cross-linkers greatly increased 5D3 labeling in ABCG2 expressing HEK cells; however, there was no correlation between covalent dimer formation, the inhibition of transport activity, and the increase in 5D3 binding. Dithiothreitol treatment, which reduced the extracellular SS bridge-forming cysteines of ABCG2, had no effect on transport function but caused a significant decrease in 5D3 binding. When analyzing ABCG2 mutants carrying Cys-to-Ala changes in the extracellular loop, we found that the mutant C603A (lacking the intermolecular SS bond) showed comparable transport activity and 5D3 reactivity to the wild-type ABCG2. However, disruption of the intramolecular SS bridge (in C592A, C608A, or C592A/C608A mutants) in this loop abolished 5D3 binding, whereas the function of the protein was preserved. Based on these results and ab initio folding simulations, we propose a model for the large extracellular loop of the ABCG2 protein.'),(300,'sshlbjbbddd_s09','Schwede, T., Sali, A., Honig, B., Levitt, M., Berman, H. M., Jones, B., Brenner, S., Burley, S. K., Das, R., Dokholyan, N. V., Dunbrack, R. L. Jr., Fidelis, K., Fiser, A., Godzik, A., Humblet, C., Jacobson, M. P., Joachimiak, A., Krystek, S., Kortemme, T., Kryshtafovych, A., Montelione, G., Moult, J., Murray, D., Sanchez, R., Sosnick, T. R., Standley, D. M., Stouch, T., Vajda, S., Westbrook, J. D., and Wilson, I. A., ','Outcome of a workshop on applications of protein models in biomedical research','Structure',17,'151','159',2009,'Proteins','Schwede, T.','no','no','no','article',19217386,'',NULL,NULL,NULL,'We describe the proceedings and conclusions from the &quot;Workshop on Applications of Protein Models in Biomedical Research&quot; (the Workshop) that was held at the University of California, San Francisco on 11 and 12 July, 2008. At the Workshop, international scientists involved with structure modeling explored (i) how models are currently used in biomedical research, (ii) the requirements and challenges for different applications, and (iii) how the interaction between the computational and experimental research communities could be strengthened to advance the field.'),(301,'glddw_jacs09','Gherghe, C. M., Leonard, C. W., Ding, F., Dokholyan, N. V., and Weeks, K. M., ','Native-like RNA tertiary structures using a sequence-encoded cleavage agent and refinement by discrete molecular dynamics','Journal of the American Chemical Society',131,'2541','2546',2009,'RNA','Dokholyan, N. V., Weeks, K. M.','yes','no','no','article',19193004,'',NULL,NULL,NULL,'The difficulty of analyzing higher order RNA structure, especially for folding intermediates and for RNAs whose functions require domains that are conformationally flexible, emphasizes the need for new approaches for modeling RNA tertiary structure accurately. Here, we report a concise approach that makes use of facile RNA structure probing experiments that are then interpreted using a computational algorithm, carefully tailored to optimize both the resolution and refinement speed for the resulting structures, without requiring user intervention. The RNA secondary structure is first established using SHAPE chemistry. We then use a sequence-directed cleavage agent, which can be placed arbitrarily in many helical motifs, to obtain high quality inter-residue distances. We interpret this in-solution chemical information using a fast, coarse grained, discrete molecular dynamics engine in which each RNA nucleotide is represented by pseudoatoms for the phosphate, ribose, and nucleobase groups. By this approach, we refine base paired positions in yeast tRNA<sup>Asp</sup> to 4 A rmsd without any preexisting information or assumptions about secondary or tertiary structures. This blended experimental and computational approach has the potential to yield native-like models for the diverse universe of functionally important RNAs whose structures cannot be characterized by conventional structural methods.'),(296,'dd_pnas08','Ding, F., and Dokholyan, N. V., ','Dynamical roles of metal ions and the disulfide bond in Cu, Zn superoxide dismutase folding and aggregation','Proceedings of the National Academy of Sciences USA',105,'19696','19701',2008,'Proteins: Aggregation','Dokholyan, N. V.','yes','no','no','article',19052230,'',NULL,NULL,NULL,'Misfolding and aggregation of Cu, Zn superoxide dismutase (SOD1) is implicated in neuronal death in amyotrophic lateral sclerosis. Each SOD1 monomer binds to 1 copper and 1 zinc ion and maintains its disulfide bond (Cys-57-Cys-146) in the reducing cytoplasm of cell. Mounting experimental evidence suggests that metal loss and/or disulfide reduction are important for initiating misfolding and aggregation of SOD1. To uncover the role of metals and the disulfide bond in the SOD1 folding, we systemically study the folding thermodynamics and structural dynamics of SOD1 monomer and dimer with and without metal binding and under disulfide-intact or disulfide reduced environments in computational simulations. We use all-atom discrete molecular dynamics for sampling. Our simulation results provide dynamical evidence to the stabilizing role of metal ions in both dimer and monomer SOD1. The disulfide bond anchors a loop (Glu-49 to Asn-53) that contributes to the dimer interface. The reduction of the disulfide bond in SOD1 with metal ions depleted results in a flexible Glu-49-Asn-53 loop, which, in turn, disrupts dimer formation. Interestingly, the disulfide bond reduction does not affect the thermostability of monomer SOD1 as significantly as the metal ions do. We further study the structural dynamics of metal-free SOD1 monomers, the precursor for aggregation, in simulations and find inhomogeneous local unfolding of &beta;-strands. The strands protected by the metal-binding and electrostatic loops are found to unfold first after metal loss, leading to a partially unfolded &beta;-sheet prone to aggregation. Our simulation study sheds light on the critical role of metals and disulfide bond in SOD1 folding and aggregation.'),(295,'hashcadr_jbc08','He, L., Aleksandrov, L. A., Serohijos, A. W. R., Hegedus, T., Cui, L., Aleksandrov, A. A., Dokholyan, N. V., and Riordan, J. R.,','Multiple membrane-cytoplasmic domain contacts in CFTR','Journal of Biological Chemistry',283,'26383','26390',2008,'Proteins: Folding','Riordan, J. R.','yes','no','no','article',18658148,'',NULL,NULL,NULL,'The cystic fibrosis transmembrane conductance regulator (CFTR) is a unique ATP-binding cassette (ABC) ion channel mutated in patients with cystic fibrosis. The most common mutation, deletion of phenylalanine 508 (F508) and many other disease-associated mutations occur in the nucleotide binding domains (NBD) and the cytoplasmic loops (CL) of the membrane-spanning domains (MSD). A recently constructed computational model of the CFTR three-dimensional structure, supported by experimental data (Serohijos, A. W., Hegedus, T., Aleksandrov, A. A., He, L., Cui, L., Dokholyan, N. V., and Riordan, J. R. (2008) Proc. Natl. Acad. Sci. U. S. A. 105, 3256-3261) revealed that several of these mutations including F508 disrupted interfaces between these domains. Here we have used cysteine cross-linking experiments to verify all NBD/CL interfaces predicted by the structural model and observed that their cross-linking has a variety of different effects on channel gating. The interdomain contacts comprise aromatic clusters important for stabilization of the interfaces and also involve the Q-loops and X-loops that are in close proximity to the ATP binding sites. Cross-linking of all domain-swapping contacts between NBDs andMSDcytoplasmic loops in opposite halves of the protein rapidly and reversibly arrest single channel gating while those in the same halves have lesser impact. These results reinforce the idea that mediation of regulatory signals between cytoplasmic- and membrane-integrated domains of the CFTR channel apparently relies on an array of precise but highly dynamic interdomain structural joints.'),(229,'dss_pnas02','Dokholyan, N. V., Shakhnovich, B. E., and Shakhnovich, E. I., ','Expanding protein universe and its origin from the biological Big Bang','Proceedings of the National Academy of Sciences USA',99,'','14136',2002,'Proteins: Evolution','Dokholyan, N. V.','no','no','no','article',12384571,'',NULL,NULL,NULL,'The bottom-up approach to understanding the evolution of organisms is by studying molecular evolution. With the large number of protein structures identified in the past decades, we have discovered peculiar patterns that nature imprints on protein structural space in the course of evolution. In particular, we have discovered that the universe of protein structures is organized hierarchically into a scale-free network. By understanding the cause of these patterns, we attempt to glance at the very origin of life.'),(230,'ddbss_bj02','Ding, F., Dokholyan, N. V., Buldyrev, S. V., Stanley, H. E., and Shakhnovich, E. I., ','Direct molecular dynamics observation of protein folding transition state ensemble','Biophysical Journal',83,'3525','3532',2002,'Proteins: Folding','Ding, F.','no','no','no','article',12496119,'',NULL,NULL,NULL,'The concept of the protein transition state ensemble (TSE), a collection of the conformations that have 50% probability to convert rapidly to the folded state and 50% chance to rapidly unfold, constitutes the basis of the modern interpretation of protein engineering experiments. It has been conjectured that conformations constituting the TSE in many proteins are the expanded and distorted forms of the native state built around a specific folding nucleus. This view has been supported by a number of on-lattice and off-lattice simulations. Here we report a direct observation and characterization of the TSE by molecular dynamic folding simulations of the C-Src SH3 domain, a small protein that has been extensively studied experimentally. Our analysis reveals a set of key interactions between residues, conserved by evolution, that must be formed to enter the kinetic basin of attraction of the native state.'),(227,'dlds_pnas02','Dokholyan, N. V., Li, L., Ding, F., and Shakhnovich, E. I., ','Topological determinants of protein folding','Proceedings of the National Academy of Sciences USA',99,'8637','8641',2002,'Proteins: Folding','Dokholyan, N. V.','no','no','no','article',12084924,'',NULL,NULL,NULL,'The folding of many small proteins is kinetically a two-state process that represents overcoming the major free-energy barrier. A kinetic characteristic of a conformation, its probability to descend to the native state domain in the amount of time that represents a small fraction of total folding time, has been introduced to determine to which side of the free-energy barrier a conformation belongs. However, which features make a protein conformation on the folding pathway become committed to rapidly descending to the native state has been a mystery. Using two small, well characterized proteins, CI2 and C-Src SH3, we show how topological properties of protein conformations determine their kinetic ability to fold. We use a macroscopic measure of the protein contact network topology, the average graph connectivity, by constructing graphs that are based on the geometry of protein conformations. We find that the average connectivity is higher for conformations with a high folding probability than for those with a high probability to unfold. Other macroscopic measures of protein structural and energetic properties such as radius of gyration, rms distance, solvent-accessible surface area, contact order, and potential energy fail to serve as predictors of the probability of a given conformation to fold.'),(228,'vdpk_pre02','Vendruscolo, M., Dokholyan, N. V., Paci, E., and Karplus, M., ','A small-world view of the amino acids that play a key role in protein folding','Physical Review E',65,'61910','',2002,'Proteins: Folding','','no','no','no','article',12188762,'',NULL,NULL,NULL,'We use geometrical considerations to provide a different perspective on the fact that a few selected amino acids, the so-called &quot;key residues,&quot; act as nucleation centers for protein folding. By constructing graphs corresponding to protein structures we show that they have the &quot;small-world&quot; feature of having a limited set of vertices with large connectivity. These vertices correspond to the key residues that play the role of &quot;hubs&quot; in the network of interactions that stabilize the structure of the transition state.'),(226,'dms_pa02','Dokholyan, N. V., Mirny, L. A., and Shakhnovich, E. I., ','Understanding conserved amino acids in proteins','Physica A',314,'600','606',2002,'Proteins: Evolution','Dokholyan, N. V.','no','no','no','article',0,'',NULL,NULL,NULL,'It has been conjectured that evolution exerted pressure to preserve amino acids bearing thermodynamic, kinetic, and functional roles. In this letter we show that the physical requirement to maintain protein stability gives rise to a sequence conservatism pattern that is in remarkable agreement with that found in natural proteins. Based on the physical properties of amino acids, we propose a model of evolution that explains conserved amino acids across protein families sharing the same fold.'),(225,'bdbss_jmb02','Borreguero, J. M., Dokholyan, N. V., Buldyrev, S. V., Stanley, H. E., and Shakhnovich, E. I., ','Thermodynamics and folding kinetics analysis of the SH3 domain from Discrete Molecular Dynamics','Journal of Molecular Biology',318,'863','876',2002,'Proteins: Folding','Borreguero, J. M.','no','no','no','article',12054829,'',NULL,NULL,NULL,'We perform a detailed analysis of the thermodynamics and folding kinetics of the SH3 domain fold with discrete molecular dynamic simulations. We propose a protein model that reproduces some of the experimentally observed thermodynamic and folding kinetic properties of proteins. Specifically, we use our model to study the transition state ensemble of the SH3 fold family of proteins, a set of unstable conformations that fold to the protein native state with probability 1/2. We analyze the participation of each secondary structure element formed at the transition state ensemble. We also identify the folding nucleus of the SH3 fold and test extensively its importance for folding kinetics. We predict that a set of amino acid contacts between the RT-loop and the distal hairpin are the critical folding nucleus of the SH3 fold and propose a hypothesis that explains this result.'),(224,'dpbs_pre02','Dokholyan, N. V., Pitard, E., Buldyrev, S. V., and Stanley, H. E., ','Glassy behavior of a homopolymer from molecular dynamics simulations','Physical Review E',65,'030801(R)','',2002,'Proteins: Polymers','','no','no','no','article',11909019,'',NULL,NULL,NULL,'We study at- and out-of-equilibrium dynamics of a single homopolymer chain at low temperature using molecular dynamics. The main quantities of interest are the average root mean square displacement of the monomers below the &theta; point, and the structure factor, as a function of time. The observation of these quantities show a close resemblance to those measured in structural glasses and suggest that the polymer chain in its low temperature phase is in a glassy phase, with its dynamics dominated by traps. In equilibrium, at low temperature, we observe the trapping of the monomers and a slowing down of the overall motion of the polymer as well as non-exponential relaxation of the structure factor. Out of equilibrium, at low temperatures, we compute the two-time quantities and observe breaking of ergodicity in a range of waiting times, with the onset of aging.'),(221,'ds_jmb01','Dokholyan, N. V., and Shakhnovich, E. I., ','Understanding hierarchical protein evolution from first principles','Journal of Molecular Biology',312,'289','307',2001,'Proteins: Evolution','Dokholyan, N. V.','no','no','no','article',11545603,'',NULL,NULL,NULL,'We propose a model that explains the hierarchical organization of proteins in fold families. The model, which is based on the evolutionary selection of proteins by their native state stability, reproduces patterns of amino acids conserved across protein families. Due to its dynamic nature, the model sheds light on the evolutionary time-scales. By studying the relaxation of the correlation function between consecutive mutations at a given position in proteins, we observe separation of the evolutionary time-scales: at short time intervals families of proteins with similar sequences and structures are formed, while at long time intervals the families of structurally similar proteins that have low sequence similarity are formed. We discuss the evolutionary implications of our model. We provide a &quot;profile&quot; solution to our model and find agreement between predicted patterns of conserved amino acids and those actually observed in nature.'),(207,'dbss_fd98','Dokholyan, N. V., Buldyrev, S. V., Stanley, H. E., and Shakhnovich, E. I., ','Molecular dynamics studies of folding of a protein-like model','Folding and Design',3,'577','587',1998,'Proteins: Folding','Dokholyan, N. V.','no','no','no','article',9889167,'',NULL,NULL,NULL,'<strong>Background:</strong> Many attempts have been made to resolve in time the folding of model proteins in computer simulations. Different computational approaches have emerged. Some of these approaches suffer from insensitivity to the geometrical properties of the proteins (lattice models), whereas others are computationally heavy (traditional molecular dynamics).<br><br><strong>Results:</strong> We used the recently proposed approach of Zhou and Karplus to study the folding of a protein model based on the discrete time molecular dynamics algorithm. We show that this algorithm resolves with respect to time the folding <img src=papers/dbss_fd98.png width=50 height=10> unfolding transition. In addition, we demonstrate the ability to study the core of the model protein.<br><br><strong>Conclusions:</strong> The algorithm along with the model of interresidue interactions can serve as a tool for studying the thermodynamics and kinetics of protein models.'),(217,'dbss_jmb00','Dokholyan, N. V., Buldyrev, S. V., Stanley, H. E., and Shakhnovich, E. I., ','Identifying the protein folding nucleus using molecular dynamics','Journal of Molecular Biology',296,'1183','1188',2000,'Proteins: Folding','Dokholyan, N. V.','no','no','no','article',10698625,'',NULL,NULL,NULL,'Molecular dynamics simulations of folding in an off-lattice protein model reveal a nucleation scenario, in which a few well-defined contacts are formed with high probability in the transition state ensemble of conformations. Their appearance determines folding cooperativity and drives the model protein into its folded conformation. Amino acid residues participating in those contacts may serve as &quot;accelerator pedals&quot; used by molecular evolution to control protein folding rate.'),(218,'d_rrdsp00','Dokholyan, N. V., ','The protein folding problem','Recent Research Developments in Statistical Physics',1,'77','84',2000,'Proteins: Folding','Dokholyan, N. V.','no','no','no','article',0,'',NULL,NULL,NULL,'We present an overview of the protein folding problem. We discuss difficulties associated with this problem, its importance, the various approaches to its solution, and recent advances in the field. We discuss two important aspects of protein folding properties: its folding thermodynamics and kinetics. We also describe a popular protein folding scenario &#8211; folding via nucleation.'),(220,'sdbs_pre01','Scala, A., Dokholyan, N. V., Buldyrev, S. V., and Stanley, H. E., ','Thermodynamically important contacts in folding of model proteins','Physical Review E',63,'32901','',2001,'Proteins: Folding','Dokholyan, N. V.','no','no','no','article',11308693,'',NULL,NULL,NULL,'We introduce a quantity, the entropic susceptibility, that measures the thermodynamic importance &#8211; for the folding transition &#8211; of the contacts between amino acids in model proteins. Using this quantity, we find that only one equilibrium run of a computer simulation of a model protein is sufficient to select a subset of contacts that give rise to the peak in the specific heat observed at the folding transition. To illustrate the method, we identify thermodynamically important contacts in a model 46-mer. We show that only about 50% of all contacts present in the protein native state are responsible for the sharp peak in the specific heat at the folding transition temperature, while the remaining 50% of contacts do not affect the specific heat.'),(233,'bdehkms_pa03','Buldyrev, S. V., Dokholyan, N. V., Erramilli, S., Hong, M. K., Kim, J. Y., Malescio, G., and Stanley, H. E., ','Hierarchy in social organization','Physica A',330,'659','665',2003,'Complex systems','Buldyrev, S. V.','no','no','no','article',0,'',NULL,NULL,NULL,'We find that area and population distributions of nations follow an inverse power-law, as is known for cities, but with a different exponent. To interpret this result, we develop a growth model based on the geometrical properties of partitioning of the plane. The substantial agreement between the model and the actual nation distributions motivates the hypothesis that the distribution of aggregates of organisms is related to land partitioning. To test this hypothesis we follow the development of bacterial colonies of <i>Escherichia coli</i>, which, compared to humans, are on a completely different level of complexity. We find that the distributions of <i>E. coli</i> colonies follow an inverse power law with exponent similar to that of nations.'),(222,'kbdhlps_pa02','King, P. R., Buldyrev, S. V., Dokholyan, N. V., Havlin, S., Lopez, E., Paul, G., and Stanley, H.E.,','Uncertainty in oil production predicted by percolation theory','Physica A',306,'376','380',2002,'Random systems: Percolation','King, P. R.','no','no','no','article',0,'',NULL,NULL,NULL,'In this paper, we apply scaling laws from percolation theory to the problem of estimating the time for a fluid injected into an oil field to breakthrough into a production well. The main contribution is to show that when these previously published results are used on realistic data they are in good agreement with results calculated in a more conventional way but they can be obtained significantly more quickly. As a result they may be used in practical engineering circumstances and aid decision making for real field problems.'),(219,'kbdhlpsv_pg01','King, P. R., Buldyrev, S. V., Dokholyan, N. V., Havlin, S., Lee, Y., Paul, G., Stanley, H. E., and Vandesteeg, N.,','Predicting oil recovery using percolation theory','Petroleum Geoscience',7,'S105','S107',2001,'Random systems: Percolation','King, P. R.','no','no','no','article',0,'',NULL,NULL,NULL,'In this paper we apply scaling laws from percolation theory to the problem of estimating the time for a fluid injected into an oil field to breakthrough into a production well. The main contribution is to show that, when these previously published results are used on realistic data, they are in good agreement with results calculated in a more conventional way but they can be obtained significantly more quickly. As a result they may be used in practical engineering circumstances and aid decision-making for real field problems.'),(216,'abdhklps_pre00','Andrade, J. S. Jr., Buldyrev, S. V., Dokholyan, N. V., Havlin, S., King, P. R., Lee, Y., Paul, G., and Stanley, H. E., ','Flow between two sites on a percolation cluster','Physical Review E',62,'8270','8281',2000,'Random systems: Percolation','','no','no','no','article',11138125,'',NULL,NULL,NULL,'We study the flow of fluid in porous media in dimensions <i>d</i>=2 and 3. The medium is modeled by bond percolation on a lattice of <i>L<sup>d</sup></i> sites, while the flow front is modeled by tracer particles driven by a pressure difference between two fixed sites (&quot;wells&quot;) separated by Euclidean distance <i>r</i>. We investigate the distribution function of the shortest path connecting the two sites, and propose a scaling ansatz that accounts for the dependence of this distribution (i) on the size of the system <i>L</i> and (ii) on the bond occupancy probability <i>p</i>. We confirm by extensive simulations that the ansatz holds for d=2 and 3. Further, we study two dynamical quantities: (i) the minimal traveling time of a tracer particle between the wells when the total flux is constant and (ii) the minimal traveling time when the pressure difference is constant. A scaling ansatz for these dynamical quantities also includes the effect of finite system size <i>L</i> and off-critical bond occupation probability <i>p</i>. We find that the scaling form for the distribution functions for these dynamical quantities for d=2 and 3 is similar to that for the shortest path, but with different critical exponents. Our results include estimates for all parameters that characterize the scaling form for the shortest path and the minimal traveling time in two and three dimensions; these parameters are the fractal dimension, the power law exponent, and the constants and exponents that characterize the exponential cutoff functions.'),(215,'pbdhkls_pre00','Paul, G., Buldyrev, S. V., Dokholyan, N. V., Havlin, S., King, P. R., Lee, Y.,  and Stanley, H. E.,','Dependence of conductance on percolation backbone mass','Physical Review E',61,'3435','3440',2000,'Random systems: Percolation','Paul, G.','no','no','no','article',11088120,'',NULL,NULL,NULL,'We study &lang;&sigma;(<i>M<sub>B</sub>, r</i>)&rang; the average conductance of the backbone, defined by two points separated by Euclidean distance <i>r</i>, of mass <i>M<sub>B</sub></i> on two-dimensional percolation clusters at the percolation threshold. We find that with increasing <i>M<sub>B</sub></i> and for fixed <i>r</i>, &lang;(&sigma;<i>M<sub>B</sub> ,r</i>)&rang; asymptotically <i>decreases</i> to a constant, in contrast with the behavior of homogeneous systems and nonrandom fractals (such as the Sierpinski gasket) in which conductance increases with increasing <i>M<sub>B</sub></i>. We explain this behavior by studying the distribution of shortest paths between the two points on clusters with a given <i>M<sub>B</sub></i>. We also study the dependence of conductance on <i>M<sub>B</sub></i> above the percolation threshold and find that (i) slightly above <i>p<sub>c</sub></i>, the conductance first decreases and then increases with increasing <i>M<sub>B</sub></i> and (ii) further above <i>p<sub>c</sub></i>, the conductance increases monotonically for all values of <i>M<sub>B</sub></i>, as is the case for homogeneous systems.'),(213,'kbdhlps_pa99','King, P. R., Buldyrev, S. V., Dokholyan, N. V., Havlin, S., Lee, Y., Paul, G., and Stanley, H. E.,','Application of statistical physics to the oil industry: Predicting oil recovery using percolation theory','Physica A',274,'60','66',1999,'Random systems: Percolation','King, P. R.','no','no','no','article',0,'',NULL,NULL,NULL,'In this paper we apply scaling laws from percolation theory to the problem of estimating the time for a fluid injected into an oil field (for the purposes of recovering the oil) to breakthrough into a production well. The main contribution is to show that percolation theory, when applied to a realistic model, can be used to obtain the same results as calculated in a more conventional way but significantly more quickly. Specifically, we found that a previously proposed scaling form for the breakthrough time distribution when applied to a real oil field is in good agreement with more time consuming simulation results. Consequently these methods can be used in practical engineering circumstances to aid decision making for real field problems.'),(212,'labdhkps_pre99','Lee, Y., Andrade, J. S. Jr., Buldyrev, S. V., Dokholyan, N. V., Havlin, S., King, P. R., Paul, G., and Stanley, H. E., ','Traveling time and traveling length in critical percolation clusters','Physical Review E',60,'3425','3428',1999,'Random systems: Percolation','Lee, Y.','no','no','no','article',11970164,'',NULL,NULL,NULL,'We study traveling time and traveling length for tracer dispersion in two-dimensional bond percolation, modeling flow by tracer particles driven by a pressure difference between two points separated by Euclidean distance <i>r</i>. We find that the minimal traveling time <i>t<sub>min</sub></i> scales as <i>t<sub>min</sub></i> &#8764; <i>r<sup>1.33</sup></i>, which is different from the scaling of the most probable traveling time, <i>t</i> &#8764; <i>r<sup>1.64</sup></i>. We also calculate the length of the path corresponding to the minimal traveling time and find <i>l<sub>min</sub></i> &#8764; <i>r<sup>1.13</sup></i> and that the most probable traveling length scales as <i>l</i> &#8764; <i>r<sup>1.21</sup>. We present the relevant distribution functions and scaling relations.'),(211,'kabdlhs_pa99','King, P. R., Andrade, J. S. Jr., Buldyrev, S. V., Dokholyan, N. V., Lee, Y., Havlin, S., and Stanley, H. E., ','Percolation and oil recovery','Physica A',266,'107','114',1999,'Random systems: Percolation','King, P. R.','no','no','no','article',0,'',NULL,NULL,NULL,'One particular practical problem in oil recovery is to predict the time to breakthrough of a fluid injected in one well and the subsequent decay in the production rate of oil at another well. Because we only have a stochastic view of the distribution of rock properties we need to predict the uncertainty in the breakthrough time and post-breakthrough behaviour in order to calculate the economic risk. In this paper we use percolation theory to predict (i) the distribution of the chemical path (shortest path) between two points (representing well pairs) at a given Euclidean separation and present a scaling hypothesis for this distribution which is confirmed by numerical simulation, (ii) the distribution of breakthrough times which can be calculated algebraically rather than by very time consuming direct numerical simulation of large numbers of realisations.'),(206,'dlbhks_jsp98','Dokholyan, N. V., Lee, Y., Buldyrev, S. V., Havlin, S., King, P. R., and Stanley, H. E.,','Scaling of the distribution of shortest paths in percolation','Journal of Statistical Physics',93,'603','613',1999,'Random systems: Percolation','Dokholyan, N. V.','no','no','no','article',0,'',NULL,NULL,NULL,'We present a scaling hypothesis for the distribution function of the shortest paths connecting any two points on a percolating cluster which accounts for (i) the effect of the finite size of the system and (ii) the dependence of this distribution on the site occupancy probability <i>p</i>. We test the hypothesis for the case of two-dimensional percolation.'),(210,'dbhkls_pa99','Dokholyan, N. V., Buldyrev, S. V., Havlin, S., King, P. R., Lee, Y., and Stanley, H. E., ','Distribution of shortest paths in percolation','Physica A',266,'53','59',1999,'Random systems: Percolation','Dokholyan, N. V.','no','no','no','article',0,'',NULL,NULL,NULL,'We present a scaling Ansatz for the distribution function of the shortest paths connecting any two points on a percolating cluster which accounts for (i) the effect of the finite size of the system, and (ii) the dependence of this distribution on the site occupancy probability <i>p</i>. We present evidence supporting the scaling Ansatz for the case of two-dimensional percolation.'),(209,'bdhss_pa99','Buldyrev, S. V., Dokholyan, N. V., Havlin, S., Stanley, H. E., and Stanley, H. R. H., ','Expansion of tandem repeats and oligomer clustering in coding and noncoding DNA sequences','Physica A',273,'19','32',1999,'Bioinformatics','Buldyrev, S. V.','no','no','no','article',0,'',NULL,NULL,NULL,'We review recent studies of distribution of dimeric tandem repeats and short oligomer clustering in DNA sequences. We find that distribution of dimeric tandem repeats in coding DNA is exponential, while in noncoding DNA it often has long power-law tails. We explain this phenomenon using mutation models based on random multiplicative processes. We also develop a clustering measure based on percolation theory that quantifies the degree of clustering of short oligomers. We find that mono-, di-, and tetramers cluster more in noncoding DNA than in coding DNA. However trimers have some degree of clustering in coding DNA and noncoding DNA. We relate this phenomena to modes of tandem repeat expansion.'),(205,'hjdsshe_nimpr98','Hong, M. K., Jeung, A. G., Dokholyan, N. V., Smith, T. I., Schwettman, H. A., Huie, P., and Erramilli, S., ','Imaging single living cells with a scanning near-field infrared microscope based on a free electron laser','Nuclear Instruments and Methods in Physics Research',0,'246','255',1999,'Cellular motility','Erramilli, S.','no','no','no','article',0,'',NULL,NULL,NULL,'We report the first sub-wavelength mid-infrared images under water, and describe an application to obtaining images of single living cells in water using the Stanford Free Electron Laser (FEL). Spatial resolution is enhanced at the peak of infrared absorption of water. Images were obtained of single motile fibroblasts with the FEL wavelength tuned to absorption peaks of both protein and lipid molecules. Analysis of the unexpectedly strong absorption due to lipid molecules in motile fibroblasts suggests that the concentration of lipid molecules in lamellopodia is consistent with membrane flow.'),(214,'dbhs_jtb00','Dokholyan, N. V., Buldyrev, S. V., Havlin, S., and Stanley, H. E., ','Distributions of dimeric tandem repeats in noncoding and coding DNA sequences','Journal of  Theoretical Biology',202,'273','282',2000,'Bioinformatics','Dokholyan, N. V.','no','no','no','article',10666360,'',NULL,NULL,NULL,'We study the length distribution functions for the 16 possible distinct dimeric tandem repeats in DNA sequences of diverse taxonomic partitions of GenBank (known human and mouse genomes, and complete genomes of <i>Caenorhabditis elegans</i> and yeast). For coding DNA, we find that all 16 distribution functions are exponential. For non-coding DNA, the distribution functions for most of the dimeric repeats have surprisingly long tails, that fit a power-law function. We hypothesize that: (i) the exponential distributions of dimeric repeats in protein coding sequences indicate strong evolutionary pressure against tandem repeat expansion in coding DNA sequences; and (ii) long tails in the distributions of dimers in non-coding DNA may be a result of various mutational mechanisms. These long, non-exponential tails in the distribution of dimeric repeats in non-coding DNA are hypothesized to be due to the higher tolerance of non-coding DNA to mutations. By comparing genomes of various phylogenetic types of organisms, we find that the shapes of the distributions are not universal, but rather depend on the specific class of species and the type of a dimer.'),(201,'dj_plb94','Dokholyan, N. V., and Jikia, G. V., ','Single top quark production and V<sub>tb</sub> CKM matrix element measurement in high energy e<sup>+</sup>e<sup>-</sup> collisions','Physics Letters B',336,'251','256',1994,'High energy physics','Dokholyan, N. V.','no','no','no','article',0,'',NULL,NULL,NULL,'A new method for the determination of the CKM mixing matrix element V<sub>tb</sub> has been proposed. It has been shown, that at future colliders one will measure the <i>tb</i>-mixing element with an accuracy 12-28 %'),(202,'dbhs_prl97','Dokholyan, N. V., Buldyrev, S. V., Havlin, S., and Stanley, H. E., ','Distribution of base pair repeats in coding and noncoding DNA sequences','Physical Review Letters',79,'5182','5185',1997,'Bioinformatics','Dokholyan, N. V.','no','no','no','article',0,'',NULL,NULL,NULL,'We analyze the histograms for the lengths of the 16 possible distinct repeats of identical dimers, known as dimeric tandem repeats, in DNA sequences. For coding regions, the probability of finding a repetitive sequence of l copies of a particular dimer decreases exponentially as l increases. For the noncoding regions, the distribution functions for most of the 16 dimers have long tails and can be approximated by power-law functions, while for coding DNA, they can be well fit by a first order Markov process. We propose a model, based on known biophysical processes, which leads to the observed probability distribution functions for noncoding DNA. We argue that this difference in the shape of the distribution functions between coding and noncoding DNA arises from the fact that noncoding DNA is more tolerant to evolutionary mutational alterations than coding DNA.'),(203,'dbhs_pa98','Dokholyan, N. V., Buldyrev, S. V., Havlin, S., and Stanley, H. E., ','Model of unequal chromosomal crossing over in DNA sequences','Physica A',249,'594','599',1998,'Bioinformatics','Dokholyan, N. V.','no','no','no','article',0,'',NULL,NULL,NULL,'It is known that some dimeric tandem repeats (DTR) are very abundant in noncoding DNA. We find that certain DTR length distribution functions in noncoding DNA can be fit by a power law function. We analyze a simplified model of unequal chromosomal crossing over and find that it produces a stable power law length distribution function, with the exponent  &micro; = 2. Although the exponent predicted by this model differs from those observed in nature, we argue that the biophysical process underlying this model provides the major contribution to the DTR length distribution function.'),(208,'sdbhs_jbsd99','Stanley, H. R. H., Dokholyan, N. V., Buldyrev, S. V., Havlin, S., and Stanley, H. E., ','Clustering of identical oligomers in coding and noncoding DNA sequences','Journal of Biomolecular Structure and Dynamics',17,'79','87',1999,'Bioinformatics','Stanley, H. R. H.','no','no','no','article',10496423,'',NULL,NULL,NULL,'We develop a quantitative method for analyzing repetitions of identical short oligomers in coding and noncoding DNA sequences. We analyze sequences presently available in the GenBank separately for primate, mammal, vertebrate, rodent, invertebrate and plant taxonomic partitions. We find that some oligomers &quot;cluster&quot; more than they would if randomly distributed, while other oligomers &quot;repel&quot; each other. To quantify this degree of clustering, we define clustering measures. We find that (i) clustering significantly differs in coding and noncoding DNA; (ii) in most cases, monomers, dimers and tetramers cluster in noncoding DNA but appear to repel each other in coding DNA. (iii) The degree of clustering for different sources (primates, invertebrates, and plants) is more conserved among these sources in the case of coding DNA than in the case of noncoding DNA. (iv) In contrast to other oligomers, we find that trimers always prefer to cluster. (v) Clustering of each particular oligomer is conserved within the same organism.'),(204,'bdghpsv_pa98','Buldyrev, S. V., Dokholyan, N. V., Goldberger, A. L., Havlin, S., Peng, C.-K., Stanley, H. E., and Viswanathan, G. M., ','Analysis of DNA sequences using methods of statistical physics','Physica A',249,'430','438',1998,'Bioinformatics','Buldyrev, S. V.','no','no','no','article',0,'',NULL,NULL,NULL,'We review the present status of the studies of DNA sequences using methods of statistical physics. We present evidence, based on systematic studies of the entire GenBank database, supporting the idea that the DNA sequence in genes containing noncoding regions is correlated, and that the correlation is remarkably long range, i.e., base pairs thousands of base pairs distant are correlated. We do not find such a long-range correlation in the coding regions of the DNA. We discuss the mechanisms of molecular evolution that may lead to the presence of long-range power-law correlations in noncoding DNA and their absence in coding DNA. One such mechanism is the simple repeat expansion, which recently has attracted the attention of the biological community in conjunction with genetic diseases. We also review new tools &#8212; e.g., detrended  fluctuation analysis &#8212; that are useful for studies of complex hierarchical DNA structure.'),(304,'rsxmd_plscb09','Ramachandran, S., Serohijos, A. W. R., Xu, L., Meissner, G., and Dokholyan, N. V.,','A Structural Model of the Pore-Forming Region of the\nSkeletal Muscle Ryanodine Receptor (RyR1)','Public Library of Science Computational Biology',5,'e1000367','',2009,'Proteins: Transmembrane','Dokholyan, N. V.,','no','no','no','article',19390614,'',NULL,NULL,NULL,'Ryanodine Receptors (RyRs) are ion channels that regulate muscle contraction by releasing calcium ions from intracellular stores into the cytoplasm. Mutations in skeletal muscle RyR (RyR1) give rise to congenital diseases such as the Central Core Disease. The absence of high-resolution structures of RyR1 has limited our understanding of channel function and disease mechanisms at the molecular level. Here, we report a structural model of the pore-forming region of RyR1 and electrophysiological studies on a Central Core Disease mutant RyR1-G4898R. Molecular dynamics simulations on the structural model show preferential localization of Ca<sup>2+</sup> over K<sup>+</sup> in the selectivity filter. We observe high ion binding to the residues D4899, E4900, D4938 and D4945 along the pore, which are experimentally known to be critical for channel function and selectivity. Furthermore, simulations on the mutant RyR1-D4899Q show a loss of preference to Ca<sup>2+</sup> in the selectivity filter as seen experimentally. Electrophysiological experiments on RyR1-G4898R show constitutively open channels that conduct K<sup>+</sup> but not Ca<sup>2+</sup>. Our simulations with G4898R likewise show a decrease in preference of Ca<sup>2+</sup> over K<sup>+</sup> in the selectivity filter. Together, the computational and experimental results shed light on the functioning of the RyR1 pore.'),(232,'lbdghks_pre03','Lopez, E., Buldyrev, S. V., Dokholyan, N. V., Goldmakher, L., Havlin, S., King, P. R., and Stanley, H. E.,','Postbreakthrough behavior in flow through porous media','Physical Review E',67,'56314','',2003,'Random systems: Percolation','Lopez, E.,','no','no','no','article',0,'',NULL,NULL,NULL,'We numerically simulate the traveling time of a tracer in convective flow between two points (injection and extraction) separated by a distance <i>r</i> in a model of porous media, <i>d</i>=2 percolation. We calculate and analyze the traveling time probability density function for two values of the fraction of connecting bonds <i>p</i>: the homogeneous case <i>p</i>=1 and the inhomogeneous critical threshold case <i>p=p<sub>c</sub></i> . We analyze both constant current and constant pressure conditions at <i>p=p<sub>c</sub></i> . The homogeneous <i>p</i>=1 case serves as a comparison base for the more complicated <i>p=p<sub>c</sub></i> situation. We find several regions in the probability density of the traveling times for the homogeneous case (<i>p</i>=1) and also for the critical case (<i>p=p<sub>c</sub></i>) for both constant pressure and constant current conditions. For constant pressure, the first region <i>I<sub>P</sub></i> corresponds to the short times before the flow breakthrough occurs, when the probability distribution is strictly zero. The second region <i>II<sub>P</sub></i> corresponds to numerous fast flow lines reaching the extraction point, with the probability distribution reaching its maximum. The third region <i>III<sub>P</sub></i> corresponds to intermediate times and is characterized by a power-law decay. The fourth region <i>IV<sub>P</sub></i> corresponds to very long traveling times, and is characterized by a different power-law decaying tail. The power-law characterizing region <i>IV<sub>P</sub></i> is related to the multifractal properties of flow in percolation, and an expression for its dependence on the system size <i>L</i> is presented. The constant current behavior is different from the constant pressure behavior, and can be related analytically to the constant pressure case. We present theoretical arguments for the values of the exponents characterizing each region and crossover times. Our results are summarized in two scaling assumptions for the traveling time probability density; one for constant pressure and one for constant current. We also present the production curve associated with the probability of traveling times, which is of interest to oil recovery.'),(237,'mbsd_prl03','Marques, M. I., Borreguero, J. M., Stanley, H. E., and Dokholyan, N. V.,','Possible mechanism for cold denaturation of proteins at high pressure','Physical Review Letters',91,'138103','',2003,'Proteins: Folding','Marques, M. I.,','no','no','no','article',14525339,'',NULL,NULL,NULL,'We study cold denaturation of proteins at high pressures. Using multicanonical Monte Carlo simulations of a model protein in a water bath, we investigate the effect of water density fluctuations on protein stability. We find that above the pressure where water freezes to the dense ice phase (&#8776; 2 kbars) the mechanism for cold denaturation with decreasing temperature is the loss of local low-density water structure. We find our results in agreement with data of bovine pancreatic ribonuclease A.'),(223,'kbdhlps_pa02a','King, P. R., Buldyrev, S. V., Dokholyan, N. V., Havlin, S., Lopez, E., Paul, G., and Stanley, H.E.,','Using percolation theory to predict oil field performance','Physica A',314,'103','108',2002,'Random systems: Percolation','King, P. R.,','no','no','no','article',0,'',NULL,NULL,NULL,'In this paper, we apply scaling laws from percolation theory to the problem of estimating the time for a fluid injected into an oil field to breakthrough into a production well. The main contribution is to show that when these previously published results are used on realistic data they are in good agreement with results calculated in a more conventional way but they can be obtained significantly more quickly. As a result they may be used in practical engineering circumstances and aid decision making for real field problems.'),(693,'dbbdss_me03','Dokholyan, N. V., Borreguero, J. M., Buldyrev, S. V., Ding, F., Stanley, H. E., and Shakhnovich, E. I.,','Identifying importance of amino acids for protein folding from crystal structures','Methods in Enzymology',374,'618','640',2003,'Proteins: Folding','Dokholyan, N. V.,','no','no','no','book',14696390,'','Carter Jr. C. W., and Sweet, R. M.,','Macromolecular Crystallography D','Elsevier',''),(694,'ds_mbiu06','Dokholyan, N. V., and Shakhnovich, E. I.,','Scale-free evolution: from proteins to organisms','Power Laws, Scale-free Networks and Genome Biology',0,'86','105',2006,'Proteins: evolution','Dokholyan, N. V.,','no','no','no','book',0,'','Koonin, E. V., Wolf, Y. I., and Karev, G. P.,','Molecular Biology Intelligence Unit','Eurekah.com and Springer',''),(695,'ds_bmpbe07','Dokholyan, N. V., and Shakhnovich, E. I.,','Towards unifying protein evolution theory','Structural approaches to sequence evolution: Molecules, networks, populations',0,'113','126',2007,'Proteins: evolution','Dokholyan, N. V.,','no','no','no','book',0,'','Bastolla, U., Porto, M., Roman, H. E., and Vendruscolo, M.,','Biological and Medical Physics, Biomedical Engineering','Springer',''),(696,'d_08','Dokholyan, N. V.,','Protein designability and engineering','Structural Bioinformatics',0,'961','982',2008,'Proteins: Design','Dokholyan, N. V.,','no','no','no','book',0,'','Bourne. P., and Gu, J.,','','Wiley & Sons',''),(697,'crsdtbwc_pohmccc09','Chaney, S. G., Ramachandran, S., Sharma, S., Dokholyan, N. V., Temple, B., Bhattacharyya, D., Wu, Y., and Campbell, S.,','Differences in Conformation and Conformational Dynamics Between Cisplatin and Oxaliplatin DNA Adducts','Platinum and Other Heavy Metal Compounds in Cancer Chemotherapy',0,'157','169',2009,'DNA: Drugs','Chaney, S. G.,','no','no','no','book',0,'','Bonetti, A., Leone, R., Muggia, F. M., and Howell, S. B.,','Cancer Drug Discovery and Development','Humana Press',''),(306,'stled_pccp09','Serohijos, A. W. R., Tsygankov, D., Liu, S., Elston, T. C., and Dokholyan, N. V.','Multiscale approaches for studying energy transduction in dynein','Physical Chemistry Chemical Physics',11,'4840','4850',2009,'Proteins: Mechanism','Dokholyan, N. V.','no','no','no','article',19506759,'',NULL,NULL,NULL,'Cytoplasmic dynein is an important motor that drives all minus-end directed movement along microtubules. Dynein is a complex motor whose processive motion is driven by ATP-hydrolysis. Dynein&rsquo;s run length has been measured to be several millimetres with typical velocities in the order of a few nanometres per second. Therefore, the average time between steps is a fraction of a second. When this time scale is compared with typical time scales for protein side chain and backbone movements (&#126;10<sup>-9</sup> s and &#126;10<sup>-5</sup> s, respectively), it becomes clear that a multi-timescale modelling approach is required to understand energy transduction in this protein. Here, we reviewrecent efforts to use computational and mathematical modelling to understand various aspects of dynein&rsquo;s chemomechanical cycle. First, we describe a structural model of dynein&rsquo;s motor unit showing a heptameric organization of the motor subunits. Second, we describe our molecular dynamics simulations of the motor unit that are used to investigate the dynamics of the various motor domains. Third, we present a kinetic model of the coordination between the two dynein heads. Lastly, we investigate the various potential geometries of the dimer during its hydrolytic and stepping cycle.'),(307,'kdtd_psfb10','Kesner, B. A., Ding, F., Temple, B. S., and Dokholyan, N. V.','N-terminal strands of filamin Ig domains act as a conformational switch under biological forces','Proteins: Structure, Function, and Bioinformatics',78,'12','24',2010,'Proteins: Structure','Dokholyan, N. V.','no','no','no','article',19514078,'',NULL,NULL,NULL,'Conformational changes of filamin A under stress have been postulated to play crucial roles in signaling pathways of cell responses. Direct observation of conformational changes under stress is beyond the resolution of current experimental techniques. On the other hand, computational studies are mainly limited to either traditional molecular dynamics simulations of short durations and high forces or simulations of simplified models. Here we perform all-atom discrete molecular dynamics (DMD) simulations to study thermally and force-induced unfolding of filamin A. The high conformational sampling efficiency of DMD allows us to observe force-induced unfolding of filamin A Ig domains under physiological forces. The computationally identified critical unfolding forces agree well with experimental measurements. Despite a large heterogeneity in the population of force-induced intermediate states, we find a common initial unfolding intermediate in all the Ig domains of filamin, where the N-terminal strand unfolds. We also study the thermal unfolding of several filamin Ig-like domains. We find that thermally induced unfolding features an early-stage intermediate state similar to the one observed in force-induced unfolding and characterized by the N-terminal strand being unfurled. We propose that the N-terminal strand may act as a conformational switch that unfolds under physiological forces leading to exposure of cryptic binding sites, removal of native binding sites, and modulating the quaternary structure of domains.'),(308,'ksrcd_pnas09','Kota, P., Summers, D. W., Ren, H-Y., Cyr, D. M. and Dokholyan, N. V.','Identification of a consensus motif in substrates bound by a type I Hsp40','Proceedings of the National Academy of Sciences USA',106,'11073','11078',2009,'Proteins: Structure','Dokholyan, N. V.','yes','no','no','article',19549854,'',NULL,NULL,NULL,'Protein aggregation is a hallmark of a large and diverse number of conformational diseases. Molecular chaperones of the Hsp40 family (Escherichia coli DnaJ homologs) recognize misfolded disease proteins and suppress the accumulation of toxic protein species. Type I Hsp40s are very potent at suppressing protein aggregation and facilitating the refolding of damaged proteins. Yet, the molecular mechanism for the recognition of nonnative polypeptides by Type I Hsp40s such as yeast Ydj1 is not clear. Here we computationally identify a unique motif that is selectively recognized by Ydj1p. The motif is characterized by the consensus sequence GX[LMQ]{P}X{P}{CIMPVW}, where [XY] denotes either X or Y and {XY} denotes neither X nor Y. We further verify the validity of the motif by site-directed mutagenesis and show that substrate binding by Ydj1 requires recognition of this motif. A yeast proteome screen revealed that many proteins contain more than one stretch of residues that contain the motif and are separated by varying numbers of amino acids. In light of our results, we propose a 2-site peptide-binding model and a plausible mechanism of peptide presentation by Ydj1p to the chaperones of the Hsp70 family. Based on our results, and given that Ydj1p and its human ortholog Hdj2 are functionally interchangeable, we hypothesize that our results can be extended to understanding human diseases.'),(698,'d_nic08','Dokholyan, N. V.','Simplified approaches to complex biological systems','Proceedings of the From Computational Biophysics to Systems Biology (CBSB08) Symposium, May 19-21, 2008',40,'85','88',2008,'Proteins: Folding','Dokholyan, N. V.','no','no','no','conf',0,'','Hansmann, U. H. E., Meinke, J. H., Mohanty, S., Nadler, W. and Zimmermann, O.','','',''),(699,'dghklopttt_jad04','Dokholyan, N. V., Glabe, C., Holtzman, D., Koudinov, A., Lomakin, A., Oyler, N., Paganini, N., Teplow, D., Thirumalai, D., and Tycko, R.','Live discussion: Protein folding and neurodegeneration: Biophysics to the rescue&#63;','Journal of Alzheimer&#39;s Disease',6,'99','105',2004,'Proteins: Aggregation','','no','no','no','conf',0,'','','','',''),(700,'dbss_aip00','Dokholyan, N. V., Buldyrev, S. V., Stanley, H. E., and Shakhnovich, E. I.','Kinetics of the protein folding transition.','AIP Proceedings of the Third Tohwa University Conference on Statistical Physics, November 8-12, 1999',0,'419','425',2000,'Proteins: Folding','','no','no','no','conf',0,'','','','',''),(701,'ds_pisp00','Dokholyan, N. V., and Shakhnovich, E. I.','Two models of amino acid conservation in proteins.','Proceedings of the International School of Physics &quot;Enrico Fermi&quot; Course CXLV: &quot;Protein folding, evolution and design&quot;, Varenna, Italy',0,'227','245',2000,'Proteins: Evolution','','no','no','no','conf',0,'','','','',''),(305,'wzjhyrccd_jbc09','Wilcox, K. C., Zhou, L., Jordon, J., Huang, Y., Yu, Y., Redler, R. L., Chen, X., Caplow, M., and Dokholyan, N. V.','Modifications of SOD1 in human erythrocytes: A possible role in ALS','Journal of Biological Chemistry',284,'13940','13947',2009,'Proteins: Aggregation','','yes','no','no','article',19299510,'','','','','Over 100 mutations in Cu/Zn superoxide dismutase (SOD1) result in familial amyotrophic lateral sclerosis (FALS). Dimer dissociation is the first step in SOD1 aggregation, and studies suggest nearly every amino acid residue in Cu/Zn superoxide dismutase (SOD1) is dynamically connected to the dimer interface. Post-translational modifications of SOD1 residues might be expected to have similar effects to mutations, but few have been identified. Here we show, using SOD1 isolated from human erythrocytes, that human SOD1 is phosphorylated at threonine 2 and glutathionylated at cysteine 111.  A second SOD1 phosphorylation was observed and mapped to either Thr-58 or Ser-59. Cysteine-111 glutathionylation promotes SOD1 monomer formation &#8211; a necessary initiating step in SOD1 aggregation &#8211; by causing a two-fold increase in the K<sub>d</sub>. This change in the dimer stability is expected to result in a 67% increase in monomer concentration &#8211; 315 nM rather than 212 nM at physiological SOD1 concentrations. Because protein glutathionylation is associated with redox regulation, our finding that glutathionylation promotes SOD1 monomer formation supports a model in which increased oxidative stress promotes SOD1 aggregation.'),(309,'ypld_pnas09','Yin, S., Proctor, E. A., Lugovskoy, A. A. and Dokholyan, N. V.','Fast screening of protein surfaces using geometric invariant fingerprints','Proceedings of the National Academy of Sciences USA',106,'16622','16626',2009,'Proteins: Methods','Dokholyan, N. V.','yes','no','yes','article',19805347,'',NULL,NULL,NULL,'We develop a rapid and efficient method for the comparison of protein local surface similarities utilizing geometric invariants (fingerprints). By combining fast fingerprint comparison with explicit alignment, we successfully screen the entire Protein Data Bank for proteins that possess local surface similarities. Our method is independent of sequence and fold similarities, and has potential application to protein structure annotation and protein-protein interface design.'),(707,'kmtd_mbe10','Kesner, B. A., Milgram, S. L., Temple, B. S. and Dokholyan, N. V.','Isoform divergence of the filamin family of proteins','Molecular Biology and Evolution',27,'283','295',2010,'Proteins: Assembly','Dokholyan, N. V.','yes','no','no','article',19805437,'','','','','The vertebrate filamin family (A, B and C) is part of the spectrin family of actin cross-linking proteins. Family members share high sequence similarity (>64%) and have both common and isoform distinct functionality. To identify the basis for isoform specific functionality, we perform an evolutionary trace of chordate filamin at the granularity of single residues. Our trace methodology is constrained to focus on neo-functionality by requiring that one isoform remain the ancestral type while at least one isoform has an accepted mutation. We call divergence meeting these characteristics \"class-distinctive.\" To obtain a temporal and spatial context for class-distinctive residues, we derive an all-atom model of full-length filamin A by homology modeling and joining individual domains. We map onto our model both conserved and class-distinctive residues along with the period (Teleostei, Amphibian, Mammalian) in which they diverged. Our phylogenetic analysis suggests that filamins diverged from a common ancestral gene between urochordate and vertebrate lineages. Filamins also diverged the most just after gene duplication, in the Teleostei period, with filamin C remaining closest to ancestral filamin. At the residue level, domains with well-characterized interfaces, IgFLN 17 and IgFLN 21, have diverged in potentially critical residues in their adhesion protein binding interfaces, signifying that isoforms may bind or regulate ligand binding differentially. Similarly, isoform divergence in a region associated with F-actin binding regulation suggests that isoforms differentially regulate F-actin binding. In addition, we observe some class-distinctive residues in the vicinity of missense mutations that cause filamin A and B associated skeletal disorders. Our analysis, utilizing both spatial and temporal granularity, has identified potentially important residues responsible for vertebrate filamin isoform specific divergence - significantly in regions where few binding partners have been discovered to date - and suggests yet to be discovered filamin binding partners, and isoform specific differential regulation with these binding partners. '),(708,'ydd_pspma09','Yin, S., Ding, F. and Dokholyan, N. V.','Modeling mutations in proteins using Medusa and discrete molecular dynamics','Protein Structure Prediction: Method and Algorithms',0,'','',2009,'Proteins: Design','Dokholyan, N. V.','no','no','no','book',0,'','Rangwala, H. and Karypis, G.','','Wiley & Sons',NULL),(709,'ydd_ivmp09','Yin, S., Ding, F. and Dokholyan, N. V.','Computational evaluation of protein stability change upon mutations','<i>In Vitro</i> Mutagenesis Protocols',0,'','',2009,'Proteins: Design','Dokholyan, N. V.','no','no','no','book',0,'','Braman, J.,','','Humana Press',NULL),(710,'td_pccp10','Tsao, D. and Dokholyan, N. V.','Macromolecular crowding induces polypeptide compaction and decreases folding cooperativity','Physical Chemistry Chemical Physics',12,'3491','3500',2010,'Proteins: Crowding','Dokholyan, N. V.','no','no','no','article',20355290,'','','','','A cell&#39;s interior is comprised of macromolecules that can occupy up to 40% of its available volume.  Such crowded environments can influence the stability of proteins and their rates of reaction.  Using discrete molecular dynamics simulations, we investigate how both the size and number of neighboring crowding reagents affect the thermodynamic and folding properties of structurally diverse proteins.  We find that crowding induces higher compaction of proteins.  We also find that folding becomes less cooperative with the introduction of crowders into the system.  The crowders may induce alternative non-native protein conformations, thus creating barriers for protein folding in highly crowded media.'),(711,'gkgdst_paejp10','Gaillard, E., Kota, P., Gentzsch, M., Dokholyan, N. V., Stutts, M. J. and Tarran, R.','The Regulation of Epithelial Na<sup>+</sup> Channel and Airway Surface Liquid Volume by Extracellular Proteases','Pflugers Archives-European Journal of Physiology',460,'1','17',2010,'Proteins: Diseases','Tarran, R.','no','no','no','article',20401730,'','','','','Mammalian airways are protected from infection by this thin film of airway surface liquid (ASL) which covers airway epithelial surfaces and acts as a lubricant to keep mucus from adhering to the epithelial surface.  Precise regulation of ASL volume is essential for efficient mucus clearance and too great a reduction in ASL volume causes mucus dehydration and mucus stasis which contributes to chronic airway infection.  The epithelial Na<sup>+</sup> channel (ENaC) is the rate limiting step that governs Na<sup>+</sup> absorption in the airways. Recent <i>in vitro</i> and <i>in vivo</i> data have demonstrated that ENaC is a critical determinant of ASL volume and hence, mucus clearance.  ENaC must be cleaved by either intracellular furin-type proteases or extracellular serine proteases to be active and conduct Na<sup>+</sup>, and this process can be inhibited by protease inhibitors. ENaC can be regulated by multiple pathways and once proteolytically cleaved, ENaC may then be inhibited by intracellular second messengers such as cAMP and PIP2.  In the airways, however, regulation of ENaC by proteases seems to be the predominant mode of regulation since knockdown of either endogenous serine proteases such as prostasin, or inhibitors of ENaC proteolysis such as SPLUNC1 have large effects on ENaC activity in airway epithelia. In this review, we shall discuss how ENaC is proteolytically cleavaged, how this process can regulate ASL volume and how its failure to operate correctly may contribute to chronic airway disease.'),(712,'hddw_rna10','Hajdin, C. E., Ding, F., Dokholyan, N. V., and Weeks, K. M.','On the significance of an RNA tertiary structure prediction','RNA',16,'1340','1349',2010,'RNA','Dokholyan, N. V.','no','no','no','article',20498460,'','','','','Tertiary structure prediction is important for understanding structure-function relationships for RNAs whose structures are unknown and for characterizing RNA states recalcitrant to direct analysis. However, it is unknown what root-mean-square deviation (RMSD) corresponds to a statistically significant RNA tertiary structure prediction. We use discrete molecular dynamics to generate RNA-like folds for structures up to 161 nts that have complex tertiary interactions and then determine the RMSD distribution between these decoys.  These distributions are Gaussian-like. The mean RMSD increases with RNA length and is smaller if secondary structure constraints are imposed while generating decoys. The compactness of RNA molecules with true tertiary folds is intermediate between closely packed spheres and a freely jointed chain. We use this scaling relationship to define an expression relating RMSD with the confidence that a structure prediction is better than that expected by chance. This is the prediction significance, and corresponds to a p-value. For a 100-nucleotide RNA, the RMSD of predicted structures should be within 25  of the accepted structure to reach the p  0.01 level if the secondary structure is predicted de novo and within 14  if secondary structure information is used as a constraint. This p-value approach should be useful for evaluating diverse RNA structure prediction and molecular modeling algorithms.'),(713,'ldd_plscb10','Lakhani, V. V., Ding, F. and Dokholyan, N. V.','Poly-glutamine induced misfolding of huntingtin exon1 is modulated by the flanking sequences','Public Library of Science Computational Biology',6,'e1000772','',2010,'Proteins: Aggregation','Dokholyan, N. V.','no','no','no','article',20442863,'','','','','Polyglutamine (polyQ) expansion in exon1 (XN1) of the huntingtin protein is linked to Huntington&prime;s disease. When the number of glutamines exceeds a threshold of approximately 36 &ndash; 40 repeats, XN1 can readily form amyloid aggregates similar to those associated with disease. Many experiments suggest that misfolding of monomeric XN1 plays an important role in the length-dependent aggregation. Elucidating misfolding of XN1 monomer can help determine the molecular mechanism of XN1 aggregation and potentially help develop strategies to inhibit XN1 aggregation. The flanking sequences surrounding the polyQ region can play a critical role in determining the structural rearrangement and aggregation mechanism of XN1. Few experiments have studied XN1 in its entirety, with all flanking regions. To obtain structural insights into the misfolding of XN1 toward amyloid aggregation, we perform molecular dynamics simulations on monomeric XN1 with full flanking regions, a variant missing the poly-proline regions, which are hypothesized to prevent aggregation, and an isolated polyQ peptide (Q<sub>n</sub>). For each of these three constructs, we study glutamine repeat lengths of 23, 36, 40 and 47. We find that polyQ peptides have a positive correlation between their probability to form a &beta;-rich misfolded state and their expansion length. We also find that the flanking regions of XN1 affect its probability to form a &beta;-rich state compared to the isolated polyQ. Particularly, the poly-proline regions form polyproline type II helices and decrease the probability of the polyQ region to form a &beta;-rich state. Additionally, by lengthening polyQ, the first N-terminal 17 residues are more likely to adopt a &beta;-sheet conformation rather than an &alpha;-helix conformation. Therefore, our molecular dynamics study provides a structural insight of XN1 misfolding and elucidates the possible role of the flanking sequences in XN1 aggregation.'),(714,'pdd_wires11','Proctor, E. A., Ding, F., and Dokholyan, N. V.','Discrete Molecular Dynamics','Wiley Interdisciplinary Reviews: Computational Molecular Science',1,'80','92',2011,'Methods','Dokholyan, N. V.','no','no','no','article',0,'','','','','Discrete Molecular Dynamics (DMD) has emerged as a simplification of traditional Molecular Dynamics (MD). DMD employs discrete step function potentials in place of the continuous potential used in traditional MD. As a result, the simulation engine solves the ballistic equations of motion for only those particles participating in a collision, instead of solving Newtons equations of motion for every particle in the system. Because fewer calculations are performed per time step, the DMD technique allows for longer time and length-scales to become accessible in the simulation of large biomolecules. The use of coarse-grained models extends the computational advantage of this method. Although some accuracy is sacrificed to speed, because of the usefulness of DMD to the simulation of many particles at longer timescales the technique has seen application to diverse molecular systems.'),(715,'kdkdh_nb10','Karginov, A. V., Ding, F., Kota, P., Dokholyan, N. V., and Hahn, K. M.','Engineered allosteric activation of kinases in living cells','Nature Biotechnology',28,'743','748',2010,'Biotechnology','Hahn, K. M. and Dokholyan, N. V.','yes','no','no','article',20581846,'','','','','Studies of cellular and tissue dynamics benefit greatly from tools that can control protein activity with specificity and precise timing in living systems. We describe here a new approach to confer allosteric regulation specifically on the catalytic activity of kinases. A highly conserved portion of the kinase catalytic domain is modified with a small protein insert that inactivates catalytic activity, but does not affect other protein interactions. Catalytic activity is restored by addition of rapamycin or non-immunosuppresive analogs (Fig. 1A). We demonstrate the approach by specifically activating focal adhesion kinase (FAK) within minutes in living cells, thereby demonstrating a novel role for FAK in regulation of membrane dynamics. Molecular modeling and mutagenesis indicate that the protein insert reduces activity by increasing the flexibility of the catalytic domain. Drug binding restores activity by increasing rigidity. Successful regulation of Src and p38 suggest that modification of this highly conserved site will be applicable to other kinases.'),(716,'jlywbsdk_jmb10','Jha, R. K., Leaver-Fay, A., Yin, S., Wu, Y., Butterfoss,G. L., Szyperski, T., Dokholyan, N. V., and Kuhlman, B.','Computational design of a PAK1 binding protein','Journal of Molecular Biology',400,'257','270',2010,'Proteins: Design','Kuhlman, B.','no','no','no','article',20460129,'','','','','We describe a computational protocol, called DDMI, for redesigning scaffold proteins to bind to a specified region on a target protein.  The DDMI protocol is implemented within the Rosetta molecular modeling program and uses rigid-body docking, sequence design, and gradient-based minimization of backbone and side chain torsion angles to design low energy interfaces between the scaffold and target protein.  We find that iterative rounds of sequence design and conformational optimization are needed to produce models that have calculated binding energies that are similar to binding energies calculated for native complexes in the protein data bank. We also show that additional conformation sampling with molecular dynamics can be iterated with sequence design to further lower the computed energy of the designed complexes. To experimentally test the DDMI protocol we redesigned the human hyperplastic discs protein (\"scaffold\") to bind to the kinase domain of p21-activated kinase 1 (\"target\", PAK1).  Six designs were experimentally characterized. Two of the designs aggregated and were not characterized further. Of the remaining four designs, three bound to the \"target\" with affinities tighter than 350 uM. The best performing design, called Spider Roll, bound with an affinity of 100 uM.  Mutagenesis studies confirm that Spider Roll binds the target patch on PAK1. Additionally, Spider Roll binds to full length PAK1 in its activated state, but does not bind PAK1 when it forms an auto-inhibited conformation that blocks the Spider Roll target site. The ability to rationally design the site of novel protein-protein interactions is an important step towards creating new proteins that are useful as therapeutics or molecular probes.'),(717,'lddw_b10','Lavender, C., Ding, F. Dokholyan, N. V., and Weeks, K. M.','Robust and generic RNA modeling using inferred constraints: A structure for the hepatitis C virus IRES pseudoknot domain','Biochemistry',49,'4931','4933',2010,'RNA','Dokholyan, N. V.','yes','no','no','article',20545364,'','','','','RNA function is dependent on its structure, yet three-dimensional folds for most biologically important RNAs are unknown. We develop a generic discrete molecular dynamics (DMD)-based modeling system that uses long-range constraints inferred from diverse biochemical or bioinformatic analyses to create statistically significant (p < 0.01) native-like folds for RNAs of known structure ranging from 45 to 158 nucleotides. We then predict the unknown structure of the hepatitis C virus IRES pseudoknot domain. The resulting RNA model rationalizes independent solvent accessibility and cryo-electron microscopy structure information. The pseudoknot positions the AUG start codon near the mRNA channel and is tRNA-like, suggesting the IRES employs molecular mimicry as a functional strategy.'),(718,'akahjcgdr_jmb10','Aleksandrov, A. A., Kota, P., Aleksandrov, L. A., He, L., Jensen, T., Cui, L., Gentzsch, M., Dokholyan, N. V., and Riordan, J. R.','Regulatory insertion removal restores maturation, stability and function of &Delta;F508 CFTR','Journal of Molecular Biology',401,'194','210',2010,'Proteins: Diseases','Riordan, J. R.','no','no','yes','article',20561529,'','','','','The cystic fibrosis transmembrane conductance regulator (CFTR) epithelial anion channel is a large multi-domain membrane protein which matures inefficiently during biosynthesis. Its assembly is further perturbed by the deletion of F508 from the first nucleotide binding domain (NBD1) responsible for most cystic fibrosis. The mutant polypeptide is recognized by cellular quality control systems and is proteolyzed. CFTR NBD1 contains a 32 residue segment termed the regulatory insertion (RI) not present in other ABC transporters. We report here that RI deletion enabled &Delta;F508 CFTR to mature and traffic to the cell surface where it mediated regulated anion efflux and exhibited robust single chloride channel activity. Long term pulse-chase experiments showed that the mature &Delta;RI/&Delta;F508 had a T<sub>1/2</sub> of ~14h in cells, similar to the wild-type. RI deletion restored ATP occlusion by NBD1 of &Delta;F508 CFTR and had a strong thermo-stabilizing influence on the channel with gating up to at least 40<sup>o</sup>C. None of these effects of RI removal were achieved by deletion of only portions of RI. Dynamics simulations of NBD1 indicated that RI might indirectly influence the interaction of NBD1 with the rest of the protein by attenuating the coupling of the F508 containing loop with the F1-like ATP-binding core subdomain so that RI removal overcame the perturbations caused by F508 deletion. Restriction of RI to a particular conformational state may ameliorate the impact of the disease-causing mutation.'),(719,'tmd_plso10','Tsao, D., Minton, A. P., and Dokholyan, N. V.','A Didactic Model of Macromolecular Crowding Effects on\nProtein Folding','Public Library of Science ONE',5,'e11936','',2010,'Proteins: Crowding','Dokholyan, N. V.','no','no','no','article',0,'','','','','A didactic model is presented to illustrate how the effect of macromolecular crowding on protein folding and association is modeled using current analytical theory and discrete molecular dynamics.  While analytical treatments of crowding may consider the effect as a potential of average force acting to compress a polypeptide chain into a compact state, the use of simulations enables the presence of crowding reagents to be treated explicitly.  Using an analytically solvable toy model for protein folding, an approximate statistical thermodynamic method is directly compared to simulation in order to gauge the effectiveness of current analytical crowding descriptions.  Both methodologies are in quantitative agreement under most conditions, indication that both current theory and simulation methods are capable of recapitulating aspects of protein folding even by utilizing a simplistic protein model.'),(733,'pdd_jmb11','Proctor, E. A., Ding, F. and Dokholyan, N. V.','Structural and thermodynamic effects of post-translational modifications in mutant and wild type Cu, Zn Superoxide Dismutase','Journal of Molecular Biology',408,'555','567',2011,'Proteins: Diseases','Dokholyan, N. V.','yes','no','yes','article',0,'','','','','Aggregation of Cu, Zn superoxide dismutase (SOD1) is implicated in Amyotrophic Lateral Sclerosis (ALS). Glutathionylation and phosphorylation of SOD1 is omnipresent in the human body, even in healthy individuals, and has been shown to increase SOD1 dimer dissociation, which is the first step on the pathway toward SOD1 aggregation. We find that post-translational modification of SOD1, especially glutathionylation, promotes dimer dissociation. We discover an intermediate state in the pathway to dissociation, a conformational change that invol a &#39;loosening&#39; of the &beta;-barrels and a loss or shift of dimer interface interactions. In modified SOD1, this intermediate state is stabilized as compared to unmodified SOD1. The presence of post-translational modifications could explain the environmental factors involved in the speed of disease progression. Because post-translational modifications such as glutathionylation are often induced by oxidative stress, post-translational modification of SOD1 could be a factor in the occurrence of sporadic cases of ALS, which make up 90% of all cases of the disease.'),(720,'dyd_jcim10','Ding, F., Yin, S., and Dokholyan, N. V.','Rapid flexible docking using a stochastic rotamer library of ligands','Journal of Chemical Information and Modeling',50,'1623','1632',2010,'Methods','Dokholyan, N. V.','no','no','no','article',20712341,'','','','','Existing flexible docking approaches model the ligand and receptor flexibility either separately or in a loosely-coupled manner, which captures the conformational changes inefficiently. Here, we propose a flexible docking approach, MedusaDock, which models both ligand and receptor flexibility simultaneously with sets of discrete rotamers. We develop an algorithm to build the ligand rotamer library on-the-fly during docking simulations. MedusaDock benchmarks demonstrate a rapid sampling efficiency and high prediction accuracy in both self-docking (to the co-crystallized state) and cross-docking (to a state co-crystallized with a different ligand), the latter of which mimics the virtual-screening procedure in computational drug discovery. We also perform a virtual-screening test of four flexible kinase targets including cyclin-dependent kinase 2, vascular endothelial growth factor receptor 2, HIV reverse transcriptase, and HIV protease. We find significant improvements of virtual-screening enrichments when compared to rigid-receptor methods. The predictive power of MedusaDock in cross-docking and preliminary virtual-screening benchmarks highlights the importance to model both ligand and receptor flexibility simultaneously in computational docking.'),(722,'rkdd_psfb11','Ramachandran, S., Kota, P., Ding, F. and Dokholyan, N. V.','Automated minimization of steric clashes in protein structures','Proteins: Structure, Function and Bioinformatics',79,'261','270',2011,'Proteins: Structure','Dokholyan, N.V.','yes','no','no','article',21058396,'','','','','Molecular modeling of proteins including homology modeling, structure determination, and knowledge-based protein design requires tools to evaluate and refine three-dimensional protein structures. Steric clash is one of the artifacts prevalent in low-resolution structures and homology models. Steric clashes arise due to the unnatural overlap of any two non-bonding atoms in a protein structure. Usually, removal of severe steric clashes in some structures is challenging since many existing refinement programs do not accept structures with severe steric clashes. Here, we present a quantitative approach of identifying steric clashes in proteins by defining clashes based on the Van der Waals repulsion energy of the clashing atoms. We also define a metric for quantitative estimation of the severity of clashes in proteins by performing statistical analysis of clashes in high-resolution protein structures. We describe a rapid, automated and robust protocol, Chiron, which efficiently resolves severe clashes in low-resolution structures and homology models with minimal perturbation in the protein backbone. Benchmark studies highlight the efficiency and robustness of Chiron compared to other widely used methods. We provide Chiron as an automated web server to evaluate and resolve clashes in protein structures that can be further used for more accurate protein design.'),(724,'yd_psfb11','Yin, S., and Dokholyan, N. V.','Fingerprint-based structure retrieval using electron density','Proteins: Structure, Function, and Bioinformatics',79,'1002','1009',2011,'Proteins: Methods','Dokholyan, N. V.','yes','no','no','article',21287628,'','','','','We present a computational approach that can quickly search a large protein structural database to identify structures that fit a given electron density, such as determined by cryo-electron microscopy. We use geometric invariants (fingerprints) constructed using 3D Zernike moments to describe the electron density, and reduce the problem of fitting of the structure to the electron density to simple fingerprint comparison. Using this approach, we are able to screen the entire Protein Data Bank and identify structures that fit two experimental electron densities determined by cryo-electron microscopy.'),(725,'rvsd_plscb11','Ramachandran, S., Vogel, L., Strahl, B. D. and Dokholyan, N.V.','Thermodynamic stability of histone H3 is a necessary but not sufficient driving force for its evolutionary conservation','Public Library of Science Computational Biology',7,'e1001042','',2011,'Proteins: Evolution','Dokholyan, N. V.','yes','no','no','article',0,'','','','','Determining the forces that conserve amino acid positions in proteins across species is a fundamental pursuit of molecular evolution. Evolutionary conservation is driven by either a protein&#39;s function or its thermodynamic stability.  Highly conserved histone proteins offer a platform to evaluate these driving forces. While the conservation of histone H3 and H4 &#34;tail&#34; domains and surface residues are driven by functional importance, the driving force behind the conservation of buried histone residues has not been examined. Using a computational approach, we determined the thermodynamically preferred amino acids at each buried position in H3 and H4.  In agreement with what is normally observed in proteins, we find a significant correlation between thermodynamic stability and evolutionary conservation in the buried residues in H4.  In striking contrast, we find that thermodynamic stability of buried H3 residues does not correlate with evolutionary conservation.  Given that these H3 residues are not post-translationally modified and only regulate H3-H3 and H3-H4 stabilizing interactions, our data imply an unknown function responsible for driving conservation of these buried H3 residues.'),(726,'kzskdyhd_jacs11','Karginov, A. V., Zou, Y., Shirvanyants, D., Kota, P., Dokholyan, N. V., Young, D. D., Hahn, K. M., and Deiters, A.','Light regulation of protein dimerization and kinase activity in living cells using photocaged rapamycin and engineered FKBP','Journal of the American Chemical Society',133,'420','423',2011,'Biotechnology','Dokholyan, N. V.','no','no','no','article',0,'','','','','We developed a new system for light-induced protein dimerization in living cells using a novel photocaged analogue of rapamycin together with an engineered rapamycin binding domain. Using focal adhesion kinase as a target, we demonstrated successful light-mediated regulation of protein interaction and localization in living cells. Modification of this approach enabled light-triggered activation of a protein kinase and initiation of kinase-induced phenotypic changes in vivo.'),(727,'kd_m11','Kota, P., and Dokholyan, N. V.','Approaches for probing the sequence space of substrates recognized by molecular chaperones','Methods',53,'318','324',2011,'Proteins: Folding','Dokholyan, N. V.','no','no','no','article',0,'','','','','Neurodegeneration, the progressive loss of function in neurons that eventually leads to their death, is the cause of many neurodegenerative disorders including Alzheimers, Parkinsons, and Huntingtons diseases. Protein aggregation is a hallmark of most neurodegenerative diseases, where unfolded proteins form intranuclear, cytosolic, and extracellular insoluble aggregates in neurons. Mounting evidence from studies in neurodegenerative disease models shows that molecular chaperones, key regulators of protein aggregation and degradation, play critical roles in the progression of neurodegeneration.  Although chaperones exhibit promiscuity in their substrate specificity, specific molecular features are required for substrate recognition. Understanding the basis for substrate recognition by chaperones will aid in the development of therapeutic strategies that regulate chaperone expression levels in order to combat neurodegeneration. Many experimental techniques, including alanine scanning mutagenesis and phage display library screening, have been developed and applied to understand the basis of substrate recognition by chaperones. Here, we present computational algorithms that can be applied to rapidly screen the sequence space of potential substrates to determine the sequence and structural requirements for substrate recognition by chaperones.'),(728,'std_mmb11','Serohijos, A.W.R., Thibodeau, P.H. and Dokholyan, N.V.','Molecular modeling tools and approaches for CFTR and cystic fibrosis','Cystic Fibrosis: Methods and Protocols',741,'347','363',2011,'Proteins: Diseases','Dokholyan, N. V.','no','no','no','book',0,'','Amaral, M.D. and Kunzelmann, K.','Methods in Molecular Biology','Humana press','Cystic fibrosis is a multi-faceted disease resulting from the dysfunction of the CFTR channel. Understanding the structural basis of channel function and the structural origin of the defect is imperative in the development of therapeutic strategies. Here, we describe molecular modeling tools that, in conjunction with complementary experimental tools, lead to significant findings on CFTR channel function and on the effect of the pathogenic mutant F508del. '),(729,'dd_rsap12','Ding, F and Dokholyan, N.V.','Multiscale modeling of RNA structure and dynamics','RNA 3D Structure Analysis and Prediction',27,'167','184',2012,'RNA','Dokholyan, N. V.','yes','no','no','book',0,'','Leontis, N. and Westhof, E.','Nucleic Acids and Molecular Biology','Springer','We have developed a multiscale approach for RNA folding using dis- crete molecular dynamics (DMD), a rapid conformational sampling algorithm. We use a coarse-grained representation to effectively model RNA structures. Bench- mark studies suggest that the DMD-based RNA model is able to accurately fold small RNA molecules (<50 nucleotides). However, the large conformational space and force field inaccuracies make it difficult to computationally identify the native states of large RNA molecules. We devised an automated modeling approach for prediction of large and complex RNA structures using experimentally derived structural constraints and tested it on several RNA molecules with known experi- mental structures. In all cases, we were able to bias the DMD simulations to the native states of these RNA molecules. Therefore, a combination of experimental and computational approaches has the potential to yield native-like models for the diverse universe of functionally important RNAs, whose structures cannot be characterized by conventional structural methods.'),(730,'rd_pmbts12','Redler, R. and Dokholyan, N.V.','The complex molecular biology of amyotrophic lateral sclerosis (ALS)','Progress in Molecular Biology and Translational Science',107,'215','262',2012,'Proteins: Diseases','Dokholyan, N. V.','no','no','no','article',22482452,'','Teplow, D.B.','','Academic press','Amyotrophic lateral sclerosis (ALS) is an adult-onset neurodegenerative disorder that causes selective death of motor neurons followed by paralysis and death. A subset of ALS cases is caused by mutations in the gene for Cu, Zn superoxide dismutase (SOD1), which impart a toxic gain of function to this antioxidant enzyme. This neurotoxic property is widely believed to stem from an increased propensity to misfold and aggregate caused by decreased stability of the native homodimer or a tendency to lose stabilizing posttranslational modifications. Study of the molecular mechanisms of SOD1-related ALS has revealed a complex array of interconnected pathological processes, including glutamate excitotoxicity, dysregulation of neurotrophic factors and axon guidance proteins, axonal transport defects, mitochondrial dysfunction, deficient protein quality control, and aberrant RNA processing. Many of these pathologies are directly exacerbated by misfolded and aggregated SOD1 and/or cytosolic calcium overload, suggesting the primacy of these events in disease etiology and their potential as targets for therapeutic intervention.'),(731,'bdu_mbapaff10','Betnel, M., Dokholyan, N.V. and Urbanc, B.','From disordered amyloid &alpha;-proteins to soluble oligomers and protofibrils using discrete molecular dynamics','Alzheimer&#39;s disease: Molecular basis of amyloid-beta protein aggregation and fibril formation - Insights into low molecular weight and cytotoxic aggregates from computer simulations',7,'333','357',2013,'Proteins: Diseases','Dokholyan, N. V.','yes','no','no','book',0,'','Derreumaux, P.','','Imperial College Press',NULL),(732,'hrpdm_plso11','Hadi-Alijanvand, H., Rouhani, M., Proctor, E. A., Dokholyan, N. V. and Moosavi-Mohavedi, A. A.','A folding pathway-dependent score to recognize membrane proteins','Public Library of Science ONE',6,'e16778','',2011,'Proteins: Folding','Moosavi-Mohavedi, A.A.','no','no','no','article',0,'','','','','While various approaches exist to study protein localization, it is still a challenge to predict where proteins localize.  Here, we consider a mechanistic viewpoint for membrane localization. Taking into account the steps for the folding pathway of -helical membrane proteins and relating biophysical parameters to each of these steps, we create a score capable of predicting the propensity for membrane localization and call it FP3mem. This score is driven from the principal component analysis (PCA) of the biophysical parameters related to membrane localization. FP3mem allow us to rationalize the colocalization of a number of channel proteins with the Cav1.2 channel by their fewer propensities for membrane localization.'),(734,'sad_b11','Shirvanyants, D., Alexandrova, A. N. and Dokholyan, N. V.','Rigid substructure search','Bioinformatics',27,'1327','1329',2011,'Proteins: Methods','Dokholyan, N. V.','no','no','no','article',0,'','','','','Motivation: Identifying the location of binding sites on proteins is of fundamental importance for a wide range of applications, including molecular docking, de novo drug design, structure identification, and comparison of functional sites.  Here we present Erebus, a web-server that searches the entire Protein Data Bank for a given sub-structure defined by a set of atoms of interest, such as the binding scaffolds for small molecules. The identified substructure contains atoms having the same names, belonging to same amino acids, and separated by the same distances (within a given tolerance) as the atoms of the query structure. The accuracy of a match is measured by the root-mean-square deviation or by the normal weight with a given variance. Tests show that our approach can reliably locate rigid binding scaffolds of drugs and metal ions.<br>Availability and Implementation: We provide this service through a web server at http://erebus.dokhlab.org'),(735,'tsgdd_nar11','Tsao, D., Shabalina, S. A., Gauthier, J., Dokholyan, N. V., and Diatchenko, L.','Disruptive mRNA folding increases translational efficiency of catechol O-methyltransferase variant','Nucleic Acids Research',39,'6201','6212',2011,'RNA','Dokholyan, N. V.','no','no','no','article',0,'','','','','Catechol-O-methyltransferase (COMT) is a major enzyme controlling catecholamine levels that plays a central role in cognition, affective mood and pain perception.  There are three common COMT haplotypes in the human population reported to have functional effects, divergent in two synonymous and one nonsynonymous position.  We demonstrate that one of the haplotypes, carrying the non-synonymous variation known to code for a less stable protein, exhibits increased protein expression in vitro. This increased protein expression, which would compensate for lower protein stability, is solely produced by a synonymous variation (C166T) situated within the haplotype and located in the 5 region of the RNA transcript.  Based on mRNA secondary structure predictions, we suggest that structural destabilization near the start codon caused by the T allele could be related to the observed increase in COMT expression.  Our folding simulations of the tertiary mRNA structures demonstrate that destabilization by the T allele lowers the folding transition barrier thus decreasing the probability of occupying its native state. These data suggest a novel structural mechanism whereby functional synonymous variations near the translation initiation codon affect the translation efficiency via entropy-driven changes in mRNA dynamics and present another example of stable compensatory genetic variations in the human population.'),(736,'tld_cpl11','Tsao, D., Liu, S., and Dokholyan, N. V.','Regioselectivity of catechol O-methyltransferase confers enhancement of catalytic activity','Chemical Physics Letters',506,'135','138',2011,'Proteins: Function','Dokholyan, N. V.','no','no','no','article',0,'','','','','Catechol O-methyltransferase (COMT) metabolizes catechol moieties by methylating a single hydroxyl group at the meta- or para- hydroxyl position. Hydrophobic amino acids near the active site of COMT influence the regioselectivity of this reaction.  Our sequence analysis highlights their importance by showing that these residues are highly conserved throughout evolution.  Reaction barriers calculated in the gas phase reveal a lower barrier during methylation at the meta- position, suggesting that the observed meta-regioselectivity of COMT can be attributed to the substrate itself, and that COMT has evolved residues to orient the substrate in a manner that increases the rate of catalysis.'),(737,'d_rrb11','Dokholyan, N. V.','Towards quantitative biochemistry: Research and Reports in Biochemistry','Research and Reports in Biochemistry',1,'1','2',2011,'Biochemistry','Dokholyan, N. V.','no','no','no','editorial',0,'','','','',''),(738,'tsde_bj11','Tsygankov, D., Serohijos, A. W. R., Dokholyan, N. V. and Elston, T. C.','A physical model reveals the mechanochemistry responsible for dynein&#39;s processive motion','Biophysical Journal',101,'144','150',2011,'Proteins: Function','Elston, T.C.','yes','no','no','article',0,'','','','','The molecular motor dynein is associated with various cellular activities, such as directed transport along microtubules and the rhythmic beating of the axoneme. Because of the size and complexity of the protein, a detailed understanding of the mechanochemistry that drives dyneins processive motion is lacking. To overcome this deficiency, we developed the first computational model for two-headed dynein that couples conformational changes of the motors subunits to the biochemical steps involved in ATP hydrolysis. Analysis of the model provides several novel insights into how the protein functions: structural constraints limit the motion of the free microtubule binding domain to one dimension, increasing the efficiency with which this domain finds a binding site; in addition to the power stroke of the bound head, recovery of the free head to a pre-power stroke conformation is required for this head to reach a forward binding site; the order in which the power stroke and recovery transitions occur affects the probability of back stepping; the existence of multiple equilibria in the motors bending energy provides a mechanism for processive back-stepping. Our computational model represents the first complete mechanochemical description of the motor protein dynein and should motivate novel experimental investigations to test its predictions.'),(739,'kdrd_b11','Kota, P., Ding, F., Ramachandran, S. and Dokholyan, N. V.','Gaia: Automated quality assessment of protein structure models','Bioinformatics',27,'2209','2215',2011,'Proteins: Methods','Dokholyan, N. V.','yes','no','no','article',0,'','','','','Motivation: Increasing use of structural modeling for understanding structure-function relationships in proteins has led to the need to ensure that the protein models being used are of acceptable quality. Quality of a given protein structure can be assessed by comparing various intrinsic structural properties of the protein to those observed in high resolution protein structures.  Results: In this study, we present tools to compare a given structure to high resolution crystal structures. We assess packing by calculating the total void volume, the percentage of unsatisfied hydrogen bonds, the number of steric clashes, and the scaling of the accessible surface area. We assess covalent geometry by determining bond lengths, angles, dihedrals, and rotamers. The statistical parameters for the above measures, obtained from high resolution crystal structures enable us to provide a quality-score that points to specific areas where a given protein structural model needs improvement. Availability and Implementation: We provide these tools that appraise protein structures in the form of a web server Gaia (http://chiron.dokhlab.org). Gaia evaluates the packing and covalent geometry of a given protein structure and provides quantitative comparison of the given structure to high resolution crystal structures. '),(740,'rwpfcd_b11','Redler, R. L., Wilcox, K. C., Proctor, E. A., Fee, L., Caplow, M., and Dokholyan, N. V.','Glutathionylation at cys 111 triggers dissociation of wild type and fals mutant SOD1 dimers','Biochemistry',50,'7057','7066',2011,'','Dokholyan, N. V.','no','no','yes','article',0,'','','','','Mutation of the ubiquitous cytosolic enzyme Cu/Zn superoxide dismutase (SOD1) is hypothesized to cause familial amyotrophic lateral sclerosis (FALS) through structural destabilization leading to misfolding and aggregation. Considering the late onset of symptoms as well as the phenotypic variability among patients with identical SOD1 mutations, it is clear that non-genetic factor(s) impact ALS etiology and disease progression. Here we examine the effect of Cys-111 glutathionylation, a physiologically prevalent post-translational oxidative modification, on the stabilities of wild type SOD1 and two phenotypically diverse FALS mutants, A4V and I112T. Glutathionylation results in profound destabilization of SOD1WT dimers, increasing the equilibrium dissociation constant Kd to ~10-20 M, comparable to that of the aggressive A4V mutant. SOD1A4V is further destabilized by glutathionylation, experiencing an approximately 30-fold increase in Kd. Dissociation kinetics of glutathionylated SOD1WT and SOD1A4V are unchanged, as measured by surface plasmon resonance, indicating that glutathionylation destabilizes these variants by decreasing association rate. In contrast, SOD1I112T has a modestly increased dissociation rate but no change in Kd when glutathionylated. Using computational structural modeling, we show that the distinct effects of glutathionylation on different SOD1 variants correspond to changes in composition of the dimer interface. Our experimental and computational results show that Cys-111 glutathionylation induces structural rearrangements that modulate stability of both wild type and FALS mutant SOD1. The distinct sensitivities of SOD1 variants to glutathionylation, a modification that acts in part as a coping mechanism for oxidative stress, suggest a novel mode by which redox regulation and aggregation propensity interact in ALS.'),(741,'brspkbbscdc_po11','Bhattacharyya, D., Ramachandran, S., Sharma, S., Pathmasiri, W., King, C. L., Baskerville-Abraham, I., Boysen, G., Swenberg, J. A., Campbell, S. L., Dokholyan, N. V. and Chaney, S. G.','Flanking Bases Influence the Nature of DNA Distortion by Oxaliplatin 1,2-Intrastrand (GG) Cross-links','Public Library of Science ONE',6,'e23582','',2011,'DNA: Drugs','Campbell, S. L., Dokholyan, N.V. and Chaney, S. G.','yes','no','no','article',0,'','','','','The differences in efficacy and molecular mechanisms of platinum anti-cancer drugs cisplatin (CP) and oxaliplatin (OX) are thought to be partially due to the differences in the DNA conformations of the CP and OX adducts that form on adjacent guanines on DNA, which in turn influence the binding of damage-recognition proteins that control downstream effects of the adducts. Here we report a comprehensive comparison of the structural distortion of DNA caused by CP and OX adducts in the TGGT sequence context using nuclear magnetic resonance (NMR) spectroscopy and molecular dynamics (MD) simulations. When compared to our previous studies in other sequence contexts, these structural studies help us understand the effect of the sequence context on the conformation of Pt-GG DNA adducts. We find that both the sequence context and the type of Pt-GG DNA adduct (CP vs OX) play an important role in the conformation and the conformational dynamics of Pt-DNA adducts, possibly explaining their influence on the ability of many damage-recognition proteins to bind to Pt-DNA adducts. '),(742,'hylsdt_jcim11','Hsieh, J.-H., Yin, S., Liu, S., Sedykh, A., Dokholyan, N. V., and Tropsha, A.','Combined application of cheminformatics- and physical force field-based scoring functions improves binding affinity prediction for CSAR datasets','Journal of Chemical Information and Modeling',51,'2027','2035',2011,'Drugs','Dokholyan, N. V.','no','no','no','article',0,'','','','','The curated CSAR-NRC benchmark sets provide valuable opportunity for testing or comparing the performance of both existing and novel scoring functions. We apply two different scoring functions, both independently and in combination, to predict binding affinity of ligands in the CSAR-NRC datasets. One, reported here for the first time, employs multiple chemical-geometrical descriptors of the protein-ligand interface to develop Quantitative Structure  Binding Affinity Relationships (QSBAR) models; these models are then used to predict binding affinity of ligands in the external dataset. Second is a physical force field-based scoring function, MedusaScore. We show that both individual scoring functions achieve statistically significant prediction accuracies with the squared correlation coefficient (R2) between actual and predicted binding affinity of 0.44/0.53 (Set1/Set2) with QSBAR models and 0.34/0.47 (Set1/Set2) with MedusaScore. Importantly, we find that the combination of QSBAR models and MedusaScore into consensus scoring function affords higher prediction accuracy than any of the contributing methods achieving R2 of 0.45/0.58 (Set1/Set2). Furthermore, we identify several chemical features and non-covalent interactions that may be responsible for the inaccurate prediction of binding affinity for several ligands by the scoring functions employed in this study.'),(743,'grkdsh_bba11','Gyimesi, G., Ramachandran, S., Kota, P., Dokholyan, N. V., Sarkadi, B., and Hegedus, T.','ATP hydrolysis at one of the two sites in ABC transporters initiates transport related conformational transitions','Biochimica et Biophysica Acta - Biomembranes',1808,'2954','2964',2011,'Proteins: Function','Dokholyan, N. V.','yes','no','no','article',0,'','','','','ABC transporters play important roles in all types of organisms by participating in both physiological and pathological processes. In order to modulate the function and stability of ABC transporters, detailed knowledge regarding their structure and dynamics is necessary. Recently published atomic resolution X-ray structures of ABC exporter proteins indicate three major conformations, a nucleotide-bound bottom-closed state with the two nucleotide binding domains (NBDs) tightly closed, and two nucleotide-free conformations, the bottom-closed conformation where the NBDs are close to each other, and the bottom-open where the two NBDs are widely separated. Since there are experimental indications that a large distance between NBDs is not a requisite for function, it still remains a question how the widely open conformation should be interpreted.  It is questioned whether or not hydrolysis at one of the sites can drive conformational transitions while the NBDs remain in contact.  To extend our knowledge on these questions, we have investigated the dynamic properties of the bacterial Sav1866 transporter in different states using molecular dynamics (MD) simulations. We have demonstrated that the replacement of ATP by ADP at a nucleotide binding site in the bottom-closed holo conformation alters the correlated motion patterns of the NBDs as well as the transmembrane domains (TMD). The results suggest that the hydrolysis of even a single nucleotide could directly lead to extracellular closure, driving the transport cycle. Essential dynamics analysis of simulations suggest that the replacement of one of the ATP molecules can drive the system towards a bottom-closed apo conformation similar to that observed in the X-ray structure of the bacterial ABC transporter, MsbA. We have also found significant structural instability of the bottom-open form of the transporters in MD simulations.  Our results suggest that ATP hydrolysis at one of the sites promotes transport related conformational changes that do not lead to a bottom-open apo conformation, and that the bottom-closed conformation is physiologically more relevant for describing the structure of the apo state.'),(744,'tdd_plso11','Tsao, D., Diatchenko, L., and Dokholyan, N. V.','Structural mechanism of S-adenosyl methionine binding to catechol O-methyltransferase','Public Library of Science ONE',6,'e24287','',2011,'Proteins: Function','Dokholyan, N. V.','no','no','no','article',0,'','','','','	Methyltransferases possess a homologous domain that requires both a divalent metal cation and S-adenosyl-L-methionine (SAM) to catalyze its reactions.   The kinetics of several methyltransferases has been well characterized; however, the details regarding their structural mechanisms have remained unclear to date.  Using catechol O-methyltransferase (COMT) as a model, we perform discrete molecular dynamics and computational docking simulations to elucidate the initial stages of cofactor binding.  We find that COMT binds SAM via an induced-fit mechanism, where SAM adopts a different docking pose in the absence of metal and substrate in comparison to the holoenzyme.  Flexible modeling of the active site side-chains is essential for observing the lowest energy state in the apoenzyme; rigid docking tools are unable to recapitulate the pose unless the appropriate side-chain conformations are given a priori.  From our docking results, we hypothesize that the metal reorients SAM in a conformation suitable for donating its methyl substituent to the recipient ligand.  The proposed mechanism enables a general understanding of how divalent metal cations contribute to methyltransferase function.'),(745,'sydggmdd_s11','Serohijos, A.W.R., Yin, S., Ding, F., Gauthier, J., Gibson, D., Maixner, W., Dokholyan, N. V., and Diatchenko, L.','Structural basis for &mu;-opioid receptor binding and activation','Structure',19,'1683','1690',2011,'Proteins: Function','Dokholyan, N. V.','yes','no','no','article',0,'','','','','Opioids that stimulate the -opioid receptor (MOR1) are the most frequently prescribed and effective analgesics.  Here we present a structural model of MOR1. Molecular dynamics simulations show a ligand-dependent increase in the conformational flexibility of the third intracellular loop that couples with the G-protein complex. These simulations likewise identified residues that form frequent contacts with ligands.  We validated the binding residues using site-directed mutagenesis coupled with radioligand binding and functional assays.   The model was used to blindly screen a library of ~1.2 million compounds.   From the thirty-four compounds predicted to be strong binders, the top three candidates were examined using biochemical assays. One of these compounds showed high efficacy and potency. Post hoc testing revealed this compound to be nalmefene, a potent clinically used antagonist, thus further validating the model.  In summary, the MOR1 model provides a tool for elucidating the structural mechanism of ligand-initiated cell signaling and screening for novel opioid receptor ligands.'),(746,'dd_cmbs12','Ding, F., and Dokholyan, N.V.','Discrete molecular dynamics simulation of biomolecules.','Computational Modeling of Biological Systems: From Molecules to Pathways',0,'55','73',2012,'Proteins: Methods','Dokholyan, N. V.','yes','no','no','book',0,'','','','',''),(747,'rd_cmbs12','Ramachandran, S., and Dokholyan, N.V.','Homology modeling: Generating structural models to understand protein function and mechanism.','Computational Modeling of Biological Systems: From Molecules to Pathways',0,'97','116',2012,'Proteins: Structure','Dokholyan, N. V.','yes','no','no','book',0,'','Dokholyan, N.V.','','Springer',''),(748,'ckdf_cfmbpcts11','Chong, P.A., Kota, P., Dokholyan, N.V. and Forman-Kay, J.D.','Dynamics intrinsic to CFTR function and stability.','Cystic Fibrosis: Molecular Basis, Physiological Changes, and Therapeutic strategies.',3,'a009522','',2013,'Proteins: Diseases','Dokholyan, N. V.','yes','no','no','book',0,'','Riordan, J.R., Boucher, R.C., and Quinton, P.M.','','Cold Spring Harbor Press',''),(749,'dpddd_s11','Dagliyan, O., Proctor, E. A., D&#39;Auria, K., Ding, F. and Dokholyan N. V.','Structural and Dynamic Determinants of Protein-peptide Recognition','Structure',19,'1837','1845',2011,'Proteins: Methods','Ding, F. and Dokholyan, N.V.','yes','yes','no','article',0,'','','','','Protein-peptide interactions play important roles in many cellular processes, including signal transduction, trafficking, and immune recognition. Protein conformational changes upon binding, an ill-defined peptide binding surface, and the large number of peptide degrees of freedom make the prediction of protein-peptide interactions particularly challenging. To address these challenges, we perform rapid molecular dynamics simulations in order to examine the energetic and dynamic aspects of protein-peptide binding. We find that, in most cases, we recapitulate the native binding sites and native-like poses of protein-peptide complexes. Inclusion of electrostatic interactions in simulations significantly improves the prediction accuracy. Our results also highlight the importance of protein conformational flexibility, especially side-chain movement, which allows the peptide to optimize its conformation. Our findings not only demonstrate the importance of sufficient sampling of the protein and peptide conformations, but also reveal the possible effects of electrostatics and conformational flexibility on peptide recognition. '),(750,'hywldt_jcim12','Hsieh, J-H., Yin, S., Wang, X.S., Liu, S., Dokholyan, N.V. and Tropsha, A.','Cheminformatics meets molecular simulations: A combined application of knowledge based and physical force field-based pose scoring functions improves the accuracy of virtual screening','Journal of Chemical Information and Modeling',52,'16','28',2012,'Drugs: Screening','Tropsha, A.','yes','no','no','article',0,'','','','','Poor performance of scoring functions, which is a well-known bottleneck in structure-based virtual screening, is most frequently manifested in their inability to discriminate between true ligands versus known non-binders (therefore designated as binding decoys). This deficiency leads to a large number of false positive hits resulting from virtual screening. We have hypothesized that filtering out docking poses recognized as non-native (i.e., pose decoys) should improve the performance of virtual screening in terms of improved identification of true binders. Using several concepts from the field of cheminformatics, we have developed a novel approach to identifying pose decoys from an ensemble of poses generated by computational docking procedures. We demonstrate that the use of pose filters leads to significant improvement of hit rates in virtual screening studies employing a physical force field-based scoring function (MedusaScore) for 12 of the 13 benchmark sets from the clustered version of Database of Useful Decoys (DUD). This new combined scoring function outperforms several conventional structure-based scoring functions, including XSCORE::HMSCORE, ChemScore, PLP, and Chemgauss3, in six out of 13 data sets at early stage of VS (up 1% decoys of the screening database). We compare our method with several VS methods that were recently reported to have good performances on the same DUD data sets. Compared with two ligand-based methods (FieldScreen and FLAP::LBX), we find that the retrieved ligands using our method are chemically more diverse. We also compare our method with FLAP::RBLB, a high-performance VS method that also utilizes both the receptor and the cognate ligand structures. Interestingly, we find that the top ligands retrieved using our method are highly complementary to those retrieved using FLAP::RBLB, hinting effective directions for best VS applications. We suggest that this integrative virtual screening approach combining cheminformatics and molecular mechanics methodologies may be applied to a broad variety of protein targets to improve the outcome of structure-based drug discovery studies.'),(751,'pytd_bj12','Proctor, E. A., Yin, S., Tropsha, A., and Dokholyan, N. V.','Discrete molecular dynamics distinguishes native-like binding poses from decoys in difficult targets','Biophysical Journal',102,'144','151',2012,'Drugs','Dokholyan, N. V.','yes','no','no','article',0,'','','','','Virtual screening is one of the major tools used in computer-aided drug discovery. In structure-based virtual screening, the scoring function is critical to identifying the correct docking pose and accurately predicting the binding affinities of compounds. However, the performance of existing scoring functions has been shown to be uneven for different targets, and some important drug targets have proven especially challenging. In these targets, scoring functions cannot accurately identify the native or near-native binding pose of the ligand from among decoy poses, affecting both the accuracy of the binding affinity prediction and the ability of virtual screening to identify true binders in chemical libraries. Here, we present an approach to discriminate native poses from decoys in difficult targets where several scoring functions failed to correctly identify the native pose. Our approach employs Discrete Molecular Dynamics simulations in order to incorporate protein-ligand dynamics and the entropic effects of binding. We analyze a collection of poses generated by docking and find that the residence time of the ligand in the native and native-like binding poses is distinctly longer than that in decoy poses. This finding suggests that molecular simulations offer a unique approach to distinguishing the native (or native-like) binding pose from decoys that cannot be accomplished with scoring functions evaluating static structures. The success of our method emphasizes the importance of protein-ligand dynamics in the accurate determination of binding pose, an aspect that is not addressed in typical docking and scoring protocols.'),(753,'dfnd_jmb12','Ding, F., Furukawa, Y., Nukina, N., and Dokholyan, N. V.','Local unfolding of Cu, Zn Superoxide Dismutase monomer determines the morphology of fibrillar aggregates','Journal of Molecular Biology',421,'548','560',2012,'Proteins: Aggregation','Dokholyan, N. V.','yes','no','no','article',0,'','','','','Aggregation of Cu, Zn Superoxide Dismutase (SOD1) is often found in Amyotrophic Lateral Sclerosis (ALS) patients. The fibrillar aggregates formed by wildtype and various disease-associated mutants have recently been found to have distinct cores and morphologies. Previous computational and experimental studies of wildtype SOD1 suggest that the apo-monomer, highly aggregation-prone, displays substantial local unfolding dynamics. The residual folded structure of locally unfolded apoSOD1 corresponds to peptide segments forming the aggregation core as identified by a combination of proteolysis and mass spectroscopy. Therefore, we hypothesize that the destabilization of apoSOD1 caused by various mutations leads to distinct local unfolding dynamics. The partially unfolded structure, exposing the hydrophobic core and backbone hydrogen bond donors and acceptors, is prone to aggregate. The peptide segments in the residual folded structures form the building block for aggregation, which in turn determines the morphology of the aggregates. To test this hypothesis, we apply a multiscale simulation approach to study the aggregation of three typical SOD1 variants: wildtype, G37R, and I149T. Each of these SOD1 variants has distinct peptide segments forming the core structure and features different aggregate morphologies. We perform atomistic molecular dynamics simulations to study the conformational dynamics of apoSOD1 monomer, and coarse-grained molecular dynamics simulations to study the aggregation of partially unfolded SOD1 monomers. Our computational studies of monomer local unfolding and the aggregation of different SOD1 variants are consistent with experiments, supporting the hypothesis of the formation of aggregation building blocks via apo-monomer local unfolding as the mechanism of SOD1 fibrillar aggregation. '),(754,'cbbbccdddfllmmppsshrrsssstwwwlw_rna12','Cruz, J. A., Blanchet, M-F., Boniecki, M., Bujnicki, J. M., Chen, S-J., Cao, S., Das, R., Ding, F., Dokholyan, N. V., Flores, S. C., Lavender C. A., Lisi, V., Major, F., Mikolajczak, K., Patel D. J., Philips, A., Puton, T., Santalucia, J., Siyenji, F., Hermann, T., Rother, K., Rother, M., Serganov, S., Skorupski, M., Soltysinski, T., Sripakdeevong, P., Tuszynska, I., Weeks, K. M., Waldsich, C., Wildauer, M., Leontis, N. B. and Westhof, E.','RNA-Puzzles: A CASP-like evaluation of RNA three-dimensional structure prediction','RNA',18,'610','625',2012,'RNA','Dokholyan, N. V.','no','no','no','article',0,'','','','','We report the results of a first, collective, blind experiment in RNA three-dimensional (3D) structure prediction, encompassing three prediction puzzles. The goals are to assess the leading edge of RNA structure prediction techniques; compare existing methods and tools; and evaluate their relative strengths, weaknesses, and limitations in terms of sequence length and structural complexity. The results should give potential users insight into the suitability of available methods for different applications and facilitate efforts in the RNA structure prediction community in ongoing efforts to improve prediction tools. We also report the creation of an automated evaluation pipeline to facilitate the analysis of future RNA structure prediction exercises.'),(755,'nbssrsdmbtkmnwpfmrlhbczwhdwmkss_mt12','Nakayama, T., Butler, J. S., Sehgal, A., Severgnini, M., Racie, T., Sharman J., Ding, F., Morskaya, S. S., Brodsky, J., Tchangov, L., Kosovrasti, V., Meys, M., Nechev, L., Wang, G., Peng, C. G., Fang, Y., Maier, M., Rajeev, K. G., Li, R., Hettinger, J., Barros, S., Clausen, V., Zhang, X., Wang, Q., Hutabarat, R., Dokholyan, N. V., Wolfrum, C., Manoharan, M., Kotelianski, V., Stoffel, M. and Sah, D. W. Y.','Harnessing a Physiologic Mechanism for siRNA Delivery with Mimetic Lipoprotein Particles','Molecular Therapy',20,'1582','1589',2012,'Biotechnology','Nakayama, T.,','yes','no','no','article',0,'','','','','Therapeutics based on RNA interference (RNAi) have emerged as a potential new class of drugs for treating human disease by silencing the target mRNA, thereby reducing levels of the corresponding pathogenic protein. The major challenge for RNAi therapeutics is the development of safe delivery vehicles for small interfering RNAs (siRNAs). We previously showed that cholesterol-conjugated siRNAs (chol-siRNA) associate with plasma lipoprotein particles and distribute primarily to the liver after systemic administration to mice. We further demonstrated enhancement of silencing by administration of chol-siRNA pre-associated with isolated HDL or LDL. In this study, we investigated mimetic lipoprotein particle prepared from recombinant apolipoprotein A1 (apoA) and apolipoprotein E3 (apoE) as a delivery vehicle for chol-siRNAs. We show that apoE-containing particle (E-lip) is highly effective in functional delivery of chol-siRNA to mouse liver. E-lip delivery was found to be considerably more potent than apoA-containing particle (A-lip). Furthermore, E-lip mediated delivery was not significantly affected by high endogenous levels of plasma LDL. These results demonstrate that E-lip has substantial potential as delivery vehicles for lipophilic conjugates of siRNAs.'),(756,'sdtrd_jcpb12','Shirvanyants, D., Ding,  F., Tsao, D., Ramachandran, S. and Dokholyan, N. V.','DMD: an efficient and versatile simulation method for fine protein characterization','Journal of Physical Chemistry B',116,'8375','8382',2012,'Methods','Dokholyan, N. V.','yes','no','no','article',0,'','','','','Until now it has been impractical to observe protein folding in silico for proteins larger than 50 residues. Limitations of both force field accuracy and computational efficiency make the folding problem very challenging. Here we employ discrete molecular dynamics (DMD) simulations with an all-atom force field to fold fast-folding proteins. We extend the DMD force field by introducing long-range electrostatic interactions to model salt-bridges and a sequence-dependent semi-empirical potential accounting for natural tendencies of certain amino acid sequences to form specific secondary structures. We enhance the computational performance by parallelizing the DMD algorithm. Using a small number of commodity computers, we achieve sampling quality and folding accuracy comparable to the explicit-solvent simulations performed on high-end hardware. We demonstrate that DMD can be used to observe equilibrium folding of villin headpiece and WW domain, study two-state folding kinetics and sample near-native states in ab initio folding of proteins of ~100 residues.'),(757,'d_jpcb12','Dokholyan, N. V.','Physical microscopic model of proteins under force','Journal of Physical Chemistry B',116,'6806','6809',2012,'Proteins: Folding','Dokholyan, N. V.','no','no','no','article',0,'','','','','Nature has evolved proteins to counter-act forces applied on living cells, and designed proteins that can sense forces. One can appreciate Natures ingenuity in evolving these proteins to be highly sensitive to force and to have a high dynamic force range at which they operate.  To achieve this level of sensitivity, many of these proteins are comprised of multiple domains and linking peptides connecting these domain, each of them have their own force response regimes. Here, using a simple model of a protein, we address the question of how each individual domain responds to force. We also ask how multi-domain proteins respond to forces.  We find that the end-to-end distance of individual domains under force scales linearly with force. In multi-domain proteins, we find that the force response has a rich range: at low force, extension is predominantly governed by weaker linking peptides or domain intermediates, while at higher force, the extension is governed by unfolding of individual domains. Overall, the force extension curve comprises multiple sigmoidal transition governed by unfolding of linking peptides and domains.  Our study provides a basic framework for the understanding of protein response to force, and allows for interpretation experiments in which force is used to study the mechanical properties of multi-domain proteins.'),(758,'pkda_po12','Pulicherla, N., Kota, P., Dokholyan, N. V. and Asokan, A.','Intra- and inter-subunit disulfide bond formation is nonessential in adeno-associated viral capsids','Public Library of Science One',7,'e32163','',2012,'Proteins: Structure','Dokholyan, N. V.','yes','no','no','article',0,'','','','','The capsid proteins of adeno-associated viruses (AAV) have five conserved cysteine residues. Structural analysis of AAV serotype 2 reveals that Cys289 and Cys361 are located adjacent to each other within each monomer, while Cys230 and Cys394 are located on opposite edges of each subunit and juxtaposed at the pentamer interface. The Cys482 residue is located at the base of a surface loop within the trimer region. Although plausible based on molecular dynamics simulations, intra- or inter-subunit disulfides have not been observed in structural studies. In the current study, we generated a panel of Cys-to-Ser mutants to interrogate the potential for disulfide bond formation in AAV capsids. The C289S, C361S and C482S mutants were similar to wild type AAV with regard to titer and transduction efficiency. However, AAV capsid protein subunits with C230S or C394S mutations were prone to proteasomal degradation within the host cells. Proteasomal inhibition partially blocked degradation of mutant capsid proteins, but failed to rescue infectious virions. While these results suggest that the Cys230/394 pair is critical, a C394V mutant was found viable, but not the corresponding C230V mutant. Although the exact nature of the structural contribution(s) of Cys230 and Cys394 residues to AAV capsid formation remains to be determined, these results support the notion that disulfide bond formation within the Cys289/361 or Cys230/394 pair appears to be nonessential. These studies represent an important step towards understanding the role of inter-subunit interactions that drive AAV capsid assembly.'),(759,'akcjarhgadr_jmb12','Aleksandrov, A. A., Kota, P., Cui, L., Jensen, T., Alekseev, A. E., Reyes, S., He, L., Gentzsch, M., Aleksandrov, L. A., Dokholyan, N. V. and Riordan, J. R.','Allosteric modulation balances thermodynamic stability and restores function of DelF508 CFTR','Journal of Molecular Biology',419,'41','60',2012,'Proteins: Diseases','Dokholyan, N. V.','yes','no','no','article',0,'','','','',''),(760,'dlwd_nm12','Ding, F., Lavender, C. A., Weeks, K. M. and Dokholyan, N. V.','Three-dimensional RNA structure refinement by hydroxyl radical probing','Nature Methods',9,'603','608',2012,'RNA','Dokholyan, N. V.','yes','no','no','article',0,'','','','','Molecular modeling guided by experimentally-derived structural information is an attractive approach for three-dimensional structure determination of complex RNAs that are not amenable to study by high-resolution methods. Hydroxyl radical probing (HRP), performed routinely in many laboratories, provides a measure of solvent accessibility at individual nucleotides. HRP measurements have, to date, only been used to evaluate RNA models qualitatively. Here, we report development of a quantitative structure refinement approach using HRP measurements to drive discrete molecular dynamics simulations for RNAs ranging in size from 80 to 230 nucleotides. HRP reactivities were first used to identify RNAs that form extensive helical packing interactions. For these RNAs, we achieved highly significant structure predictions, given inputs of RNA sequence and base pairing. This HRP-directed tertiary structure refinement approach generates robust structural hypotheses useful for guiding explorations of structure-function interrelationships in RNA.'),(761,'ltbbbbcddeeggghlllnppprrrssstttww_ps12','Liberles, D.A., Teichmann, S.A., Bahar, I., Bastolla, U., Bloom, J., Bornberg-Bauer, E., Colwell, L.J., de Koning, A.P.J., Dokholyan, N.V., Echave, J., Elofson, A., Gerloff, D.L., Goldstein, R.A., Grahnen, J.A., Holder, M.T., Lakner, C., Lartillot, N., Lovell, S.C., Naylor, G., Perica, T., Pollock, D.D., Pupko, T., Regan, L., Roger, A., Rubinstein, N., Shakhnovich, E.I., Sjolander, K., Sunyaev, S., Teufel, A.I., Thorne, J.L., Thornton, J.W., Weinreich, D.M. and Whelan, S.','The interface of protein structure, protein biophysics, and molecular evolution','Protein Science',21,'769','785',2012,'Proteins: Evolution','Liberles, D.A., and Teichmann, S.A.','yes','no','no','article',0,'','','','','The interface of protein structural biology, protein biophysics, molecular evolution, and molecular population genetics forms the foundations for a mechanistic understanding of many aspects of protein biochemistry. Current efforts in interdisciplinary protein modeling are in their infancy and the state-of-the art of such models is described. Beyond the relationship between amino acid substitution and static protein structure, protein function, and corresponding organismal fitness, other considerations are also discussed. More complex mutational processes such as insertion and deletion and domain rearrangements and even circular permutations should be evaluated. The role of intrinsically disordered proteins is still controversial, but may be increasingly important to consider. Protein geometry and protein dynamics as a deviation from static considerations of protein structure are also important. Protein expression level is known to be a major determinant of evolutionary rate and several considerations including selection at the mRNA level and the role of interaction specificity are discussed. Lastly, the relationship between modeling and needed high-throughput experimental data as well as experimental examination of protein evolution using ancestral sequence resurrection and in vitro biochemistry are presented, towards an aim of ultimately generating better models for biological inference and prediction.'),(762,'twrsrsmmdd_mp12','Tsao, D., Wieskopf, J. S., Rashid, N., Sorge, R. E., Redler, R. L., Segall, S. K., Mogil, J. S., Maixner, W., Dokholyan, N. V. and Diatchenko, L.','Serotonin-induced hypersensitivity via inhibition of catechol O-methyltransferase activity','Molecular Pain',8,'25','',2012,'Proteins: Diseases','Diatchenko, L. and Dokholyan, N. V.','no','no','no','article',0,'','','','','The subcutaneous and systemic injection of serotonin reduces cutaneous and visceral pain thresholds and increases responses to noxious stimuli.  Different subtypes of 5-hydroxytryptamine (5-HT) receptors are suggested to be associated with different types of pain responses. Here we show that serotonin also inhibits catechol O-methyltransferase (COMT), an enzyme that contributes to modultion the perception of pain, via non-competitive binding to the site bound by catechol substrates with a binding affinity comparable to the binding affinity of catechol itself (Ki = 44 M).  Using computational modeling, biochemical tests and cellular assays we show that serotonin actively competes with the methyl donor S-adenosyl-L-methionine (SAM) within the catalytic site.  Binding of serotonin to the catalytic site inhibits the access of SAM, thus preventing methylation of COMT substrates.  The results of in vivo animal studies show that serotonin-induced pain hypersensitivity in mice is reduced by either SAM pretreatment or by the combined administration of selective antagonists for 2- and 3-adrenergic receptors, which have been previously shown to mediate COMT-dependent pain signaling.  Our results suggest that inhibition of COMT via serotonin binding contributes to pain hypersensitivity, providing additional strategies for the treatment of clinical pain conditions.'),(763,'kgdgsd_b12','Kota, P., Garcia-Caballero, A., Dang, H., Gentzsch, M., Stutts, M. J. and Dokholyan, N. V.','Energetic and structural basis for activation of the epithelial sodium channel by matriptase','Biochemistry',51,'3460','3469',2012,'Proteins: Function','Dokholyan, N. V.','yes','no','no','article',0,'','','','','Limited proteolysis, accomplished by endopeptidases, is a ubiquitous phenomenon underlying the regulation and activation of many enzymes, receptors and other proteins synthesized as inactive precursors. Serine proteases are one of the largest and conserved families of endopeptidases involved in diverse cellular activities including wound healing, blood coagulation and immune responses. Heteromeric ,,-epithelial sodium channels (ENaC) associated with diseases like cystic fibrosis and Liddles syndrome, are irreversibly stimulated by membrane-anchored proteases (MAPs) and furin-like convertases. Matriptase/Channel activating protease-3 (CAP3) is one of the several MAPs that potently activate ENaC. Despite identification of protease cleavage sites, the basis for enhanced susceptibility of - and -ENaC to proteases remains elusive. Here, we elucidate the energetic and structural bases for activation of ENaC by CAP3. We find a region near the -ENaC furin site that is previously unidentified as a critical cleavage site for CAP3-mediated stimulation. We also report that CAP3 mediates cleavage of ENaC at basic residues downstream of the furin site. Our results indicate that surface proteases alone are sufficient to fully activate uncleaved ENaC, and explain how ENaC in epithelia expressing surface-active proteases can appear refractory to soluble proteases. Our results support a model in which proteases prime ENaC for activation by cleaving at the furin site, and cleavage at downstream sites is accomplished by membrane surface proteases or extracellular soluble proteases. Based on our results, we propose a dynamics-driven anglerfish mechanism that explains less stringent sequence requirements for substrate recognition and cleavage by matriptase compared to furin.'),(764,'kprd_fknbm12','Kotelnikova, E. A., Pyatnitskiy, M. A., Redler, R. L. and Dokholyan, N. V.','Role of Ca2+-mediated signaling in ALS pathology','From Knowledge Networks to Biological Models',49,'24','72',2012,'Proteins: Diseases','Dokholyan, N. V.','yes','no','no','book',0,'','','','','Familial amyotrophic lateral sclerosis (fALS) is a hereditary disorder of motor neurons that is caused by mutation in Cu, Zn superoxide dismutase (SOD1) in a subset of cases. The onset of the disease is relatively late, usually at age 50 or later, and is associated with interrelated molecular mechanisms of neurodegeneration. One of the mechanisms that can promote ALS progression is increased intracellular calcium concentration. The only market-available drug for ALS targets glutamate receptors and slows disease in part by mitigating excitotoxicity, a process in which persistent stimulation of glutamate receptors leads to pathologically high calcium concentration. To dissect the potential contributions of calcium mishandling to ALS, we have processed several publically available expression datasets related to fALS and analyzed the differential expression of genes related to calcium homeostasis. We find that SOD1-related fALS is associated with changes in expression of numerous genes related to calcium handling. Several genes which are down-regulated in fALS are targets of the repressor element-1 transcription factor/neuron restrictive silencer factor (REST/NRSF) transcription factor, which is normally inactivated in neuronal tissue. Our meta-analysis shows that changes in gene expression occurring in SOD1-related fALS promote calcium mishandling through dysregulation of multiple pathways, and that aberrant REST/NRSF activity may underlie some errors in calcium homeostasis.'),(765,'sdsda_bj12','Sparta, M., Ding, F., Shirvanyants, D., Dokholyan, N. V., and Alexandrova, A. N.','Hybrid dynamics simulation engine for metalloproteins','Biophysical Journal',103,'767','776',2012,'Proteins: Folding','Dokholyan, N. V., Alexandrova, A. N.','yes','no','no','article',0,'','','','','Quality computational description of metalloproteins is a great challenge, due to the vast span of time and length scales characteristic of their existence. We present an efficient new method that allows for robust characterization of metalloproteins. It combines quantum mechanical (QM) description of the metal containing active site, and extensive dynamics of the protein captured by discrete molecular dynamics (DMD), QM/DMD. DMD samples the entire protein, including the backbone, and most of the active site, except for the immediate coordination region of the metal. QM operates on the part of the protein of electronic and chemical significance, which may include tens to hundreds of atoms. The breathing quantum-classical boundary provides a continuous mutual feedback between the two machineries. We test QM/DMD using the Fe-containing electron transporter protein, rubredoxin, and its three mutants as a model. QM/DMD can provide a reliable balanced description of metalloproteins structure, dynamics, and electronic structure, in a reasonable amount of time. As an illustration of QM/DMD capabilities, we then predict the structure of the Ca2+ form of the enzyme catechol O-methyl transferase, which, unlike the native Mg2+ form, is catalytically inactive. The Mg2+ site is ochtahedral, but the Ca2+ is 7-coordinate and features the misalignment of the reacting parts of the system. The change is facilitated by the backbone adjustment. QM/DMD is ideal and fast for providing this level of structural insight.'),(766,'dd_rnanano12','Ding, F. and Dokholyan, N. V.','RNA three-dimensional structure determination using experimental constraints','RNA Nanotechnology and Therapeutics',0,'159','176',2013,'RNA','Dokholyan, N. V.','no','no','no','book',0,'','Guo, P.','','Taylor and Francis','RNA structure determination is one of the major challenges in structural biology. Many RNAs are not amenable to high-resolution structure characterization by either x-ray or NMR methods because of their conformational flexibility or large size. Recently, novel computational methods to determine RNA structures have begun to emerge, but have often been limited to small RNAs with simple topologies due to either sampling problems or inaccuracy in force field. We developed a discrete molecular dynamics (DMD)-based RNA modeling approach, which allowed robust recapitulation of 3D structure of small RNA structures (< 50 nts). To fold large RNAs with complex 3D structures, we proposed to incorporate experimentally-derived structural information into modeling. Using various types experimentally-derived structural information to bias DMD simulations, we were able to recapitulate 3D structure of RNAs with complex topologies and lengths up to 230 nts. We expect a broad application of our experimentally-driven RNA modeling approach for generating robust structural hypotheses that are useful for guiding explorations of structure-function relationships in RNA.'),(767,'rtacd_b12','Ramachandran, S., Temple, B., Alexandrova, A. N., Chaney, S. G., and Dokholyan, N. V.','Recognition of platinum-DNA adducts by HMGB1a','Biochemistry',51,'7608 ','7617',2012,'DNA','Dokholyan, N. V., Chaney, S. G.','yes','no','no','article',0,'','','','','Cisplatin (CP) and oxaliplatin (OX), platinum (Pt) based drugs used widely in chemotherapy form adducts on intrastrand guanines (5GG) in genomic DNA. DNA damage-recognition proteins, transcription factors, mismatch repair proteins and DNA polymerases discriminate between CP- and OX-GG DNA adducts, which could partly account for differences in efficacy, toxicity and mutagenicity of CP and OX. In addition, differential recognition of CP- and OX-GG adducts is highly dependent on the sequence context of the Pt-GG adduct. In particular, the DNA binding protein domain HMGB1a binds to cisplatin (CP)-GG DNA adducts with up to 53-fold greater affinity than to oxaliplatin (OX)-GG adducts in the TGGA sequence context, but shows much smaller differences in binding in the AGGC or TGGT sequence contexts. Here, simulations of HMGB1a-Pt-DNA complex in the three sequence contexts revealed higher number of interface contacts for CP-DNA in the TGGA sequence context compared to OX-DNA. However, the number of interface contacts was similar in the TGGT and AGGC sequence contexts. The higher number of interface contacts in CP-TGGA sequence context corresponded to higher roll of the Pt-GG base-pair step. Furthermore, geometric analysis of stacking of phenylalanine 37 in HMGB1a (Phe37) with the platinated-guanines revealed more favorable stacking modes correlated with higher roll of the Pt-GG base-pair step in the TGGA sequence context. These data are consistent with our previous MD simulations showing that the CP-TGGA complex was able to sample higher roll angles than the OX-TGGA complex or either CP- or OX-DNA complexes in the AGGC or TGGT sequences. We infer that the high binding affinity of HMGB1a for CP-TGGA is due to the greater flexibility of CP-TGGA compared to OX-TGGA and other Pt-DNA adducts. This increased flexibility is reflected in the ability of CP-TGGA to sample higher roll-angles, which allows for a higher number of interface contacts between the Pt-DNA adduct and HMGB1a.'),(768,'hpgdm_ploso12','Hadi-Alijanvand, H., Proctor, E. A., Goliaei, B., Dokholyan, N. V., and Moosavi-Movahedi, A. A.','Thermal unfolding pathway of PHD2 catalytic domain in three different PHD2 species: Computational approaches','Public Library of Science ONE',7,'e47061\0\0','',2012,'Proteins: Folding','Moosavi-Movahedi, A. A.','no','no','no','article',0,'','','','','Prolyl hydroxylase domain 2 containing protein (PHD2) is a key protein in regulation of angiogenesis and metastasis. In normoxic condition, PHD2 triggers the degradation of hypoxia-inducible factor 1 (HIF-1?) that induces the expression of hypoxia response genes. Therefore the correct function of PHD2 would inhibit angiogenesis and consequent metastasis of tumor cells in normoxic condition.     PHD2 mutations were reported in some common cancers. However, high levels of HIF-1? protein were observed even in normoxic metastatic tumors with normal expression of wild type PHD2. PHD2 malfunctions due to protein misfolding may be the underlying reason of metastasis and invasion in such cases.  In this study, we scrutinize the unfolding pathways of the PHD2 catalytic domains possible species and demonstrate the properties of their unfolding states by computational approaches. Our study introduces the possibility of aggregation disaster for the prominent species of PHD2 during its partial unfolding. This may justify PHD2 inability to regulate HIF-1? level in some normoxic tumor types.'),(769,'lhkmfdw_b13','Leonard, C. W., Hajdin, F. Karabiber, C. E., Mathews, D. H., Favorov, O., Dokholyan, N. V., and Weeks, K. M.','Principles for understanding the errors of SHAPE-directed RNA structure modeling','Biochemistry',52,'588','595',2013,'RNA','Weeks, K. M.','no','no','no','article',0,'','','','','Accurate RNA structure modeling is an important, incompletely solved, challenge.  Single-nucleotide resolution SHAPE (selective 2\'-hydroxyl acylation analyzed by primer extension) yields an experimental measurement of local nucleotide flexibility that can be incorporated as pseudo-free energy change constraints to direct secondary structure predictions. Prior work from our laboratory has emphasized both the overall accuracy of this approach and the need for nuanced interpretation of some apparent discrepancies between modeled and accepted structures. Recent studies [Kladwang et al., Biochemistry 50:8049 (2011) and Nat. Chem. 3:954 (2011)] yielded poorer RNA secondary structure predictions than expected based on prior benchmarking efforts. To understand the features that led to these divergent results, we re-examined four RNAs yielding the poorest results in this recent work  tRNAPhe, adenine and di-cyclic-GMP riboswitches, and 5S rRNA. For two RNAs, tRNAPhe and the adenine riboswitch, secondary structure predictions are nearly perfect if no experimental information is included but were rendered inaccurate by the Kladwang et al. SHAPE data. An additional focus of the recent work was to compare SHAPE-directed structure prediction with the mutate-and-map method, the latter providing an information-rich view of RNA interactions by obtaining SHAPE profiles for hundreds of RNA variants. When best practices and identical RNAs were used, single-sequence SHAPE prediction of canonical base pairs was nearly indistinguishable from the mutate-and-map approach with the exception of a single helix in the 5S rRNA. SHAPE-directed secondary structure modeling recovered ~93% of individual base pairs and greater than 90% of helices in the four RNAs. The field of experimentally-directed RNA secondary structure prediction is entering a phase focused on the most difficult prediction challenges: We outline five constructive principles for guiding this field forward.'),(770,'hkacjdr_fasebj13','He, L., Kota, P., Aleksandrov, A. A., Cui, L., Jensen, T., Dokholyan, N. V., and Riordan, J. R.','Correctors of DeltaF508 CFTR restore global conformational maturation without thermally stabilizing the mutant protein','FASEB Journal',27,'536','545',2013,'Proteins: Diseases','Riordan, J. R.','no','no','no','article',0,'','','','','Most cystic fibrosis is caused by the deletion of a single amino acid (F508) from CFTR and the resulting misfolding and destabilization of the protein. Compounds identified by high throughput screening to improve DeltaF508 CFTR maturation have already entered clinical trials and it is important to understand their mechanisms of action to further improve their efficacy. Here we showed that several of these compounds including the investigational drug VX-809 caused a much greater increase (5-10 fold) in maturation at 27oC than at 37oC (<2 fold) and the mature product remained short-lived (T1/2 ~4.5 h) and thermally unstable even though its overall conformational state was similar to wild-type as judged by resistance to proteolysis and inter-domain cross-linking. Consistent with its inability to restore thermodynamic stability, VX-809 stimulated maturation 2-5 fold beyond that caused by several different stabilizing modifications of NBD1 and the NBD1/CL4 interface. The compound also promoted maturation of several disease-associated processing mutants on the CL4 side of this interface. Although these effects may reflect an interaction of VX-809 with this interface, an interpretation supported by computational docking, it also rescued maturation of mutants in other cytoplasmic loops, either by allosteric effects or via additional sites of action. In addition to revealing the capabilities and some of the limitations of this important investigational drug, these findings clearly demonstrate that DeltaF508 CFTR can be completely assembled and evade cellular quality control systems while remaining thermodynamically unstable.'),(771,'clbsddj_jacs12','Cole, D. I., Legassie,  J. D., Bonifacio, L. N., Sekaran, V. G., Ding, F., Dokholyan, N. V. and Jarstfer, M. B.','New models of tetrahymena telomerase RNA  from experimentally derived constraints and modeling','Journal of the American Chemical Society',134,'20070','20080',2012,'RNA','Dokholyan, N. V., Jarstfer, M. B.','yes','no','no','article',0,'','','','','The telomerase ribonucleoprotein complex ensures complete replication of eukaryotic chromosomes. Telomerase RNA, TER, provides the template for replicating the G-rich strand of telomeric DNA, provides an anchor site for telomerase-associated proteins, and participates in catalysis through several incompletely characterized mechanisms. A major impediment towards understanding the non-templating roles is the absence of high content structural information for TER within the telomerase complex. Here, we used selective 2-hydroxyl acylation analyzed by primer extension (SHAPE) to examine the structure of Tetrahymena TER free in solution and bound to tTERT in the minimal telomerase RNP. We discovered a striking difference in the two conformations and established direct evidence for base pair triples in the tTER pseudoknot. We then used the SHAPE data and available FRET data to model the structure of tTER using discrete molecular dynamics simulations. The resulting tTER structure was docked with a homology model of tTERT to characterize the conformational changes of tTER that attend binding to tTERT. Free in solution, tTER contains four pairing regions: stems I, II, and IV, which are present in the commonly accepted structure, and stem III, a large paired region that encompasses the template and pseudoknot domains. Our interpretation of the data and subse- quent modeling affords a molecular model for telomerase assemblage in which a large stem III of tTER unwinds to allow proper association of the template with the tTERT active site and formation of the pseudoknot. Additionally, analysis of our SHAPE data and previous enzymatic footpinting allows us to propose a model for stem IV function in which tTERT is activated by binding stem IV in the major grove of the helix-capping loop.'),(772,'pkdcd_jmb13','Proctor, E. A., Kota, P., Demarest, S. J., Caravella, J. A., and Dokholyan, N. V.','Metric to distinguish closely related domain families using sequence information','Journal of Molecular Biology',425,'475','478',2013,'Proteins: Evolution','Dokholyan, N. V.','no','no','no','article',0,'','','','','Engineered antibodies are emerging as a promising class of therapeutic biomolecules, as well as having applications in medical research. Knowledge of conserved functional and structural regions within antibody domains is imperative in order to rationally design stable and specific antibodies. Of particular interest for the design of therapeutics are antibody variable and constant domains, which are responsible for antigen binding and immune response. These antibody domains are part of the larger immunoglobulin (Ig) V-class and C-class families, respectively. We find that, although both classes belong to the Ig-fold superfamily, the sets of conserved residue positions and identities differ between these classes. We exploit these evolutionary differences to derive a metric based on sequence positional entropy that distinguishes C-class from V-class sequences utilizing only sequence information. By distinguishing different domain families using sequence information alone, we enable the application of domain-specific design strategies without the need for secondary or tertiary structural information.'),(773,'dd_jcim13','Ding F. and Dokholyan N. V.','Incorporating backbone flexibility in MedusaDock improves ligand binding pose prediction in the CSAR2011 docking benchmark','Journal of Chemical Information and Modeling',53,'1871','1879',2012,'Methods','Dokholyan, N. V.','no','no','no','article',0,'','','','','Solution of the structures of ligand-receptor complexes via computational docking is an integral step in many structural modeling efforts as well as in rational drug discovery. A major challenge in ligand-receptor docking is the modeling of both receptor and ligand flexibilities in order to capture receptor conformational changes induced by ligand binding. In the molecular docking suite MedusaDock both ligand and receptor side chain flexibilities are modeled simultaneously with sets of discrete rotamers, where the ligand rotamer library is generated on the fly in a stochastic manner. Here, we introducebackbone flexibility into MedusaDock by implementing ensemble docking in a sequential manner for a set of distinct receptor backbone conformations. We generate corresponding backbone ensembles to capture backbone changes upon binding to different ligands, as observed experimentally. We develop a simple clustering and ranking approach to select the top poses as blind predictions. We applied our method in the CSAR2011 benchmark exercise. In 28 out of 35 cases (80%) where the ligand-receptor complex structures were released, we were able to predict near-native poses (< 2.5  RMSD), the highest success rate reported for CSAR2011. This result highlights the importance of modeling receptor backbone flexibility to the accurate docking of ligands to flexible targets. We expect a broach application of our fully flexible docking approach in biological studies as well as in rational drug design.'),(774,'pkdcd_psfb13','Proctor, E. A., Kota, P., Demarest, S. J., Caravella, J. A., and Dokholyan, N. V.','Highly covarying residues have a functional role in antibody constant domains','Proteins: Structure, Function, and Bioinformatics',81,'884','895',2013,'Proteins: Evolution','Dokholyan, N. V.','no','no','no','article',0,'','','','','The ability to generate and design antibodies recognizing specific targets has revolutionized the pharmaceutical industry and medical imaging. Engineering antibody therapeutics in some cases requires modifying their constant domains to enable new and altered interactions. Engineering novel specificities into antibody constant domains has proved challenging due to the complexity of inter-domain interactions. Covarying networks of residues that tend to cluster on the protein surface and near binding sites have been identified in some proteins. However, the underlying role these networks play in the protein resulting in their conservation remains unclear in most cases. Resolving their role is crucial, because residues in these networks are not viable design targets if their role is to maintain the fold of the protein. Conversely, these networks of residues are ideal candidates for manipulating specificity if they are primarily involved in binding, such as the myriad inter-domain interactions maintained within antibodies. Here, we identify networks of evolutionarily-related residues in C-class antibody domains by evaluating covariation, a measure of propensity with which residue pairs vary dependently during evolution. We computationally test whether mutation of residues in these networks affects stability of the folded antibody domain, determining their viability as design candidates. We find that members of covarying networks cluster at domain-domain interfaces, and that mutations to these residues are diverse and frequent during evolution, precluding their importance to domain stability. These results indicate that networks of covarying residues exist in antibody domains for functional reasons unrelated to thermodynamic stability, making them ideal targets for antibody design.'),(775,'rdwd_b13','Ramachandran, S., Ding, F., Weeks, K., and Dokholyan, N. V.','Statistical analysis of SHAPE-directed RNA secondary structure modeling','Biochemistry',52,'596','599',2013,'RNA','Dokholyan, N. V.','no','no','no','article',0,'','','','','The ability to predict RNA secondary structure is fundamental for understanding and manipulating RNA function. The structural information obtained from selective 2\'-hydroxyl acylation analyzed by primer extension (SHAPE) experiments greatly improves the accuracy of RNA secondary structure prediction. Recently, Das and colleagues [Kladwang  et al., Biochemistry 50:8049 (2011)] proposed a bootstrapping approach to estimate the variance and helix-by-helix confidence levels of predicted secondary structures based on resampling (randomizing and summing) the measured SHAPE data. We show that the specific resampling approach described by Kladwang  et al. introduces systematic errors and underestimates confidence in secondary structure prediction using  SHAPE data. Instead, a leave-data-out jackknife approach better estimates the influence of a given experimental dataset on SHAPE-directed secondary structure modeling. Even when 35\\% of the data were left out in the jackknife approach, the confidence levels of SHAPE-directed secondary structure prediction were significantly higher than those calculated by Das and colleagues using bootstrapping. Helix confidence levels were thus significantly underestimated in the recent study, and resampling approach implemented by Kladwang et al. is not an appropriate metric for assigning confidences in SHAPE-directed secondary structure modeling.'),(776,'rcxmsdm_jbc13','Ramachandran, S., Chakraborty, A., Xu, L., Mei, Y., Samso, M., Dokholyan, N. V., and Meissner, G.','Structural determinants of skeletal muscle ryanodine receptor gating','Journal of Biological Chemistry',288,'6154','6165',2013,'Proteins: Structure','Dokholyan, N. V.','no','no','no','article',23319589,'','','','','Ryanodine receptor type 1 (RyR1) releases Ca2+ from intracellular stores upon nerve impulse to trigger skeletal muscle contraction. Effector binding at the cytoplasmic domain tightly controls gating of the pore domain of RyR1 to release Ca2+. However, the molecular mechanism that links effector binding to channel gating is unknown due to lack of structural data. Here we used a combination of computational and electrophysiological methods and cryoEM densities to generate structural models of the open and closed states of RyR1. Using our structural models, we identified an interface between the pore-lining helix (Y4912-E4948) and a linker helix (V4830-V4841) that lies parallel to the cytoplasmic membrane leaflet. To test the hypothesis that this interface controls RyR1 gating, we designed mutations in the linker helix to stabilize either the open (V4830W, T4840W) or closed (H4832W, G4834W) states and validated them using single channel experiments. To further confirm this interface, we designed mutations in the pore-lining helix to stabilize the closed state (Q4947N, Q4947T, Q4947S), which we also validated using single channel experiments. Channel conductance and selectivity of the mutations that we designed in the linker and pore-lining helices were indistinguishable from RyR1-WT, demonstrating our ability to modulate RyR1 gating without affecting ion permeation. Our integrated computational and experimental approach significantly advances understanding the structure and function of an unusually large ion channel.'),(777,'dskdcfshhd_pnas13','Dagliyan, O., Shirvanyants, D., Karginov, A., Ding, F., Fee, L., Chandrasekaran, S. N., Freisinger, C. M., Smolen, G. A., Huttenlocher, A., Hahn, K. M., and Dokholyan, N. V.','Rational design of a ligand-controlled protein conformational switch','Proceedings of the National Academy of Sciences USA',110,'6800','6804',2013,'Proteins: Design, Biotechnology','Dokholyan, N. V.','no','no','no','article',0,'','','','','Design of a regulatable multi-state protein is a challenge for protein engineering. Here we design a protein with a novel topology, called uniRapR, whose conformation is controlled by the binding of a small molecule. We confirm switching and control ability of uniRapR in silico, in vitro and in vivo. As a proof of concept, uniRapR is used as an artificial regulatory domain to control activity of kinases. By activating Src kinase using uniRapR in single cells and whole organism, we observe two novel phenotypes consistent with its role in metastasis. Activation of Src kinase leads to rapid induction of protrusion with polarized spreading in HeLa cells, and morphological changes with loss of cell-cell contacts in the epidermal tissue of zebrafish. The rational creation of uniRapR exemplifies the strength of computational protein design, and offers a powerful means for targeted activation of many pathways to study signaling in living organisms.'),(778,'bclpdz_acd13','Barison, N., Cendron, L., Loconte, V., Proctor, E. A., Dokholyan, N. V. and Zanotti, G.','Protein HP1028 from the human pathogen Helicobacter pylori belongs to the lipocalin\'s family','Acta Crystallographica Section D',69,'1387','1394',2013,'','Dokholyan, N. V.','no','no','no','article',0,'','','','','Helicobacter pylori is a bacterial pathogen which causes severe diseases, including gastritis, ulcer and gastric cancer. Although this bacterium is extensively studied, the physiological function of a large part of the proteins coded in its genome is unknown. HP1028 is a protein relevant for colonization and for the survival of the bacterium in the stomach, but its function is not clearly understood. Bioinformatics studies and DLS analysis suggest that HP1028 is a monomeric protein of 35 kDa secreted in the H. pylori periplasm. We determined the crystal structure of HP1028 at 2.6 resolution using SAD method. The three-dimensional structure of the protein reveals that it belongs to the family of lipocalins, a group of proteins that bind and transport small molecules, often hydrophobic. The structure of the HP1028 together with the possible localization of the mature protein in the bacterial periplasm and the position of hp1028 gene in the bacterial genome point to a role in H. pylori chemotaxis.'),(780,'hdbogsldf_bpj13','Hudson, N. E., Ding, F., Bucay, I., O\'Brien III, E. T., Gorkun, O. V., Superfine, R., Lord, S. T., Dokholyan, N. V. and Falvo, M. R.','Sub-microsecond elastic recoil reveals molecular origins of fibrin fiber mechanics','Biophysical Journal',104,'2671','2680',2013,'','Dokholyan, N. V.','no','no','no','article',0,'','','','','Fibrin fibers form the structural scaffold of blood clots. Thus, their mechanical properties are of central importance to understanding hemostasis and thrombotic disease. Recent studies have revealed that fibrin fibers are elastomeric despite their high degree of molecular ordering. These results have inspired a variety of molecular models for fibrins elasticity, ranging from reversible protein unfolding to rubber-like elasticity. An important property that has not been explored is the timescale of elastic recoil, a parameter that is critical for fibrins mechanical function and places a temporal constraint on molecular models of fiber elasticity. Using high-frame-rate imaging and atomic force microscopy-based nanomanipulation, we measured the recoil dynamics of individual fibrin fibers and found that the recoil was orders of magnitude faster than anticipated from models involving protein refolding. We also performed steered discrete molecular-dynamics simulations to investigate the molecular origins of the observed recoil. Our results point to the unstructured C regions of the otherwise structured fibrin molecule as being responsible for the elastic recoil of the fibers.'),(785,'fmddt_jcim13','Fourches, D., Muratov, E., Ding, F., Dokholyan, N. V., Tropsha, A.','Predicting binding affinity of CSAR ligands using both structure-based and ligand-based approaches','Journal of Chemical Information and Modeling',53,'1915','1922',2013,'','Dokholyan, N. V.','no','no','no','article',0,'','','','','We report on the prediction accuracy of ligand-based (2D QSAR) and structure-based (MedusaDock) methods used both independently and in consensus for ranking the congeneric series of ligands binding to three protein targets (UK, ERK2, and CHK1) from the CSAR 2011 benchmark exercise. An ensemble of predictive QSAR models was developed using known binders of these three targets extracted from the publicly available ChEMBL database. Selected models were used to predict the binding anity of CSAR compounds toward the corresponding targets and rank them accordingly; the overall ranking accuracy evaluated by Spearman correlation was as high as 0.78 for UK, 0.60 for ERK2, and 0.56 for CHK1, placing our predictions in the top 10% among all the participants. In parallel, MedusaDock, designed to predict reliable docking poses, was also used for ranking the CSAR ligands according to their docking scores; the resulting accuracy (Spearman correlation) for UK, ERK2, and CHK1 were 0.76, 0.31, and 0.26, respectively. In addition, performance of several consensus approaches combining MedusaDock- and QSAR-predicted ranks altogether has been explored; the best approach yielded Spearman correlation coefficients for UK, ERK2, and CHK1 of 0.82, 0.50, and 0.45, respectively. This study shows that (i) externally validated 2D QSAR models were capable of ranking CSAR ligands at least as accurately as more computationally intensive structure-based approaches used both by us and by other groups and (ii) ligand-based QSAR models can complement structure-based approaches by boosting the prediction performances when used in consensus.'),(787,'fbbdwtdlp_bc13','Freeman, T. C. Jr., Black, J. L., Bray H. G., Dagliyan, O., Wu, Y. I., Tripathy, A., Dokholyan, N. V., Leisner, T. M., Parise, L. V.','Identification of novel integrin binding partners for CIB1: structural and thermodynamic basis of CIB1 promiscuity','Biochemistry',52,'7082','7090',2013,'','Dokholyan, N. V.','no','no','no','article',0,'','','','','The short cytoplasmic tails of the &#945;&#8722; and &#946;&#8722;chains of integrin adhesion receptors regulate integrin activation and cell signaling. Signicantly less is known about proteins that bind to &#945;&#8722;integrin cytoplasmic tails (CTs) as opposed to &#946;-CTs to regulate integrins. Calcium and integrin binding protein 1 (CIB1) was previously identied as an &#945;IIb binding partner that inhibits agonist-induced activation of the platelet-specic integrin, &#945;IIb&#946;3. A sequence alignment of all &#945;&#8722;integrin CTs revealed that key residues in the CIB1 binding site of &#945;IIb are well-conserved, and was used to delineate a consensus binding site (I/L-x-x-x-L/M-W/Y-K-x-G-F-F). Because the CIB1 binding site of &#945;IIb is conserved in all &#945;&#8722;integrins and CIB1 expression is ubiquitous, we asked if CIB1 could interact with other &#945;-integrin CTs. We predicted that multiple &#945;&#8722;integrin CTs were capable of binding to the same hydrophobic binding pocket on CIB1 with docking models generated by all-atom replica exchange discrete molecular dynamics. After demonstrating novel in vivo interactions between CIB1 and other whole integrin complexes with co&#8722;immunoprecipitations, we validated the modeled predictions with solid-phase competitive binding assays, which showed that other &#945;&#8722;integrin CTs compete with the &#945;IIb CT for binding to CIB1 in vitro. Isothermal titration calorimetry measurements indicated that this binding is driven by hydrophobic interactions and depends on residues in the CIB1 consensus binding site. These new mechanistic details of CIB1&#8722;integrin binding imply that CIB1 could bind to all integrin complexes and act as a broad regulator of integrin function.'),(784,'khdkzdhp_cnb13','Kummer, L., Hsu, C-W., Dagliyan, O., Kaufholz, M., Zimmermann, B., Dokholyan, N. V., Hahn, K. M., and Pl&#252ckthun, A.','Knowledge-based design of a biosensor to quantify localized ERK activation in living cells','Chemistry & Biology',20,'847','856',2013,'','Dokholyan, N. V.','no','no','no','article',0,'','','','','Investigation of protein activation in living cells is fundamental to understanding how proteins are infuenced by the full complement of upstream regulatorsthey experience. Here, we describe the generation of a biosensor based on the DARPin binding scaffold suited for intracellular applications. Combining library selection and knowledge-based design, we created an ERK activity biosensor by derivatizing a DARPin specific for phosphorylated ERK with a solvatochromatic merocyanine dye, whosefluorescence increases upon pERK binding. The biosensor specifically responded to pERK2, recognized by itsconformation, but not to ERK2 or other closely related mitogen-activated kinases tested. Activated endogenous ERK was visualized in mouse embryo fibroblasts, revealing greater activation in thenucleus, perinuclear regions, and especially the nucleoli. The DARPin-based biosensor will serve as a useful tool for studying biological functions of ERK in vitro and in vivo.'),(788,'kbdt_psfb13','Kimura, S., Broglia, R. A., Dokholyan, N. V., Tiana, G.','The maturation of HIV-1 protease precursor studied by discrete molecular dynamics','Proteins: Structure, Function, and Bioinformatics',82,'633','639',2014,'Proteins: Diseases','Dokholyan, N. V. and Tiana G.','yes','no','no','article',24123234,'','','','','The equilibrium properties of a HIV-1-protease precursor are studied by means of an efficient molecular dynamics scheme, which allows for the simulation of the folding of the protein monomers and their dimerization into an active form and compare them with those of the mature protein. The results of the model provide, with atomic detail, an overall account of several experimental findings, including the NMR conformation of the mature dimer, the calorimetric properties of the sys- tem, the effects of the precursor tail on the dimerization constant, the secondary chemical shifts of the monomer, and the paramagnetic relaxation enhancement data associated with the conformations of the precursor. It is found that although the mature protein can dimerize in a unique, single way, the precursor populates several dimeric conformations in which mono- mers are always native-like, but their binding can be non-native.'),(487,'hsntwsdb_plscb08','Hao, J., Serohijos, A. W. R., Newton, G., Tassone, G., Wang,. Z., Sgroi, D. C., Dokholyan, N. V., Basilion, J. P.','Identification and Rational Redesign of Peptide Ligands to CRIP1, A Novel Biomarker for Cancers','Plos Comp. Biol.',4,'e1000138-1','e1000138-1',2008,'','Dokholyan, N. V.','no','no','no','article',0,'','','','','Cysteine-rich intestinal protein 1 (CRIP1) has been identified as a novel marker for early detection of cancers. Here we report on the use of phage display in combination with molecular modeling to identify a high-affinity ligand for CRIP1. Panning experiments using a circularized C7C phage library yielded several consensus sequences with modest binding affinities to purified CRIP1. Two sequence motifs, A1 and B5, having the highest affinities for CRIP1, were chosen for further study. With peptide structure information and the NMR structure of CRIP1, the higher-affinity A1 peptide was computationally redesigned, yielding a novel peptide, A1M, whose affinity was predicted to be much improved. Synthesis of the peptide and saturation and competitive binding studies demonstrated approximately a 1028-fold improvement in the affinity of A1M compared to that of either A1 or B5 peptide. These techniques have broad application to the design of novel ligand peptides.'),(789,'rsddkkprtd_jmcb14','Redler, R.L., Shirvanyants, D., Dagliyan, O., Ding, F., Kim, D.N., Kota, P., Proctor, E.A., Ramachandran, S., Tandon, A., and Dokholyan, N.V.','Computational approaches to understanding protein aggregation in neurodegeneration.','Journal of Molecular Cell Biology',6,'104','115',2014,'Proteins: Aggregation','Dokholyan N. V.','no','no','no','article',24620031,'','','','','The generation of toxic non-native protein conformers has emerged as a unifying thread among disorders such as Alzheimer\'s disease, Parkinson\'s disease, and amyotrophic lateral sclerosis. Atomic-level detail regarding dynamical changes that facilitate protein aggregation, as well as the structural features of large-scale ordered aggregates and soluble non-native oligomers, would contribute significantly to current understanding of these complex phenomena and offer potential strategies for inhibiting formation of cytotoxic species. However, experimental limitations often preclude the acquisition of high-resolution structural and mechanistic information for aggregating systems. Computational methods, particularly those combine both all-atom and coarse-grained simulations to cover a wide range of time and length scales, have thus emerged as crucial tools for investigating protein aggregation. Here we review the current state of computational methodology for the study of protein self-assembly, with a focus on the application of these methods toward understanding of protein aggregates in human neurodegenerative disorders.'),(791,'ftisdjhdnakbs_jmb14','Furukawa, Y., Teraguchi, S., Ikegami, T. Dagliyan, O., Jin, L., Hall, D., Dokholyan, N. V., Namba, K., Akira, S., Kurosaki, T. Baba, Y., Standley, D. M.','Intrinsic disorder mediates cooperative signal transduction in STIM1','Journal of Molecular Biology',426,'2082','2097',2014,'Proteins: Structure','Dokholyan, N. V.','no','no','no','article',24650897,'','','','','Intrinsically disordered domains have been reported to play important roles in signal transduction networks by introducing cooperativity into protein-protein interactions. Unlike intrinsically disordered domains that become ordered upon binding, the EF-SAM domain in the stromal interaction molecule (STIM) 1 is distinct in that it is ordered in the monomeric state and partially unfolded in its oligomeric state, with the population of the two states depending on the local Ca(2+) concentration. The oligomerization of STIM1, which triggers extracellular Ca(2+) influx, exhibits cooperativity with respect to the local endoplasmic reticulum Ca(2+) concentration. Although the physiological importance of the oligomerization reaction is well established, the mechanism of the observed cooperativity is not known. Here, we examine the response of the STIM1 EF-SAM domain to changes in Ca(2+) concentration using mathematical modeling based on in vitro experiments. We find that the EF-SAM domain partially unfolds and dimerizes cooperatively with respect to Ca(2+) concentration, with Hill coefficients and half-maximal activation concentrations very close to the values observed in vivo for STIM1 redistribution and extracellular Ca(2+) influx. Our mathematical model of the dimerization reaction agrees quantitatively with our analytical ultracentrifugation-based measurements and previously published free energies of unfolding. A simple interpretation of these results is that Ca(2+) loss effectively acts as a denaturant, enabling cooperative dimerization and robust signal transduction. We present a structural model of the Ca(2+)-unbound EF-SAM domain that is consistent with a wide range of evidence, including resistance to proteolytic cleavage of the putative dimerization portion.'),(792,'rffcd_bc14','Redler, R. L., T. Fee, l., Fay, J., Caplow, M., Dokholyan, N. V.','Non-native soluble oligomers of Cu/Zn superoxide dismutase (SOD1) contain a conformational epitope linked to cytotoxicity in amyotrophic lateral sclerosis (ALS)','Biochemistry',53,'2423','2432',2014,'Proteins: Aggregation','Dokholyan, N. V.','no','no','no','article',24660965,'','','','','Soluble misfolded Cu/Zn superoxide dismutase (SOD1) is implicated in motor neuron death in amyotrophic lateral sclerosis (ALS); however, the relative toxicities of the various non-native species formed by SOD1 as it misfolds and aggregates are unknown. Here, we demonstrate that early stages of SOD1 aggregation involve the formation of soluble oligomers that contain an epitope specific to disease- relevant misfolded SOD1; this epitope, recognized by the C4F6 antibody, has been proposed as a marker of toxic species. Formation of potentially toxic oligomers is likely to be exacerbated by an oxidizing cellular environment, as evidenced by increased oligomerization propensity and C4F6 reactivity when oxidative modification by glutathione is present at Cys-111. These findings suggest that soluble non-native SOD1 oligomers, rather than native-like dimers or monomers, share structural similarity to pathogenic misfolded species found in ALS patients and therefore represent potential cytotoxic agents and therapeutic targets in ALS.'),(793,'shhdtkh_plosone14','Szollosi, D., Horvath, T., Han, K. -H., Dokholyan, N. V., Tompa, P., Kalmar L., Hegedus, T. Z.','Discrete molecular dynamics can predict helical prestructured motifs in disordered proteins','Public Library of Science ONE',9,'e95995','',2014,'','Dokholyan, N. V.','no','no','no','article',24763499,'','','','','Intrinsically disordered proteins (IDPs) lack a stable tertiary structure, but their short binding regions termed Prestructured Motifs (PreSMo) can form transient secondary structure elements in solution. Although disordered proteins are crucial in many biological processes and designing strategies to modulate their function is highly important, both experimental and computational tools to describe their conformational ensembles and the initial steps of folding are sparse. Here we report that discrete molecular dynamics (DMD) simulations combined with replica exchange (RX) method efficiently samples the conformational space and detects regions populating a-helical conformational states in disordered protein regions. While the available computational methods predict secondary structural propensities in IDPs based on the observation of protein-protein interactions, our ab initio method rests on physical principles of protein folding and dynamics. We show that RX-DMD predicts a-PreSMos with high confidence confirmed by comparison to experimental NMR data. Moreover, the method also can dissect a-PreSMos in close vicinity to each other and indicate helix stability. Importantly, simulations with disordered regions forming helices in X-ray structures of complexes indicate that a preformed helix is frequently the binding element itself, while in other cases it may have a role in initiating the binding process. Our results indicate that RX-DMD provides a breakthrough in the structural and dynamical characterization of disordered proteins by generating the structural ensembles of IDPs even when experimental data are not available.'),(794,'srmxmd_bj14','Shirvanyants, D., Ramachandran, S., Mei, Y., Xu, L., Meissner, G., Dokholyan, N. V.','Pore dynamics and conductance of RyR1 transmembrane domain','Biophysical Journal',106,'2375','2384',2014,'','Dokholyan, N. V.','no','no','no','article',24896116,'','','','','Ryanodine receptors (RyR) are calcium release channels, playing a major role in the regulation of muscular contraction. Mutations in skeletal muscle RyR (RyR1) are associated with congenital diseases such as malignant hyperthermia and central core disease (CCD). The absence of high-resolution structures of RyR1 has limited our understanding of channel function and disease mechanisms at the molecular level. Previously, we have reported a hypothetical structure of the RyR1 pore-forming region, obtained by homology modeling and supported by mutational scans, electrophysiological measurements, and cryo-electron microscopy. Here, we utilize the expanded model encompassing six transmembrane helices to calculate the RyR1 pore region conductance, to analyze its structural stability, and to hypothesize the mechanism of the Ile4897 CCD-asso- ciated mutation. The calculated conductance of the wild-type RyR1 suggests that the proposed pore structure can sustain ion currents measured in single-channel experiments. We observe a stable pore structure on timescales of 0.2 ms, with multiple cations occupying the selectivity filter and cytosolic vestibule, but not the inner chamber. We further suggest that stability of the selectivity filter critically depends on the interactions between the I4897 residue and several hydrophobic residues of the neighboring subunit. Loss of these interactions in the case of polar substitution I4897T results in destabilization of the selectivity filter, a possible cause of the CCD-specific reduced Ca2 conductance.'),(795,'ttskppogbedsc_structure14','Thompson, P. M., Tolbert, C. E., Shen, K., Kota, P., Palmer, S. M., Plevock, K. M., Orlova, A., Glakin, V. E., Burridge, K., Egelman, E. H., Dokholyan, N. V., Superfine, R., Campbell, S. L.','Identification of a new actin binding surface on vinculin that mediates mechanical cell and focal adhesion properties','Structure',22,'697','706',2014,'Proteins: Structure','Dokholyan, N. V.','no','no','no','article',24685146,'','','','','Vinculin, a cytoskeletal scaffold protein essential for embryogenesis and cardiovascular function, localizes to focal adhesions and adherens junctions, connecting cell surface receptors to the actin cytoskeleton. While vinculin interacts with many adhesion proteins, its interaction with filamentous actin regulates cell morphology, motility, and mechanotransduction. Disruption of this interaction lowers cell traction forces and enhances actin flow rates. Although a model for the vinculin:actin complex exists, we recently identified actin-binding deficient mutants of vinculin outside sites predicted to bind actin and developed an alternative model to better define this actin-binding surface, using negative-stain electron microscopy (EM), discrete molecular dynamics, and mutagenesis. Actin-binding deficient vinculin variants expressed in vinculin knockout fibroblasts fail to rescue cell-spreading defects and reduce cellular response to external force. These findings highlight the importance of this actin-binding surface and provide the molecular basis for elucidating additional roles of this interaction, including actin-induced conformational changes that promote actin bundling.'),(797,'cqfedrdbg_sciencetm14','Cholon, D. M., Quinney, N. L., Fulcher, M. L., Esther, C. R., Das, J., Randell, S. H., Dokholyan, N. V., Boucher, R. C., and Gentzsch M.','Potentiator ivacaftor abrogates pharmacological correction of &#916F508 CFTR in cystic fibrosis','Science Translational Medicine',6,'246ra96','',2014,'','Dokholyan, N. V.','yes','no','no','article',25101886,'','','','','Cystic Fibrosis (CF) is caused by mutations in the CF transmembrane conductance regulator (CFTR). Newly developed correctors such as lumacaftor (VX-809) that improve CFTR maturation and trafficking and potentiators such as ivacaftor (VX-770) that enhance channel activity may provide important advances in CF therapy. Although VX-770 has demonstrated substantial clinical efficacy in the small subset of patients with a mutation (G551D) that affects only channel activity, a single compound is not sufficient to treat patients with the more common CFTR mutation, F508. Thus, patients with F508 will likely require treatment with both correctors and potentiators to achieve clinical benefit. However, whereas the effectiveness of acute treatment with this drug combination has been demonstrated in vitro, the impact of chronic therapy has not been established. In studies of human primary airway epithelial cells, we found that both acute and chronic treatment with VX-770 improved CFTR function in cells with the G551D mutation, consistent with clinical studies. In contrast, chronic VX-770 administration caused a dose-dependent reversal of VX-809-mediated CFTR correction in F508 homozygous cultures. This result reflected the destabilization of corrected F508 CFTR by VX-770, dramatically increasing its turnover rate. Chronic VX-770 treatment also reduced mature wild- type CFTR levels and function. These findings demonstrate that chronic treatment with CFTR potentiators and correctors may have unexpected effects that cannot be predicted from short-term studies. Combining of these drugs to successfully rescue F508 CFTR may require changes in dosing and/or development of new potentiator compounds that do not interfere with CFTR stability.'),(799,'kbcdhgkgsd_jbc14','Kota, P., Buchner, G., Chakrabarty, H., Dang, Y. L., Hong, H., Garcia, G. J. M., Kubelka, J., Gentzsch, M., Stutts, M. J., and Dokholyan, N. V.','The N-terminal domain allosterically regulates cleavage and activation of the epithelial sodium channel','Journal of Biological Chemistry',289,'23029','23042',2014,'','Dokholyan, N. V.','no','no','no','article',24973914,'','','','','The epithelial sodium channel (ENaC) is activated upon endoproteolytic cleavage of specific segments in the extracellular domains of the - and - subunits. Cleavage is accomplished by intracellular proteases prior to membrane insertion and by surface-expressed or extracellular soluble proteases once ENaC resides at the cell surface. These cleavage events are partially regulated by intracellular signaling through an unknown allosteric mechanism. Here we show that the intracellular N-terminus of -ENaC undergoes secondary structural transitions upon interaction with phosphoinositides. From ab initio folding simulations of the N-termini in the presence and absence of phosphatidylinositol 4,5 bisphosphate (PI(4,5)P2 or PIP2) we found that PIP2 increases alpha-helical propensity in the N-terminus of -ENaC. Electrophysiology and mutation experiments revealed that a highly conserved cluster of lysines in the -ENaC N-terminus regulates accessibility of extracellular cleavage sites in -ENaC. We also show that conditions that decrease PIP2 or enhance ubiquitination sharply limit access of the -ENaC extracellular domain to proteases. Further, the efficiency of allosteric control of ENaC proteolysis is dependent on Tyr370 in -ENaC. Our findings provide an allosteric mechanism for ENaC activation regulated by the N-termini and sheds light on potentially general mechanism of channel and receptor activation.'),(800,'nrpda_jmb14','Nedd, S., Redler, R. L., Proctor, E. A., Dokholyan, N. V., and Alexandrova, A. N.','Cu,Zn-Superoxide dismutase without Zn is folded but catalytically inactive','Journal of Molecular Biology',426,'4112','4124',2014,'Proteins: Aggregation','Dokholyan, N. V. and Alexandrova, A. N.','yes','no','no','article',25083917,'','','','','Amyotrophic lateral sclerosis has been linked to the gain of aberrant function of superoxide dismutase, Cu,Zn-SOD1 upon protein misfolding. The mechanism of SOD1 misfolding is thought to involve mutations leading to the loss of Zn, followed by protein unfolding and aggregation. We show that the removal of Zn from SOD1 may not lead to an immediate unfolding but immediately deactivates the enzyme through a combination of subtle structural and electronic effects. Using quantum mechanics/discrete molecular dynamics, we showed that both Zn-less wild-type (WT)-SOD1 and its D124N mutant that does not bind Zn have at least metastable folded states. In those states, the reduction potential of Cu increases, leading to the presence of detectable amounts of Cu(I) instead of Cu(II) in the active site, as confirmed experimentally. The Cu(I) protein cannot participate in the catalytic Cu(I)Cu(II) cycle. However, even without the full reduction to Cu(I), the Cu site in the Zn-less variants of SOD1 is shown to be catalytically incompetent: unable to bind superoxide in a way comparable to the WT-SOD1. The changes are more radical and different in the D124N Zn-less mutant than in the Zn-less WT-SOD1, suggesting D124N being perhaps not the most adequate model for Zn-less SOD1. Overall, Zn in SOD1 appears to be influencing the Cu site directly by adjusting its reduction potential and geometry. Thus, the role of Zn in SOD1 is not just structural, as was previously thought; it is a vital part of the catalytic machinery.'),(801,'hflkgbdw_pnas14','Homan, P., Favorov, O. V., Lavender, C. A., Kursun, O., Ge, X., Busan, S., Dokholyan, N. V., and Weeks, K. M.','Single-molecule correlated chemical probing of RNA','Proceedings of the National Academy of Sciences USA',111,'13858','13863',2014,'RNA','Dokholyan, N. V.','yes','no','yes','article',25205807,'','','','','Complex higher-order RNA structures play critical roles in all facets of gene expression; however, the through-space interac- tion networks that define tertiary structures and govern sampling of multiple conformations are poorly understood. Here we de- scribe single-molecule RNA structure analysis in which multiple sites of chemical modification are identified in single RNA strands by massively parallel sequencing and then analyzed for correlated and clustered interactions. The strategy thus identifies RNA inter- action groups by mutational profiling (RING-MaP) and makes possible two expansive applications. First, we identify through- space interactions, create 3D models for RNAs spanning 80265 nucleotides, and characterize broad classes of intramolecular inter- actions that stabilize RNA. Second, we distinguish distinct confor- mations in solution ensembles and reveal previously undetected hidden states and large-scale structural reconfigurations that occur in unfolded RNAs relative to native states. RING-MaP single-mole- cule nucleic acid structure interrogation enables concise and facile analysis of the global architectures and multiple conformations that govern function in RNA.'),(802,'hpydc_bpj14','Hasan, S. S., Proctor, E. A., Yamashita, E., Dokholyan, N. V., and Cramer, W. A.','Traffic within the cytochrome b6f lipoprotein complex: Gating of the quinone portal','Biophysical Journal',107,'1620','1628',2014,'Proteins: Mechanism','Dokholyan, N. V.','yes','no','no','article',25296314,'','','','','The cytochrome bc complexes b6f and bc1 catalyze proton-coupled quinol/quinone redox reactions to generate a transmembrane proton electrochemical gradient. Quinol oxidation on the electrochemically positive (p) interface of the complex occurs at the end of a narrow quinol/quinone entry/exit Qp portal, 11  long in bc complexes. Superoxide, which has multiple signaling functions, is a by-product of the p-side quinol oxidation. Although the transmembrane core and the chemistry of quinone redox reactions are conserved in bc complexes, the rate of superoxide generation is an order of magnitude greater in the b6f complex, implying that functionally significant differences in structure exist between the b6f and bc1 complexes on the p-side. A unique structure feature of the b6f p-side quinol oxidation site is the presence of a single chlorophyll-a molecule whose function is unrelated to light harvesting. This study describes a cocrystal structure of the cytochrome b6f complex with the quinol analog stigmatellin, which partitions in the Qp portal of the bc1 complex, but not effectively in b6f. It is inferred that the Qp portal is partially occluded in the b6f complex relative to bc1. Based on a discrete molecular-dynamics analysis, occlusion of the Qp portal is attributed to the presence of the chlorophyll phytyl tail, which increases the quinone residence time within the Qp portal and is inferred to be a cause of enhanced superoxide production. This study attributes a novel (to our knowledge), structure-linked function to the otherwise enigmatic chlorophyll-a in the b6f complex, which may also be relevant to intracellular redox signaling.'),(803,'htrddw_bc14','Homan, P., Tandon, A., Rice, G. M., Ding, F., Dokholyan, N. V., and Weeks, K. M.','RNA tertiary structure analysis and refinement by 2\'-hydroxyl molecular interference','Biochemistry',53,'6825','6833',2014,'RNA','Dokholyan, N. V.','yes','no','no','article',25341083,'','','','','We introduce a melded chemical and computational approach for probing and modeling higher-order intramolecular tertiary interactions in RNA. 2\'-Hydroxyl molecular interference (HMX) identifies nucleotides in highly packed regions of an RNA by exploiting the ability of bulky adducts at the 2\'-hydroxyl position to disrupt overall RNA structure. HMX was found to be exceptionally selective for quantitative detection of higher-order and tertiary interactions. When incorporated as experimental constraints in discrete molecular dynamics simulations, HMX information yielded accurate three-dimensional models, emphasizing the power of molecular interference to guide RNA tertiary structure analysis and fold refinement. In the case of a large, multidomain RNA, the Tetrahymena group I intron, HMX identified multiple distinct sets of tertiary structure interaction groups in a single, concise experiment'),(804,'pkahrd_cs15','Proctor, E. A., Kota, P., Aleksandrov, A. A., He, L., Riordan, J. R., Dokholyan, N. V.','Rational coupled dynamics network manipulation rescues disease-relevant mutant cystic fibrosis transmembrane regulator','Chemical Science',6,'1237','1246',2015,'Proteins: Allostery','Dokholyan, N. V.','yes','no','no','article',25685315,'','','','','Many cellular functions necessary for life are tightly regulated by protein allosteric conformational change, and correlated dynamics between protein regions has been found to contribute to the function of proteins not previously considered allosteric. The ability to map and control such dynamic coupling would thus create opportunities for the extension of current therapeutic design strategy. Here, we present an approach to determine the networks of residues involved in the transfer of correlated motion across a protein, and apply our approach to rescue disease-causative mutant cystic fibrosis transmembrane regulator (CFTR) ion channels, F508 and I507, which together constitute over 90% of cystic fibrosis cases. We show that these mutations perturb dynamic coupling within the first nucleotide-binding domain (NBD1), and uncover a critical residue that mediates trans-domain coupled dynamics. By rationally designing a mutation to this residue, we improve aberrant dynamics of mutant CFTR as well as enhance surface expression and function of both mutants, demonstrating the rescue of a disease mutation by rational correction of aberrant protein dynamics.'),(805,'csggmdd_pnbp15','Convertino, M., Samoshkin, A., Gauthier, J., Gold, M. S., Maixner, W., Dokholyan, N. V., and Diatchenko, L.','&#956-Opioid receptor 6-transmembrane isoform: a potential therapeutic target for new effective opioids','Progress in Neuro-Psychopharmacology & Biological Psychiatry',62,'61','67',2015,'Proteins: Diseases','Dokholyan, N. V.','no','no','no','article',25485963,'','','','','The &#956-opioid receptor (MOR) is the primary target for opioid analgesics. MOR induces analgesia through the inhibition of second messenger pathways and the modulation of ion channels activity. Nevertheless, cellular excitation has also been demonstrated, and proposed to mediate reduction of therapeutic efficacy and opioid-induced hyperalgesia upon prolonged exposure to opioids. In this mini-perspective, we review the recently identified, functional MOR isoform subclass, which consists of six transmembrane helices (6TM) and may play an important role in MOR signaling. There is evidence that 6TM MOR signals through very different cellular pathways and may mediate excitatory cellular effects rather than the classic inhibitory effects produced by the stimulation of the major (7TM) isoform. Therefore, the development of 6TM and 7TM MOR selective compounds represents a new and exciting opportunity to better understand the mechanisms of action and the pharmacodynamic properties of a new class of opioids.'),(806,'kd_me15','Krokhotin, A. and Dokholyan, N. V.','Computational methods toward accurate RNA structure prediction using coarse-grained and all-atom models','Methods in Enzymology',553,'65','89',2015,'RNA','Dokholyan, N. V.','no','no','no','book',0,'','D. H. Burke and S.-J. Chen','Macromolecular crystallography D.','Elsevier Academic Press','Computational methods can provide significant insights into RNA structure and dynam- ics, bridging the gap in our understanding of the relationship between structure and biological function. Simulations enrich and enhance our understanding of data derived on the bench, as well as provide feasible alternatives to costly or technically challenging experiments. Coarse-grained computational models of RNA are especially important in this regard, as they allow analysis of events occurring in timescales relevant to RNA bio- logical function, which are inaccessible through experimental methods alone. We have developed a three-bead coarse-grained model of RNA for discrete molecular dynamics simulations. This model is efficient in de novo prediction of short RNA tertiary structure, starting from RNA primary sequences of less than 50 nucleotides. To complement this model, we have incorporated additional base-pairing constraints and have developed a bias potential reliant on data obtained from hydroxyl probing experiments that guide RNA folding to its correct state. By introducing experimentally derived constraints to our computer simulations, we are able to make reliable predictions of RNA tertiary structures up to a few hundred nucleotides. Our refined model exemplifies a valuable benefit achieved through integration of computation and experimental methods.'),(807,'bzszsmdd_cc15','Brown, K. A., Zou, D., Shirvanyants, D., Zhang, J., Samanta, P. K., Dokholyan, N. V., and Deiters, A.','Light-cleavable rapamycin dimer as an optical trigger for protein dimerization','Chemical Communications',51,'5702','5705',2015,'Proteins: Design','Dokholyan, N. V.','yes','no','no','article',25716548,'','','','','Rapamycin-induced protein heterodimerization of FKBP12 and FRB is one of the most commonly employed switches to conditionally control biological processes. We developed an optically activated rapamycin dimer that does not induce FKBP12-FRB dimerization until exposed to light, and applied it to control kinase, protease, and recombinase function.'),(808,'mabbbccccccfddddkklmmmmmmpppsssttwxxzzzw_rna15','Miao, Z., Adamiak, R. W., Blanchet, M. F., Boniecki, M., Bujnicki, J. M., Chen, S. J., Cheng, C., Chojnowski, G., Chou, F. C., Cordero, P., Cruz, J. A., Ferr&eacute -D\'Amar&eacute , A. R., Das, R., Ding, F., Dokholyan, N. V., Dunin-Horkawicz, S., Kladwang, W., Krokhotin, A., Lach, G., Magnus, M., Major, F., Mann, T. H., Masquida, B., Matelska, D., Meyer, M., Peselis, A., Popenda, M., Purzycka, K. J., Serganov, A., Stasiewicz, J., Szachniuk, M., Tandon, A., Tian, S., Wang, J., Xiao, Y., Xu, X., Z','RNA-Puzzles Round II: assessment of RNA structure prediction programs applied to three large RNA structures','RNA',21,'1066','1084',2015,'RNA','Dokholyan, N. V.','no','no','no','article',25883046,'','','','','This paper is a report of a second round of RNA-Puzzles, a collective and blind experiment in three-dimensional (3D) RNA structure prediction. Three puzzles, Puzzles 5, 6, and 10, represented sequences of three large RNA structures with limited or no homology with previously solved RNA molecules. A lariat-capping ribozyme, as well as riboswitches complexed to adenosylcobalamin and tRNA, were predicted by seven groups using RNAComposer, ModeRNA/SimRNA, Vfold, Rosetta, DMD, MC-Fold, 3dRNA, and AMBER refinement. Some groups derived models using data from state-of-the-art chemical-mapping methods (SHAPE, DMS, CMCT, and mutate-and-map). The comparisons between the predictions and the three subsequently released crystallographic structures, solved at diffraction resolutions of 2.5-3.2 , were carried out automatically using various sets of quality indicators. The comparisons clearly demonstrate the state of present-day de novo prediction abilities as well as the limitations of these state-of-the-art methods. All of the best prediction models have similar topologies to the native structures, which suggests that computational methods for RNA structure prediction can already provide useful structural information for biological problems. However, the prediction accuracy for non-Watson-Crick interactions, key to proper folding of RNAs, is low and some predicted models had high Clash Scores. These two difficulties point to some of the continuing bottlenecks in RNA structure prediction. All submitted models are available for download at http://ahsoka.u-strasbg.fr/rnapuzzles/.'),(809,'khd_bi15','Krokhotin, A., Houlihan, K., and Dokholyan, N. V.','iFoldRNA v2: Folding RNA with Constraints','Bioinformatics',31,'2891','2893',2015,'RNA','Dokholyan, N. V.','no','no','no','article',25910700,'','','','','A key to understanding RNA function is to uncover its complex  three-dimensional  structure. Experimental methods used for determining RNA three-dimensional structures are technologically challenging and laborious, which makes the development of computational prediction methods of substantial interest. Previously we developed the iFoldRNA server that allows accurate prediction of short (<50 nucleotides) tertiary RNA structures starting from primary sequences. Here we present a new version of the iFoldRNA server that permits the prediction of tertiary structure of RNAs as long as a few  hundred  nucleotides. This substantial increase in the server capacity is achieved by utilization of experimental information such as base-pairing and hydroxyl-radical  probing. We demonstrate a significant benefit provided by integration of experimental data and computational methods.'),(810,'mxmgwpdm_jbc15','Mei, Y., Xu, L., Mowrey, D. D., Giraldez, R. M., Wang, Y., Pasek, D. A., Dokholyan, N. V., and Meissner G.','Channel gating dependence on pore lining helix glycine residues in skeletal muscle ryanodine receptor','Journal of Biological Chemistry',290,'17535','17545',2015,'Proteins: Function','Dokholyan, N. V.','no','no','no','article',25998124,'','','','','Type 1 ryanodine receptors (RyR1s) release Ca2+ from the sarcoplasmic reticulum to initiate skeletal muscle contraction. The role of RyR1-G4934 and -G4941 in the pore-lining helix in channel gating and ion permeation was probed by replacing them with amino acid residues of increasing side chain volume. RyR1-G4934A, -G4941A and -G4941V mutant channels exhibited a caffeine-induced Ca2+ release response in HEK293 cells and bound the RyR-specific ligand [3H]ryanodine. In single channel recordings, significant differences in the number of channel events and mean open and close times were observed between wild type (WT) and RyR1-G4934A and -G4941A. RyR1-G4934A had reduced K+ conductance and ion selectivity compared to WT. Mutations further increasing the side chain volume at these positions (G4934V and G4941I) resulted in reduced caffeine-induced Ca2+ release in HEK293 cells, low [3H]ryanodine binding levels, and channels that were not regulated by Ca2+ and did not conduct Ca2+ in single channel measurements.  Computational predictions of the thermodynamic impact of mutations on protein stability indicated that while the G4934A mutation was tolerated, the G4934V mutation decreased protein stability by introducing clashes with neighboring amino acid residues. In similar fashion the G4941A mutation did not introduce clashes, while the G4941I mutation resulted in intersubunit clashes among the mutated isoleucines. Co-expression of RyR1-WT with RyR1-G4934V or -G4941I partially restored the WT phenotype, which suggested lessening of amino acid clashes in heterotetrameric channel complexes. The results indicate that both glycines are important for RyR1 channel function by providing flexibility and minimizing amino acid clashes.'),(811,'wcdd_pcb15','Williams, B. II, Convertino, M., Das, J., and Dokholyan, N. V.','ApoE4-specific misfolded intermediate identified by molecular dynamics simulations','Public Library of Science Computational Biology',11,'e1004359','',2015,'Proteins: Folding','Dokholyan, N. V.','yes','no','no','article',0,'','','','','The increased risk of developing Alzheimers disease (AD) is associated with the APOE gene, which encodes for three variants of Apolipoprotein E, namely E2, E3, E4, differing only by two amino acids at positions 112 and 158. ApoE4 is known to be the strongest risk factor for AD onset, while ApoE3 and ApoE2 are considered to be the AD-neutral and AD-protective isoforms, respectively. It has been hypothesized that the ApoE isoforms may contribute to the development of AD by modifying the homeostasis of ApoE physiological partners and AD-related proteins in an isoform-specific fashion. Here we find that, despite the high sequence similarity among the three ApoE variants, only ApoE4 exhibits a misfolded intermediate state characterized by isoform-specific domain-domain interactions in molecular dynamics simulations. The existence of an ApoE4-specific intermediate state can contribute to the onset of AD by altering multiple cellular pathways involved in ApoE-dependent lipid transport efficiency or in AD-related protein aggregation and clearance. We present what we believe to be the first structural model of an ApoE4 misfolded intermediate state, which may serve to elucidate the molecular mechanism underlying the role of ApoE4 in AD pathogenesis. The knowledge of the structure for the ApoE4 folding intermediate provides a new platform for the rational design of alternative therapeutic strategies to fight AD.'),(812,'msspsrgtcpsfgokmzdrmd_pain15','Meloto, C. B., Segell, S. K., Parisien, M., Shabalina S. A., Rizzatti-Barbosa, C. M., Gauthier, J., Tsao, D., Convertino, M., Piltonen, M. H., Slade, G. D., Fillingim, R. B., Greenspan, J. D., Ohrbach, R., Knott, C., Maixner, W., Zaykin, D., Dokholyan, N. V., Reenil&auml I., M&aumlnnist&ouml; P. T., and Diatchenko L.','COMT gene locus: new functional variants','Pain',156,'2072','2083',2015,'Proteins: Diseases','Dokholyan, N. V.','yes','no','no','article',26207649,'','','','','Catechol-O-methyltransferase (COMT) metabolizes catecholaminergic neurotransmitters. Numerous studies have linked COMT to pivotal brain functions such as mood, cognition, response to stress, and pain. Both nociception and risk of clinical pain have been associated with COMT genetic variants, and this association was shown to be mediated via adrenergic pathways. Here we show that association studies between COMT polymorphic markers and pain phenotypes in two independent cohorts identified a new functional marker, rs165774, situated in the 3\' untranslated region of a newfound splice variant, (a)-COMT. Sequence comparisons showed that the (a)-COMT transcript is highly conserved in primates, and deep sequencing data demonstrated that (a)-COMT is expressed across several human tissues, including the brain. In silico analyses showed that the (a)-COMT enzyme features a distinct C-terminus structure, capable of stabilizing substrates in its active site. In vitro experiments demonstrated not only that (a)-COMT is catalytically active, but also that it displays unique substrate specificity, exhibiting enzymatic activity with dopamine but not epinephrine. They also established that the pain protective A allele of rs165774 coincides with lower COMT activity, suggesting contribution to decreased pain sensitivity through increased dopaminergic rather than decreased adrenergic tone, characteristic of reference isoforms. Our results provide evidence for an essential role of the (a)-COMT isoform in nociceptive signaling and suggest that genetic variations in (a)-COMT isoforms may contribute to individual variability in pain phenotypes.'),(814,'wxrlyzklzcsjdc_cpcb15','Wang, L., Xie, L., Ramachandran, S., Lee, Y., Yan, Z., Zhou, L., Krajewski, K., Liu, F., Zhu, C., Chen, D. J., Strahl, B. D., Jin, J., Dokholyan, N. V., Chen, X.','Non-canonical bromodomain within DNA-PKcs promotes DNA damage response and radioresistance through recognizing an IR-induced acetyl-lysine on H2AX','Chemistry &amp Biology',22,'849','861',2015,'Proteins: Mechanism','Dokholyan, N. V. and Chen, X.','yes','no','no','article',26119999,'','','','','Regulatory mechanisms underlying &gamma;H2AX induction and the associated cell fate decision during DNA damage response (DDR) remain obscure. Here, we discover a bromodomain (BRD)-like module in DNA-PKcs (DNA-PKcs-BRD) that specifically recognizes H2AX acetyl-lysine 5 (K5ac) for sequential induction of &gamma;H2AX and concurrent cell fate decision(s). First, top-down mass spectrometry of radiation-phenotypic, full-length H2AX revealed a radiation-inducible, K5ac-dependent induction of &gamma;H2AX. Combined approaches of sequence-structure modeling/docking, site-directed mutagenesis, and biochemical experiments illustrated that through docking on H2AX K5ac, this non-canonical BRD determines not only the H2AX-targeting activity of DNA-PKcs but also the over-activation of DNA-PKcs in radioresistant tumor cells, whereas a Kac antagonist, JQ1, was able to bind to DNA-PKcs-BRD, leading to re-sensitization of tumor cells to radiation. This study elucidates the mechanism underlying the H2AX-dependent regulation of DNA-PKcs in ionizing radiation-induced, differential DDR, and derives an unconventional, non-catalytic domain target in DNA-PKs for overcoming resistance during cancer radiotherapy.'),(815,'mpgcadjdm_jp15','Martin, L. J., Piltonen, M. H., Gauthier, J., Convertino, M., Acland, E. L., Dokholyan, N. V., Jeffrey, S. M., Diatchenko, L., Maixner, W.','Differences in the antinociceptive effects and binding properties of propranolol and bupranolol enantiomers','The Journal of Pain',16,'1321','1333',2015,'Proteins: Diseases','Dokholyan, N. V.','yes','no','no','article',26456674,'','','','','Recent efforts have suggested that the &beta;-adrenergic receptor (&beta;AR) system may be a novel and viable therapeutic target for pain reduction, however most of the work to date has focused on the &beta;2adrenergic receptor. Here, we compared the antinociceptive effects of enantiomeric configurations of propranolol and bupranolol, two structurally similar non-selective  &beta;-blocking drugs against mouse models of inflammatory and chronic pain.  In addition, we calculated in silico docking and measured the binding properties of propranolol and bupranolol for all three &beta;ARs. Of the examined agents, S-bupranolol is superior in terms of its antinociceptive effect, and exhibited fewer side effects than propranolol or its associated enantiomers.  In contrast to propranolol, S-bupranolol exhibited negligible &beta;AR intrinsic agonist activity and displayed a full competitive antagonist profile at &beta;1/&beta;2/&beta;3ARs producing a unique blockade of &beta;3ARs. We have shown that S-bupranolol is an effective antinociceptive in mice without negative side effects. The distinctive profile of S-bupranolol is most likely mediated by its negligible &beta;AR intrinsic agonist activity and unique blockade of &beta;3AR. These findings suggest that S-bupranolol and not propranolol may represent a new and effective treatment for a variety of painful conditions.'),(816,'csvgfssmdd_ploso15','Convertino, M., Samoshkin, A., Viet, C. T., Gauthier, J., Fraine, S. P. L., Sharif-Naeini, R., Schimdt, B. L., Maixner, W., Diatchenko, L., and Dokholyan, N. V.','Differential regulation of 6- and 7-transmembrane helix variants of &mu;-opioid receptor in response to morphine stimulation','Public Library of Science One',10,'e0142826','',2015,'Proteins: Function, Proteins: Mechanism, Proteins: Diseases, Drugs','Dokholyan, N. V.','yes','no','no','article',26554831,'','','','','The pharmacological effect of opioids originates, at the cellular level, by their interaction with the &mu;-opioid receptor (mOR) resulting in the regulation of voltage-gated Ca2+ channels and inwardly rectifying K+ channels that ultimately modulate the synaptic transmission. Recently, an alternative six trans-membrane helix isoform of mOR, (6TM-mOR) has been identified, but its function and signaling are still largely unknown. Here, we present the structural and functional mechanisms of 6TM-mOR signaling activity upon binding to morphine. Our data suggest that despite the similarity of binding modes of the alternative 6TM-mOR and the dominant seven trans-membrane helix variant (7TM-mOR), the interaction with morphine generates different dynamic responses in the two receptors, thus, promoting the activation of different mOR-specific signaling pathways. We characterize a series of 6TM-mOR-specific cellular responses, and observed that they are significantly different from those for 7TM-mOR. Morphine stimulation of 6TM-mOR does not promote a cellular cAMP response, while it increases the intracellular Ca2+ concentration and reduces the cellular K+ conductance. Our findings indicate that 6TM-mOR has a unique contribution to the cellular opioid respons'),(817,'rddd_jme16','Redler, R. L., Das, J., Diaz J. R., and Dokholyan, N. V.','Protein destabilization as a common factor in diverse inherited disorders','Journal of Molecular Evolution',82,'11','16',2016,'Proteins: Diseases','Dokholyan, N. V.','no','no','yes','article',26584803,'','','','','Protein destabilization by amino acid substitutions is proposed to play a prominent role in widespread inherited human disorders, not just those known to involve protein misfolding and aggregation. To test this hypothesis, we computationally evaluate the effects on protein stability of all possible amino acid substitutions in 20 disease-associated proteins with multiple identified pathogenic missense mutations. For 18 of the 20 proteins studied, substitutions at known positions of pathogenic mutations are significantly more likely to destabilize the native protein fold (as indicated by more positive values of &Delta;&Delta;G). Thus, positions identified as sites of disease-associated mutations, as opposed to non-disease-associated sites, are predicted to be more vulnerable to protein destabilization upon amino acid substitution. This finding supports the notion that destabilization of native protein structure underlies the pathogenicity of broad set of missense mutations, even in cases where reduced protein stability and/or aggregation are not characteristic of the disease state.'),(818,'pd_cosb15','Proctor, E. A., and Dokholyan, N. V.','Applications of Discrete Molecular Dynamics in biology and medicine','Current Opinion in Structural Biology',37,'9','13',2016,'Methods','Dokholyan, N. V.','no','no','no','article',26638022,'','','','','Discrete Molecular Dynamics (DMD) is a physics-based simulation method using discrete energetic potentials rather than traditional continuous potentials, allowing microsecond time scale simulations of biomolecular systems to be performed on personal computers rather than supercomputers or specialized hardware. With the ongoing explosion in processing power even in personal computers, applications of DMD have similarly multiplied. In the past two years, researchers have used DMD to model structures of disease-implicated protein folding intermediates, study assembly of protein complexes, predict proteinprotein binding conformations, engineer rescue mutations in disease-causative protein mutants, design a protein conformational switch to control cell signaling, and describe the behavior of polymeric dispersants for environmental cleanup of oil spills, among other innovative applications.'),(819,'scvwkmpsmkmsdd_sr15','Samoshkin, A., Convertino, M., Viet, C. T., Wieskopf, J. S., Kambur, O., Marcovitz, J., Patel, P., Stone, L. S., Maixner, W., Kalso, E., Mogil, J. S., Schmidt, B. L., Dokholyan, N. V., and L. Diatchenko, L.','Structural and functional interaction between six transmembrane &mu;-opioid receptor isoform and &beta;2-adrenoreceptors','Scientific Reports',5,'18198','',2015,'Proteins: Function, Proteins: Mechanism, Proteins: Diseases, Drugs','Dokholyan, N. V.','yes','no','no','article',26657998,'','','','','The primary molecular target for clinically used opioids is the &mu;-opioid receptor (MOR). Besides the major seven-transmembrane (7TM) receptors, the MOR gene codes for alternatively spliced six-transmembrane (6TM) isoforms, the biological and clinical significance of which remains unclear. Here, we show that the otherwise exclusively intracellular localized 6TM-MOR translocates to the plasma membrane upon coexpression with &beta;2-adrenergic receptors (&beta;2-ARs) through an interaction with the fifth and sixth helices of &beta;2-AR. Coexpression of the two receptors in BE(2)-C neuroblastoma cells potentiates calcium responses to a 6TM-MOR ligand, and this calcium response is completely blocked by a selective &beta;2-antagonist in BE(2)-C cells, and in trigeminal and dorsal root ganglia. Co-administration of 6TM-MOR and &beta;2-AR ligands leads to substantial analgesic synergy and completely reverses opioid-induced hyperalgesia in rodent behavioral models. Together, our results provide evidence that the heterodimerization of 6TM-MOR with &beta;2-AR underlies a molecular mechanism for 6TM cellular signaling, presenting a unique functional responses to opioids. This signaling pathway may contribute to the hyperalgesic effects of opioids that can be efficiently blocked by &beta;2-AR antagonists, providing a new avenue for opioid therapy.'),(820,'adb_acscb16','Allen, S. E., Dokholyan, N. V. and Bowers, A. A.','Dynamic docking of conformationally constrained macrocycles: methods and applications','ACS Chemical Biology',11,'10','24',2016,'Drugs, Methods','Dokholyan, N. V.','no','no','no','article',26575401,'','','','','Any natural products consist of large and flexible macrocycles that engage their targets via multiple contact points. This combination of contained flexibility and large contact area often allows natural products to bind at target surfaces rather than deep pockets, making them attractive scaffolds for inhibiting protein-protein interactions and other challenging therapeutic targets. The increasing ability to manipulate such compounds either biosynthetically or via semi-synthetic modification means that these compounds can now be considered as starting points for medchem campaigns, rather than solely as ends. Modern medchem benefits substantially from rational improvements made on the basis of molecular docking. As such, docking methods have been enhanced in recent years to deal with the complicated binding modalities and flexible scaffolds of macrocyclic natural products and natural product-like structures. Here, we comprehensively review methods for treating and docking these large macrocyclic scaffolds and discuss some of the resulting advances in medicinal chemistry.'),(821,'mabdmsdsyd_nrdd09','Michino, M., Abola, E., Brooks III, C. L., Scott Dixon, J., Moult, J., Stevens, R. C., Olson, A., Jurkowski, W., Elofsson, A., Filipek, S., Pogozheva, I., Lomize, A., Maigret, B., Horst, J., Roy, A., Bernard, B., Iyer, S., Zhang, Y., Samudrala, R., Sezerman, O.U., Nikiforovich, G.V., Taylor, C.M., Costanzi, S., Vorobjev, Y., Bakulina, N., Solovyev, V.,  Kanou, K.,  Takaya, D.,  Terashi, G., Takeda-Shitaka, M., Umeyama, H., Goddard III, W.A., Li, Y., Kim, S.-K., Trzaskowski, B., Abrol, R., Griffith, A., Katritch, V., Rueda, M., Abagyan, R., Davis, I., Barth, P., Baker, D., Feig, M., Brylinski, M., Zhou, H.,  Lee, S.Y.,  Skolnick, J., Ostopovici-Halip, L., Bologa, C., Lam, P., Abagyan, R., Dawson, E.S., Kaufmann, K., Woetzel, N., Meiler, J., Ding, F., Serohijos, A., Yin, S., Dokholyan, N.V., Rodriguez, D., Guti&eactue;rrez-de-Ter&aacute;n, H., and Xhaard, H.','Community-wide assessment of GPCR structure modelling and ligand docking: GPCR Dock 2008','Nature Reviews Drug Discovery',8,'455','463',2009,'Drugs: Screening','Dokholyan, N. V.','yes','no','no','article',19461661,'','','','','Recent breakthroughs in the determination of the crystal structures of G protein-coupled receptors (GPCRs) have provided new opportunities for structure-based drug design strategies targeting this protein family. With the aim of evaluating the current status of GPCR structure prediction and ligand docking, a community-wide, blind prediction assessment  GPCR Dock 2008  was conducted in coordination with the publication of the crystal structure of the human adenosine A2A receptor bound to the ligand ZM241385. Twenty-nine groups submitted 206 structural models before the release of the experimental structure, which were evaluated for the accuracy of the ligand binding mode and the overall receptor model compared with the crystal structure. This analysis highlights important aspects for success and future development, such as accurate modelling of structurally divergent regions and use of additional biochemical insight such as disulphide bridges in the extracellular loops.'),(822,'krksadkrd_str11','Kufareva, I., Rueda, M., Katritch, V., Stevens, R. C., Abagyan, R., Yoshikawa, Y., Furuya, T., Lee, H., Roy, A., Grime, J., Rebehmed, J., Zhang, Y., Roumen, L., de Esch, I. J. P., Leurs, R., de Graaf, C., Li, Y., Hou, T., Mysinger, M. M., Weiss, D. R., Irwin, J.J., Shoichet, B. K., McRobb, F. M., Capuano, B., Crosby, I. T., Chalmers, D. K., Yuriev, E., Wang, Q.,  Mach, R. H., Reichert, D. E., Chuang, G.-Y., Rognan, D., Simms, J., Sexton, P., Wootten, D., Latek, D., Ghoshdastider, U., Filipek, S., LenServern, Kirkpatrick, A., Trzaskowski, B., Griffith, A., Kim, S.-K., Abrol, R., Goddard III, W. A., Vaidehi, N., Lam, A., Bhattacharya, S., Li, H., Balaraman, G., Niesen, M., Pal, S., Solovyev, V., Vorobjev, Y., Bakulina, N., Beuming, T., Costanzi, S., Shi, L., Higgs, C., Salam, N., Lupyan, D., Sherman, W., Ding, F., Kota, P., Ramachandran, S., Dokholyan, N. V., Carlsson, J., Coleman, R.G., Fan, H., Schlessinger, A., Irwin, J. J., Sali, A., Shoichet, B. K., Tikhonova, I., Pogozheva, I., Lomize, A., Hall, N. E., Muddassa, M., Zhang, Y., Nim Pae, A., Lee, J., Lopez, L., Obiol-Pardo, C., Selent, J., Mahboob, S., Werner, T., Bret Church, W., Brylinski, M., Ando, T., Guerler, A., Zhou, H., Skolnick, J., Xhaard, H., Jurkowski, W., Elofsson, A., Murad, A. K., Drwal, M., Dupree, T. B., Griffith, R., Ostopovici-Halip, L., Bologa, C., Chen, K. M., Sun, J., Barth, P., Yarov-Yarovoy, V., Baker, D., Vroling, B., Sanders, M. P. A., Nabuurs, S. B., and Nikiforovich, G. V.','Status of GPCR modeling and docking as reflected by community-wide GPCR dock 2010 assessment','Structure',19,'1108','1126',2011,'Drugs: Screening','Dokholyan, N. V.','yes','no','no','article',21827947,'','','','','The community-wide GPCR Dock assessment is conducted to evaluate the status of molecular modeling and ligand docking for human G protein-coupled receptors. The present round of the assessment was based on the recent structures of dopamine D3 and CXCR4 chemokine receptors bound to small molecule antagonists and CXCR4 with a synthetic cyclopeptide. Thirty-five groups submitted their receptor-ligand complex structure predictions prior to the release of the crystallographic coordinates. With closely related homology modeling templates, as for dopamine D3 receptor, and with incorporation of biochemical and QSAR data, modern computational techniques predicted complex details with accuracy approaching experimental. In contrast, CXCR4 complexes that had less-characterized interactions and only distant homology to the known GPCR structures still remained very challenging. The assessment results provide guidance for modeling and crystallographic communities in method development and target selection for further expansion of the structural coverage of the GPCR universe.'),(823,'pftrfzlmdld_pnas16','Proctor, E. A., Fee, L., Tao, Y., Redler, R. L., Fay, J. M., Zhang, Y., Lv, Z., Mercer, I. P., Deshmukh, M., Lyubchenko, Y. L., and Dokholyan, N. V.','Non-native SOD1 trimer is toxic to motor neurons in a model of amyotrophic lateral sclerosis','Proceedings of the National Academy of Sciences USA',113,'614','619',2016,'Proteins: Diseases, Proteins: Aggregation','Dokholyan, N. V.','yes','no','yes','article',26719414,'','','','','Since the linking of mutations in the gene sod1 to ALS in 1993, researchers have sought the connection between Cu,Zn superoxide dismutase (SOD1) and motor neuron death. Disease-linked mutations tend to destabilize the native dimeric structure of SOD1, and plaques containing misfolded and aggregated SOD1 have been found in the motor neurons of ALS patients. Despite advances in understanding of ALS disease progression and SOD1 folding and stability, cytotoxic species and mechanisms remain unknown, greatly impeding the search for and design of therapeutic interventions. Here, we definitively link cytotoxicity associated with SOD1 aggregation in amyotrophic lateral sclerosis (ALS) to a non-native trimeric SOD1 species. We develop methodology for the incorporation of low-resolution experimental data into simulations toward the structural modeling of meta-stable, multi-domain aggregation intermediates. We apply this methodology to derive the structure of a SOD1 trimer, which we validate in vitro and in hybridized motor neurons. We show that SOD1 mutants designed to promote trimerization increase cell death. Further, we demonstrate that the cytotoxicity of the designed mutants correlates with trimer stability, providing a direct link between the presence of misfolded oligomers and neuron death. Identification of cytotoxic species is the first and critical step in elucidating the molecular etiology of ALS, and the ability to manipulate formation of these species will provide an avenue for the development of future therapeutic strategies.'),(824,'sdpdhsd_jbc16','Shobair, M., Dagliyan, O., Kota, P., Dang, Y. L., He, H., Stutts, M. J., and Dokholyan, N. V.','Gain-of-function mutation W493R in the epithelial sodium channel allosterically reconfigures intersubunit coupling','Journal of Biological Chemistry (Paper of the Week)',291,'3682','3692',2016,'Proteins: Allostery','Dokholyan, N. V.','no','no','no','article',26668308,'','','','','Sodium absorption in epithelial cells is rate-limited by the epithelial sodium channel (ENaC) activity in lung, kidney and the distal colon. Pathophysiological conditions, such as cystic fibrosis and Liddle syndrome, result from water-electrolyte imbalance partly due to malfunction of ENaC regulation. Because the quaternary structure of ENaC is yet undetermined, the bases of pathologically linked mutations in ENaC subunits &alpha;, &beta; and &gamma; are largely unknown. Here we present a structural model of heterotetrameric ENaC &alpha;1&beta;&alpha;2&gamma; that is consistent with previous cross-linking results and site-directed mutagenesis experiments. By using this model, we show that the disease-causing mutation &alpha;W493R rewires structural dynamics of the intersubunit interfaces &alpha;1&beta; and &alpha;2&gamma;. Changes in dynamics can allosterically propagate to the channel gate. We demonstrate that cleavage of the &gamma;-subunit, which is critical for full channel activation, does not mediate activation of ENaC by &alpha;W493R. Our molecular dynamics simulations lead us to identify a channel-activating electrostatic interaction between &alpha;2R493 and &gamma;E348 at the &alpha;2&gamma; interface. By neutralizing a sodium binding acidic patch at the &alpha;1&beta; interface, we reduced ENaC activation by &alpha;W493R by more than 2-fold. By combining homology modeling, molecular dynamics, cysteine cross-linking and voltage clamp experiments, we propose a dynamics-driven model for the gainof-function in ENaC by &alpha;W493R. Our integrated computational and experimental approach advances our understanding of structure, dynamics, and function of ENaC in its disease-causing state.'),(825,'cddc_sd16','Chandrasekaran, S. N., Das, J., Dokholyan, N. V. and Carter Jr., C. W.','A modified PATH algorithm rapidly generates transition states comparable to those found by other well established algorithms','Structural Dynamics',3,'012101','',2016,'Methods','Carter Jr., C. W.','no','no','no','article',0,'','','','','PATH rapidly computes a path and transition state between crystal structures by minimizing the Onsager-Machlup action. It requires input parameters whose range of values can generate different transition-state structures that cannot be uniquely com- pared with those generated by other methods. We outline modifications to estimate these input parameters to circumvent these difficulties, and validate the PATH tran- sition states by showing consistency between transition-states derived by different algorithms for unrelated protein systems. Although functional protein conforma- tional change trajectories are to a degree stochastic, they nonetheless pass through a well-defined transition state whose detailed structural properties can rapidly be identified using PATH.'),(826,'hpddm_psfb16','Hadi-Alijanvand, H., Proctor, E. A., Ding, F., Dokholyan, N. V. and Moosavi-Movahedi, A. A.','A hidden aggregation-prone structure in the heart of hypoxia inducible factor prolyl hydroxylase','Proteins',84,'611','623',2016,'Proteins: Structure','Dokholyan, N. V. and Moosavi-Movahedi, A. A.','yes','no','no','article',26868435,'','','','','Prolyl hydroxylase domain-containing protein 2 (PHD2), as one of the most important regulators of angiogenesis and metastasis of cancer cells, is a promising target for cancer therapy drug design. Progressive studies imply that abnormality in PHD2 function may be due to misfolding. Therefore, study of the PHD2 unfolding pathway paves the way for a better understanding of the influence of PHD2 mutations and cancer cell metabolites on the protein folding pathway. We study the unfolding of the PHD2 catalytic domain using differential scanning calorimetry (DSC), fluorescence spectroscopy, and discrete molecular dynamics simulations (DMD). Using computational and experimental techniques, we find that PHD2 undergoes four transitions along the thermal unfolding pathway. To illustrate PHD2 unfolding events in atomic detail, we utilize DMD simulations. Analysis of computational results indicates an intermediate species in the PHD2 unfolding pathway that may enhance aggregation propensity, explaining mutation-independent PHD2 malfunction.'),(827,'d_cr16','Dokholyan, N. V.','Controlling allosteric networks in proteins','Chemical Reviews',116,'6463','6487',2016,'Proteins: Allostery','Dokholyan, N. V.','no','no','no','article',26894745,'','','','','Allosteric transition, defined as conformational changes induced by ligand binding, is one of the fundamental properties of proteins. Allostery has been observed and characterized in many proteins, and has been recently utilized to control protein function via regulation of protein activity. Here, we review the physical and evolutionary origin of protein allostery, as well as its importance to protein regulation, drug discovery, and biological processes in living systems. We describe recently developed approaches to identify allosteric pathways, connected sets of pairwise interactions that are responsible for propagation of conformational change from the ligand-binding site to a distal functional site. We then present experimental and computational protein engineering approaches for control of protein function by modulation of allosteric sites. As an example of application of these approaches, we describe a synergistic computational and experimental approach to rescue the cystic fibrosis-associated protein cystic fibrosis transmembrane conductance regulator, which upon deletion of a single residue misfolds and causes disease. This example demonstrates the power of allosteric manipulation in proteins to both elucidate mechanisms of molecular function and to develop therapeutic strategies that rescue those functions. Allosteric control of proteins provides a tool to shine a light on the complex cascades of cellular processes and facilitate unprecedented interrogation of biological systems.'),(828,'pcpdt_jcim16','Politi, R., Convertino, M., Popov, K., Dokholyan, N. V. and Tropsha, A.','Docking and scoring with target-specific pose classifier succeeds in native-like pose identification but not binding affinity prediction in the CSAR 2014 benchmark exercise','Journal of Chemical Information and Modeling',56,'1032','1041',2016,'Drugs: Screening','Dokholyan N. V. and Tropsha, A.','yes','no','no','article',27050767,'','','','','The CSAR 2014 exercise provided an important benchmark for testing current approaches for pose identification and ligand ranking using three X-ray characterized proteins: Factor Xa (FXa), Spleen Tyrosine Kinase (SYK), and tRNA Methyltransferase (TRMD). In Phase 1 of the exercise, we employed Glide and MedusaDock docking software, both individually and in combination, with the special target-specific pose classifier trained to discriminate native-like from decoy poses. All approaches succeeded in the accurate detection of native and native-like poses. We then used Glide SP and MedusaScore scoring functions individually and in combination with the pose-scoring approach to predict relative binding affinities of the congeneric series of ligands in Phase 2 of the exercise. Similar to other participants in the CSAR 2014 exercise, we found that our models showed modest prediction accuracy. Quantitative Structure Activity Relationship (QSAR) models developed for the FXa ligands using available bioactivity data from ChEMBL showed relatively low prediction accuracy for the CSAR 2014 ligands of the same target. Interestingly, QSAR models built with CSAR data only yielded Spearman correlation coefficient as high as &rho;=0.69 for FXa, and &rho;=0.79 for SYK based on 5-fold cross-validation. Virtual screening of the DUD library using the FXa structure was successful in discriminating between active compounds and decoys in spite of poor ranking accuracy of the underlying scoring functions. Our results suggest that two of the three common tasks associated with molecular docking, i.e., native-like pose identification and virtual screening, but not binding affinity prediction could be accomplished successfully for the CSAR 2014 challenge dataset.'),(829,'cdd_acscb16','Convertino, M., Das, J., and Dokholyan, N. V.','Pharmacological chaperones: design and development of new therapeutic strategies for the treatment of conformational diseases','ACS Chemical Biology',11,'1471','1489',2016,'Drugs','Dokholyan N. V.','no','no','no','article',27097127,'','','','','Errors in protein folding may result in premature clearance of structurally aberrant proteins, or in the accumulation of toxic misfolded species and/or protein aggregates. These pathological events lead to a large range of conditions known as conformational diseases. Several research groups have presented possible therapeutic solutions for their treatment by developing novel compounds, known as pharmacological chaperones. These cell-permeable molecules selectively provide a molecular scaffold around which misfolded proteins can recover their native folding and, thus, their biological activities. Here, we review therapeutic strategies, clinical potentials, and cost-benefit impacts of several classes of pharmacological chaperones for the treatment of a series of conformational diseases.'),(831,'cd_mmb16','Convertino, M., and Dokholyan, N. V.','Computational modeling of small molecule ligand binding interactions and affinities','Methods in Molecular Biology',1414,'23','32',2016,'Methods','Dokholyan N. V.','no','no','no','book',27094283,'','','','','Understanding and controlling biological phenomena via structure-based drug screening efforts often critically rely on accurate description of protein-ligand interactions. However, most of the currently available computational techniques are affected by severe deficiencies in both protein and ligand conformational sampling as well as in the scoring of the obtained docking solutions. To overcome these limitations, we have recently developed MedusaDock, a novel docking methodology, which simultaneously models ligand and receptor flexibility. Coupled with MedusaScore, a physical force field-based scoring function that accounts for the protein-ligand interaction energy, MedusaDock, has reported the highest success rate in the CSAR 2011 exercise. Here, we present a standard computational protocol to evaluate the binding properties of the two enantiomers of the non-selective &beta;-blocker propanolol in the &beta;2 adrenergic receptor\'s binding site. We describe details of our protocol, which have been successfully applied to several other targets.'),(832,'zmd_mmb16','Zhu, C., Mowrey, D. D., and Dokholyan, N. V.','Computational protein design through grafting and stabilization','Methods in Molecular Biology: Computational Protein Design',1529,'227','241',2017,'Methods','Dokholyan N. V.','no','no','no','book',27914054,'','','','',''),(833,'grhqcscddrc_ajplung16','Gentzsch, M., Ren, H. Y., Houck, S., Quinney, N., Cholon, D., Sopha, P., Chaudhry, I., Das, J., Dokholyan, N. V., Randell, S., and Cyr, D.','Restoration of R117H CFTR folding and function in human airway cells through combination treatment with VX-809 and VX-770','American Journal of Physiology - Lung Cellular and Molecular Physiology',311,'L550','L559',2016,'Proteins: Diseases','Dokholyan N. V., Randall, S. and Cyr, D.','no','no','no','article',27402691,'','','','','Cystic fibrosis (CF) is a lethal recessive genetic disease caused primarily by the F508del mutation in the CF transmembrane conductance regulator (CFTR). The potentiator VX-770 was the first CFTR modulator approved by the FDA for treatment of CF patients with the gating mutation G551D. Orkambi is a drug containing VX-770 and corrector VX809, and is approved for treatment of CF patients homozygous for F508del, which has folding and gating defects. At least 30% of CF patients are heterozygous for the F508del mutation with the other allele encoding for one of many different rare CFTR mutations. Treatment of heterozygous F508del patients with VX-809 and VX-770 has had limited success, so it is important to identify heterozygous patients that respond to CFTR modulator therapy. R117H is a more prevalent rare mutation found in over 2000 CF patients. In this study we investigated the effectiveness of VX-809/VX-770 therapy on restoring CFTR function in human bronchial epithelial (HBE) cells from R117H/F508del CF patients. We found that VX-809 stimulated more CFTR activity in R117H/F508del HBEs than in F508del/F508del HBEs. R117H expressed exclusively in immortalized HBEs exhibited a folding defect, was retained in the ER, and degraded prematurely. VX-809 corrected the R117H folding defect and restored channel function. Because R117 is involved in ion conductance, VX-770 acted additively with VX-809 to restore CFTR function in chronically treated R117H/F508del cells. Although treatment of R117H patients with VX-770 has been approved, our studies indicate that Orkambi may be more beneficial for rescue of CFTR function in these patients.'),(834,'fzptckd_struct16','Fay, J. M., Zhu, C., Proctor, E. A., Tao, Y., Cui, W., Ke, H., and Dokholyan, N. V.','A phosphomimetic mutation stabilizes SOD1 and rescues cell viability in the context of an ALS-associated mutation','Structure',24,'1898','1906',2016,'Proteins: Diseases','Dokholyan N. V.','no','no','no','article',27667694,'','','','','The majority of amyotrophic lateral sclerosis (ALS) related mutations in the enzyme Cu, Zn superoxide dismutase (SOD1), as well as a post-translational modification, glutathionylation, destabilize the protein and lead to a misfolded oligomer, that is toxic to motor neurons. The biophysical role of another physiological SOD1 modification, T2-phosphorylation, has remained a mystery. Here, we find that a phosphomimetic mutation, T2D, thermodynamically stabilizes SOD1 even in the context of a strongly SOD1-destabilizing mutation, A4V, one of the most prevalent and aggressive ALS-associated mutations in North America. This stabilization protects against formation of toxic SOD oligomers and positively impacts motor neuron survival in cellular assays. We solve the crystal structure of T2D-SOD1 and explain its stabilization effect using DMD simulations. These findings imply that T2-phosphorylation may be a plausible innate cellular protection response against SOD1-induced cytotoxicity, and stabilizing SOD1 native conformation might offer us viable pharmaceutical strategies to fight currently incurable ALS.'),(835,'jmafd_cps16','Jacob, R. B., Michaels, K., Anderson, C., Fay, J. M., and Dokholyan, N. V.','Harnessing Natures Diversity: Discovering organophosphate bioscavenger characteristics among low molecular weight proteins','Scientific Reports',6,'37175','',2016,'','Dokholyan N. V.','yes','no','no','article',27845442,'','','','','Organophosphate poisoning can occur from exposure to agricultural pesticides or chemical weapons. This exposure inhibits acetylcholinesterase resulting in increased acetylcholine levels within the synaptic cleft causing loss of muscle control, seizures, and death. Mitigating the effects of organophosphates in our bodies is critical and yet an unsolved challenge. Here, we present a computational strategy that integrates structure mining and modeling approaches, using which we identify novel candidates capable of interacting with a serine hydrolase probe (with equilibrium binding constants ranging from 4 to 120 &mu;M). One candidate Smu. 1393c catalyzes the hydrolysis of the organophosphate omethoate (k<sub>cat</sub>/K<sub>m</sub> of (2.0&plusmn;1.3)10<sup>-1</sup> M<sup>-1</sup>s<sup>-1</sup>) and paraoxon (k<sub>cat</sub>/K<sub>m</sub> of (4.6&plusmn;0.8)103 M<sup>-1</sup>s<sup>-1</sup>), V- and G-agent analogs respectively. In addition, Smu. 1393c protects acetylcholinesterase activity from being inhibited by two organophosphate simulants. We demonstrate that the utilized approach is an efficient and highly-extendable framework for the development of prophylactic therapeutics against organophosphate poisoning and other important targets. Our findings further suggest currently unknown molecular evolutionary rules governing natural diversity of the protein universe, which make it capable of recognizing previously unseen ligands.'),(836,'kmwd_rna17','Krokhotin, A., Mustoe, A. M., Weeks, K. M., and Dokholyan, N. V.','Direct identification of base-paired RNA nucleotides by correlated chemical probing','RNA',23,'6','13',2017,'RNA','Dokholyan N. V.','yes','no','no','article',27803152,'','','','','Many RNA molecules fold into complex secondary and tertiary structures that play critical roles in biological function. Among the best-established methods for examining RNA structure are chemical probing experiments, which can report on local nucleotide structure in a concise and extensible manner. While probing data are highly useful for inferring overall RNA secondary structure, these data do not directly measure through-space base pairing interactions. We recently introduced an approach for single-molecule correlated chemical probing with dimethyl sulfate (DMS) that measures RNA interaction groups by mutational profiling (RING-MaP). RING-MaP experiments reveal diverse through-space interactions corresponding to both secondary and tertiary structure. Here we develop a framework for using RING-MaP data to directly and robustly identify canonical base pairs in RNA. When applied to three representative RNAs, this framework identified 20-50% of accepted base pairs with a <10% false discovery rate, allowing detection of 88% of duplexes containing four or more base pairs, including pseudoknotted pairs. We further show that base pairs determined from RING-MaP analysis significantly improve secondary structure modeling. RING-MaP based correlated chemical probing represents a direct, experimentally concise, and accurate approach for detection of individual base pairs and helices, and should greatly facilitate structure modeling for complex RNAs.'),(837,'dtcssdh_science16','Dagliyan, O., Tarnawski, M., Chu, P-H., Shirvanyants, D., Schlichting, I., Dokholyan, N. V. and Hahn, K. M.','Engineering extrinsic disorder to control protein activity in living cells','Science',354,'1441','1444',2016,'Proteins: Design','Dokholyan N. V. and Hahn, K.','yes','no','yes','article',0,'','','','','Optogenetic and chemogenetic control of proteins has revealed otherwise inaccessible facets of signaling dynamics. Here, we use light- or ligand-sensitive domains to modulate the structural disorder of diverse proteins, thereby generating robust allosteric switches. Sensory domains were inserted into nonconserved, surface-exposed loops that were tight and identified computationally as allosterically coupled to active sites. Allosteric switches introduced into motility signaling proteins (kinases, guanosine triphosphatases, and guanine exchange factors) controlled conversion between conformations closely resembling natural active and inactive states, as well as modulated the morphodynamics of living cells. Our results illustrate a broadly applicable approach to design physiological protein switches.'),(838,'trcttpdwc_struct17','Thompson, P. M., Ramachandran, S., Case, L. B., Tolbert, C. E., Tandon, A., Pershad, M., Dokholyan, N. V., Waterman, C. M., and Campbell, S. L.','A structural model for vinculin insertion into PIP<sub>2</sub>-containing membranes and the effect of insertion on vinculin activation and localization','Structure',25,'264','275',2017,'Proteins: Structure','Dokholyan N. V., Waterman, C. M. and Campbell, S. L.','yes','no','no','article',28089450,'','','','','Vinculin is a highly conserved scaffolding protein that localizes to focal adhesions (FA) and adherens junctions, where it binds actin and links the actin cytoskeleton to the adhesive structure. Vinculin also binds to phosphatidylinositol 4,5-bisphosphate (PIP<sub>2</sub>). However, despite recent structural studies, it is unclear how vinculin associates with PIP<sub>2</sub> in the context of a lipid bilayer and how binding to PIP<sub>2</sub> regulates vinculin localization, activation, turnover at FAs and cell motility. Herein, we incorporate lipid-association data, nuclear magnetic resonance (NMR), and computational modeling to generate an experimentally-driven structural model for PIP<sub>2</sub>-dependent interaction of vinculin with the lipid bilayer. According to our model, two basic patches on the vinculin tail drive membrane association: the basic collar specifically recognizes the PIP<sub>2</sub> head group, while the basic ladder drives association with the lipid bilayer. As mutations within the basic collar impair vinculin-mediated actin crosslinking, basic ladder mutations that disrupt PIP<sub>2</sub>-dependent liposome association were made to evaluate the cellular consequences of vinculin binding to PIP<sub>2</sub>. Results obtained from cellular expression of the PIP<sub>2</sub> deficient vinculin mutants suggest that PIP<sub>2</sub>-binding is not required for localization of vinculin to FAs or FA strengthening. Rather. PIP<sub>2</sub>-binding is specifically required for vinculin activation and turnover at FAs to promote its association with the force-transduction FA nanodomain.'),(839,'maabbbbbcccfdddddgkkklmmmppppprrssstttwwwxxzzw_rna17','Miao, Z., Adamiak, R. W., Antczak, M., Batey, R. T., Becka, A. J., Biesiada, M., Boniecki, M. J., Bujnicki, J., Chen, S. J., Cheng, C. Y., Chou, F. C., Ferr&eacute;-D\'Amar&eacute; A. R., Das, R., Dawson W. K., Ding, F., Dokholyan, N. V., Dunin-Horkawicz, S., Geniesse, C., Kappel, K., Kladwang, W., Krokhotin, A., Lach, G. E., Major, F., Mann, T. H., Magnus, M., Pachulska-Wieczorek, K., Patel, D. J., Piccirilli, J. A., Popenda, M., Purzycka, K. J., Ren, A., Rice, G. M., Santalcia, J. Jr., Sarzynska, J., Szachniuk, M., Tandon, A., Trausch, J. J., Tian, S., Wang, J., Weeks, K. M., Williams B. 2nd, Xiao, Y., Xu, X., Zhang, D., Zok, T., and Westhof, E.','RNA-Puzzles Round III: 3D RNA structure prediction of five riboswitches and one ribozyme','RNA',23,'655','672',2017,'RNA','Dokholyan N. V.','no','no','no','article',28138060,'','','','','RNA-Puzzles is a collective experiment in blind 3D RNA structure prediction. We report here a third round of RNA-Puzzles. Five puzzles, 4, 8, 12, 13, 14, all structures of riboswitch aptamers and puzzle 7, a ribozyme structure, are included in this round of the experiment. The riboswitch structures include biological binding sites for small molecules (S-adenosyl methionine, cyclic diadenosine monophosphate, 5-amino 4-imidazole carboxamide riboside 5\'-triphosphate, glutamine) and proteins (YbxF) and one set describes large conformational changes between ligand-free and ligand-bound states; the Varkud satellite ribozyme is the most recently solved structure of a known large ribozyme. All the puzzles have established biological functions and require structural understanding to appreciate their molecular mechanisms. Through the use of fast-track experimental data, including multidimensional chemical mapping, and accurate prediction of RNA secondary structure, a large portion of the contacts in 3D have been predicted correctly leading to similar topologies for the top ranking predictions. Template-based and homology-derived predictions could predict structures to particularly high accuracies. However, achieving biological insights from de novo prediction of RNA 3D structures still depends on the size and complexity of the RNA. Blind computational predictions of RNA structures already appear to provide useful structural information in many cases. Similar to the previous RNA-Puzzles Round II experiment, the prediction of non-Watson-Crick interactions and the observed high atomic clash scores reveal notable need for algorithm of improvement. All prediction models and assessment results are available at http://ahsoka.u-strasbg.fr/rnapuzzles/.'),(840,'dh_cell17','Dokholyan, N. V. and Hahn, K. M.','Stealthy control of proteins and cellular networks in live cells','Cell Systems',4,'5','',2017,'Proteins: Design','Dokholyan N. V. and Hahn, K.','no','no','no','article',0,'','','','',''),(841,'dkygwdwdkh_acssb17','Dagliyan, O., Karginov, A. V., Yagishita, S., Gale, M. E., Wang, H., DerMardirossian, C., Wells, C. M., Dokholyan, N. V., Kasai, H. and Hahn, K. M.','Engineering Pak1 allosteric switches','ACS Synthetic Biology',6,'1257','1262',2017,'Proteins: Design','Dokholyan N. V., Kasai, H. and Hahn, K.','no','no','no','article',28365983,'','','','','p21-activated kinases (PAKs) are important regulators of cell motility and morphology. It has been challenging to interrogate their functions because cells adapt to genetic manipulation of PAK, and because inhibitors act on multiple PAK isoforms. Here we describe genetically encoded PAK1 analogues that can be selectively activated by the membrane-permeable small molecule rapamycin. An engineered domain inserted away from the active site responds to rapamycin to allosterically control activity of the PAK1 isoform. To examine the mechanism of rapamycin-induced PAK1 activation, we used molecular dynamics with graph theory to predict amino acids involved in allosteric communication with the active site. This analysis revealed allosteric pathways that were exploited to generate kinase switches. Activation of PAK1 resulted in transient cell spreading in metastatic breast cancer cells, and long-term dendritic spine enlargement in mouse hippocampal CA1 neurons.'),(842,'mclmdd_pnas17','Mu, X., Choi, S., Lang, L., Mowrey, D., Dokholyan, N. V., Danielsson, J., and Oliveberg, M.','Physicochemical code for quinary protein interactions in E. coli','Proceedings of the National Academy of Sciences USA',114,'e4556','e4563',2017,'Proteins: Mechanism','Dokholyan N. V., and Danielsson Oliveberg, M.','no','no','no','article',28536196,'','','','','How proteins sense and navigate the cellular interior to find their functional partners remain poorly understood. An intriguing aspect of this search is that it relies on diffusive encounters with the crowded cellular background, made up by protein surfaces that are largely non-conserved. The question is then if/how this protein search is amenable to selection and biological control? To shed light on this issue, we examined the motions of three evolutionary divergent proteins in the E. coli cytoplasm by in-cell NMR. The results show that the diffusive in-cell motions, after all, follows simplistic physical-chemical rules: the proteins reveal a common dependence on (i) net-charge density, (ii) surface hydrophobicity and (iii) the electric dipole moment. The bacterial protein is here biased to move relatively freely in the bacterial interior, whereas the human counterparts more easily stick. Even so, the in- cell motions respond predictably to surface mutation, allowing us to tune and intermix the proteins behaviour at will. The findings show how evolution can swiftly optimise the diffuse background of protein-encounter complexes by just single point mutations, and provide a rational framework for adjusting the cytoplasmic motions of individual proteins, e.g. for rescuing poor in-cell NMR signals and for optimising protein therapeutics.'),(843,'bppdb_sa17','Brodie, N. I., Popov, K. I., Petrotchenko, E. V., Dokholyan, N. V., and Borchers, C. H.','Solving protein structures using short-distance crosslinking constraints as a guide for discrete molecular dynamics simulations','Science Advances',3,'e1700479','',2017,'Proteins: Structure','Dokholyan N. V. and Borchers, C. H.','yes','no','no','article',0,'','','','','We present here an integrated experimental and computational approach for de novo protein structure determination in which short-distance crosslinking data is incorporated into rapid discrete molecular dynamic (DMD) simulations as constraints, reducing the conformational space and achieving the correct protein folding on practical time scales.  We tested our approach on myoglobin and FKBP -- models for alpha-helix and beta-sheet rich proteins, respectively -- and found that the lowest-energy structures obtained were in excellent agreement with the crystal structure and with hydrogen/deuterium exchange, surface modification, and long-distance crosslinking validation data.  Our approach is readily applicable to other proteins with unknown structures.'),(844,'wlwdgl_bj17','Woods, C. T., Lackey, L., Williams, B., Dokholyan, N. V., Gotz, D. and Laederach, A.','Comparative visualization of the RNA suboptimal conformational ensemble <i>in vivo</i>','Biophysical Journal',113,'290','301',2017,'RNA','Dokholyan N. V., Gotz, D. and Laederach, A.','no','no','no','article',28625696,'','','','','Motivation: When a RiboNucleic Acid (RNA) molecule folds, it often does not adopt a single, well-defined conformation. The folding energy landscape of an RNA is highly dependent on its nucleotide sequence and molecular environment. Cellular molecules sometimes alter the energy landscape, thereby changing the ensemble of likely low-energy conformations. The effects of these energy landscape changes on the conformational ensemble are particularly challenging to visualize for large RNAs. We have created a robust approach for visualizing the conformational ensemble of RNAs that is well-suited for <i>in vitro</i> vs. <i>in vivo</i> comparisons. <br> Method: Our method creates a stable map of conformational space for a given RNA sequence. We first identify single point mutations in the RNA that maximally sample suboptimal conformational space based on the ensemble\'s partition function. Then, we cluster these diverse ensembles to identify the most diverse partition functions for Boltzmann stochastic sampling. By using a novel nestedness distance metric, we iteratively add mutant suboptimal ensembles to converge on a stable 2-dimensional map of conformational space. We then compute the SHAPE-directed ensemble for the RNA folding under different conditions, and we project these ensembles on the map to visualize. <br>Results: To validate our approach, we established a conformational map of the Vibrio vunificus add adenine riboswitch that reveals five classes of structures. In the presence of adenine, projection of the SHAPE-directed sampling correctly identified the &quot;On&quot; conformation; without the ligand, only &quot;Off&quot; conformations were visualized. We also collected whole-transcript <i>in vitro</i> and <i>in vivo</i> SHAPE-MaP for human &beta;-actin messenger RNA (mRNA) that revealed similar global folds in both conditions. Nonetheless, a comparison of <i>in vitro</i> and <i>in vivo</i> data revealed that specific regions exhibited significantly different SHAPE-MaP profiles indicative of structural rearrangements, including rearrangement consistent with binding of the zipcode protein in a region distal to the stop codon.'),(845,'dachrd_ploscb17','Das, J., Aleksandrov, A. A., Cui, L., He, L., Riordan, J. R., and Dokholyan, N. V.','Transmembrane helical interactions in the CFTR channel pore','Public Library of Science Computational Biology',13,'e1005594','',2017,'Proteins: Structure','Riordan J. R. and Dokholyan N. V.','no','no','no','article',28640808,'','','','','Mutations in the Cystic Fibrosis Transmembrane Conductance Regulator (CFTR) gene affect CFTR protein biogenesis or its function as a chloride channel, resulting in dysregulation of epithelial fluid transport in the lung, pancreas and other organs in cystic fibrosis (CF). Development of pharmaceutical strategies to treat CF requires understanding of the mechanisms underlying channel function. However, incomplete 3D structural information on the unique ABC ion channel, CFTR, hinders elucidation of its functional mechanism and correction of cystic fibrosis causing mutants. Several CFTR homology models have been developed using bacterial ABC transporters as templates but these have low sequence similarity to CFTR and are not ion channels. Here, we refine an earlier model in an outward (OWF) and develop an inward (IWF) facing model employing an integrated experimental-molecular dynamics simulation (200 ns) approach. Our IWF structure agrees well with a recently solved cryo-EM structure of a CFTR IWF state. We utilize cysteine cross-linking to verify positions and orientations of residues within trans-membrane helices (TMHs) of the OWF conformation and to reconstruct a physiologically relevant pore structure. Comparison of pore profiles of the two conformations reveal a radius sufficient to permit passage of hydrated Cl- ions in the OWF but not the IWF model. To identify structural determinants that distinguish the two conformations and possible rearrangements of TMHs within them responsible for channel gating, we perform cross-linking by bifunctional reagents of multiple predicted pairs of cysteines in TMH 6 and 12 and 6 and 9. To determine whether the effects of cross-linking on gating observed are the result of switching of the channel from open to close state, we also treat the same residue pairs with monofunctional reagents in separate experiments. Both types of reagents prevent ion currents indicating that pore blockage is primarily responsible.'),(846,'kd_plr17','Krokhotin, A., and Dokholyan, N. V.','Protein folding: Over half a century lasting quest','Physics of Life Reviews',21,'72','74',2017,'','Dokholyan N. V.','no','no','no','article',28599786,'','','','',' Comment on &#34;There and back again: Two views on the protein folding puzzle&#34; by Alexei V. Finkelstein et al.'),(847,'mxmpmd_jbc17','Mowrey, D. D., Xu, L., Mei, Y., Pasek D. A., Meissner, G. and Dokholyan, N. V.','Ion-pulling simulations provide insights into the mechanisms of channel opening of the skeletal muscle ryanodine receptor','Journal of Biological Chemistry',292,'12947','12958',2017,'Proteins: Mechanism','Dokholyan N. V.','no','no','no','article',28584051,'','','','','The type 1 ryanodine receptor (RyR1) mediates Ca<sup>2+</sup> release from the sarcoplasmic reticulum to initiate skeletal muscle contraction and is associated with muscle diseases, malignant hyperthermia, and central core disease. understand RyR1 channel function, we investigated the molecular mechanisms of channel gating and ion permeation. An adequate model of channel gating requires accurate, high-resolution models of both open and closed states of the channel. To this end, we generated an open-channel RyR1 model using molecular simulations to pull Ca<sup>2+</sup> through the pore constriction site of a closed-channel RyR1 structure determined at 3.8 &#8491; resolution. Importantly, we find that our open-channel model is consistent with the RyR1 and cardiac RyR (RyR2) open- channel structures reported while this paper was in preparation. Both our model and the published structures show similar rotation of the upper portion of the pore-lining S6 helix away from the four-fold channel axis and twisting of I4937 at the channel constriction site out of the channel pore. These motions result in a minimum open-channel pore radius of ~3 &#8491; formed by Q4933, rather than I4937 in the closed-channel structure. We also present functional support for our model by muta- tions around the closed- and open-channel constriction sites (Q4933 and I4937). Our results indicate that use of ion-pulling simulations produces a RyR1 open-channel model, which can provide in- sights into the mechanisms of channel opening complementing those from the structural data.'),(848,'dkdykd_sr17','Dakal, T. C., Kala, D., Dhiman, G., Yadav, V., Krokhotin, A., and Dokholyan, N. V.','Predicting the functional consequences of non-synonymous single nucleotide polymorphisms in IL8 gene','Scientific Reports',7,'6525','',2017,'','Dokholyan N. V.','no','no','no','article',0,'','','','','Here we report an in-silico approach for identification, characterization and validation of deleterious non-synonymous SNPs (nsSNPs) in the interleukin-8 gene using three steps. In first step, sequence homology-based genetic analysis of a set of 50 coding SNPs associated with 41 rsIDs using SIFT (Sorting Intolerant from Tolerant) and PROVEAN (Protein Variation Effect Analyzer) identified 23 nsSNPs to be putatively damaging/deleterious in at least one of the two tools used. Subsequently, structure-homology based PolyPhen-2 (Polymorphism Phenotyping)  analysis predicted 9 of 23 nsSNPs (K4T, E31A, E31K, S41Y, I55N, P59L, only nsSNPs that score damaging/deleterious prediction in both sequence and structural homology-based approach will be considered as high-confidence nsSNPs. In step 2, based on conservation of amino acid residues, stability analysis, structural superimposition, RSMD and docking analysis, the possible structural-functional relationship was ascertained for high-confidence nsSNPs. Finally, in a separate analysis (step 3), the IL-8 deregulation has also appeared to be an important prognostic marker for detection of patients with gastric and lung cancer. This study, for the first time, provided in-depth insights on the effects of amino acid substitutions on IL-8 protein structure, function and disease association.'),(849,'drjadpds_sr17','Dronamraju, R., Ramachandran, S., Jha, D. K., Adams, A. T., DiFiore, J. V., Parra, M. A., Dokholyan, N. V. and Strahl, B. D.','Redundant functions for Nap1 and Chz1 in H2A.Z deposition','Scientific Reports',0,'','',2017,'','Dokholyan N. V.','no','no','no','article',0,'','','','','H2A.Z is a histone H2A variant that contributes to transcriptional regulation, DNA damage response and limits heterochromatin spreading. In <i>Saccharomyces cerevisiae</i>,  H2A.Z is deposited by the SWR-C complex, which relies on several histone chaperones including Nap1 and Chz1 to deliver H2A.Z-H2B dimers to SWR-C. However, the mechanisms by which Nap1 and Chz1 cooperate to bind H2A.Z and their contribution to H2A.Z deposition in chromatin is not well understood. Using structural modeling and molecular dynamics simulations, we identify a series of H2A.Z residues that form a chaperone specific binding surface. Mutation of these residues revealed different surface requirements for Nap1 and Chz1 interaction with H2A.Z. Consistent with this result, we found that loss of Nap1 or Chz1 individually resulted in mild defects in H2A.Z deposition, but that deletion of both Nap1 and Chz1 resulted in a significant reduction of H2A.Z deposition at promoters and led to heterochromatin spreading. Together, our findings reveal unique H2A.Z surface dependences for Nap1 and Chz1 and a redundant role for these chaperones in H2A.Z deposition.'),(850,'wcdd_bj17','Williams, B., Convertino, M., Das, J. and Dokholyan, N. V.','Molecular mechanisms of the R61T mutation in Apolipoprotein E4: A dynamic rescue','Biophysical Journal',0,'','',2017,'','Dokholyan N. V.','no','no','no','article',0,'','','','','The apolipoprotein E4 (ApoE4) gene is the strongest genetic risk factor for Alzheimers disease (AD). With respect to the other common isoforms of this protein (ApoE2 and ApoE3), ApoE4 is characterized by lower stability that underlies the formation of a stable interaction between the proteins N- and C-terminal domains. AD-related cellular dysfunctions have been linked to this ApoE4 misfolded state. In this regard, it has been reported that the mutation R61T is able to rescue the deleterious cellular effects of ApoE4 by preventing the formation of the misfolded intermediate state. However, a clear description of the structural features at the basis of the R61T-ApoE4 mutants protective effect is still missing. Recently, using extensive molecular dynamics simulations, we have identified a structural model of an ApoE4 misfolded intermediate state. Building on our previous work, here, we explore the dynamical changes induced by the R61T mutation in the ApoE4 native and misfolded states. Notably, we do not observe any local changes in the domains in the R61T-ApoE4 system, rather a general loss of correlated movements in the entire protein structure. More specifically, we detect increased dynamics in the hinge region, which is essential for ApoE4 domain-domain interaction. Consistent with previously reported data on altered phospholipid and receptor binding, we hypothesize that mutations destabilizing the ApoE4 intermediate state change hinge region dynamics which propagates to distal functional regions of the protein and modifies ApoE4s functional properties. This unique behavior of the ApoE4 hinge region provides a novel understanding of ApoE4s role in AD.');
/*!40000 ALTER TABLE `pubs` ENABLE KEYS */;
UNLOCK TABLES;

--
-- Table structure for table `pubs_with_news`
--

DROP TABLE IF EXISTS `pubs_with_news`;
/*!40101 SET @saved_cs_client     = @@character_set_client */;
/*!40101 SET character_set_client = utf8 */;
CREATE TABLE `pubs_with_news` (
  `pub_id` int(10) unsigned NOT NULL AUTO_INCREMENT,
  `code` varchar(60) DEFAULT NULL,
  `authors` varchar(500) DEFAULT NULL,
  `title` varchar(500) DEFAULT NULL,
  `journal` varchar(250) DEFAULT NULL,
  `volume` int(5) DEFAULT '0',
  `fpage` varchar(10) DEFAULT NULL,
  `lpage` varchar(10) DEFAULT NULL,
  `year` int(11) DEFAULT '2014',
  `topic` varchar(100) DEFAULT NULL,
  `cauthors` varchar(100) DEFAULT 'Dokholyan, N. V.',
  `suppinfo` varchar(3) DEFAULT 'yes',
  `cover` varchar(3) DEFAULT 'no',
  `f1000` varchar(3) DEFAULT 'no',
  `type` varchar(15) DEFAULT 'article',
  `pmid` int(10) unsigned DEFAULT '0',
  `pmcid` varchar(20) DEFAULT NULL,
  `editors` varchar(200) DEFAULT NULL,
  `series` varchar(100) DEFAULT NULL,
  `publisher` varchar(100) DEFAULT NULL,
  `abstract` text,
  `news_release` varchar(500) DEFAULT NULL,
  PRIMARY KEY (`pub_id`)
) ENGINE=InnoDB AUTO_INCREMENT=2 DEFAULT CHARSET=utf8;
/*!40101 SET character_set_client = @saved_cs_client */;

--
-- Dumping data for table `pubs_with_news`
--

LOCK TABLES `pubs_with_news` WRITE;
/*!40000 ALTER TABLE `pubs_with_news` DISABLE KEYS */;
INSERT INTO `pubs_with_news` VALUES (1,'ttskppogbedsc_structure14','Thompson, P. M., Tolbert, C. E., Shen, K., Kota, P., Palmer, S. M., Plevock, K. M., Orlova, A., Glakin, V. E., Burridge, K., Egelman, E. H., Dokholyan, N. V., Superfine, R., Campbell, S. L.','Identification of a new actin binding surface on vinculin that mediates mechanical cell and focal adhesion properties','Structure',0,'','',2014,'Proteins: Structure','Dokholyan, N. V.','no','no','no','article',0,'','','','','','http://www.newswise.com/articles/unc-researchers-develop-insights-into-protein-linked-to-cell-movement');
/*!40000 ALTER TABLE `pubs_with_news` ENABLE KEYS */;
UNLOCK TABLES;

--
-- Table structure for table `users`
--

DROP TABLE IF EXISTS `users`;
/*!40101 SET @saved_cs_client     = @@character_set_client */;
/*!40101 SET character_set_client = utf8 */;
CREATE TABLE `users` (
  `username` varchar(30) NOT NULL,
  `password` varchar(32) DEFAULT NULL,
  `userid` varchar(32) DEFAULT NULL,
  `userlevel` tinyint(1) unsigned NOT NULL,
  `email` varchar(50) DEFAULT NULL,
  `timestamp` int(11) unsigned NOT NULL,
  PRIMARY KEY (`username`)
) ENGINE=MyISAM DEFAULT CHARSET=utf8;
/*!40101 SET character_set_client = @saved_cs_client */;

--
-- Dumping data for table `users`
--

LOCK TABLES `users` WRITE;
/*!40000 ALTER TABLE `users` DISABLE KEYS */;
INSERT INTO `users` VALUES ('dokh','5a78b024726b16421d0e7bae53bd0eb8','f7bb47cc98a04925b1d6336bfb0f6f9d',9,'dokh@med.unc.edu',1313788368);
/*!40000 ALTER TABLE `users` ENABLE KEYS */;
UNLOCK TABLES;
/*!40103 SET TIME_ZONE=@OLD_TIME_ZONE */;

/*!40101 SET SQL_MODE=@OLD_SQL_MODE */;
/*!40014 SET FOREIGN_KEY_CHECKS=@OLD_FOREIGN_KEY_CHECKS */;
/*!40014 SET UNIQUE_CHECKS=@OLD_UNIQUE_CHECKS */;
/*!40101 SET CHARACTER_SET_CLIENT=@OLD_CHARACTER_SET_CLIENT */;
/*!40101 SET CHARACTER_SET_RESULTS=@OLD_CHARACTER_SET_RESULTS */;
/*!40101 SET COLLATION_CONNECTION=@OLD_COLLATION_CONNECTION */;
/*!40111 SET SQL_NOTES=@OLD_SQL_NOTES */;

-- Dump completed on 2017-08-26 11:05:54
